# Ontology comparison

## Left
- Ontology IRI: `http://purl.obolibrary.org/obo/cl/cl-base.owl`
- Version IRI: `http://purl.obolibrary.org/obo/cl/cl-base/cl/releases/2025-07-30/cl-base.owl/cl/cl-base.owl`
- Loaded from: `file:/work/src/ontology/tmp/current-base-release.obo`

## Right
- Ontology IRI: `http://purl.obolibrary.org/obo/cl/cl-base.owl`
- Version IRI: `http://purl.obolibrary.org/obo/cl/cl-base/cl/releases/2025-10-16/cl-base.owl/cl/cl-base.owl`
- Loaded from: `file:/work/src/ontology/../../cl-base.obo`

### Ontology imports 



### Ontology annotations 
#### Removed
- [versionInfo](http://www.w3.org/2002/07/owl#versionInfo) "2025-07-30"^^[string](http://www.w3.org/2001/XMLSchema#string) 

#### Added
- [versionInfo](http://www.w3.org/2002/07/owl#versionInfo) "2025-10-16"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### Bergmann glial cell `http://purl.obolibrary.org/obo/CL_0000644`

#### Added
- [Bergmann glial cell](http://purl.obolibrary.org/obo/CL_0000644) [description](http://purl.org/dc/terms/description) "Bergmann glial cells, also known as Bergmann glia or Bergmann astrocytes, are specialized astrocytes located in the cerebellum, specifically in the Purkinje cell layer. They were discovered and named after the German histologist Karl Bergmann and are known for their characteristic morphology. Unique to Bergmann glial cells is their long, fibrous processes, which extend through the molecular layer to the cerebellar surface, aligning parallel to the Purkinje cells' dendrites. These extensions play a detailed role in the structural organization and function of the cerebellum.
The primary function of Bergmann glial cells is to support and nourish the adjacent neuronal cells, especially the Purkinje cells. Their strategic arrangements around synapses or nerve junctions also indicate their role in facilitating communication between neurons and in modulating synaptic plasticity. The glial cells remove excess neurotransmitters from the synaptic clefts and thus help maintain homeostasis. An additional important role of Bergmann glia is in the guided migration of granule cells during cerebellar development. This guidance is critical for the proper formation and arrangement of the cerebellar circuitry.
In addition to these roles, recent studies have revealed a vital function of Bergmann glial cells in maintaining the overall health and functioning of the cerebellum. In instances of cerebellar injury or disease, these cells respond proactively and change their morphology and activity, a process known as reactive gliosis. This response seems to protect and restore the neural circuits. However, aberrant activation of Bergmann glial cells has been associated with several neurodegenerative diseases and disorders, such as spinocerebellar ataxia, underscoring their significance in the pathobiology of the nervous system.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1007/s12035-013-8405-y"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/S0074-7742(08)60530-9"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1080/14734220600724569"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1002/jnr.10197"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/nn1750"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [Bergmann glial cell](http://purl.obolibrary.org/obo/CL_0000644) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0000644"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "Bergmann glial cell on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### Bergmann glial cell (Mmus) `http://purl.obolibrary.org/obo/CL_4307026`
#### Removed
- [Bergmann glial cell (Mmus)](http://purl.obolibrary.org/obo/CL_4307026) [IAO_0000028](http://purl.obolibrary.org/obo/IAO_0000028) "Bergman glial cell (Mmus)"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [Bergmann glial cell (Mmus)](http://purl.obolibrary.org/obo/CL_4307026) [label](http://www.w3.org/2000/01/rdf-schema#label) "Bergman glial cell (Mmus)"^^[string](http://www.w3.org/2001/XMLSchema#string) 

#### Added
- [Bergmann glial cell (Mmus)](http://purl.obolibrary.org/obo/CL_4307026) [IAO_0000028](http://purl.obolibrary.org/obo/IAO_0000028) "Bergmann glial cell (Mmus)"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [Bergmann glial cell (Mmus)](http://purl.obolibrary.org/obo/CL_4307026) [label](http://www.w3.org/2000/01/rdf-schema#label) "Bergmann glial cell (Mmus)"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### Betz upper motor neuron `http://purl.obolibrary.org/obo/CL_4023052`

#### Added
- [Betz upper motor neuron](http://purl.obolibrary.org/obo/CL_4023052) SubClassOf [BFO_0000002](http://purl.obolibrary.org/obo/BFO_0000002) 


### Bm1 B cell `http://purl.obolibrary.org/obo/CL_0000961`
#### Removed
- [Bm1 B cell](http://purl.obolibrary.org/obo/CL_0000961) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A follicular B cell that is IgD-positive, CD23-negative, and CD38-negative. This naive cell type is activated in the extrafollicular areas through interaction with interdigitating dendritic cells and antigen-specific CD4-positive T cells."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "GOC:msz"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:17992590"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:8006591"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "GOC:dsd"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "GO_REF:0000031"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:18432934"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [Bm1 B cell](http://purl.obolibrary.org/obo/CL_0000961) EquivalentTo [follicular B cell](http://purl.obolibrary.org/obo/CL_0000843) and ([lacks_plasma_membrane_part](http://purl.obolibrary.org/obo/CL_4030046) some [PR_000001408](http://purl.obolibrary.org/obo/PR_000001408)) and ([lacks_plasma_membrane_part](http://purl.obolibrary.org/obo/CL_4030046) some [PR_000001880](http://purl.obolibrary.org/obo/PR_000001880)) and ([RO_0002104](http://purl.obolibrary.org/obo/RO_0002104) some [GO_0071738](http://purl.obolibrary.org/obo/GO_0071738)) 

- [Bm1 B cell](http://purl.obolibrary.org/obo/CL_0000961) SubClassOf [fraction F mature B cell](http://purl.obolibrary.org/obo/CL_0002056) 

#### Added
- [Bm1 B cell](http://purl.obolibrary.org/obo/CL_0000961) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A follicular B cell that is IgD-positive, CD23-negative, and CD38-negative. This naive cell type is activated in the extrafollicular areas through interaction with interdigitating dendritic cells and antigen-specific CD4-positive T cells."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:31681331"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "GOC:msz"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:17992590"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:11564773"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:8006591"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "GOC:dsd"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "GO_REF:0000031"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:19350211"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:18432934"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [Bm1 B cell](http://purl.obolibrary.org/obo/CL_0000961) EquivalentTo [follicular B cell](http://purl.obolibrary.org/obo/CL_0000843) and ([lacks_plasma_membrane_part](http://purl.obolibrary.org/obo/CL_4030046) some [PR_000001408](http://purl.obolibrary.org/obo/PR_000001408)) and ([lacks_plasma_membrane_part](http://purl.obolibrary.org/obo/CL_4030046) some [PR_000001880](http://purl.obolibrary.org/obo/PR_000001880)) and ([RO_0002104](http://purl.obolibrary.org/obo/RO_0002104) some [GO_0071738](http://purl.obolibrary.org/obo/GO_0071738)) and ([RO_0002162](http://purl.obolibrary.org/obo/RO_0002162) some [NCBITaxon_9606](http://purl.obolibrary.org/obo/NCBITaxon_9606)) 

- [Bm1 B cell](http://purl.obolibrary.org/obo/CL_0000961) SubClassOf [lacks_plasma_membrane_part](http://purl.obolibrary.org/obo/CL_4030046) some [PR_000001880](http://purl.obolibrary.org/obo/PR_000001880) 

- [Bm1 B cell](http://purl.obolibrary.org/obo/CL_0000961) SubClassOf [RO_0002162](http://purl.obolibrary.org/obo/RO_0002162) some [NCBITaxon_9606](http://purl.obolibrary.org/obo/NCBITaxon_9606) 


### Bm2 B cell `http://purl.obolibrary.org/obo/CL_0000962`
#### Removed
- [Bm2 B cell](http://purl.obolibrary.org/obo/CL_0000962) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A follicular B cell that is IgD-positive and CD23-positive and CD38-positive. This naive cell type is activated in the extrafollicular areas via interaction with dendritic cells and antigen specific T cells."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "GOC:msz"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:17992590"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:8006591"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "GOC:dsd"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "GO_REF:0000031"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:18432934"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [Bm2 B cell](http://purl.obolibrary.org/obo/CL_0000962) EquivalentTo [follicular B cell](http://purl.obolibrary.org/obo/CL_0000843) and ([RO_0002104](http://purl.obolibrary.org/obo/RO_0002104) some [GO_0071738](http://purl.obolibrary.org/obo/GO_0071738)) and ([RO_0002104](http://purl.obolibrary.org/obo/RO_0002104) some [PR_000001408](http://purl.obolibrary.org/obo/PR_000001408)) and ([RO_0002104](http://purl.obolibrary.org/obo/RO_0002104) some [PR_000001880](http://purl.obolibrary.org/obo/PR_000001880)) 

#### Added
- [Bm2 B cell](http://purl.obolibrary.org/obo/CL_0000962) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A follicular B cell that is IgD-positive and CD23-positive and CD38-positive. This naive cell type is activated in the extrafollicular areas via interaction with dendritic cells and antigen specific T cells."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:31681331"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "GOC:msz"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:17992590"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:11564773"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:8006591"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "GOC:dsd"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "GO_REF:0000031"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:19350211"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:18432934"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [Bm2 B cell](http://purl.obolibrary.org/obo/CL_0000962) EquivalentTo [follicular B cell](http://purl.obolibrary.org/obo/CL_0000843) and ([RO_0002104](http://purl.obolibrary.org/obo/RO_0002104) some [GO_0071738](http://purl.obolibrary.org/obo/GO_0071738)) and ([RO_0002104](http://purl.obolibrary.org/obo/RO_0002104) some [PR_000001408](http://purl.obolibrary.org/obo/PR_000001408)) and ([RO_0002104](http://purl.obolibrary.org/obo/RO_0002104) some [PR_000001880](http://purl.obolibrary.org/obo/PR_000001880)) and ([RO_0002162](http://purl.obolibrary.org/obo/RO_0002162) some [NCBITaxon_9606](http://purl.obolibrary.org/obo/NCBITaxon_9606)) 


### Bm2' B cell `http://purl.obolibrary.org/obo/CL_0000964`
#### Removed
- [Bm2' B cell](http://purl.obolibrary.org/obo/CL_0000964) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A germinal center B cell that founds a germinal center, and has the phenotype IgD-positive, CD38-positive, and CD23-negative."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "GOC:msz"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:17992590"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:8006591"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "GOC:dsd"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "GO_REF:0000031"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "GOC:tfm"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:18432934"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [Bm2' B cell](http://purl.obolibrary.org/obo/CL_0000964) EquivalentTo [germinal center B cell](http://purl.obolibrary.org/obo/CL_0000844) and ([lacks_plasma_membrane_part](http://purl.obolibrary.org/obo/CL_4030046) some [PR_000001880](http://purl.obolibrary.org/obo/PR_000001880)) and ([RO_0002104](http://purl.obolibrary.org/obo/RO_0002104) some [GO_0071738](http://purl.obolibrary.org/obo/GO_0071738)) and ([RO_0002104](http://purl.obolibrary.org/obo/RO_0002104) some [PR_000001408](http://purl.obolibrary.org/obo/PR_000001408)) and ([RO_0002215](http://purl.obolibrary.org/obo/RO_0002215) some [GO_0002634](http://purl.obolibrary.org/obo/GO_0002634)) 

#### Added
- [Bm2' B cell](http://purl.obolibrary.org/obo/CL_0000964) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A germinal center B cell that founds a germinal center, and has the phenotype IgD-positive, CD38-positive, and CD23-negative."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:31681331"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "GOC:msz"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:17992590"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:11564773"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:8006591"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "GOC:dsd"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "GO_REF:0000031"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:19350211"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "GOC:tfm"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:18432934"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [Bm2' B cell](http://purl.obolibrary.org/obo/CL_0000964) EquivalentTo [germinal center B cell](http://purl.obolibrary.org/obo/CL_0000844) and ([lacks_plasma_membrane_part](http://purl.obolibrary.org/obo/CL_4030046) some [PR_000001880](http://purl.obolibrary.org/obo/PR_000001880)) and ([RO_0002104](http://purl.obolibrary.org/obo/RO_0002104) some [GO_0071738](http://purl.obolibrary.org/obo/GO_0071738)) and ([RO_0002104](http://purl.obolibrary.org/obo/RO_0002104) some [PR_000001408](http://purl.obolibrary.org/obo/PR_000001408)) and ([RO_0002162](http://purl.obolibrary.org/obo/RO_0002162) some [NCBITaxon_9606](http://purl.obolibrary.org/obo/NCBITaxon_9606)) and ([RO_0002215](http://purl.obolibrary.org/obo/RO_0002215) some [GO_0002634](http://purl.obolibrary.org/obo/GO_0002634)) 


### Bm3 B cell `http://purl.obolibrary.org/obo/CL_0000965`
#### Removed
- [Bm3 B cell](http://purl.obolibrary.org/obo/CL_0000965) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A germinal center B cell that is rapidly dividing and has the phenotype IgD-negative, CD38-positive, and CD77-positive. Somatic hypermutation of the immunoglobulin V gene region can occur during proliferation of this cell type."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "GOC:msz"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:8006591"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "GOC:dsd"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "GO_REF:0000031"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:18432934"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [Bm3 B cell](http://purl.obolibrary.org/obo/CL_0000965) EquivalentTo [germinal center B cell](http://purl.obolibrary.org/obo/CL_0000844) and ([lacks_plasma_membrane_part](http://purl.obolibrary.org/obo/CL_4030046) some [GO_0071738](http://purl.obolibrary.org/obo/GO_0071738)) and ([RO_0002104](http://purl.obolibrary.org/obo/RO_0002104) some [PR_000001408](http://purl.obolibrary.org/obo/PR_000001408)) and ([RO_0002104](http://purl.obolibrary.org/obo/RO_0002104) some [PR_000003516](http://purl.obolibrary.org/obo/PR_000003516)) and ([RO_0002215](http://purl.obolibrary.org/obo/RO_0002215) some [GO_0016446](http://purl.obolibrary.org/obo/GO_0016446)) 

#### Added
- [Bm3 B cell](http://purl.obolibrary.org/obo/CL_0000965) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A germinal center B cell that is rapidly dividing and has the phenotype IgD-negative, CD38-positive, and CD77-positive. Somatic hypermutation of the immunoglobulin V gene region can occur during proliferation of this cell type."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:31681331"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "GOC:msz"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:11564773"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:8006591"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "GOC:dsd"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "GO_REF:0000031"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:19350211"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:18432934"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [Bm3 B cell](http://purl.obolibrary.org/obo/CL_0000965) EquivalentTo [germinal center B cell](http://purl.obolibrary.org/obo/CL_0000844) and ([lacks_plasma_membrane_part](http://purl.obolibrary.org/obo/CL_4030046) some [GO_0071738](http://purl.obolibrary.org/obo/GO_0071738)) and ([RO_0002104](http://purl.obolibrary.org/obo/RO_0002104) some [PR_000001408](http://purl.obolibrary.org/obo/PR_000001408)) and ([RO_0002104](http://purl.obolibrary.org/obo/RO_0002104) some [PR_000003516](http://purl.obolibrary.org/obo/PR_000003516)) and ([RO_0002162](http://purl.obolibrary.org/obo/RO_0002162) some [NCBITaxon_9606](http://purl.obolibrary.org/obo/NCBITaxon_9606)) and ([RO_0002215](http://purl.obolibrary.org/obo/RO_0002215) some [GO_0016446](http://purl.obolibrary.org/obo/GO_0016446)) 


### Bm4 B cell `http://purl.obolibrary.org/obo/CL_0000966`
#### Removed
- [Bm4 B cell](http://purl.obolibrary.org/obo/CL_0000966) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A germinal center B cell that has the phenotype CD77-negative, IgD-negative, and CD38-positive. These cells have undergone somatic mutation of the B cell receptor."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "GOC:msz"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:8006591"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "GOC:dsd"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "GO_REF:0000031"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:18432934"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [Bm4 B cell](http://purl.obolibrary.org/obo/CL_0000966) EquivalentTo [germinal center B cell](http://purl.obolibrary.org/obo/CL_0000844) and ([lacks_plasma_membrane_part](http://purl.obolibrary.org/obo/CL_4030046) some [GO_0071738](http://purl.obolibrary.org/obo/GO_0071738)) and ([lacks_plasma_membrane_part](http://purl.obolibrary.org/obo/CL_4030046) some [PR_000003516](http://purl.obolibrary.org/obo/PR_000003516)) and ([RO_0002104](http://purl.obolibrary.org/obo/RO_0002104) some [PR_000001408](http://purl.obolibrary.org/obo/PR_000001408)) and ([RO_0002353](http://purl.obolibrary.org/obo/RO_0002353) some [GO_0016446](http://purl.obolibrary.org/obo/GO_0016446)) 

#### Added
- [Bm4 B cell](http://purl.obolibrary.org/obo/CL_0000966) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A germinal center B cell that has the phenotype CD77-negative, IgD-negative, and CD38-positive. These cells have undergone somatic mutation of the B cell receptor."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:31681331"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "GOC:msz"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:11564773"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:8006591"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "GOC:dsd"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "GO_REF:0000031"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:19350211"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:18432934"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [Bm4 B cell](http://purl.obolibrary.org/obo/CL_0000966) EquivalentTo [germinal center B cell](http://purl.obolibrary.org/obo/CL_0000844) and ([lacks_plasma_membrane_part](http://purl.obolibrary.org/obo/CL_4030046) some [GO_0071738](http://purl.obolibrary.org/obo/GO_0071738)) and ([lacks_plasma_membrane_part](http://purl.obolibrary.org/obo/CL_4030046) some [PR_000003516](http://purl.obolibrary.org/obo/PR_000003516)) and ([RO_0002104](http://purl.obolibrary.org/obo/RO_0002104) some [PR_000001408](http://purl.obolibrary.org/obo/PR_000001408)) and ([RO_0002162](http://purl.obolibrary.org/obo/RO_0002162) some [NCBITaxon_9606](http://purl.obolibrary.org/obo/NCBITaxon_9606)) and ([RO_0002353](http://purl.obolibrary.org/obo/RO_0002353) some [GO_0016446](http://purl.obolibrary.org/obo/GO_0016446)) 


### Bm5 B cell `http://purl.obolibrary.org/obo/CL_0000967`
#### Removed
- [Bm5 B cell](http://purl.obolibrary.org/obo/CL_0000967) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A memory B cell arising in the germinal center that is IgD-negative and has undergone somatic mutation of the variable region of the immunoglobulin heavy and light chain genes."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "GOC:msz"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:8006591"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "GOC:dsd"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "GO_REF:0000031"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "GOC:tfm"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:18432934"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [Bm5 B cell](http://purl.obolibrary.org/obo/CL_0000967) EquivalentTo [memory B cell](http://purl.obolibrary.org/obo/CL_0000787) and ([lacks_plasma_membrane_part](http://purl.obolibrary.org/obo/CL_4030046) some [GO_0071738](http://purl.obolibrary.org/obo/GO_0071738)) and ([RO_0002353](http://purl.obolibrary.org/obo/RO_0002353) some [GO_0002314](http://purl.obolibrary.org/obo/GO_0002314)) and ([RO_0002353](http://purl.obolibrary.org/obo/RO_0002353) some [GO_0016446](http://purl.obolibrary.org/obo/GO_0016446)) 

#### Added
- [Bm5 B cell](http://purl.obolibrary.org/obo/CL_0000967) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A memory B cell arising in the germinal center that is IgD-negative and has undergone somatic mutation of the variable region of the immunoglobulin heavy and light chain genes."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:31681331"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "GOC:msz"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:11564773"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:8006591"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "GOC:dsd"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "GO_REF:0000031"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:19350211"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "GOC:tfm"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:18432934"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [Bm5 B cell](http://purl.obolibrary.org/obo/CL_0000967) EquivalentTo [memory B cell](http://purl.obolibrary.org/obo/CL_0000787) and ([lacks_plasma_membrane_part](http://purl.obolibrary.org/obo/CL_4030046) some [GO_0071738](http://purl.obolibrary.org/obo/GO_0071738)) and ([RO_0002162](http://purl.obolibrary.org/obo/RO_0002162) some [NCBITaxon_9606](http://purl.obolibrary.org/obo/NCBITaxon_9606)) and ([RO_0002353](http://purl.obolibrary.org/obo/RO_0002353) some [GO_0002314](http://purl.obolibrary.org/obo/GO_0002314)) and ([RO_0002353](http://purl.obolibrary.org/obo/RO_0002353) some [GO_0016446](http://purl.obolibrary.org/obo/GO_0016446)) 


### CD11c-low plasmacytoid dendritic cell `http://purl.obolibrary.org/obo/CL_0000989`
#### Removed
- [CD11c-low plasmacytoid dendritic cell](http://purl.obolibrary.org/obo/CL_0000989) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "CD11c-low plasmacytoid dendritic cell is a leukocyte that is CD11c-low, CD45R-positive, GR1-positive and CD11b-negative."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "GOC:amm"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:17850486"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:15771572"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "GOC:dsd"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "GO_REF:0000031"^^[string](http://www.w3.org/2001/XMLSchema#string) 

#### Added
- [CD11c-low plasmacytoid dendritic cell](http://purl.obolibrary.org/obo/CL_0000989) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "CD11c-low plasmacytoid dendritic cell is a leukocyte that is CD11c-low, CD45R-positive, GR1-positive and CD11b-negative."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "GOC:amm"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:17850486"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:19243617"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:15771572"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "GOC:dsd"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "GO_REF:0000031"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:34603298"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [CD11c-low plasmacytoid dendritic cell](http://purl.obolibrary.org/obo/CL_0000989) SubClassOf [RO_0002162](http://purl.obolibrary.org/obo/RO_0002162) some [NCBITaxon_10090](http://purl.obolibrary.org/obo/NCBITaxon_10090) 


### CD11c-negative plasmacytoid dendritic cell `http://purl.obolibrary.org/obo/CL_0000991`
#### Removed
- [CD11c-negative plasmacytoid dendritic cell](http://purl.obolibrary.org/obo/CL_0000991) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "CD11c-negative plasmacytoid dendritic cell is a leukocyte is CD11c-negative, CD45RA-positive, CD85g-positive(ILT7), CD123-positive, CD303-positive."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "GOC:amm"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:17850486"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:15771572"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "GO_REF:0000031"^^[string](http://www.w3.org/2001/XMLSchema#string) 

#### Added
- [CD11c-negative plasmacytoid dendritic cell](http://purl.obolibrary.org/obo/CL_0000991) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "CD11c-negative plasmacytoid dendritic cell is a leukocyte is CD11c-negative, CD45RA-positive, CD85g-positive(ILT7), CD123-positive, CD303-positive."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "GOC:amm"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:17850486"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:19243617"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:29313948"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:15771572"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "GO_REF:0000031"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [CD11c-negative plasmacytoid dendritic cell](http://purl.obolibrary.org/obo/CL_0000991) SubClassOf [RO_0002162](http://purl.obolibrary.org/obo/RO_0002162) some [NCBITaxon_9606](http://purl.obolibrary.org/obo/NCBITaxon_9606) 


### CD16-negative, CD56-bright natural killer cell, human `http://purl.obolibrary.org/obo/CL_0000938`

#### Added
- [CD16-negative, CD56-bright natural killer cell, human](http://purl.obolibrary.org/obo/CL_0000938) [description](http://purl.org/dc/terms/description) "The CD16-negative, CD56-bright natural killer (NK) cell is a subtype of human natural killer cells that plays a pivotal role in the innate immune response. These cells are identified primarily by the lack of CD16, a low-affinity receptor for the Fc region of immunoglobulins G, and high expression of CD56 protein, a neural cell adhesion molecule, on the cell surface. This unique receptor pattern characterizes the regulatory role of these cells in the immune system and distinguishes them from their counterparts - the CD16-positive, CD56-dim NK cells, which have cytotoxic killing actions instead.
CD16-negative, CD56-bright NK cells are specialized in immunomodulation and this trait is manifested primarily through the release of substantial amounts of cytokines, including TNF, IFN-gamma, GM-CSF, and especially IL-10, when they interact with specific target cells or in response to stimulation by cytokines such as IL-12, IL-15, and IL-18. Thus, these cells form a significant link between innate and adaptive immunity.
Furthermore, these cells have tropism for secondary lymphoid tissues, where they frequently interact with dendritic cells, contributing to the modulation of dendritic cell maturation and function. This intimate crosstalk between CD16-negative, CD56-bright NK cells and dendritic cells is crucial for the orchestration of both innate and adaptive immune responses. Hence, while the term 'natural killer cell' may suggest a primary role in cell killing, this particular subset of NK cells exert their effects mainly through immune regulation and interaction with other immune cells.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.4049/jimmunol.1502570"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.3389/fimmu.2014.00159"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.4049/jimmunol.173.10.6418"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [CD16-negative, CD56-bright natural killer cell, human](http://purl.obolibrary.org/obo/CL_0000938) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0000938"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "CD16-negative, CD56-bright natural killer cell, human on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### CD4-positive helper T cell `http://purl.obolibrary.org/obo/CL_0000492`

#### Added
- [CD4-positive helper T cell](http://purl.obolibrary.org/obo/CL_0000492) [description](http://purl.org/dc/terms/description) "The CD4-positive helper T cells are a subpopulation of T lymphocytes, which are part of the adaptive immune system. These cells play a vital role in establishing and maximizing the capability of the immune system to combat infections. Like all T lymphocytes, CD4+ helper T cells are uniquely characterized by the expression of a T-cell receptor (TCR) complex on their cell surface. The subset of CD4+ cells express the CD4 glycoprotein on their cell surface that serves as a co-receptor for the T-cell receptor. The CD4 co-receptor enables these cells to interact with antigen-presenting cells and perform their influential role in adaptive immune response.
CD4+ helper T cells are primarily responsible for assisting other cells in the immunological processes. This involves recognition, interaction, and processing of antigens. Once an antigen-presenting cell presents a foreign antigen such as peptides derived from pathogens or transformed cells to the CD4+ helper T cell, it becomes activated, proliferates, and differentiates into subtypes such as Th1, Th2, Th17, or Treg, each with a specific role in the immune response. Th1 cells produces various cytokines. including IFN‚Äê&gamma;, IL-2 and TNF-&beta;, and aid cell-mediated immune responses against intracellular pathogens. Th2 cells produce IL-4, IL-5 and IL13 and assist in antibody-mediated immune responses. 
Th17 cells release IL-17 and convey protective immunity against extracellular bacteria and fungi as well as contributing to inflammation and autoimmunity, while Treg cells are immunosuppressive cells and maintain immune homeostasis and provide tolerance to self-antigens. Finally, follicular T helper (Tfh) cells are thought to help B cells build up antibody responses by assisting germinal center formation, affinity maturation, and possibly immunoglobulin class switch recombination 
In addition to their critical role in immune response, CD4+ helper T cells are notably the main target of HIV infection, where the virus depletes the number of these cells, resulting in the deterioration of the immune system, characterized in the progression to AIDS. Thus, the presence and functionality of CD4+ helper T cells are not only necessary for a healthy immune response but also crucial in the context of immunodeficiency diseases.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1002/ctd2.199"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/t-helper-cell"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1111/imcb.12662"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/s43018-023-00521-2"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1146/annurev-immunol-103019-085803"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [CD4-positive helper T cell](http://purl.obolibrary.org/obo/CL_0000492) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0000492"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "CD4-positive helper T cell on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### CD4-positive, CD25-positive, alpha-beta regulatory T cell `http://purl.obolibrary.org/obo/CL_0000792`

#### Added
- [CD4-positive, CD25-positive, alpha-beta regulatory T cell](http://purl.obolibrary.org/obo/CL_0000792) [description](http://purl.org/dc/terms/description) "The CD4-positive, CD25-positive, alpha-beta regulatory T (Treg) cell serves as a vital element in the mammalian immune system, playing a pivotal role in maintaining immune homeostasis and preventing autoimmunity. This particular T regulatory cell is characterized by CD4 proteins and a high level of CD25 (the alpha chain of the interleukin-2 receptor) on its cell surface. The T-cell receptor (TCR) of these Treg cells is composed of alpha and beta protein chains, making them a subtype of the alpha-beta T cell population that recognize and bind to antigens presented by MHC class II molecules.
The primary function of CD4-positive, CD25-positive, alpha-beta Treg cells is regulating immune responses, ensuring that the immune response is proportionate and preventing autoimmunity, maintaining peripheral immune tolerance, and limiting chronic inflammation. They achieve this regulation by suppressing or downregulating the proliferation of effector immune cells, which are activated in response to antigens, to prevent their excessive or uncontrolled activity. 
CD4-positive, CD25-positive, alpha-beta Treg cells can suppress immune responses via a variety of mechanisms, including the release of inhibitory cytokines, in particular IL-10 and TGF&beta;, or through cytolysis of effector immune cells (e.g. antibody-producing B cells) involving perforin and granzymes. They can also modulate the maturation or function of dendritic cells, which are key antigen-presenting cells, and thereby reduce their ability to stimulate effector T cells.
This downregulation also suppresses effector T cell functions against self-antigens&mdash;proteins that are innate to the body&mdash;to prevent autoimmunity. In essence, Treg cells act as a policing force within the immune system to maintain balance and prevent overreaction which can potentially lead to damaging effects.
Besides their crucial role in preventing autoimmunity, CD4-positive, CD25-positive, alpha-beta Treg cells contribute to other functions such as tumor immune evasion, maternal-fetal tolerance, and transplantation tolerance. By suppressing the cytotoxic T cell response, these cells may inadvertently help tumor cells evade destruction, contributing to cancer progression. However, this characteristic can be potentially manipulated for therapeutic applications in immune-related diseases or transplantation, further highlighting the potential and importance of CD4-positive, CD25-positive, alpha-beta regulatory T cells in immune regulation.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.3389/fimmu.2019.03100"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.3389/fimmu.2021.750542"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1146/annurev-immunol-042718-041717"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/nri2343"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.immuni.2019.01.020"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [CD4-positive, CD25-positive, alpha-beta regulatory T cell](http://purl.obolibrary.org/obo/CL_0000792) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0000792"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "CD4-positive, CD25-positive, alpha-beta regulatory T cell on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### CD8-alpha alpha positive, gamma-delta intraepithelial T cell `http://purl.obolibrary.org/obo/CL_0000802`

#### Added
- [CD8-alpha alpha positive, gamma-delta intraepithelial T cell](http://purl.obolibrary.org/obo/CL_0000802) [description](http://purl.org/dc/terms/description) "The CD8-alpha alpha positive, gamma-delta intraepithelial T cell are a subset of intraepithelial lymphocytes (IELs), a highly heterogeneous population of immune cell that resides within the epithelial layer of mucosal barriers. CD8-alpha alpha positive, gamma-delta intraepithelial T cell are predominantly found in the gastrointestinal tract. They are characterized by the expression of both CD8-alpha-alpha homodimer and the gamma-delta (&gamma;&delta;) variant of the T cell receptor (TCR). CD8-alpha alpha positive, gamma-delta are thought to represent the most typical &gamma;&delta; IEL subset in the intestine.
The IELs in the intestines are constantly and directly exposed to antigens in the gut lumen and form the first line of immune defense against invading pathogens. Unlike conventional T cells, IELs in the intestine are antigen-experienced and do not require priming. CD8-alpha alpha positive, gamma-delta intraepithelial T cells fall into a subgroup of &lsquo;natural&rsquo; IELs, which acquire their activated phenotype in an &lsquo;alternative&rsquo; self-antigen-based maturation process in the thymus. 
CD8-alpha alpha positive, gamma-delta intraepithelial T cell have several key functions. They help maintain the integrity of the epithelium and repair the epithelial barriers in response to physical and inflammatory injury by secreting various factors, including TGF&beta;1, TGF&beta;3 and keratinocyte growth factor (KGF). These cells also provide immediate, front-line defense against pathogens. They retain the ability to respond directly to stress-associated and infected cells independent of antigen-presenting cells and major histocompatibility complex molecules. They do this through recognition of antigens present on the surfaces of stressed or infected cells. Once activated, these cells initiate a rapid immune response by releasing cytotoxic granules and engaging death receptors, processes that eliminate harmful cells. Furthermore, CD8-alpha alpha positive, gamma-delta intraepithelial T cells have been implicated in regulatory functions, including in antibody class switching and immunoglobulin A (IgA) production and IL-10-dependent oral tolerance. 
Dysregulation CD8-alpha alpha positive, gamma-delta intraepithelial T cells is linked to several inflammatory disorders including celiac disease, inflammatory bowel disease, and others. Therefore, the CD8-alpha alpha positive, gamma-delta intraepithelial T cell plays a significant role in not only direct pathogen defense but also in the maintenance of epithelial tissue integrity and homeostasis.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/nri.2017.101"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.3390/cells9030686"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/nri3007"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.3389/fimmu.2022.1059042"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [CD8-alpha alpha positive, gamma-delta intraepithelial T cell](http://purl.obolibrary.org/obo/CL_0000802) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0000802"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "CD8-alpha alpha positive, gamma-delta intraepithelial T cell on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### CD8-alpha-alpha-positive, alpha-beta intraepithelial T cell `http://purl.obolibrary.org/obo/CL_0000915`

#### Added
- [CD8-alpha-alpha-positive, alpha-beta intraepithelial T cell](http://purl.obolibrary.org/obo/CL_0000915) [description](http://purl.org/dc/terms/description) "The CD8-alpha-alpha-positive, alpha-beta intraepithelial T cell, often abbreviated as CD8&alpha;&alpha; IEL, is a specialized immune cell found within the epithelial layer of mucosal surfaces, such as the intestines. CD8&alpha;&alpha; IEL have been studied extensively in mice, but equivalent cells are thought to be present in humans, albeit at lower numbers. 
These cells represent a subset of intraepithelial T lymphocytes characterized by the expression of both CD8-alpha-alpha homodimer and the alpha-beta variant of the T cell receptor (TCR). Other features shared with intraepithelial T lymphocytes include the expression of T-bet, a master transcription factor that controls expression of proinflammatory cytokines, natural killer cell receptors, and an activated phenotype (high expression of CD69). 
Functionally, CD8&alpha;&alpha; IELs are thought to play a role in maintaining epithelial integrity, local immune surveillance, and protection against microbial pathogens. Upon detecting alterations associated with infection or inflammation, these cells orchestrate immune responses to restore tissue homeostasis, primarily through the secretion of cytokines and chemokines, and directly eliminating infected cells. 
CD8&alpha;&alpha; IELs have been shown require &beta;2 microglobulin-dependent MHC class I expression for their generation and/or homeostasis. They share some characteristics with natural killer cells and might have similar suppressive and regulatory roles which they perform through the production of IL-10 and TGF&beta;. For example, they have been shown the prevent induced colitis, indicating that by modulating immune responses, CD8&alpha;&alpha; IELs prevent excessive and damaging immune activation that can lead to chronic inflammatory conditions or autoimmunity.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1111/j.1600-065X.2006.00488.x"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.coi.2019.04.010"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1093/cei/uxad049"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.3389/fimmu.2022.1059042"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [CD8-alpha-alpha-positive, alpha-beta intraepithelial T cell](http://purl.obolibrary.org/obo/CL_0000915) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0000915"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "CD8-alpha-alpha-positive, alpha-beta intraepithelial T cell on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### CD8-alpha-beta-positive, alpha-beta intraepithelial T cell `http://purl.obolibrary.org/obo/CL_0000796`

#### Added
- [CD8-alpha-beta-positive, alpha-beta intraepithelial T cell](http://purl.obolibrary.org/obo/CL_0000796) [description](http://purl.org/dc/terms/description) "CD8-alpha-beta-positive, alpha-beta intraepithelial T cells are subpopulation of &alpha;&beta; intraepithelial lymphocytes (IELs) which are predominately found in the intraepithelial layer of the intestine, skin and lung. They are characterized by the cell surface expression of both &alpha;&beta; T cell receptors (TCRs) and the CD8 &alpha;&beta; co-receptors, which are responsible for recognizing specific antigens expressed on the surface of pathogenic cells. 
The primary function of CD8-alpha-beta-positive, alpha-beta intraepithelial T cells revolves around providing immune protection by targeting and eliminating cells infected with viruses or other pathogens. The TCR-CD8 complex on their surfaces recognizes major histocompatibility complex (MHC) class I molecules on infected cells. After recognition, the T cells bind to the infected cells and release cytotoxic agents that trigger apoptosis, leading to the death of the infected cells and effectively curbing the spread of the invading pathogen. This process also helps create a memory response for the same pathogens, enabling quicker responses in subsequent encounters.
With a majority of pathogens entering the body through the epithelial surfaces like skin and mucous membranes, having these T cells strategically located in these layers enables an effective first line of defense. Apart from pathogen detection and removal, they also contribute to maintaining tissue integrity and providing a surveillance mechanism against tumorigenic cells.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/s41385-020-0294-6"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/s41385-018-0016-5"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.3389/fimmu.2022.1059042"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [CD8-alpha-beta-positive, alpha-beta intraepithelial T cell](http://purl.obolibrary.org/obo/CL_0000796) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0000796"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "CD8-alpha-beta-positive, alpha-beta intraepithelial T cell on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### CD8-positive, alpha-beta T cell `http://purl.obolibrary.org/obo/CL_0000625`

#### Added
- [CD8-positive, alpha-beta T cell](http://purl.obolibrary.org/obo/CL_0000625) [description](http://purl.org/dc/terms/description) "CD8-positive, alpha-beta T cells, also referred to as CD8+ T cells, are integral components of the body's adaptive immune system. They are characterized by the presence of the CD8 glycoprotein on their cell surface and the alpha and beta chains in their T cell receptors (TCRs). CD8+ T cells primarily function as cytotoxic T cells (CTLs), their specialized role in the immune response being to detect and eliminate cells presenting foreign or abnormal antigens on their surfaces.
Na&iuml;ve CD8+ T cells patrol the central lymphoid organs, checking peptide&ndash; MHC class I molecule complexes for the presence of their cognate antigen on antigen-presenting cells. Recognition of the foreign antigen by the TCR and interaction with MHC class I molecules triggers a series of events leading to the activation of the CD8+ T cell. Once activated, the CD8+ T cell divides, matures, and differentiates into effector T cells capable of destroying the target cells. This cytotoxic function is carried out by the release of substances like perforin and granzymes, which induce apoptosis, or programmed cell death, in the target cells.
In addition to their cytotoxic function, CD8+ T cells are known to release various cytokines, including interferon-gamma (IFN-&gamma;) and tumor necrosis factor-alpha (TNF-&alpha;). These are potent mediators of the immune response and have roles in shaping the immune response and the inflammatory reaction. 
Following the resolution of an immune response, a small subset of activated CD8+ T cells persists as memory T cells, providing a quicker and more vigorous response upon subsequent encounters with the same antigen. These capabilities make CD8-positive, alpha-beta T cells indispensable for the body's defense against infections and malignant transformations.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.immuni.2011.07.010"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/s41577-021-00574-3"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.3389/fimmu.2018.02692"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.3389/fimmu.2018.02826"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [CD8-positive, alpha-beta T cell](http://purl.obolibrary.org/obo/CL_0000625) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0000625"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "CD8-positive, alpha-beta T cell on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### CD8-positive, alpha-beta cytotoxic T cell `http://purl.obolibrary.org/obo/CL_0000794`

#### Added
- [CD8-positive, alpha-beta cytotoxic T cell](http://purl.obolibrary.org/obo/CL_0000794) [description](http://purl.org/dc/terms/description) "The CD8-positive, alpha-beta cytotoxic T cell plays a crucial role in the immune response system of vertebrates. A key feature of these cells is the expression of CD8, a transmembrane glycoprotein that serves as a co-receptor for the T cell receptor (TCR). Notably, the alpha-beta refers to the TCR heterodimer on the T cell, utilized to recognize the antigens displayed by Major Histocompatibility Complex (MHC) class I molecules on antigen-presenting cells.
Functioning primarily to combat intracellular pathogens, CD8+ T cells primarily seek and destroy cells infected by viruses or transformed by cancer. The TCR on the CD8+ T cell recognizes and binds to the antigen-MHC I complex displayed on the infected cell&rsquo;s surface. Following this interaction, along with the engagement of co-stimulatory molecules, the T cell becomes activated. Upon activation, the CD8+ T cell proliferates and differentiates into cytotoxic T lymphocytes (CTL), ready to survey the body for the recognized antigen. 
CTLs employ two major cytotoxic mechanisms: the release of cytotoxic granules and the expression of death-inducing ligands on their surface. Cytotoxic granules contain perforin, a protein that forms pores in the targeted cell membrane, and granzymes, enzymes that trigger apoptosis inside the targeted cell through the activation of caspases. The Fas ligand and TRAIL death-inducing ligands on CTLs also bind to their respective receptors on the target cell and initiate a caspase cascade leading to cell death of the target. This cytotoxic function of the CD8+ alpha-beta T cell is critical in the control of infections and in cancer surveillance.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.3389/fmolb.2021.628332"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/cytotoxic-t-cell"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/s12276-023-01105-x"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.3389/fimmu.2022.883010"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [CD8-positive, alpha-beta cytotoxic T cell](http://purl.obolibrary.org/obo/CL_0000794) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0000794"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "CD8-positive, alpha-beta cytotoxic T cell on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### Cajal-Retzius cell `http://purl.obolibrary.org/obo/CL_0000695`

#### Added
- [Cajal-Retzius cell](http://purl.obolibrary.org/obo/CL_0000695) [description](http://purl.org/dc/terms/description) "Cajal-Retzius cells are unique, early-born neurons found within the developing cerebral cortex and hippocampus. They are named after Santiago Ramon y Cajal and Gustaf Retzius, who first discovered them independently in the late 19th century. These neurons are characterized by horizontally oriented bipolar or multipolar cell bodies, facilitating their function in structuring the architecture of the cortex.
The primary role of Cajal-Retzius cells is to mediate the correct formation of the cortical layers in the developing brain. They accomplish this chiefly through the secretion of the extracellular matrix protein Reelin, which interacts with receptors on migrating neurons to control neuronal migration and position. The Reelin signal provided by Cajal-Retzius cells is essential to establish the cortical plate, where new neurons arising from the ventricular zone accumulate and form the recognizable layered structure of the cortical region. Disruptions in Reelin signaling can lead to reeler-like phenotypes characterized by inversion of cortical layers, demonstrating the importance of these cells during normal brain development.
In addition to layer formation, Cajal-Retzius cells also play a role in synaptic plasticity, which is a key aspect of learning, memory, and cognitive function. It has been suggested that these cells are involved in establishing the first functional synapses in the developing cortex, providing initial neural network activity. Furthermore, they have been linked to the modulation of gamma-aminobutyric acid (GABA) trafficking, excitatory-inhibitory balance, and thus, the fine-tune of the neural network activity during early postnatal life. Despite their disappearance shortly after birth in rodents and during early adolescence in humans, Cajal-Retzius cells lay the groundwork for the structure and functionality of the brain's cortical region.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1523/JNEUROSCI.1407-14.2014"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/S0166-2236(97)01164-8"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1111/j.1440-169X.2009.01102.x"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1155/2014/597395"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/nrn2009"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [Cajal-Retzius cell](http://purl.obolibrary.org/obo/CL_0000695) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0000695"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "Cajal-Retzius cell on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### D1/D2-hybrid medium spiny neuron `http://purl.obolibrary.org/obo/CL_4030050`
#### Removed
- [D1/D2-hybrid medium spiny neuron](http://purl.obolibrary.org/obo/CL_4030050) SubClassOf [BFO_0000002](http://purl.obolibrary.org/obo/BFO_0000002) 

#### Added
- [D1/D2-hybrid medium spiny neuron](http://purl.obolibrary.org/obo/CL_4030050) [RO_0002175](http://purl.obolibrary.org/obo/RO_0002175) [NCBITaxon_9606](http://purl.obolibrary.org/obo/NCBITaxon_9606) 

- [D1/D2-hybrid medium spiny neuron](http://purl.obolibrary.org/obo/CL_4030050) [in_subset](http://www.geneontology.org/formats/oboInOwl#inSubset) [human_reference_atlas](http://purl.obolibrary.org/obo/cl/cl-base#human_reference_atlas) 


### G1 retinal ganglion cell `http://purl.obolibrary.org/obo/CL_0003002`

#### Added
- [G1 retinal ganglion cell](http://purl.obolibrary.org/obo/CL_0003002) SubClassOf [RO_0002162](http://purl.obolibrary.org/obo/RO_0002162) some [NCBITaxon_9986](http://purl.obolibrary.org/obo/NCBITaxon_9986) 


### G10 retinal ganglion cell `http://purl.obolibrary.org/obo/CL_0003013`

#### Added
- [G10 retinal ganglion cell](http://purl.obolibrary.org/obo/CL_0003013) SubClassOf [RO_0002162](http://purl.obolibrary.org/obo/RO_0002162) some [NCBITaxon_9986](http://purl.obolibrary.org/obo/NCBITaxon_9986) 


### G11 retinal ganglion cell `http://purl.obolibrary.org/obo/CL_0003014`

#### Added
- [G11 retinal ganglion cell](http://purl.obolibrary.org/obo/CL_0003014) SubClassOf [RO_0002162](http://purl.obolibrary.org/obo/RO_0002162) some [NCBITaxon_9986](http://purl.obolibrary.org/obo/NCBITaxon_9986) 


### G2 retinal ganglion cell `http://purl.obolibrary.org/obo/CL_0003003`

#### Added
- [G2 retinal ganglion cell](http://purl.obolibrary.org/obo/CL_0003003) SubClassOf [RO_0002162](http://purl.obolibrary.org/obo/RO_0002162) some [NCBITaxon_9986](http://purl.obolibrary.org/obo/NCBITaxon_9986) 


### G3 retinal ganglion cell `http://purl.obolibrary.org/obo/CL_0003004`

#### Added
- [G3 retinal ganglion cell](http://purl.obolibrary.org/obo/CL_0003004) SubClassOf [RO_0002162](http://purl.obolibrary.org/obo/RO_0002162) some [NCBITaxon_9986](http://purl.obolibrary.org/obo/NCBITaxon_9986) 


### G4 retinal ganglion cell `http://purl.obolibrary.org/obo/CL_0003005`

#### Added
- [G4 retinal ganglion cell](http://purl.obolibrary.org/obo/CL_0003005) SubClassOf [RO_0002162](http://purl.obolibrary.org/obo/RO_0002162) some [NCBITaxon_9986](http://purl.obolibrary.org/obo/NCBITaxon_9986) 


### G5 retinal ganglion cell `http://purl.obolibrary.org/obo/CL_0003008`

#### Added
- [G5 retinal ganglion cell](http://purl.obolibrary.org/obo/CL_0003008) SubClassOf [RO_0002162](http://purl.obolibrary.org/obo/RO_0002162) some [NCBITaxon_9986](http://purl.obolibrary.org/obo/NCBITaxon_9986) 


### G6 retinal ganglion cell `http://purl.obolibrary.org/obo/CL_0003009`

#### Added
- [G6 retinal ganglion cell](http://purl.obolibrary.org/obo/CL_0003009) SubClassOf [RO_0002162](http://purl.obolibrary.org/obo/RO_0002162) some [NCBITaxon_9986](http://purl.obolibrary.org/obo/NCBITaxon_9986) 


### G7 retinal ganglion cell `http://purl.obolibrary.org/obo/CL_0003010`

#### Added
- [G7 retinal ganglion cell](http://purl.obolibrary.org/obo/CL_0003010) SubClassOf [RO_0002162](http://purl.obolibrary.org/obo/RO_0002162) some [NCBITaxon_9986](http://purl.obolibrary.org/obo/NCBITaxon_9986) 


### G8 retinal ganglion cell `http://purl.obolibrary.org/obo/CL_0003011`

#### Added
- [G8 retinal ganglion cell](http://purl.obolibrary.org/obo/CL_0003011) SubClassOf [RO_0002162](http://purl.obolibrary.org/obo/RO_0002162) some [NCBITaxon_9986](http://purl.obolibrary.org/obo/NCBITaxon_9986) 


### G9 retinal ganglion cell `http://purl.obolibrary.org/obo/CL_0003012`

#### Added
- [G9 retinal ganglion cell](http://purl.obolibrary.org/obo/CL_0003012) SubClassOf [RO_0002162](http://purl.obolibrary.org/obo/RO_0002162) some [NCBITaxon_9986](http://purl.obolibrary.org/obo/NCBITaxon_9986) 


### GABA-Glut neuron `http://purl.obolibrary.org/obo/CL_0008060`
#### Removed
- [GABA-Glut neuron](http://purl.obolibrary.org/obo/CL_0008060) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A neuron that releases both gamma-aminobutyric acid and glutamate as vesicular neurotransmitters."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:37124720"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:38092916"^^[string](http://www.w3.org/2001/XMLSchema#string) 

#### Added
- [GABA-Glut neuron](http://purl.obolibrary.org/obo/CL_0008060) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A neuron that releases both gamma-aminobutyric acid and glutamate as vesicular neurotransmitters."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:29924991"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:37124720"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:38092916"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### GABAergic interneuron `http://purl.obolibrary.org/obo/CL_0011005`

#### Added
- [GABAergic interneuron](http://purl.obolibrary.org/obo/CL_0011005) [RO_0002175](http://purl.obolibrary.org/obo/RO_0002175) [NCBITaxon_9606](http://purl.obolibrary.org/obo/NCBITaxon_9606) 

- [GABAergic interneuron](http://purl.obolibrary.org/obo/CL_0011005) [in_subset](http://www.geneontology.org/formats/oboInOwl#inSubset) [human_reference_atlas](http://purl.obolibrary.org/obo/cl/cl-base#human_reference_atlas) 

- [GABAergic interneuron](http://purl.obolibrary.org/obo/CL_0011005) [description](http://purl.org/dc/terms/description) "GABAergic interneurons, a major subtype of neurons, are characterized by the secretion of gamma-aminobutyric acid (GABA) &ndash; the primary inhibitory neurotransmitter in the central nervous system (CNS) of mammals. These cells play a crucial role in balancing the excitatory and inhibitory neurotransmission within neural networks. GABAergic neurons are particularly important for maintaining homeostasis within the brain, and any imbalance in GABAergic neurotransmission is often associated with neurological disorders such as epilepsy, schizophrenia, and mood disorders, highlighting their significance in the nervous system.
The primary function of GABAergic interneurons is to inhibit neural transmission or modulate the degree of activation of other neurons. They achieve this task by releasing GABA neurotransmitter, which binds to GABA receptors on other cells, resulting in a decrease of neuronal excitability. They form extensive circuits and are intricately involved in various neural processes including learning and memory, sensory perception, and motor control among others. Additionally, subsets of GABAergic interneurons serve as pacemakers in various brain rhythms thus contributing to cognitive processes such as attention and sensory perception.
GABAergic interneurons are not a homogenous group, they vary in terms of morphology, distribution, and function in the brain. For example, interneurons in the hippocampus regulate the transmission of information by inhibiting the flow of excitatory signals as part of a feedback loop. Other types, found predominantly in the cerebral cortex (like basket cells), exert control over a larger network of cells. Thus, despite being inhibitory in nature, their impact on neuronal circuitry broadens the diversity of information flow within the brain, hence potentiating information processing and ensuring proper nervous system function.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/s41583-019-0195-4"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.neuron.2013.01.011"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/S0074-7696(02)13011-7"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/S0893-133X(01)00225-1"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1152/physrev.00007.2017"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [GABAergic interneuron](http://purl.obolibrary.org/obo/CL_0011005) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0011005"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "GABAergic interneuron on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### GABAergic neuron `http://purl.obolibrary.org/obo/CL_0000617`

#### Added
- [GABAergic neuron](http://purl.obolibrary.org/obo/CL_0000617) [RO_0002175](http://purl.obolibrary.org/obo/RO_0002175) [NCBITaxon_9606](http://purl.obolibrary.org/obo/NCBITaxon_9606) 

- [GABAergic neuron](http://purl.obolibrary.org/obo/CL_0000617) [in_subset](http://www.geneontology.org/formats/oboInOwl#inSubset) [human_reference_atlas](http://purl.obolibrary.org/obo/cl/cl-base#human_reference_atlas) 


### GO_0002639 `http://purl.obolibrary.org/obo/GO_0002639`

#### Added
- Class: [GO_0002639](http://purl.obolibrary.org/obo/GO_0002639) 


### GO_0005983 `http://purl.obolibrary.org/obo/GO_0005983`

#### Added
- Class: [GO_0005983](http://purl.obolibrary.org/obo/GO_0005983) 


### GO_0014051 `http://purl.obolibrary.org/obo/GO_0014051`
#### Removed
- Class: [GO_0014051](http://purl.obolibrary.org/obo/GO_0014051) 



### GO_0017156 `http://purl.obolibrary.org/obo/GO_0017156`

#### Added
- Class: [GO_0017156](http://purl.obolibrary.org/obo/GO_0017156) 


### GO_0042588 `http://purl.obolibrary.org/obo/GO_0042588`

#### Added
- Class: [GO_0042588](http://purl.obolibrary.org/obo/GO_0042588) 


### GO_0046541 `http://purl.obolibrary.org/obo/GO_0046541`

#### Added
- Class: [GO_0046541](http://purl.obolibrary.org/obo/GO_0046541) 


### GO_0051024 `http://purl.obolibrary.org/obo/GO_0051024`
#### Removed
- Class: [GO_0051024](http://purl.obolibrary.org/obo/GO_0051024) 



### Gr1-high classical monocyte `http://purl.obolibrary.org/obo/CL_0002395`
#### Removed
- [Gr1-high classical monocyte](http://purl.obolibrary.org/obo/CL_0002395) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A resident monocyte that is Gr-1 high, CD43-negative, CX3CR1-negative, CD115-positive, and B220-negative."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:20628149"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:8890901"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "GOC:tfm"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "http://www.immgen.org/index_content.html"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [Gr1-high classical monocyte](http://purl.obolibrary.org/obo/CL_0002395) EquivalentTo [classical monocyte](http://purl.obolibrary.org/obo/CL_0000860) and ([lacks_plasma_membrane_part](http://purl.obolibrary.org/obo/CL_4030046) some [PR_000001014](http://purl.obolibrary.org/obo/PR_000001014)) and ([lacks_plasma_membrane_part](http://purl.obolibrary.org/obo/CL_4030046) some [PR_000001206](http://purl.obolibrary.org/obo/PR_000001206)) and ([lacks_plasma_membrane_part](http://purl.obolibrary.org/obo/CL_4030046) some [PR_000001879](http://purl.obolibrary.org/obo/PR_000001879)) and ([RO_0002104](http://purl.obolibrary.org/obo/RO_0002104) some [PR_000001012](http://purl.obolibrary.org/obo/PR_000001012)) and ([RO_0002104](http://purl.obolibrary.org/obo/RO_0002104) some [PR_000002062](http://purl.obolibrary.org/obo/PR_000002062)) and ([RO_0015015](http://purl.obolibrary.org/obo/RO_0015015) some [PR_000002980](http://purl.obolibrary.org/obo/PR_000002980)) 

#### Added
- [Gr1-high classical monocyte](http://purl.obolibrary.org/obo/CL_0002395) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A resident monocyte that is Gr-1 high, CD43-negative, CX3CR1-negative, CD115-positive, and B220-negative."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:20628149"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:8890901"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "GOC:tfm"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:27098205"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "http://www.immgen.org/index_content.html"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [Gr1-high classical monocyte](http://purl.obolibrary.org/obo/CL_0002395) EquivalentTo [classical monocyte](http://purl.obolibrary.org/obo/CL_0000860) and ([lacks_plasma_membrane_part](http://purl.obolibrary.org/obo/CL_4030046) some [PR_000001014](http://purl.obolibrary.org/obo/PR_000001014)) and ([lacks_plasma_membrane_part](http://purl.obolibrary.org/obo/CL_4030046) some [PR_000001206](http://purl.obolibrary.org/obo/PR_000001206)) and ([lacks_plasma_membrane_part](http://purl.obolibrary.org/obo/CL_4030046) some [PR_000001879](http://purl.obolibrary.org/obo/PR_000001879)) and ([RO_0002104](http://purl.obolibrary.org/obo/RO_0002104) some [PR_000001012](http://purl.obolibrary.org/obo/PR_000001012)) and ([RO_0002104](http://purl.obolibrary.org/obo/RO_0002104) some [PR_000002062](http://purl.obolibrary.org/obo/PR_000002062)) and ([RO_0002162](http://purl.obolibrary.org/obo/RO_0002162) some [NCBITaxon_10090](http://purl.obolibrary.org/obo/NCBITaxon_10090)) and ([RO_0015015](http://purl.obolibrary.org/obo/RO_0015015) some [PR_000002980](http://purl.obolibrary.org/obo/PR_000002980)) 

- [Gr1-high classical monocyte](http://purl.obolibrary.org/obo/CL_0002395) SubClassOf [RO_0002162](http://purl.obolibrary.org/obo/RO_0002162) some [NCBITaxon_10090](http://purl.obolibrary.org/obo/NCBITaxon_10090) 


### Gr1-positive, CD43-positive monocyte `http://purl.obolibrary.org/obo/CL_0002398`
#### Removed
- [Gr1-positive, CD43-positive monocyte](http://purl.obolibrary.org/obo/CL_0002398) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "An intermediate monocyte that is Gr1-positive, CD43-positive."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:20628149"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "GOC:tfm"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [Gr1-positive, CD43-positive monocyte](http://purl.obolibrary.org/obo/CL_0002398) EquivalentTo [monocyte](http://purl.obolibrary.org/obo/CL_0000576) and ([RO_0002104](http://purl.obolibrary.org/obo/RO_0002104) some [PR_000001012](http://purl.obolibrary.org/obo/PR_000001012)) and ([RO_0002104](http://purl.obolibrary.org/obo/RO_0002104) some [PR_000001879](http://purl.obolibrary.org/obo/PR_000001879)) and ([RO_0002104](http://purl.obolibrary.org/obo/RO_0002104) some [PR_000002062](http://purl.obolibrary.org/obo/PR_000002062)) and ([RO_0002104](http://purl.obolibrary.org/obo/RO_0002104) some [PR_000002980](http://purl.obolibrary.org/obo/PR_000002980)) 

#### Added
- [Gr1-positive, CD43-positive monocyte](http://purl.obolibrary.org/obo/CL_0002398) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "An intermediate monocyte that is Gr1-positive, CD43-positive."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:20628149"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "GOC:tfm"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:27098205"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [Gr1-positive, CD43-positive monocyte](http://purl.obolibrary.org/obo/CL_0002398) EquivalentTo [monocyte](http://purl.obolibrary.org/obo/CL_0000576) and ([RO_0002104](http://purl.obolibrary.org/obo/RO_0002104) some [PR_000001012](http://purl.obolibrary.org/obo/PR_000001012)) and ([RO_0002104](http://purl.obolibrary.org/obo/RO_0002104) some [PR_000001879](http://purl.obolibrary.org/obo/PR_000001879)) and ([RO_0002104](http://purl.obolibrary.org/obo/RO_0002104) some [PR_000002062](http://purl.obolibrary.org/obo/PR_000002062)) and ([RO_0002104](http://purl.obolibrary.org/obo/RO_0002104) some [PR_000002980](http://purl.obolibrary.org/obo/PR_000002980)) and ([RO_0002162](http://purl.obolibrary.org/obo/RO_0002162) some [NCBITaxon_10090](http://purl.obolibrary.org/obo/NCBITaxon_10090)) 


### Island of Calleja granule cell `http://purl.obolibrary.org/obo/CL_4030053`

#### Added
- [Island of Calleja granule cell](http://purl.obolibrary.org/obo/CL_4030053) [RO_0002175](http://purl.obolibrary.org/obo/RO_0002175) [NCBITaxon_9606](http://purl.obolibrary.org/obo/NCBITaxon_9606) 

- [Island of Calleja granule cell](http://purl.obolibrary.org/obo/CL_4030053) [in_subset](http://www.geneontology.org/formats/oboInOwl#inSubset) [human_reference_atlas](http://purl.obolibrary.org/obo/cl/cl-base#human_reference_atlas) 


### L2/3 intratelencephalic projecting glutamatergic neuron `http://purl.obolibrary.org/obo/CL_4030059`
#### Removed
- [L2/3 intratelencephalic projecting glutamatergic neuron](http://purl.obolibrary.org/obo/CL_4030059) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A transcriptomically distinct intratelencephalic-projecting glutamatergic neuron with a soma found between cortical layer 2-4. This intratelencephalic-projecting glutamatergic neuron has thin-tufted apical dendrites and extends its axonal projection into L5 in the neocortex. This neuronal type has a hyperpolarised resting membrane potential. The standard transcriptomic reference data for this cell type can be found on the CellxGene census under the collection: 'Transcriptomic cytoarchitecture reveals principles of human neocortex organization', dataset: 'Supercluster: IT-projecting excitatory neurons', Author Categories: 'CrossArea_subclass', clusters L2/3 IT."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:25622573"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:37292694"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:34004146"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:37824655"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "https://cellxgene.cziscience.com/e/c3aa4f95-7a18-4a7d-8dd8-ca324d714363.cxg/"^^[string](http://www.w3.org/2001/XMLSchema#string) 

#### Added
- [L2/3 intratelencephalic projecting glutamatergic neuron](http://purl.obolibrary.org/obo/CL_4030059) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A transcriptomically distinct intratelencephalic-projecting glutamatergic neuron with a soma found between cortical layer 2-4. This intratelencephalic-projecting glutamatergic neuron has thin-tufted apical dendrites and extends its axonal projection into L5 in the neocortex. This neuronal type has a hyperpolarised resting membrane potential."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:25622573"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:37292694"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:34004146"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:37824655"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "https://cellxgene.cziscience.com/e/c3aa4f95-7a18-4a7d-8dd8-ca324d714363.cxg/"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### L2/3 intratelencephalic projecting glutamatergic neuron (Homo sapiens) `http://purl.obolibrary.org/obo/CL_4072037`

#### Added
- Class: [L2/3 intratelencephalic projecting glutamatergic neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072037) 

- [L2/3 intratelencephalic projecting glutamatergic neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072037) [IAO_0000028](http://purl.obolibrary.org/obo/IAO_0000028) "L2/3 IT (Homo sapiens)"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [L2/3 intratelencephalic projecting glutamatergic neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072037) [RO_0002175](http://purl.obolibrary.org/obo/RO_0002175) [NCBITaxon_9606](http://purl.obolibrary.org/obo/NCBITaxon_9606) 

- [L2/3 intratelencephalic projecting glutamatergic neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072037) [contributor](http://purl.org/dc/terms/contributor) "https://orcid.org/0000-0002-5507-2103"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [L2/3 intratelencephalic projecting glutamatergic neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072037) [date](http://purl.org/dc/terms/date) "2025-09-10T09:15:51Z"^^[dateTime](http://www.w3.org/2001/XMLSchema#dateTime) 

- [L2/3 intratelencephalic projecting glutamatergic neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072037) [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "ILX:0770156"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [L2/3 intratelencephalic projecting glutamatergic neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072037) [id](http://www.geneontology.org/formats/oboInOwl#id) "CL:4072037"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [L2/3 intratelencephalic projecting glutamatergic neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072037) [in_subset](http://www.geneontology.org/formats/oboInOwl#inSubset) [cellxgene_subset](http://purl.obolibrary.org/obo/cl/cl-base#cellxgene_subset) 

- [L2/3 intratelencephalic projecting glutamatergic neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072037) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A transcriptomically distinct intratelencephalic-projecting glutamatergic neuron with a soma found between cortical layer 2-4. This intratelencephalic-projecting glutamatergic neuron has thin-tufted apical dendrites and extends its axonal projection into L5 in the neocortex. This neuronal type has a hyperpolarised resting membrane potential. The standard transcriptomic reference data for this cell type can be found on the CellxGene census under the collection: 'Transcriptomic cytoarchitecture reveals principles of human neocortex organization', dataset: 'Supercluster: IT-projecting excitatory neurons', Author Categories: 'CrossArea_subclass', clusters L2/3 IT."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:25622573"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:37292694"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:34004146"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:37824655"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "https://cellxgene.cziscience.com/e/c3aa4f95-7a18-4a7d-8dd8-ca324d714363.cxg/"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [L2/3 intratelencephalic projecting glutamatergic neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072037) [has_related_synonym](http://www.geneontology.org/formats/oboInOwl#hasRelatedSynonym) "L2/3 IT (Homo sapiens)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [has_synonym_type](http://www.geneontology.org/formats/oboInOwl#hasSynonymType) [OMO_0003000](http://purl.obolibrary.org/obo/OMO_0003000) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:34004146"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [L2/3 intratelencephalic projecting glutamatergic neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072037) [comment](http://www.w3.org/2000/01/rdf-schema#comment) "The marker set ONECUT2, COL5A2 can identify the Human cell type L2/3 intratelencephalic projecting glutamatergic neuron in the neocortex with a confidence of 0.670149932 (NS-Forest FBeta value)."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "https://doi.org/10.5281/zenodo.11165918"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [L2/3 intratelencephalic projecting glutamatergic neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072037) [label](http://www.w3.org/2000/01/rdf-schema#label) "L2/3 intratelencephalic projecting glutamatergic neuron (Homo sapiens)"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [L2/3 intratelencephalic projecting glutamatergic neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072037) SubClassOf [BFO_0000002](http://purl.obolibrary.org/obo/BFO_0000002) 

- [L2/3 intratelencephalic projecting glutamatergic neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072037) SubClassOf [L2/3 intratelencephalic projecting glutamatergic neuron](http://purl.obolibrary.org/obo/CL_4030059) 

- [L2/3 intratelencephalic projecting glutamatergic neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072037) SubClassOf [RO_0002162](http://purl.obolibrary.org/obo/RO_0002162) some [NCBITaxon_9606](http://purl.obolibrary.org/obo/NCBITaxon_9606) 


### L2/3-6 intratelencephalic projecting glutamatergic neuron `http://purl.obolibrary.org/obo/CL_4023040`

#### Added
- [L2/3-6 intratelencephalic projecting glutamatergic neuron](http://purl.obolibrary.org/obo/CL_4023040) [description](http://purl.org/dc/terms/description) "The L2/3-6 intratelencephalic projecting glutamatergic cortical neurons, often referred to by the shorthand L2/3-6 IT neurons, are essential variants of the most numerous types of excitatory neurons in the cerebral cortex. These neurons are found in layers 2, 3 to 6 of the cerebral cortex and are specifically classified under the glutamatergic neurons due to their ability to release the neurotransmitter. Therefore, they are primarily involved in producing excitatory responses, which serve as a significant component in the intricate pathways of neuronal communication and information processing within the brain.
These L2/3-6 IT neurons project intratelencephalically, implying that their axons interconnect regions within the cerebral hemispheres or between the two hemispheres, particularly to other cortical areas, the striatum, and the claustrum. This feature aids in vital brain functions such as perception, cognition, voluntary motor control, and learning. Importantly, these neurons possess a characteristic pyramidal morphology, which endows them with a high degree of connectivity to other neurons - thereby playing a crucial role in the formation and function of neural networks.
Each specific cortical layer's glutamatergic neuron subtype bears a different function; for instance, neurons in the upper layers (L2/3) generally project to other cortical areas, ultimately assisting in the integration of sensory and motor information across diverse cortical regions. Conversely, the deeper layer neurons (L6) are associated with the modulation of thalamo-cortical input and output, in addition to projecting to the thalamus, reinforcing the complexity and bilateral communication within the cerebral cortex. Thus, the L2/3-6 intratelencephalic projecting glutamatergic cortical neuron, beyond its role as an excitatory neuron, critically influences a myriad of higher brain functions, reinforcing its importance in studying neurological processes and disorders.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/nn.3917"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/nrn3469"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.neuron.2018.10.032"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1523/JNEUROSCI.0150-18.2018"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/s41586-021-03813-8"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [L2/3-6 intratelencephalic projecting glutamatergic neuron](http://purl.obolibrary.org/obo/CL_4023040) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_4023040"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "L2/3-6 intratelencephalic projecting glutamatergic neuron on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### L2/3/5 fan Martinotti sst GABAergic interneuron (Mmus) `http://purl.obolibrary.org/obo/CL_4023030`

#### Added
- [L2/3/5 fan Martinotti sst GABAergic interneuron (Mmus)](http://purl.obolibrary.org/obo/CL_4023030) SubClassOf [BFO_0000002](http://purl.obolibrary.org/obo/BFO_0000002) 


### L4 intratelencephalic projecting glutamatergic neuron `http://purl.obolibrary.org/obo/CL_4030063`
#### Removed
- [L4 intratelencephalic projecting glutamatergic neuron](http://purl.obolibrary.org/obo/CL_4030063) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A transcriptomically distinct intratelencephalic-projecting glutamatergic neuron with a soma found in cortical layer 3-4. This neuron type can have a pyramidal, star-pyramidal or spiny stellate morphology and projects its output to L2/3 and L5A/B. The standard transcriptomic reference data for this cell type can be found on the CellxGene census under the collection: 'Transcriptomic cytoarchitecture reveals principles of human neocortex organization', dataset: 'Supercluster: IT-projecting excitatory neurons', Author Categories: 'CrossArea_subclass', L4 IT."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:37292694"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:37824655"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "https://cellxgene.cziscience.com/e/c3aa4f95-7a18-4a7d-8dd8-ca324d714363.cxg/"^^[string](http://www.w3.org/2001/XMLSchema#string) 

#### Added
- [L4 intratelencephalic projecting glutamatergic neuron](http://purl.obolibrary.org/obo/CL_4030063) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A transcriptomically distinct intratelencephalic-projecting glutamatergic neuron with a soma found in cortical layer 3-4. This neuron type can have a pyramidal, star-pyramidal or spiny stellate morphology and projects its output to L2/3 and L5A/B."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:37292694"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:37824655"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "https://cellxgene.cziscience.com/e/c3aa4f95-7a18-4a7d-8dd8-ca324d714363.cxg/"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### L4 intratelencephalic projecting glutamatergic neuron (Homo sapiens) `http://purl.obolibrary.org/obo/CL_4072039`

#### Added
- Class: [L4 intratelencephalic projecting glutamatergic neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072039) 

- [L4 intratelencephalic projecting glutamatergic neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072039) [IAO_0000028](http://purl.obolibrary.org/obo/IAO_0000028) "L4 IT (Homo sapiens)"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [L4 intratelencephalic projecting glutamatergic neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072039) [RO_0002175](http://purl.obolibrary.org/obo/RO_0002175) [NCBITaxon_9606](http://purl.obolibrary.org/obo/NCBITaxon_9606) 

- [L4 intratelencephalic projecting glutamatergic neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072039) [date](http://purl.org/dc/terms/date) "2025-09-10T10:05:26Z"^^[dateTime](http://www.w3.org/2001/XMLSchema#dateTime) 

- [L4 intratelencephalic projecting glutamatergic neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072039) [id](http://www.geneontology.org/formats/oboInOwl#id) "CL:4072039"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [L4 intratelencephalic projecting glutamatergic neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072039) [in_subset](http://www.geneontology.org/formats/oboInOwl#inSubset) [cellxgene_subset](http://purl.obolibrary.org/obo/cl/cl-base#cellxgene_subset) 

- [L4 intratelencephalic projecting glutamatergic neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072039) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A transcriptomically distinct intratelencephalic-projecting glutamatergic neuron with a soma found in cortical layer 3-4. This neuron type can have a pyramidal, star-pyramidal or spiny stellate morphology and projects its output to L2/3 and L5A/B. The standard transcriptomic reference data for this cell type can be found on the CellxGene census under the collection: 'Transcriptomic cytoarchitecture reveals principles of human neocortex organization', dataset: 'Supercluster: IT-projecting excitatory neurons', Author Categories: 'CrossArea_subclass', L4 IT."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:37292694"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:37824655"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "https://cellxgene.cziscience.com/e/c3aa4f95-7a18-4a7d-8dd8-ca324d714363.cxg/"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [L4 intratelencephalic projecting glutamatergic neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072039) [has_related_synonym](http://www.geneontology.org/formats/oboInOwl#hasRelatedSynonym) "L4 IT (Homo sapiens)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [has_synonym_type](http://www.geneontology.org/formats/oboInOwl#hasSynonymType) [OMO_0003000](http://purl.obolibrary.org/obo/OMO_0003000) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:37292694"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [L4 intratelencephalic projecting glutamatergic neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072039) [comment](http://www.w3.org/2000/01/rdf-schema#comment) "The marker set RORB, ST8SIA4 can identify the Human cell type L4 intratelencephalic projecting glutamatergic neuron in the neocortex with a confidence of 0.720108696 (NS-Forest FBeta value)."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "https://doi.org/10.5281/zenodo.11165918"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [L4 intratelencephalic projecting glutamatergic neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072039) [label](http://www.w3.org/2000/01/rdf-schema#label) "L4 intratelencephalic projecting glutamatergic neuron (Homo sapiens)"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [L4 intratelencephalic projecting glutamatergic neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072039) SubClassOf [BFO_0000002](http://purl.obolibrary.org/obo/BFO_0000002) 

- [L4 intratelencephalic projecting glutamatergic neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072039) SubClassOf [L4 intratelencephalic projecting glutamatergic neuron](http://purl.obolibrary.org/obo/CL_4030063) 

- [L4 intratelencephalic projecting glutamatergic neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072039) SubClassOf [RO_0002162](http://purl.obolibrary.org/obo/RO_0002162) some [NCBITaxon_9606](http://purl.obolibrary.org/obo/NCBITaxon_9606) 


### L4 sst Martinotti interneuron (Mmus) `http://purl.obolibrary.org/obo/CL_4023031`

#### Added
- [L4 sst Martinotti interneuron (Mmus)](http://purl.obolibrary.org/obo/CL_4023031) SubClassOf [BFO_0000002](http://purl.obolibrary.org/obo/BFO_0000002) 


### L5 extratelencephalic projecting glutamatergic cortical neuron `http://purl.obolibrary.org/obo/CL_4023041`
#### Removed
- [L5 extratelencephalic projecting glutamatergic cortical neuron](http://purl.obolibrary.org/obo/CL_4023041) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A transcriptomically distinct glutamatergic neuron, with a soma found in the deeper portion of L5, that has long-range axonal projections including deep subcortical targets outside of the telencephalon and, in some cases, the spinal cord. While the L5 ET neuron projections are not limited to ET targets, they are clearly differentiated from the neuron subclasses whose projections are constrained to intratelencephalic (IT) targets.  L5 ET neurons are generally the largest excitatory cortical neurons, typically having a thick apical dendrite with a prominent dendritic tuft in layer 1 and displaying burst-firing physiological characteristics. The standard transcriptomic reference data for this cell type can be found on the CellxGene census under the collection: 'Transcriptomic cytoarchitecture reveals principles of human neocortex organization', dataset: 'Supercluster: Deep layer (non-IT) excitatory neurons ', Author Categories: 'CrossArea_subclass', clusters L5 ET."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:34616075"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:37824655"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "https://cellxgene.cziscience.com/e/d01c9dff-abd1-4825-bf30-2eb2ba74597e.cxg/"^^[string](http://www.w3.org/2001/XMLSchema#string) 

#### Added
- [L5 extratelencephalic projecting glutamatergic cortical neuron](http://purl.obolibrary.org/obo/CL_4023041) [description](http://purl.org/dc/terms/description) "The L5 extratelencephalic projecting glutamatergic cortical neuron is a highly specialized neuron located in layer 5 (L5) of the cerebral cortex. The cerebral cortex, the outermost shell of the brain, is divided into six layers based on the types of neurons they contain and the functions they perform. L5 is known for its characteristic large pyramidal neurons, among which L5 extratelencephalic projecting glutamatergic cortical neurons reside.
Functionally, the primary role of the L5 extratelencephalic projecting glutamatergic cortical neuron is to send signals from the cerebral cortex to other parts of the brain or body apart from the telencephalon - the largest part of the brain that encompasses both cerebral hemispheres. These neurons accomplish this task through their long axons that form connections or synapses with neurons in other subcortical regions. Notably, these neurons are glutamatergic, meaning they release the neurotransmitter glutamate, which is crucial for signaling in the brain as it is the primary excitatory neurotransmitter and plays a crucial role in neural plasticity, learning, and memory.
The L5 extratelencephalic projecting glutamatergic cortical neuron is pivotal in carrying out complex motor activities as the signals they transmit are often directed to areas of the brain responsible for the initiation and modulation of motor functions, such as the spinal cord or the brainstem. Alterations in their structure or function can lead to various neurological disorders like autism, epilepsy, and schizophrenia.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1523/JNEUROSCI.0150-18.2018"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.3389/fnsyn.2022.1006773"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.neuron.2021.08.030"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/s41467-020-14952-3"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1093/cercor/bhx012"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [L5 extratelencephalic projecting glutamatergic cortical neuron](http://purl.obolibrary.org/obo/CL_4023041) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A transcriptomically distinct glutamatergic neuron, with a soma found in the deeper portion of L5, that has long-range axonal projections including deep subcortical targets outside of the telencephalon and, in some cases, the spinal cord. While the L5 ET neuron projections are not limited to ET targets, they are clearly differentiated from the neuron subclasses whose projections are constrained to intratelencephalic (IT) targets. L5 ET neurons are generally the largest excitatory cortical neurons, typically having a thick apical dendrite with a prominent dendritic tuft in layer 1 and displaying burst-firing physiological characteristics."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:34616075"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:37824655"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "https://cellxgene.cziscience.com/e/d01c9dff-abd1-4825-bf30-2eb2ba74597e.cxg/"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [L5 extratelencephalic projecting glutamatergic cortical neuron](http://purl.obolibrary.org/obo/CL_4023041) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_4023041"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "L5 extratelencephalic projecting glutamatergic cortical neuron on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### L5 extratelencephalic projecting glutamatergic cortical neuron (Homo sapiens) `http://purl.obolibrary.org/obo/CL_4072043`

#### Added
- Class: [L5 extratelencephalic projecting glutamatergic cortical neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072043) 

- [L5 extratelencephalic projecting glutamatergic cortical neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072043) [IAO_0000028](http://purl.obolibrary.org/obo/IAO_0000028) "L5 ET glut (Homo sapiens)"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [L5 extratelencephalic projecting glutamatergic cortical neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072043) [RO_0002175](http://purl.obolibrary.org/obo/RO_0002175) [NCBITaxon_9606](http://purl.obolibrary.org/obo/NCBITaxon_9606) 

- [L5 extratelencephalic projecting glutamatergic cortical neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072043) [contributor](http://purl.org/dc/terms/contributor) "https://orcid.org/0000-0002-5507-2103"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [L5 extratelencephalic projecting glutamatergic cortical neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072043) [date](http://purl.org/dc/terms/date) "2025-09-16T09:26:37Z"^^[dateTime](http://www.w3.org/2001/XMLSchema#dateTime) 

- [L5 extratelencephalic projecting glutamatergic cortical neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072043) [id](http://www.geneontology.org/formats/oboInOwl#id) "CL:4072043"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [L5 extratelencephalic projecting glutamatergic cortical neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072043) [in_subset](http://www.geneontology.org/formats/oboInOwl#inSubset) [BDS_subset](http://purl.obolibrary.org/obo/cl/cl-base#BDS_subset) 

- [L5 extratelencephalic projecting glutamatergic cortical neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072043) [in_subset](http://www.geneontology.org/formats/oboInOwl#inSubset) [cellxgene_subset](http://purl.obolibrary.org/obo/cl/cl-base#cellxgene_subset) 

- [L5 extratelencephalic projecting glutamatergic cortical neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072043) [has_broad_synonym](http://www.geneontology.org/formats/oboInOwl#hasBroadSynonym) "subcerebral projection (SCPN) neuron (Homo sapiens)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.neuron.2005.08.030"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [L5 extratelencephalic projecting glutamatergic cortical neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072043) [has_related_synonym](http://www.geneontology.org/formats/oboInOwl#hasRelatedSynonym) "L5b neuron (Homo sapiens)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [has_synonym_type](http://www.geneontology.org/formats/oboInOwl#hasSynonymType) [OMO_0003000](http://purl.obolibrary.org/obo/OMO_0003000) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.neuron.2011.07.029"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [L5 extratelencephalic projecting glutamatergic cortical neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072043) [has_narrow_synonym](http://www.geneontology.org/formats/oboInOwl#hasNarrowSynonym) "pyramidal tract (PT) neuron (Homo sapiens)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/nrn3469"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [L5 extratelencephalic projecting glutamatergic cortical neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072043) [has_exact_synonym](http://www.geneontology.org/formats/oboInOwl#hasExactSynonym) "burst-firing layer 5 neuron (Homo sapiens)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.3389/fncel.2011.00001"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [L5 extratelencephalic projecting glutamatergic cortical neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072043) [has_exact_synonym](http://www.geneontology.org/formats/oboInOwl#hasExactSynonym) "thick-tufted layer 5 (TTL5) pyramidal neuron (Homo sapiens)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.3389/fncel.2015.00233"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [L5 extratelencephalic projecting glutamatergic cortical neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072043) [has_exact_synonym](http://www.geneontology.org/formats/oboInOwl#hasExactSynonym) "Pyramidal tract-like (PT-l) (Homo sapiens)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [comment](http://www.w3.org/2000/01/rdf-schema#comment) "pyramidal tract-like is used because, unlike pyramidal tract neurons in the motor cortex, neurons in the structures like the auditory cortex do not project to the spinal cord."^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:31626774"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [L5 extratelencephalic projecting glutamatergic cortical neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072043) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A transcriptomically distinct glutamatergic neuron, with a soma found in the deeper portion of L5, that has long-range axonal projections including deep subcortical targets outside of the telencephalon and, in some cases, the spinal cord. While the L5 ET neuron projections are not limited to ET targets, they are clearly differentiated from the neuron subclasses whose projections are constrained to intratelencephalic (IT) targets. L5 ET neurons are generally the largest excitatory cortical neurons, typically having a thick apical dendrite with a prominent dendritic tuft in layer 1 and displaying burst-firing physiological characteristics. The standard transcriptomic reference data for this cell type can be found on the CellxGene census under the collection: 'Transcriptomic cytoarchitecture reveals principles of human neocortex organization', dataset: 'Supercluster: Deep layer (non-IT) excitatory neurons ', Author Categories: 'CrossArea_subclass', clusters L5 ET."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:34616075"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:37824655"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "https://cellxgene.cziscience.com/e/d01c9dff-abd1-4825-bf30-2eb2ba74597e.cxg/"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [L5 extratelencephalic projecting glutamatergic cortical neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072043) [comment](http://www.w3.org/2000/01/rdf-schema#comment) "The marker set CHST8, POU3F1 can identify the Human cell type L5 extratelencephalic projecting glutamatergic cortical neuron in the neocortex with a confidence of 0.815891473 (NS-Forest FBeta value)."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "https://doi.org/10.5281/zenodo.11165918"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [L5 extratelencephalic projecting glutamatergic cortical neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072043) [label](http://www.w3.org/2000/01/rdf-schema#label) "L5 extratelencephalic projecting glutamatergic cortical neuron (Homo sapiens)"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [L5 extratelencephalic projecting glutamatergic cortical neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072043) SubClassOf [BFO_0000002](http://purl.obolibrary.org/obo/BFO_0000002) 

- [L5 extratelencephalic projecting glutamatergic cortical neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072043) SubClassOf [L5 extratelencephalic projecting glutamatergic cortical neuron](http://purl.obolibrary.org/obo/CL_4023041) 

- [L5 extratelencephalic projecting glutamatergic cortical neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072043) SubClassOf [RO_0002162](http://purl.obolibrary.org/obo/RO_0002162) some [NCBITaxon_9606](http://purl.obolibrary.org/obo/NCBITaxon_9606) 


### L5 intratelencephalic projecting glutamatergic neuron `http://purl.obolibrary.org/obo/CL_4030064`
#### Removed
- [L5 intratelencephalic projecting glutamatergic neuron](http://purl.obolibrary.org/obo/CL_4030064) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "An intratelencephalic-projecting glutamatergic neuron with a soma found in cortical layer 5. The standard transcriptomic reference data for this cell type can be found on the CellxGene census under the collection: 'Transcriptomic cytoarchitecture reveals principles of human neocortex organization', dataset: 'Supercluster: IT-projecting excitatory neurons', Author Categories: 'CrossArea_subclass', L5 IT."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:37609206"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:37292694"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:37824655"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "https://cellxgene.cziscience.com/e/c3aa4f95-7a18-4a7d-8dd8-ca324d714363.cxg/"^^[string](http://www.w3.org/2001/XMLSchema#string) 

#### Added
- [L5 intratelencephalic projecting glutamatergic neuron](http://purl.obolibrary.org/obo/CL_4030064) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "An intratelencephalic-projecting glutamatergic neuron with a soma found in cortical layer 5."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:37609206"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:37292694"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:37824655"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "https://cellxgene.cziscience.com/e/c3aa4f95-7a18-4a7d-8dd8-ca324d714363.cxg/"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### L5 intratelencephalic projecting glutamatergic neuron (Homo sapiens) `http://purl.obolibrary.org/obo/CL_4072038`

#### Added
- Class: [L5 intratelencephalic projecting glutamatergic neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072038) 

- [L5 intratelencephalic projecting glutamatergic neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072038) [IAO_0000028](http://purl.obolibrary.org/obo/IAO_0000028) "L5 IT (Homo sapiens)"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [L5 intratelencephalic projecting glutamatergic neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072038) [RO_0002175](http://purl.obolibrary.org/obo/RO_0002175) [NCBITaxon_9606](http://purl.obolibrary.org/obo/NCBITaxon_9606) 

- [L5 intratelencephalic projecting glutamatergic neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072038) [contributor](http://purl.org/dc/terms/contributor) "https://orcid.org/0000-0002-5507-2103"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [L5 intratelencephalic projecting glutamatergic neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072038) [date](http://purl.org/dc/terms/date) "2025-09-10T09:47:23Z"^^[dateTime](http://www.w3.org/2001/XMLSchema#dateTime) 

- [L5 intratelencephalic projecting glutamatergic neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072038) [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "ILX:0770157"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [L5 intratelencephalic projecting glutamatergic neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072038) [id](http://www.geneontology.org/formats/oboInOwl#id) "CL:4072038"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [L5 intratelencephalic projecting glutamatergic neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072038) [in_subset](http://www.geneontology.org/formats/oboInOwl#inSubset) [cellxgene_subset](http://purl.obolibrary.org/obo/cl/cl-base#cellxgene_subset) 

- [L5 intratelencephalic projecting glutamatergic neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072038) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "An intratelencephalic-projecting glutamatergic neuron with a soma found in cortical layer 5. The standard transcriptomic reference data for this cell type can be found on the CellxGene census under the collection: 'Transcriptomic cytoarchitecture reveals principles of human neocortex organization', dataset: 'Supercluster: IT-projecting excitatory neurons', Author Categories: 'CrossArea_subclass', L5 IT."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:37609206"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:37292694"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:37824655"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "https://cellxgene.cziscience.com/e/c3aa4f95-7a18-4a7d-8dd8-ca324d714363.cxg/"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [L5 intratelencephalic projecting glutamatergic neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072038) [has_related_synonym](http://www.geneontology.org/formats/oboInOwl#hasRelatedSynonym) "L5 IT (Homo sapiens)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [has_synonym_type](http://www.geneontology.org/formats/oboInOwl#hasSynonymType) [OMO_0003000](http://purl.obolibrary.org/obo/OMO_0003000) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:37292694"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [L5 intratelencephalic projecting glutamatergic neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072038) [comment](http://www.w3.org/2000/01/rdf-schema#comment) "The marker set ARSJ, RORB, KCNK2 can identify the Human cell type L5 intratelencephalic projecting glutamatergic neuron in the neocortex with a confidence of 0.764261336 (NS-Forest FBeta value)."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "https://doi.org/10.5281/zenodo.11165918"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [L5 intratelencephalic projecting glutamatergic neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072038) [label](http://www.w3.org/2000/01/rdf-schema#label) "L5 intratelencephalic projecting glutamatergic neuron (Homo sapiens)"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [L5 intratelencephalic projecting glutamatergic neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072038) SubClassOf [BFO_0000002](http://purl.obolibrary.org/obo/BFO_0000002) 

- [L5 intratelencephalic projecting glutamatergic neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072038) SubClassOf [L5 intratelencephalic projecting glutamatergic neuron](http://purl.obolibrary.org/obo/CL_4030064) 

- [L5 intratelencephalic projecting glutamatergic neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072038) SubClassOf [RO_0002162](http://purl.obolibrary.org/obo/RO_0002162) some [NCBITaxon_9606](http://purl.obolibrary.org/obo/NCBITaxon_9606) 


### L5 non-Martinotti sst GABAergic interneuron (Mmus) `http://purl.obolibrary.org/obo/CL_4023028`

#### Added
- [L5 non-Martinotti sst GABAergic interneuron (Mmus)](http://purl.obolibrary.org/obo/CL_4023028) SubClassOf [BFO_0000002](http://purl.obolibrary.org/obo/BFO_0000002) 


### L5 sst T-Martinotti interneuron (Mmus) `http://purl.obolibrary.org/obo/CL_4023027`

#### Added
- [L5 sst T-Martinotti interneuron (Mmus)](http://purl.obolibrary.org/obo/CL_4023027) SubClassOf [BFO_0000002](http://purl.obolibrary.org/obo/BFO_0000002) 


### L5/6 cck GABAergic interneuron (Mmus) `http://purl.obolibrary.org/obo/CL_4023071`

#### Added
- [L5/6 cck GABAergic interneuron (Mmus)](http://purl.obolibrary.org/obo/CL_4023071) SubClassOf [BFO_0000002](http://purl.obolibrary.org/obo/BFO_0000002) 


### L5/6 near-projecting glutamatergic neuron `http://purl.obolibrary.org/obo/CL_4030067`
#### Removed
- [L5/6 near-projecting glutamatergic neuron](http://purl.obolibrary.org/obo/CL_4030067) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A near-projecting glutamatergic neuron with a soma found in cortical layer 5/6."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:37292694"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:34004146"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:37824655"^^[string](http://www.w3.org/2001/XMLSchema#string) 

#### Added
- [L5/6 near-projecting glutamatergic neuron](http://purl.obolibrary.org/obo/CL_4030067) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A transcriptomically defined near-projecting glutamatergic neuron with a soma found in cortical layer 5/6."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:37292694"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:34004146"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:37824655"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### L5/6 near-projecting glutamatergic neuron (Homo sapiens) `http://purl.obolibrary.org/obo/CL_4072042`

#### Added
- Class: [L5/6 near-projecting glutamatergic neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072042) 

- [L5/6 near-projecting glutamatergic neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072042) [IAO_0000028](http://purl.obolibrary.org/obo/IAO_0000028) "L5/6 NP (Homo sapiens)"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [L5/6 near-projecting glutamatergic neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072042) [RO_0002175](http://purl.obolibrary.org/obo/RO_0002175) [NCBITaxon_9606](http://purl.obolibrary.org/obo/NCBITaxon_9606) 

- [L5/6 near-projecting glutamatergic neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072042) [contributor](http://purl.org/dc/terms/contributor) "https://orcid.org/0000-0002-5507-2103"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [L5/6 near-projecting glutamatergic neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072042) [date](http://purl.org/dc/terms/date) "2025-09-16T08:55:09Z"^^[dateTime](http://www.w3.org/2001/XMLSchema#dateTime) 

- [L5/6 near-projecting glutamatergic neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072042) [id](http://www.geneontology.org/formats/oboInOwl#id) "CL:4072042"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [L5/6 near-projecting glutamatergic neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072042) [in_subset](http://www.geneontology.org/formats/oboInOwl#inSubset) [cellxgene_subset](http://purl.obolibrary.org/obo/cl/cl-base#cellxgene_subset) 

- [L5/6 near-projecting glutamatergic neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072042) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A transcriptomically defined near-projecting glutamatergic neuron with a soma found in cortical layer 5/6 in humans."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:37292694"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:34004146"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:37824655"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "https://cellxgene.cziscience.com/e/d01c9dff-abd1-4825-bf30-2eb2ba74597e.cxg/"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [L5/6 near-projecting glutamatergic neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072042) [has_related_synonym](http://www.geneontology.org/formats/oboInOwl#hasRelatedSynonym) "L5/6 NP (Homo sapiens)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [has_synonym_type](http://www.geneontology.org/formats/oboInOwl#hasSynonymType) [OMO_0003000](http://purl.obolibrary.org/obo/OMO_0003000) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:34004146"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [L5/6 near-projecting glutamatergic neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072042) [comment](http://www.w3.org/2000/01/rdf-schema#comment) "The marker set NPSR1-AS1, TLL1 can identify the Human cell type L5/6 near-projecting glutamatergic neuron in the neocortex with a confidence of 0.899288218 (NS-Forest FBeta value)."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "https://doi.org/10.5281/zenodo.11165918"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [L5/6 near-projecting glutamatergic neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072042) [label](http://www.w3.org/2000/01/rdf-schema#label) "L5/6 near-projecting glutamatergic neuron (Homo sapiens)"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [L5/6 near-projecting glutamatergic neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072042) SubClassOf [BFO_0000002](http://purl.obolibrary.org/obo/BFO_0000002) 

- [L5/6 near-projecting glutamatergic neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072042) SubClassOf [L5/6 near-projecting glutamatergic neuron](http://purl.obolibrary.org/obo/CL_4030067) 

- [L5/6 near-projecting glutamatergic neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072042) SubClassOf [RO_0002162](http://purl.obolibrary.org/obo/RO_0002162) some [NCBITaxon_9606](http://purl.obolibrary.org/obo/NCBITaxon_9606) 


### L5/6 near-projecting glutamatergic neuron of the primary motor cortex `http://purl.obolibrary.org/obo/CL_4023043`

#### Added
- [L5/6 near-projecting glutamatergic neuron of the primary motor cortex](http://purl.obolibrary.org/obo/CL_4023043) [description](http://purl.org/dc/terms/description) "The L5/6 near-projecting glutamatergic neuron of the primary motor cortex, also known as the M1 layer 5/6 intratelencephalic pyramidal cell, is a subcategory of excitatory neurons primarily found in layers 5 and 6 of the primary motor cortex (M1), an area of the cerebrum responsible for executing voluntary motor functions. The term &ldquo;glutamatergic&rdquo; refers to these neurons releasing the neurotransmitter glutamate, which is a major excitatory neurotransmitter in the mammalian central nervous system. This function contributes to the propagation of nerve signals, playing a crucial role in learning and memory procedures.
These specialized neurons project close to their site of origin, specifically within the telencephalon or the largest previous division of the forebrain. These near projections are primarily involved in forming intratelencephalic connections. This particular characteristic distinguishes them from the long-range projecting neurons, which can bridge distant regions of the brain.
Functionally, the L5/6 near-projecting glutamatergic neurons play important roles in various aspects of cognitive performances, including motor control, decision making, and sensory processing. Studies have implicated these neurons in action selection and execution, due to their powerful excitatory influence on other neurons within the motor cortex and other parts of the telencephalon. Additionally, the near-projecting neurons play a significant role in the modulation of local cortical circuit dynamics, through their intra-cortical and cortico-striatal projections, further influencing both motor and cognitive functions.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1523/JNEUROSCI.0150-18.2018"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1523/JNEUROSCI.1188-17.2017"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/s41586-018-0642-9"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1523/JNEUROSCI.0428-23.2023"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [L5/6 near-projecting glutamatergic neuron of the primary motor cortex](http://purl.obolibrary.org/obo/CL_4023043) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_4023043"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "L5/6 near-projecting glutamatergic neuron of the primary motor cortex on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### L5/6 non-Martinotti sst GABAergic interneuron (Mmus) `http://purl.obolibrary.org/obo/CL_4023118`

#### Added
- [L5/6 non-Martinotti sst GABAergic interneuron (Mmus)](http://purl.obolibrary.org/obo/CL_4023118) SubClassOf [BFO_0000002](http://purl.obolibrary.org/obo/BFO_0000002) 


### L6 corticothalamic-projecting glutamatergic cortical neuron `http://purl.obolibrary.org/obo/CL_4023042`
#### Removed
- [L6 corticothalamic-projecting glutamatergic cortical neuron](http://purl.obolibrary.org/obo/CL_4023042) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A transcriptomically distinct corticothalamic-projecting neuron with a soma found in cortical layer 6. The standard transcriptomic reference data for this cell type can be found on the CellxGene census under the collection: 'Transcriptomic cytoarchitecture reveals principles of human neocortex organization', dataset: 'Supercluster: Deep layer (non-IT) excitatory neurons', Author Categories: 'CrossArea_subclass', clusters L6 CT."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:34616075"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:37824655"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "https://cellxgene.cziscience.com/e/d01c9dff-abd1-4825-bf30-2eb2ba74597e.cxg/"^^[string](http://www.w3.org/2001/XMLSchema#string) 

#### Added
- [L6 corticothalamic-projecting glutamatergic cortical neuron](http://purl.obolibrary.org/obo/CL_4023042) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A transcriptomically distinct corticothalamic-projecting neuron with a soma found in cortical layer 6."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:34616075"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:37824655"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "https://cellxgene.cziscience.com/e/d01c9dff-abd1-4825-bf30-2eb2ba74597e.cxg/"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### L6 corticothalamic-projecting glutamatergic cortical neuron (Homo sapiens) `http://purl.obolibrary.org/obo/CL_4072044`

#### Added
- Class: [L6 corticothalamic-projecting glutamatergic cortical neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072044) 

- [L6 corticothalamic-projecting glutamatergic cortical neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072044) [IAO_0000028](http://purl.obolibrary.org/obo/IAO_0000028) "L6 CT glut (Homo sapiens)"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [L6 corticothalamic-projecting glutamatergic cortical neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072044) [RO_0002175](http://purl.obolibrary.org/obo/RO_0002175) [NCBITaxon_9606](http://purl.obolibrary.org/obo/NCBITaxon_9606) 

- [L6 corticothalamic-projecting glutamatergic cortical neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072044) [contributor](http://purl.org/dc/terms/contributor) "https://orcid.org/0000-0002-5507-2103"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [L6 corticothalamic-projecting glutamatergic cortical neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072044) [date](http://purl.org/dc/terms/date) "2025-09-16T09:42:20Z"^^[dateTime](http://www.w3.org/2001/XMLSchema#dateTime) 

- [L6 corticothalamic-projecting glutamatergic cortical neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072044) [id](http://www.geneontology.org/formats/oboInOwl#id) "CL:4072044"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [L6 corticothalamic-projecting glutamatergic cortical neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072044) [in_subset](http://www.geneontology.org/formats/oboInOwl#inSubset) [BDS_subset](http://purl.obolibrary.org/obo/cl/cl-base#BDS_subset) 

- [L6 corticothalamic-projecting glutamatergic cortical neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072044) [in_subset](http://www.geneontology.org/formats/oboInOwl#inSubset) [cellxgene_subset](http://purl.obolibrary.org/obo/cl/cl-base#cellxgene_subset) 

- [L6 corticothalamic-projecting glutamatergic cortical neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072044) [has_exact_synonym](http://www.geneontology.org/formats/oboInOwl#hasExactSynonym) "L6 CT (Homo sapiens)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:34616075"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:28625486"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [L6 corticothalamic-projecting glutamatergic cortical neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072044) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A transcriptomically distinct corticothalamic-projecting neuron with a soma found in cortical layer 6. The standard transcriptomic reference data for this cell type can be found on the CellxGene census under the collection: 'Transcriptomic cytoarchitecture reveals principles of human neocortex organization', dataset: 'Supercluster: Deep layer (non-IT) excitatory neurons', Author Categories: 'CrossArea_subclass', clusters L6 CT."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:34616075"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:37824655"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "https://cellxgene.cziscience.com/e/d01c9dff-abd1-4825-bf30-2eb2ba74597e.cxg/"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [L6 corticothalamic-projecting glutamatergic cortical neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072044) [comment](http://www.w3.org/2000/01/rdf-schema#comment) "The marker set SEMA5B, ADAMTSL1, SULF1 can identify the Human cell type L6 corticothalamic-projecting glutamatergic cortical neuron in the neocortex with a confidence of 0.631898455 (NS-Forest FBeta value)."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "https://doi.org/10.5281/zenodo.11165918"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [L6 corticothalamic-projecting glutamatergic cortical neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072044) [label](http://www.w3.org/2000/01/rdf-schema#label) "L6 corticothalamic-projecting glutamatergic cortical neuron (Homo sapiens)"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [L6 corticothalamic-projecting glutamatergic cortical neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072044) SubClassOf [BFO_0000002](http://purl.obolibrary.org/obo/BFO_0000002) 

- [L6 corticothalamic-projecting glutamatergic cortical neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072044) SubClassOf [L6 corticothalamic-projecting glutamatergic cortical neuron](http://purl.obolibrary.org/obo/CL_4023042) 

- [L6 corticothalamic-projecting glutamatergic cortical neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072044) SubClassOf [RO_0002162](http://purl.obolibrary.org/obo/RO_0002162) some [NCBITaxon_9606](http://purl.obolibrary.org/obo/NCBITaxon_9606) 


### L6 intratelencephalic projecting Car3 glutamatergic neuron `http://purl.obolibrary.org/obo/CL_4030068`
#### Removed
- [L6 intratelencephalic projecting Car3 glutamatergic neuron](http://purl.obolibrary.org/obo/CL_4030068) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A transcriptomically distinct intratelencepalic-projecting glutamatergic neuron that expresses Car3 with a soma found in L6 . The standard transcriptomic reference data for this cell type can be found on the CellxGene census under the collection: 'Transcriptomic cytoarchitecture reveals principles of human neocortex organization', dataset: 'Supercluster: IT-projecting excitatory neurons', Author Categories: 'CrossArea_cluster', L6 IT Car3."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "doi:10.1101/2022.11.30.518285"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "doi:10.1007/s00424-024-02923-2"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:34004146"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:37824655"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [L6 intratelencephalic projecting Car3 glutamatergic neuron](http://purl.obolibrary.org/obo/CL_4030068) [has_exact_synonym](http://www.geneontology.org/formats/oboInOwl#hasExactSynonym) "L6 IT Car3 glutamatergic neuron"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [has_synonym_type](http://www.geneontology.org/formats/oboInOwl#hasSynonymType) [OMO_0003000](http://purl.obolibrary.org/obo/OMO_0003000) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "https://orcid.org/0000-0003-4389-9821"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [L6 intratelencephalic projecting Car3 glutamatergic neuron](http://purl.obolibrary.org/obo/CL_4030068) EquivalentTo [L6 intratelencephalic projecting glutamatergic neuron](http://purl.obolibrary.org/obo/CL_4030065) and ([RO_0002100](http://purl.obolibrary.org/obo/RO_0002100) some [UBERON_0005395](http://purl.obolibrary.org/obo/UBERON_0005395)) and ([RO_0002292](http://purl.obolibrary.org/obo/RO_0002292) some [PR_000004919](http://purl.obolibrary.org/obo/PR_000004919)) 

#### Added
- [L6 intratelencephalic projecting Car3 glutamatergic neuron](http://purl.obolibrary.org/obo/CL_4030068) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A transcriptomically distinct intratelencepalic-projecting glutamatergic neuron that expresses Car3 with a soma found in L6."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "doi:10.1101/2022.11.30.518285"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "doi:10.1007/s00424-024-02923-2"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:34004146"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:37824655"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [L6 intratelencephalic projecting Car3 glutamatergic neuron](http://purl.obolibrary.org/obo/CL_4030068) [has_exact_synonym](http://www.geneontology.org/formats/oboInOwl#hasExactSynonym) "L6 IT Car3 glutamatergic neuron"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [has_synonym_type](http://www.geneontology.org/formats/oboInOwl#hasSynonymType) [OMO_0003000](http://purl.obolibrary.org/obo/OMO_0003000) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "doi:10.1101/2022.11.06.515349"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [L6 intratelencephalic projecting Car3 glutamatergic neuron](http://purl.obolibrary.org/obo/CL_4030068) EquivalentTo [L2/3-6 intratelencephalic projecting glutamatergic neuron](http://purl.obolibrary.org/obo/CL_4023040) and ([RO_0002100](http://purl.obolibrary.org/obo/RO_0002100) some [UBERON_0005395](http://purl.obolibrary.org/obo/UBERON_0005395)) and ([RO_0002292](http://purl.obolibrary.org/obo/RO_0002292) some [PR_000004919](http://purl.obolibrary.org/obo/PR_000004919)) 

- [L6 intratelencephalic projecting Car3 glutamatergic neuron](http://purl.obolibrary.org/obo/CL_4030068) SubClassOf [BFO_0000002](http://purl.obolibrary.org/obo/BFO_0000002) 


### L6 intratelencephalic projecting Car3 glutamatergic neuron (Homo sapiens) `http://purl.obolibrary.org/obo/CL_4072040`

#### Added
- Class: [L6 intratelencephalic projecting Car3 glutamatergic neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072040) 

- [L6 intratelencephalic projecting Car3 glutamatergic neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072040) [IAO_0000028](http://purl.obolibrary.org/obo/IAO_0000028) "L6 IT Car3 (Homo sapiens)"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [L6 intratelencephalic projecting Car3 glutamatergic neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072040) [RO_0002175](http://purl.obolibrary.org/obo/RO_0002175) [NCBITaxon_9606](http://purl.obolibrary.org/obo/NCBITaxon_9606) 

- [L6 intratelencephalic projecting Car3 glutamatergic neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072040) [contributor](http://purl.org/dc/terms/contributor) "https://orcid.org/0000-0002-5507-2103"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [L6 intratelencephalic projecting Car3 glutamatergic neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072040) [date](http://purl.org/dc/terms/date) "2025-09-10T10:23:11Z"^^[dateTime](http://www.w3.org/2001/XMLSchema#dateTime) 

- [L6 intratelencephalic projecting Car3 glutamatergic neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072040) [id](http://www.geneontology.org/formats/oboInOwl#id) "CL:4072040"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [L6 intratelencephalic projecting Car3 glutamatergic neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072040) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A transcriptomically distinct intratelencepalic-projecting glutamatergic neuron that expresses Car3 with a soma found in L6. The standard transcriptomic reference data for this cell type can be found on the CellxGene census under the collection: 'Transcriptomic cytoarchitecture reveals principles of human neocortex organization', dataset: 'Supercluster: IT-projecting excitatory neurons', Author Categories: 'CrossArea_cluster', L6 IT Car3."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "doi:10.1101/2022.11.30.518285"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "doi:10.1007/s00424-024-02923-2"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:34004146"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:37824655"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "https://cellxgene.cziscience.com/e/c3aa4f95-7a18-4a7d-8dd8-ca324d714363.cxg/"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [L6 intratelencephalic projecting Car3 glutamatergic neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072040) [has_related_synonym](http://www.geneontology.org/formats/oboInOwl#hasRelatedSynonym) "L6 IT Car3 neuron (Homo sapiens)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [has_synonym_type](http://www.geneontology.org/formats/oboInOwl#hasSynonymType) [OMO_0003000](http://purl.obolibrary.org/obo/OMO_0003000) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:37824655"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [L6 intratelencephalic projecting Car3 glutamatergic neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072040) [has_related_synonym](http://www.geneontology.org/formats/oboInOwl#hasRelatedSynonym) "L6-IT-Car3 cell (Homo sapiens)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [has_synonym_type](http://www.geneontology.org/formats/oboInOwl#hasSynonymType) [OMO_0003000](http://purl.obolibrary.org/obo/OMO_0003000) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:37824655"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [L6 intratelencephalic projecting Car3 glutamatergic neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072040) [has_exact_synonym](http://www.geneontology.org/formats/oboInOwl#hasExactSynonym) "L6 IT Car3 glutamatergic neuron (Homo sapiens)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [has_synonym_type](http://www.geneontology.org/formats/oboInOwl#hasSynonymType) [OMO_0003000](http://purl.obolibrary.org/obo/OMO_0003000) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "doi:10.1101/2022.11.06.515349"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [L6 intratelencephalic projecting Car3 glutamatergic neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072040) [has_related_synonym](http://www.geneontology.org/formats/oboInOwl#hasRelatedSynonym) "L6 IT Car3 (Homo sapiens)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [has_synonym_type](http://www.geneontology.org/formats/oboInOwl#hasSynonymType) [OMO_0003000](http://purl.obolibrary.org/obo/OMO_0003000) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "doi:10.1101/2022.11.06.515349"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [L6 intratelencephalic projecting Car3 glutamatergic neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072040) [has_related_synonym](http://www.geneontology.org/formats/oboInOwl#hasRelatedSynonym) "L6-IT-Car3 (Homo sapiens)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [has_synonym_type](http://www.geneontology.org/formats/oboInOwl#hasSynonymType) [OMO_0003000](http://purl.obolibrary.org/obo/OMO_0003000) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "doi:10.1101/2022.11.30.518285"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [L6 intratelencephalic projecting Car3 glutamatergic neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072040) [comment](http://www.w3.org/2000/01/rdf-schema#comment) "The marker set SMYD1, THEMIS can identify the Human cell type L6 intratelencephalic projecting Car3 glutamatergic neuron in the neocortex with a confidence of 0.877752028 (NS-Forest FBeta value)."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "https://doi.org/10.5281/zenodo.11165918"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [L6 intratelencephalic projecting Car3 glutamatergic neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072040) [label](http://www.w3.org/2000/01/rdf-schema#label) "L6 intratelencephalic projecting Car3 glutamatergic neuron (Homo sapiens)"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [L6 intratelencephalic projecting Car3 glutamatergic neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072040) SubClassOf [L6 intratelencephalic projecting Car3 glutamatergic neuron](http://purl.obolibrary.org/obo/CL_4030068) 

- [L6 intratelencephalic projecting Car3 glutamatergic neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072040) SubClassOf [RO_0002162](http://purl.obolibrary.org/obo/RO_0002162) some [NCBITaxon_9606](http://purl.obolibrary.org/obo/NCBITaxon_9606) 


### L6 intratelencephalic projecting glutamatergic neuron `http://purl.obolibrary.org/obo/CL_4030065`
#### Removed
- [L6 intratelencephalic projecting glutamatergic neuron](http://purl.obolibrary.org/obo/CL_4030065) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A transcriptomically distinct intratelencephalic-projecting glutamatergic neuron with a soma found in L6 of the primary motor cortex. These cells are short untufted pyramidal cells, which could be stellate or inverted.  The standard transcriptomic reference data for this cell type can be found on the CellxGene census under the collection: 'Transcriptomic cytoarchitecture reveals principles of human neocortex organization', dataset: 'Supercluster: IT-projecting excitatory neurons', Author Categories: 'CrossArea_subclass', L6 IT."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:37609206"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:37292694"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:37824655"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "https://cellxgene.cziscience.com/e/c3aa4f95-7a18-4a7d-8dd8-ca324d714363.cxg/"^^[string](http://www.w3.org/2001/XMLSchema#string) 

#### Added
- [L6 intratelencephalic projecting glutamatergic neuron](http://purl.obolibrary.org/obo/CL_4030065) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A transcriptomically distinct intratelencephalic-projecting glutamatergic neuron with a soma found in L6 of the primary motor cortex. These cells are short untufted pyramidal cells, which could be stellate or inverted."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:37609206"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:37292694"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:37824655"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "https://cellxgene.cziscience.com/e/c3aa4f95-7a18-4a7d-8dd8-ca324d714363.cxg/"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### L6 intratelencephalic projecting glutamatergic neuron (Homo sapiens) `http://purl.obolibrary.org/obo/CL_4072041`

#### Added
- Class: [L6 intratelencephalic projecting glutamatergic neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072041) 

- [L6 intratelencephalic projecting glutamatergic neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072041) [IAO_0000028](http://purl.obolibrary.org/obo/IAO_0000028) "L6 IT (Homo sapiens)"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [L6 intratelencephalic projecting glutamatergic neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072041) [RO_0002175](http://purl.obolibrary.org/obo/RO_0002175) [NCBITaxon_9606](http://purl.obolibrary.org/obo/NCBITaxon_9606) 

- [L6 intratelencephalic projecting glutamatergic neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072041) [contributor](http://purl.org/dc/terms/contributor) "https://orcid.org/0000-0002-5507-2103"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [L6 intratelencephalic projecting glutamatergic neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072041) [date](http://purl.org/dc/terms/date) "2025-09-11T14:38:02Z"^^[dateTime](http://www.w3.org/2001/XMLSchema#dateTime) 

- [L6 intratelencephalic projecting glutamatergic neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072041) [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "ILX:0770158"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [L6 intratelencephalic projecting glutamatergic neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072041) [id](http://www.geneontology.org/formats/oboInOwl#id) "CL:4072041"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [L6 intratelencephalic projecting glutamatergic neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072041) [in_subset](http://www.geneontology.org/formats/oboInOwl#inSubset) [cellxgene_subset](http://purl.obolibrary.org/obo/cl/cl-base#cellxgene_subset) 

- [L6 intratelencephalic projecting glutamatergic neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072041) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A transcriptomically distinct intratelencephalic-projecting glutamatergic neuron with a soma found in L6 of the primary motor cortex. These cells are short untufted pyramidal cells, which could be stellate or inverted.  The standard transcriptomic reference data for this cell type can be found on the CellxGene census under the collection: 'Transcriptomic cytoarchitecture reveals principles of human neocortex organization', dataset: 'Supercluster: IT-projecting excitatory neurons', Author Categories: 'CrossArea_subclass', L6 IT."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:37609206"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:37292694"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:37824655"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "https://cellxgene.cziscience.com/e/c3aa4f95-7a18-4a7d-8dd8-ca324d714363.cxg/"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [L6 intratelencephalic projecting glutamatergic neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072041) [has_related_synonym](http://www.geneontology.org/formats/oboInOwl#hasRelatedSynonym) "L6 IT (Homo sapiens)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [has_synonym_type](http://www.geneontology.org/formats/oboInOwl#hasSynonymType) [OMO_0003000](http://purl.obolibrary.org/obo/OMO_0003000) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:37292694"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [L6 intratelencephalic projecting glutamatergic neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072041) [comment](http://www.w3.org/2000/01/rdf-schema#comment) "The marker set LINC00507, THEMIS can identify the Human cell type L6 intratelencephalic projecting glutamatergic neuron in the neocortex with a confidence of 0.641695703 (NS-Forest FBeta value)."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "https://doi.org/10.5281/zenodo.11165918"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [L6 intratelencephalic projecting glutamatergic neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072041) [label](http://www.w3.org/2000/01/rdf-schema#label) "L6 intratelencephalic projecting glutamatergic neuron (Homo sapiens)"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [L6 intratelencephalic projecting glutamatergic neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072041) SubClassOf [BFO_0000002](http://purl.obolibrary.org/obo/BFO_0000002) 

- [L6 intratelencephalic projecting glutamatergic neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072041) SubClassOf [L6 intratelencephalic projecting glutamatergic neuron](http://purl.obolibrary.org/obo/CL_4030065) 

- [L6 intratelencephalic projecting glutamatergic neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072041) SubClassOf [RO_0002162](http://purl.obolibrary.org/obo/RO_0002162) some [NCBITaxon_9606](http://purl.obolibrary.org/obo/NCBITaxon_9606) 


### L6 intratelencephalic projecting glutamatergic neuron of the primary motor cortex `http://purl.obolibrary.org/obo/CL_4023050`

#### Added
- [L6 intratelencephalic projecting glutamatergic neuron of the primary motor cortex](http://purl.obolibrary.org/obo/CL_4023050) [description](http://purl.org/dc/terms/description) "The L6 intratelencephalic projecting glutamatergic neuron of the primary motor cortex is a specialized neuron situated in layer 6 (L6) of the primary motor cortex, a region of the brain that enables voluntary muscle control, planning of movements, and execution of complex motor tasks. The word 'intratelencephalic' denotes its pattern of connectivity, projecting within the telencephalon, the anterior part of the brain which contains both the cerebral cortex and subcortical structures. These neurons are classified as glutamatergic, as they use glutamate as their main excitatory neurotransmitter. Glutamate is a key player in mediating excitatory synapses in the mammalian central nervous system. Morphologically, these untufted neurons have short apical dendrites and with fewer branches compared to other neurons.
The L6 intratelencephalic projecting neurons play a critical role in motor function and cognitive processes. By releasing glutamate, they stimulate other neurons to which it is connected, directly impacting processes of memory, learning, and synaptic plasticity. The glutamatergic neurotransmission from these neurons shapes the output of the intrinsic cortical microcircuitry and its interaction with other brain regions, subsequently influencing the timing, coordination, and strength of motor commands.
Functionally, the L6 intratelencephalic projecting neurons integrate and transmit the afferent signals coming from the cortical layers above, particularly from the apical dendrites of the pyramidal neurons in layers 2 and 3. As the deepest layer in the six-layered neocortex, L6 contributes extensively to the feedback circuitry, both locally and across distal cortical areas. Despite the limited knowledge available on specific L6 neuron subypes owing to their heterogeneity, it is clear that L6 intratelencephalic projecting glutamatergic neurons serve as crucial components in the cortex's role as the canonical center for sensorimotor integration and cognition.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1523/JNEUROSCI.2254-10.2010"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1523/JNEUROSCI.0150-18.2018"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.3389/fnana.2010.00013"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.3389/neuro.04.003.2010"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1093/cercor/bhs254"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [L6 intratelencephalic projecting glutamatergic neuron of the primary motor cortex](http://purl.obolibrary.org/obo/CL_4023050) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_4023050"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "L6 intratelencephalic projecting glutamatergic neuron of the primary motor cortex on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### L6 th sst GABAergic interneuron (Mmus) `http://purl.obolibrary.org/obo/CL_4023075`

#### Added
- [L6 th sst GABAergic interneuron (Mmus)](http://purl.obolibrary.org/obo/CL_4023075) SubClassOf [BFO_0000002](http://purl.obolibrary.org/obo/BFO_0000002) 


### L6b glutamatergic cortical neuron `http://purl.obolibrary.org/obo/CL_4023038`
#### Removed
- [L6b glutamatergic cortical neuron](http://purl.obolibrary.org/obo/CL_4023038) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A glutamatergic neuron with a soma found in cortical layer 6b. They are transcriptomically related to corticothalamic-projecting neurons but have differential projections to the thalamus or anterior cingulate. The standard transcriptomic reference data for this cell type can be found on the CellxGene census under the collection: 'Transcriptomic cytoarchitecture reveals principles of human neocortex organization', dataset: 'Supercluster: Deep layer (non-IT) excitatory neuron', Author Categories: 'CrossArea_subclass', cluster L6b."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:30382198"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:37824655"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1101/2020.10.19.34312"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "https://cellxgene.cziscience.com/e/d01c9dff-abd1-4825-bf30-2eb2ba74597e.cxg/"^^[string](http://www.w3.org/2001/XMLSchema#string) 

#### Added
- [L6b glutamatergic cortical neuron](http://purl.obolibrary.org/obo/CL_4023038) [description](http://purl.org/dc/terms/description) "The L6b glutamatergic cortical neuron, also known as the Layer 6b glutamatergic neuron, is a specific subtype of neuron predominantly located in layer 6b of the cerebral cortex. This region is the deepest layer of the grey matter in the brain, containing various types of neurons. The L6b glutamatergic cortical neurons are named for their primary excitatory neurotransmitter, glutamate, which they use to transmit signals across the synapse. A key feature of L6b glutamatergic cortical neurons is their distinctive morphological and molecular characteristics. They express various genes that are not commonly found in other types of cortical neurons, illustrating their unique genetic make-up and they also have a distinct morphology, with dendrites that span across multiple layers of the cortex. These neurons are specific to mammals and play a critical role in modulating the brain's various functions including sensory perception, motor commands, spatial reasoning, conscious thought, and language.
Functionally, L6b glutamatergic cortical neurons are heavily involved in information processing and transmission. Using glutamate as its primary neurotransmitter, these neurons can excite or stimulate other neurons, contributing to the overall activity of the cortical circuit. Their most notable role lies within their ability to communicate between cortical and subcortical structures, as the axons of L6b neurons extend out of the cortex and towards various subcortical areas, including the thalamus and cortex. Hence, these neurons can modulate the output of cortical information to these subcortical destinations.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.neuron.2011.06.039"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.celrep.2020.02.044"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.3389/fncel.2023.1257803"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1093/cercor/bhy036"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1093/cercor/bhs254"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [L6b glutamatergic cortical neuron](http://purl.obolibrary.org/obo/CL_4023038) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A glutamatergic neuron with a soma found in cortical layer 6b. They are transcriptomically related to corticothalamic-projecting neurons but have differential projections to the thalamus or anterior cingulate."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:30382198"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:37824655"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1101/2020.10.19.34312"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "https://cellxgene.cziscience.com/e/d01c9dff-abd1-4825-bf30-2eb2ba74597e.cxg/"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [L6b glutamatergic cortical neuron](http://purl.obolibrary.org/obo/CL_4023038) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_4023038"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "L6b glutamatergic cortical neuron on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### L6b glutamatergic cortical neuron (Homo sapiens) `http://purl.obolibrary.org/obo/CL_4072045`

#### Added
- Class: [L6b glutamatergic cortical neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072045) 

- [L6b glutamatergic cortical neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072045) [IAO_0000028](http://purl.obolibrary.org/obo/IAO_0000028) "L6b glut (Homo sapiens)"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [L6b glutamatergic cortical neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072045) [RO_0002175](http://purl.obolibrary.org/obo/RO_0002175) [NCBITaxon_9606](http://purl.obolibrary.org/obo/NCBITaxon_9606) 

- [L6b glutamatergic cortical neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072045) [contributor](http://purl.org/dc/terms/contributor) "https://orcid.org/0000-0002-5507-2103"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [L6b glutamatergic cortical neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072045) [date](http://purl.org/dc/terms/date) "2025-09-16T10:30:38Z"^^[dateTime](http://www.w3.org/2001/XMLSchema#dateTime) 

- [L6b glutamatergic cortical neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072045) [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "ILX:0770163"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [L6b glutamatergic cortical neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072045) [id](http://www.geneontology.org/formats/oboInOwl#id) "CL:4072045"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [L6b glutamatergic cortical neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072045) [in_subset](http://www.geneontology.org/formats/oboInOwl#inSubset) [BDS_subset](http://purl.obolibrary.org/obo/cl/cl-base#BDS_subset) 

- [L6b glutamatergic cortical neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072045) [in_subset](http://www.geneontology.org/formats/oboInOwl#inSubset) [cellxgene_subset](http://purl.obolibrary.org/obo/cl/cl-base#cellxgene_subset) 

- [L6b glutamatergic cortical neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072045) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A glutamatergic neuron with a soma found in cortical layer 6b. They are transcriptomically related to corticothalamic-projecting neurons but have differential projections to the thalamus or anterior cingulate. The standard transcriptomic reference data for this cell type can be found on the CellxGene census under the collection: 'Transcriptomic cytoarchitecture reveals principles of human neocortex organization', dataset: 'Supercluster: Deep layer (non-IT) excitatory neuron', Author Categories: 'CrossArea_subclass', cluster L6b."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:30382198"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:37824655"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1101/2020.10.19.34312"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "https://cellxgene.cziscience.com/e/d01c9dff-abd1-4825-bf30-2eb2ba74597e.cxg/"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [L6b glutamatergic cortical neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072045) [label](http://www.w3.org/2000/01/rdf-schema#label) "L6b glutamatergic cortical neuron (Homo sapiens)"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [L6b glutamatergic cortical neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072045) SubClassOf [BFO_0000002](http://purl.obolibrary.org/obo/BFO_0000002) 

- [L6b glutamatergic cortical neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072045) SubClassOf [L6b glutamatergic cortical neuron](http://purl.obolibrary.org/obo/CL_4023038) 

- [L6b glutamatergic cortical neuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072045) SubClassOf [RO_0002162](http://purl.obolibrary.org/obo/RO_0002162) some [NCBITaxon_9606](http://purl.obolibrary.org/obo/NCBITaxon_9606) 


### Lamp5 Lhx6 GABAergic interneuron (Homo sapiens) `http://purl.obolibrary.org/obo/CL_4072022`

#### Added
- Class: [Lamp5 Lhx6 GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072022) 

- [Lamp5 Lhx6 GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072022) [RO_0002175](http://purl.obolibrary.org/obo/RO_0002175) [NCBITaxon_9606](http://purl.obolibrary.org/obo/NCBITaxon_9606) 

- [Lamp5 Lhx6 GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072022) [date](http://purl.org/dc/terms/date) "2025-07-31T14:01:52Z"^^[dateTime](http://www.w3.org/2001/XMLSchema#dateTime) 

- [Lamp5 Lhx6 GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072022) [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "https://cellxgene.cziscience.com/e/e4ddac12-f48f-4455-8e8d-c2a48a683437.cxg/"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [Lamp5 Lhx6 GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072022) [id](http://www.geneontology.org/formats/oboInOwl#id) "CL:4072022"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [Lamp5 Lhx6 GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072022) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A Lamp5 Lhx6 GABAergic interneuron of Homo sapiens. The standard transcriptomic reference data for this cell type can be found on the CellxGene census under the collection: 'Transcriptomic cytoarchitecture reveals principles of human neocortex organization', dataset: 'Supercluster: CGE-derived interneurons', Author Categories: 'CrossArea_subclass', Lamp5 Lhx6"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:38712198"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:37824669"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:36007006"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:30382198"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:37824655"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [Lamp5 Lhx6 GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072022) [comment](http://www.w3.org/2000/01/rdf-schema#comment) "Despite being clustered as CGE in this dataset, other studies suggest that the cell is of MGE origin because it expresses transcription factors such as LHX6 and NKX2-1, which are hallmark indicators of MGE-derived interneurons. Fate mapping experiments in mice have demonstrated that Lamp5+/Lhx6+ interneurons in both the hippocampus and neocortex arise from the MGE, and single-cell transcriptomic comparisons further show that primate neocortical Lamp5+/Lhx6+ cells closely resemble mouse hippocampal MGE-derived interneurons rather than classic CGE-derived types. The marker set CHST9, LAMP5 can identify the Human cell type Lamp5 Lhx6 neuron in the neocortex with a confidence of 0.821678322 (NS-Forest FBeta value)."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "https://zenodo.org/records/11165918"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:32999462"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [Lamp5 Lhx6 GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072022) [label](http://www.w3.org/2000/01/rdf-schema#label) "Lamp5 Lhx6 GABAergic interneuron (Homo sapiens)"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [Lamp5 Lhx6 GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072022) SubClassOf [Lamp5 Lhx6 neuron](http://purl.obolibrary.org/obo/CL_4042013) 

- [Lamp5 Lhx6 GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072022) SubClassOf [RO_0002162](http://purl.obolibrary.org/obo/RO_0002162) some [NCBITaxon_9606](http://purl.obolibrary.org/obo/NCBITaxon_9606) 

- [Lamp5 Lhx6 GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072022) SubClassOf [RO_0015004](http://purl.obolibrary.org/obo/RO_0015004) some [CLM_1000075](http://purl.obolibrary.org/obo/CLM_1000075) 


### Lamp5 Lhx6 neuron `http://purl.obolibrary.org/obo/CL_4042013`
#### Removed
- [Lamp5 Lhx6 neuron](http://purl.obolibrary.org/obo/CL_4042013) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A transcriptomically distinct lamp5 GABAergic cortical interneuron located in the cerebral cortex that expresses Lamp5 and Lhx6. The standard transcriptomic reference data for this cell type can be found on the CellxGene census under the collection: 'Transcriptomic cytoarchitecture reveals principles of human neocortex organization', dataset: 'Supercluster: CGE-derived interneurons', Author Categories: 'CrossArea_subclass', Lamp5 Lhx6."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:37824669"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "https://cellxgene.cziscience.com/e/e4ddac12-f48f-4455-8e8d-c2a48a683437.cxg/"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:36007006"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:34004146"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:37824655"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [Lamp5 Lhx6 neuron](http://purl.obolibrary.org/obo/CL_4042013) [comment](http://www.w3.org/2000/01/rdf-schema#comment) "The marker set CHST9, LAMP5 can identify the Human cell type Lamp5 Lhx6 neuron in the neocortex with a confidence of 0.821678322 (NS-Forest FBeta value)."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "https://doi.org/10.5281/zenodo.11165918"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [Lamp5 Lhx6 neuron](http://purl.obolibrary.org/obo/CL_4042013) SubClassOf [RO_0015004](http://purl.obolibrary.org/obo/RO_0015004) some [CLM_1000075](http://purl.obolibrary.org/obo/CLM_1000075) 

#### Added
- [Lamp5 Lhx6 neuron](http://purl.obolibrary.org/obo/CL_4042013) [in_subset](http://www.geneontology.org/formats/oboInOwl#inSubset) [human_reference_atlas](http://purl.obolibrary.org/obo/cl/cl-base#human_reference_atlas) 

- [Lamp5 Lhx6 neuron](http://purl.obolibrary.org/obo/CL_4042013) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A transcriptomically distinct GABAergic interneuron marked by Lamp5 and Lhx6 expression, derived from the Medial Ganglionic Eminence (MGE), and located in the pallium."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:37824669"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "https://cellxgene.cziscience.com/e/e4ddac12-f48f-4455-8e8d-c2a48a683437.cxg/"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:36007006"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:34004146"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:37824655"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [Lamp5 Lhx6 neuron](http://purl.obolibrary.org/obo/CL_4042013) SubClassOf [medial ganglionic eminence derived GABAergic interneuron](http://purl.obolibrary.org/obo/CL_4023069) 


### M1 retinal ganglion cell `http://purl.obolibrary.org/obo/CL_0003028`

#### Added
- [M1 retinal ganglion cell](http://purl.obolibrary.org/obo/CL_0003028) SubClassOf [RO_0002162](http://purl.obolibrary.org/obo/RO_0002162) some [NCBITaxon_10090](http://purl.obolibrary.org/obo/NCBITaxon_10090) 


### M10 retinal ganglion cell `http://purl.obolibrary.org/obo/CL_0003043`

#### Added
- [M10 retinal ganglion cell](http://purl.obolibrary.org/obo/CL_0003043) SubClassOf [RO_0002162](http://purl.obolibrary.org/obo/RO_0002162) some [NCBITaxon_10090](http://purl.obolibrary.org/obo/NCBITaxon_10090) 


### M11 retinal ganglion cell `http://purl.obolibrary.org/obo/CL_0003044`

#### Added
- [M11 retinal ganglion cell](http://purl.obolibrary.org/obo/CL_0003044) SubClassOf [RO_0002162](http://purl.obolibrary.org/obo/RO_0002162) some [NCBITaxon_10090](http://purl.obolibrary.org/obo/NCBITaxon_10090) 


### M12 retinal ganglion cell `http://purl.obolibrary.org/obo/CL_0003045`

#### Added
- [M12 retinal ganglion cell](http://purl.obolibrary.org/obo/CL_0003045) SubClassOf [RO_0002162](http://purl.obolibrary.org/obo/RO_0002162) some [NCBITaxon_10090](http://purl.obolibrary.org/obo/NCBITaxon_10090) 


### M13 retinal ganglion cell `http://purl.obolibrary.org/obo/CL_0003046`

#### Added
- [M13 retinal ganglion cell](http://purl.obolibrary.org/obo/CL_0003046) SubClassOf [RO_0002162](http://purl.obolibrary.org/obo/RO_0002162) some [NCBITaxon_10090](http://purl.obolibrary.org/obo/NCBITaxon_10090) 


### M14 retinal ganglion cell `http://purl.obolibrary.org/obo/CL_0003047`

#### Added
- [M14 retinal ganglion cell](http://purl.obolibrary.org/obo/CL_0003047) SubClassOf [RO_0002162](http://purl.obolibrary.org/obo/RO_0002162) some [NCBITaxon_10090](http://purl.obolibrary.org/obo/NCBITaxon_10090) 


### M2 retinal ganglion cell `http://purl.obolibrary.org/obo/CL_0003029`

#### Added
- [M2 retinal ganglion cell](http://purl.obolibrary.org/obo/CL_0003029) SubClassOf [RO_0002162](http://purl.obolibrary.org/obo/RO_0002162) some [NCBITaxon_10090](http://purl.obolibrary.org/obo/NCBITaxon_10090) 


### M3 retinal ganglion cell `http://purl.obolibrary.org/obo/CL_0003030`

#### Added
- [M3 retinal ganglion cell](http://purl.obolibrary.org/obo/CL_0003030) SubClassOf [RO_0002162](http://purl.obolibrary.org/obo/RO_0002162) some [NCBITaxon_10090](http://purl.obolibrary.org/obo/NCBITaxon_10090) 


### M4 retinal ganglion cell `http://purl.obolibrary.org/obo/CL_0003033`

#### Added
- [M4 retinal ganglion cell](http://purl.obolibrary.org/obo/CL_0003033) SubClassOf [RO_0002162](http://purl.obolibrary.org/obo/RO_0002162) some [NCBITaxon_10090](http://purl.obolibrary.org/obo/NCBITaxon_10090) 


### M5 retinal ganglion cell `http://purl.obolibrary.org/obo/CL_0003034`

#### Added
- [M5 retinal ganglion cell](http://purl.obolibrary.org/obo/CL_0003034) SubClassOf [RO_0002162](http://purl.obolibrary.org/obo/RO_0002162) some [NCBITaxon_10090](http://purl.obolibrary.org/obo/NCBITaxon_10090) 


### M6 retinal ganglion cell `http://purl.obolibrary.org/obo/CL_0003035`

#### Added
- [M6 retinal ganglion cell](http://purl.obolibrary.org/obo/CL_0003035) SubClassOf [RO_0002162](http://purl.obolibrary.org/obo/RO_0002162) some [NCBITaxon_10090](http://purl.obolibrary.org/obo/NCBITaxon_10090) 


### M7 retinal ganglion cell `http://purl.obolibrary.org/obo/CL_0003036`

#### Added
- [M7 retinal ganglion cell](http://purl.obolibrary.org/obo/CL_0003036) SubClassOf [RO_0002162](http://purl.obolibrary.org/obo/RO_0002162) some [NCBITaxon_10090](http://purl.obolibrary.org/obo/NCBITaxon_10090) 


### M8 retinal ganglion cell `http://purl.obolibrary.org/obo/CL_0003039`

#### Added
- [M8 retinal ganglion cell](http://purl.obolibrary.org/obo/CL_0003039) SubClassOf [RO_0002162](http://purl.obolibrary.org/obo/RO_0002162) some [NCBITaxon_10090](http://purl.obolibrary.org/obo/NCBITaxon_10090) 


### M9 retinal ganglion cell `http://purl.obolibrary.org/obo/CL_0003040`

#### Added
- [M9 retinal ganglion cell](http://purl.obolibrary.org/obo/CL_0003040) SubClassOf [RO_0002162](http://purl.obolibrary.org/obo/RO_0002162) some [NCBITaxon_10090](http://purl.obolibrary.org/obo/NCBITaxon_10090) 


### Mueller cell `http://purl.obolibrary.org/obo/CL_0000636`

#### Added
- [Mueller cell](http://purl.obolibrary.org/obo/CL_0000636) [description](http://purl.org/dc/terms/description) "Mueller cells, also referred to as M≈±ller glial cells, are a type of radial glial cell found specifically within the retina, the light-sensitive tissue lining the back interior surface of the eye. They derive their name from Heinrich M&uuml;ller, who first described these cells in the mid-nineteenth century. These support cells contribute to maintaining the homeostasis and structural integrity of the retina. Mueller cells span the entire thickness of the retina, thus representing the only cell type that traverses all layers.
Mueller cells play a crucial role in maintaining the metabolic environment of retinal neurons and participating in the visual signal transduction. They have a unique organization and possess a wide range of processes, giving them direct contact with virtually every type of cell in the retina&mdash;including photoreceptors, bipolar cells, and ganglion cells. These cells actively uptake and recycle neurotransmitters, modulate chemical levels within the extracellular space, and are involved in potassium siphoning, thus supporting neuronal signaling. Due to these functions, they play a vital role in maintaining the physiological and functional stability of the retina.
Moreover, Mueller cells are also responsible for maintaining the structural integrity of the retina. They serve as the retinal framework by providing structural support to neurons and blood vessels. They contribute to the formation and maintenance of the inner limiting membrane, comprising their end-feet at the vitreous surface. Additionally, they play an essential role during injury or disease, exhibiting reactive gliosis &ndash; a process where these cells undergo morphological and functional changes to aid in repair and protection from further damage. While their protective role is important, it's also worth noting that overactivation of Mueller cells may result in retinal scarring or degeneration, reinforcing their complex and delicate role in retinal health and disease.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1002/glia.22477"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.3389/fendo.2013.00048"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/78774"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.preteyeres.2006.05.003"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1007/s10792-020-01477-3"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [Mueller cell](http://purl.obolibrary.org/obo/CL_0000636) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0000636"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "Mueller cell on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### NCBITaxon_9986 `http://purl.obolibrary.org/obo/NCBITaxon_9986`

#### Added
- Class: [NCBITaxon_9986](http://purl.obolibrary.org/obo/NCBITaxon_9986) 


### NPY interneuron `http://purl.obolibrary.org/obo/CL_4072016`

#### Added
- Class: [NPY interneuron](http://purl.obolibrary.org/obo/CL_4072016) 

- [NPY interneuron](http://purl.obolibrary.org/obo/CL_4072016) [date](http://purl.org/dc/terms/date) "2025-07-07T09:04:45Z"^^[dateTime](http://www.w3.org/2001/XMLSchema#dateTime) 

- [NPY interneuron](http://purl.obolibrary.org/obo/CL_4072016) [id](http://www.geneontology.org/formats/oboInOwl#id) "CL:4072016"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [NPY interneuron](http://purl.obolibrary.org/obo/CL_4072016) [has_related_synonym](http://www.geneontology.org/formats/oboInOwl#hasRelatedSynonym) "NPY-expressing neocortical interneuron"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:19295167"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [NPY interneuron](http://purl.obolibrary.org/obo/CL_4072016) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "An interneuron that expresses neuropeptide Y (NPY) and is mainly located in cortical layers II&ndash;III and VI, as well as in various brain regions such as the medial geniculate and inferior colliculus. The NPY cell population is physiologically diverse, displaying a range of firing behaviors, including adapting, fast-spiking, and accelerating patterns."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:19295167"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:32376782"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [NPY interneuron](http://purl.obolibrary.org/obo/CL_4072016) [has_exact_synonym](http://www.geneontology.org/formats/oboInOwl#hasExactSynonym) "NPY-expressing neuron"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:32376782"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [NPY interneuron](http://purl.obolibrary.org/obo/CL_4072016) [label](http://www.w3.org/2000/01/rdf-schema#label) "NPY interneuron"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [NPY interneuron](http://purl.obolibrary.org/obo/CL_4072016) SubClassOf [BFO_0000002](http://purl.obolibrary.org/obo/BFO_0000002) 

- [NPY interneuron](http://purl.obolibrary.org/obo/CL_4072016) SubClassOf [GABAergic interneuron](http://purl.obolibrary.org/obo/CL_0011005) 

- [NPY interneuron](http://purl.obolibrary.org/obo/CL_4072016) SubClassOf [RO_0002292](http://purl.obolibrary.org/obo/RO_0002292) some [PR_000011387](http://purl.obolibrary.org/obo/PR_000011387) 


### PATO_0001701 `http://purl.obolibrary.org/obo/PATO_0001701`

#### Added
- Class: [PATO_0001701](http://purl.obolibrary.org/obo/PATO_0001701) 


### PAX6 GABAergic interneuron `http://purl.obolibrary.org/obo/CL_4042032`
#### Removed
- [PAX6 GABAergic interneuron](http://purl.obolibrary.org/obo/CL_4042032) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A transcriptomically distinct GABAergic neuron located in the cerebral cortex that expresses the transcript PAX6. The standard transcriptomic reference data for this cell type can be found on the CellxGene census under the collection: 'Transcriptomic cytoarchitecture reveals principles of human neocortex organization', dataset: 'Supercluster: CGE-derived interneurons', Author Categories: 'CrossArea_subclass', clusters PAX6."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "https://cellxgene.cziscience.com/e/e4ddac12-f48f-4455-8e8d-c2a48a683437.cxg/"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:37824655"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [PAX6 GABAergic interneuron](http://purl.obolibrary.org/obo/CL_4042032) [label](http://www.w3.org/2000/01/rdf-schema#label) "PAX6 GABAergic cortical interneuron"^^[string](http://www.w3.org/2001/XMLSchema#string) 

#### Added
- [PAX6 GABAergic interneuron](http://purl.obolibrary.org/obo/CL_4042032) [IAO_0000028](http://purl.obolibrary.org/obo/IAO_0000028) "Pax6 interneuron"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [PAX6 GABAergic interneuron](http://purl.obolibrary.org/obo/CL_4042032) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A transcriptomically distinct GABAergic neuron, located in the cerebral cortex and derived from the CGE, that expresses the transcript PAX6."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "https://cellxgene.cziscience.com/e/e4ddac12-f48f-4455-8e8d-c2a48a683437.cxg/"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:37824655"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [PAX6 GABAergic interneuron](http://purl.obolibrary.org/obo/CL_4042032) [label](http://www.w3.org/2000/01/rdf-schema#label) "PAX6 GABAergic interneuron"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### PAX6 GABAergic interneuron (Homo sapiens) `http://purl.obolibrary.org/obo/CL_4072036`

#### Added
- Class: [PAX6 GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072036) 

- [PAX6 GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072036) [IAO_0000028](http://purl.obolibrary.org/obo/IAO_0000028) "Pax6 interneuron (Homo sapiens)"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [PAX6 GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072036) [RO_0002175](http://purl.obolibrary.org/obo/RO_0002175) [NCBITaxon_9606](http://purl.obolibrary.org/obo/NCBITaxon_9606) 

- [PAX6 GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072036) [contributor](http://purl.org/dc/terms/contributor) "https://orcid.org/0000-0002-5507-2103"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [PAX6 GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072036) [date](http://purl.org/dc/terms/date) "2025-09-10T08:47:34Z"^^[dateTime](http://www.w3.org/2001/XMLSchema#dateTime) 

- [PAX6 GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072036) [id](http://www.geneontology.org/formats/oboInOwl#id) "CL:4072036"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [PAX6 GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072036) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A transcriptomically distinct GABAergic neuron located in the cerebral cortex that expresses the transcript PAX6. The standard transcriptomic reference data for this cell type can be found on the CellxGene census under the collection: 'Transcriptomic cytoarchitecture reveals principles of human neocortex organization', dataset: 'Supercluster: CGE-derived interneurons', Author Categories: 'CrossArea_subclass', clusters PAX"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "https://cellxgene.cziscience.com/e/e4ddac12-f48f-4455-8e8d-c2a48a683437.cxg/"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:37824655"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [PAX6 GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072036) [comment](http://www.w3.org/2000/01/rdf-schema#comment) "The marker set RELN, HCRTR2 can identify the Human cell type PAX6 GABAergic interneuron in the neocortex with a confidence of 0.724125093 (NS-Forest FBeta value)."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "https://doi.org/10.5281/zenodo.11165918"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [PAX6 GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072036) [label](http://www.w3.org/2000/01/rdf-schema#label) "PAX6 GABAergic interneuron (Homo sapiens)"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [PAX6 GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072036) SubClassOf [PAX6 GABAergic interneuron](http://purl.obolibrary.org/obo/CL_4042032) 

- [PAX6 GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072036) SubClassOf [RO_0002162](http://purl.obolibrary.org/obo/RO_0002162) some [NCBITaxon_9606](http://purl.obolibrary.org/obo/NCBITaxon_9606) 


### PR_000003846 `http://purl.obolibrary.org/obo/PR_000003846`

#### Added
- Class: [PR_000003846](http://purl.obolibrary.org/obo/PR_000003846) 


### PR_A0A0B4J238 `http://purl.obolibrary.org/obo/PR_A0A0B4J238`

#### Added
- Class: [PR_A0A0B4J238](http://purl.obolibrary.org/obo/PR_A0A0B4J238) 


### PR_O76070 `http://purl.obolibrary.org/obo/PR_O76070`

#### Added
- Class: [PR_O76070](http://purl.obolibrary.org/obo/PR_O76070) 


### PR_P01282 `http://purl.obolibrary.org/obo/PR_P01282`

#### Added
- Class: [PR_P01282](http://purl.obolibrary.org/obo/PR_P01282) 


### PR_Q9UIK5 `http://purl.obolibrary.org/obo/PR_Q9UIK5`

#### Added
- Class: [PR_Q9UIK5](http://purl.obolibrary.org/obo/PR_Q9UIK5) 


### RXFP1-positive interface island D1-medium spiny neuron `http://purl.obolibrary.org/obo/CL_4030054`
#### Removed
- [RXFP1-positive interface island D1-medium spiny neuron](http://purl.obolibrary.org/obo/CL_4030054) SubClassOf [BFO_0000002](http://purl.obolibrary.org/obo/BFO_0000002) 

#### Added
- [RXFP1-positive interface island D1-medium spiny neuron](http://purl.obolibrary.org/obo/CL_4030054) [RO_0002175](http://purl.obolibrary.org/obo/RO_0002175) [NCBITaxon_9606](http://purl.obolibrary.org/obo/NCBITaxon_9606) 

- [RXFP1-positive interface island D1-medium spiny neuron](http://purl.obolibrary.org/obo/CL_4030054) [in_subset](http://www.geneontology.org/formats/oboInOwl#inSubset) [human_reference_atlas](http://purl.obolibrary.org/obo/cl/cl-base#human_reference_atlas) 


### SCN pacemaker neuron `http://purl.obolibrary.org/obo/CL_4072019`

#### Added
- Class: [SCN pacemaker neuron](http://purl.obolibrary.org/obo/CL_4072019) 

- [SCN pacemaker neuron](http://purl.obolibrary.org/obo/CL_4072019) [date](http://purl.org/dc/terms/date) "2025-07-14T10:10:28Z"^^[dateTime](http://www.w3.org/2001/XMLSchema#dateTime) 

- [SCN pacemaker neuron](http://purl.obolibrary.org/obo/CL_4072019) [id](http://www.geneontology.org/formats/oboInOwl#id) "CL:4072019"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [SCN pacemaker neuron](http://purl.obolibrary.org/obo/CL_4072019) [has_exact_synonym](http://www.geneontology.org/formats/oboInOwl#hasExactSynonym) "pacemaker neuron of the suprachiasmatic nucleus"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:20148688"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [SCN pacemaker neuron](http://purl.obolibrary.org/obo/CL_4072019) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A type of neuron located in the suprachiasmatic nucleus (SCN) that possesses intrinsic circadian rhythmicity, characterized by strong and rhythmic expression of core clock genes and the ability to independently generate and sustain circadian oscillations. In mice, the specific subtypes Avp+/Nms+, Vip+/Nms+, and Cck+/C1ql3+ neurons have the most robust circadian gene expression and contribute to synchronizing and maintaining rhythmicity within the SCN network (Wen et al., 2020)."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:32066983"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:20148688"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [SCN pacemaker neuron](http://purl.obolibrary.org/obo/CL_4072019) [label](http://www.w3.org/2000/01/rdf-schema#label) "SCN pacemaker neuron"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [SCN pacemaker neuron](http://purl.obolibrary.org/obo/CL_4072019) SubClassOf [neuron of the forebrain](http://purl.obolibrary.org/obo/CL_0012001) 

- [SCN pacemaker neuron](http://purl.obolibrary.org/obo/CL_4072019) SubClassOf [RO_0002100](http://purl.obolibrary.org/obo/RO_0002100) some [UBERON_0002034](http://purl.obolibrary.org/obo/UBERON_0002034) 


### Schlemm's canal endothelial cell `http://purl.obolibrary.org/obo/CL_4033097`
#### Removed
- [Schlemm's canal endothelial cell](http://purl.obolibrary.org/obo/CL_4033097) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A specialised endothelial cell that is part of the canal of Schlemm. This cell type shares characteristics and expresses markers of both blood and lymphatic endothelial cells (Kizhatil et al., 2014) and regulates aqueous humor outflow resistance by modulating pore formation and responding to mechanical strain (Stamer et al., 2016)."^^[string](http://www.w3.org/2001/XMLSchema#string) 

#### Added
- [Schlemm's canal endothelial cell](http://purl.obolibrary.org/obo/CL_4033097) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A specialised endothelial cell that is part of the canal of Schlemm. This cell type shares characteristics and expresses markers of both blood and lymphatic endothelial cells (Kizhatil et al., 2014) and regulates aqueous humor outflow resistance by modulating pore formation and responding to mechanical strain (Stamer et al., 2015)."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:25051267"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:25223880"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### Schwann cell precursor `http://purl.obolibrary.org/obo/CL_0002375`

#### Added
- [Schwann cell precursor](http://purl.obolibrary.org/obo/CL_0002375) [description](http://purl.org/dc/terms/description) "Schwann cell precursors are a crucial type of cell in the peripheral nervous system, derived from the neural crest during embryogenesis. The primary function of Schwann cell precursors is to differentiate into either myelinating or non-myelinating Schwann cells, both of which are critical in the proper function of the nervous system: myelinating Schwann cells cover axons in a myelin sheath, which allows for quicker and more efficient conduction of nerve signals, while non-myelinating Schwann cells envelope several axons together, effectively aiding slower nerve conduction. 
Schwann cell precursors are also possibly involved in guiding axonal growth during the early development of the nervous system, and they have an inherent potential to form other cell types such as melanocytes, parasympathetic neurons, and endoneurial fibroblasts. 
Growing evidence suggests that there is high plasticity among Schwann cell precursor derivatives during postnatal development and that they play a dynamic role not only in the development of the nervous system but also in its repair and regeneration post injury.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.3390/cells11233753"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.3389/fnmol.2019.00069"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.cdev.2021.203686"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [Schwann cell precursor](http://purl.obolibrary.org/obo/CL_0002375) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0002375"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "Schwann cell precursor on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### Sertoli cell `http://purl.obolibrary.org/obo/CL_0000216`

#### Added
- [Sertoli cell](http://purl.obolibrary.org/obo/CL_0000216) [description](http://purl.org/dc/terms/description) "Sertoli cells, also known as sustentacular cells, are epithelial cells that play a crucial part of the male reproductive system and are situated within the seminiferous tubules of the testes. Functionally, Sertoli cells play diverse and pivotal roles in supporting spermatogenesis, the intricate process of sperm production. To scaffold the developing sperm cells, Sertoli cells morphologically change, interacting directly with the germ cells to provide nutritional support, metabolic regulation, and waste removal, thereby creating an environment conducive to the development of germ cells.
In addition to their structural function, Sertoli cells also secrete several bioactive molecules with crucial implications for spermatogenesis. These include anti-mullerian hormone (AMH) during fetal life, inhibin, and androgen binding protein (ABP), as well as several growth factors and transport proteins. They also contribute to the immune-privileged status of the testes by forming the blood-testis barrier, safeguarding sperm antigens from the systemic immune response. Given their extensive roles in maintaining testicular function, impairments in Sertoli cell function can lead to male infertility and are implicated in a range of related disorders.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "https://www.ncbi.nlm.nih.gov/books/NBK560631/"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.semcdb.2021.06.016"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/s41419-019-1782-z"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [Sertoli cell](http://purl.obolibrary.org/obo/CL_0000216) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0000216"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "Sertoli cell on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### T follicular helper cell `http://purl.obolibrary.org/obo/CL_0002038`
#### Removed
- [T follicular helper cell](http://purl.obolibrary.org/obo/CL_0002038) EquivalentTo [CD4-positive helper T cell](http://purl.obolibrary.org/obo/CL_0000492) and ([BFO_0000051](http://purl.obolibrary.org/obo/BFO_0000051) some [PR_000003450](http://purl.obolibrary.org/obo/PR_000003450)) and ([lacks_plasma_membrane_part](http://purl.obolibrary.org/obo/CL_4030046) some [PR_000001203](http://purl.obolibrary.org/obo/PR_000001203)) and ([RO_0002104](http://purl.obolibrary.org/obo/RO_0002104) some [PR_000001209](http://purl.obolibrary.org/obo/PR_000001209)) and ([RO_0002104](http://purl.obolibrary.org/obo/RO_0002104) some [PR_000001860](http://purl.obolibrary.org/obo/PR_000001860)) and ([RO_0002104](http://purl.obolibrary.org/obo/RO_0002104) some [PR_000001919](http://purl.obolibrary.org/obo/PR_000001919)) and ([RO_0002215](http://purl.obolibrary.org/obo/RO_0002215) some [GO_0045830](http://purl.obolibrary.org/obo/GO_0045830)) and ([RO_0002215](http://purl.obolibrary.org/obo/RO_0002215) some [GO_0051024](http://purl.obolibrary.org/obo/GO_0051024)) 

- [T follicular helper cell](http://purl.obolibrary.org/obo/CL_0002038) SubClassOf [RO_0002215](http://purl.obolibrary.org/obo/RO_0002215) some [GO_0051024](http://purl.obolibrary.org/obo/GO_0051024) 

#### Added
- [T follicular helper cell](http://purl.obolibrary.org/obo/CL_0002038) EquivalentTo [CD4-positive helper T cell](http://purl.obolibrary.org/obo/CL_0000492) and ([BFO_0000051](http://purl.obolibrary.org/obo/BFO_0000051) some [PR_000003450](http://purl.obolibrary.org/obo/PR_000003450)) and ([lacks_plasma_membrane_part](http://purl.obolibrary.org/obo/CL_4030046) some [PR_000001203](http://purl.obolibrary.org/obo/PR_000001203)) and ([RO_0002104](http://purl.obolibrary.org/obo/RO_0002104) some [PR_000001209](http://purl.obolibrary.org/obo/PR_000001209)) and ([RO_0002104](http://purl.obolibrary.org/obo/RO_0002104) some [PR_000001860](http://purl.obolibrary.org/obo/PR_000001860)) and ([RO_0002104](http://purl.obolibrary.org/obo/RO_0002104) some [PR_000001919](http://purl.obolibrary.org/obo/PR_000001919)) and ([RO_0002215](http://purl.obolibrary.org/obo/RO_0002215) some [GO_0002639](http://purl.obolibrary.org/obo/GO_0002639)) and ([RO_0002215](http://purl.obolibrary.org/obo/RO_0002215) some [GO_0045830](http://purl.obolibrary.org/obo/GO_0045830)) 


### T-helper 2 cell `http://purl.obolibrary.org/obo/CL_0000546`

#### Added
- [T-helper 2 cell](http://purl.obolibrary.org/obo/CL_0000546) [description](http://purl.org/dc/terms/description) "T-helper 2 cells (Th2 cells) are a subset of CD4+ T lymphocytes, which play central roles in cell-mediated immunity. All T lymphocytes are uniquely characterized by the expression of a T-cell receptor (TCR) complex on their cell surface. Th2 cells express the CD4 co-receptor which enables the T-cell receptor to engage with antigen-presenting cells and perform their influential role in adaptive immune response.
Th2 cells differentiate from na&iuml;ve CD4+  T cells following TCR ligation and specific cytokine signaling. Th2 cell development and stabilization requires the master regulatory transcription factor GATA-3. Furthermore, IL-4-mediated activation of STAT6 is thought to play an important role in Th2 cell differentiation. 
The primary function of Th2 cells is to coordinate the immune response against extracellular pathogens, particularly parasitic helminths. They achieve this primarily through the secretion of their signature cytokines &ndash; the interleukins IL-4, IL-5, and IL-13. For instance, IL-4 promotes the differentiation of B cells into plasma cells that produce immunoglobulin E (IgE), a class of antibodies that mediate the immune response to parasites and are involved in allergic reactions. IL-5 promotes the activation and survival of eosinophils, a type of white blood cell that is effective against certain parasites. IL-13 works alongside IL-4 but also influences mucus production in the respiratory tract, an important defense mechanism against parasites.
Despite their critical roles in host defense, an overactive Th2 response can also lead to pathological outcomes such as allergy and asthma. These health issues are typically characterized by excessive IgE production, eosinophilic inflammation, and mucus hypersecretion, all hallmarks of a Th2 response. In these instances, the immune system reacts excessively to normally harmless substances like pollen, dust mites, or certain foods, with the Th2 cells driving this overreaction. As such, understanding the precise pathways and mechanisms through which Th2 cells operate not only provides insight into protective immune responses but also potential therapeutic targets for treating allergies and other related conditions.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/nri.2017.118"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1146/annurev-immunol-051116-052350"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.3389/fimmu.2022.974066"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.3389/fimmu.2019.02318"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [T-helper 2 cell](http://purl.obolibrary.org/obo/CL_0000546) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0000546"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "T-helper 2 cell on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### TAC3-positive striatal interneuron `http://purl.obolibrary.org/obo/CL_4042001`

#### Added
- [TAC3-positive striatal interneuron](http://purl.obolibrary.org/obo/CL_4042001) [RO_0002175](http://purl.obolibrary.org/obo/RO_0002175) [NCBITaxon_9606](http://purl.obolibrary.org/obo/NCBITaxon_9606) 

- [TAC3-positive striatal interneuron](http://purl.obolibrary.org/obo/CL_4042001) [in_subset](http://www.geneontology.org/formats/oboInOwl#inSubset) [human_reference_atlas](http://purl.obolibrary.org/obo/cl/cl-base#human_reference_atlas) 


### UBERON_0002034 `http://purl.obolibrary.org/obo/UBERON_0002034`

#### Added
- Class: [UBERON_0002034](http://purl.obolibrary.org/obo/UBERON_0002034) 


### UBERON_0002283 `http://purl.obolibrary.org/obo/UBERON_0002283`

#### Added
- Class: [UBERON_0002283](http://purl.obolibrary.org/obo/UBERON_0002283) 


### UBERON_0002477 `http://purl.obolibrary.org/obo/UBERON_0002477`

#### Added
- Class: [UBERON_0002477](http://purl.obolibrary.org/obo/UBERON_0002477) 


### UBERON_0005273 `http://purl.obolibrary.org/obo/UBERON_0005273`

#### Added
- Class: [UBERON_0005273](http://purl.obolibrary.org/obo/UBERON_0005273) 


### UBERON_0005932 `http://purl.obolibrary.org/obo/UBERON_0005932`

#### Added
- Class: [UBERON_0005932](http://purl.obolibrary.org/obo/UBERON_0005932) 


### UBERON_0005933 `http://purl.obolibrary.org/obo/UBERON_0005933`
#### Removed
- Class: [UBERON_0005933](http://purl.obolibrary.org/obo/UBERON_0005933) 



### UBERON_0013232 `http://purl.obolibrary.org/obo/UBERON_0013232`

#### Added
- Class: [UBERON_0013232](http://purl.obolibrary.org/obo/UBERON_0013232) 


### UBERON_0014717 `http://purl.obolibrary.org/obo/UBERON_0014717`

#### Added
- Class: [UBERON_0014717](http://purl.obolibrary.org/obo/UBERON_0014717) 


### UBERON_0035049 `http://purl.obolibrary.org/obo/UBERON_0035049`

#### Added
- Class: [UBERON_0035049](http://purl.obolibrary.org/obo/UBERON_0035049) 


### UBERON_8600124 `http://purl.obolibrary.org/obo/UBERON_8600124`

#### Added
- Class: [UBERON_8600124](http://purl.obolibrary.org/obo/UBERON_8600124) 


### V1delta gamma-delta T cell `http://purl.obolibrary.org/obo/CL_0020001`

#### Added
- Class: [V1delta gamma-delta T cell](http://purl.obolibrary.org/obo/CL_0020001) 

- [V1delta gamma-delta T cell](http://purl.obolibrary.org/obo/CL_0020001) [contributor](http://purl.org/dc/terms/contributor) [0009-0000-8480-9277](https://orcid.org/0009-0000-8480-9277) 

- [V1delta gamma-delta T cell](http://purl.obolibrary.org/obo/CL_0020001) [date](http://purl.org/dc/terms/date) "2025-09-29T10:40:48Z"^^[dateTime](http://www.w3.org/2001/XMLSchema#dateTime) 

- [V1delta gamma-delta T cell](http://purl.obolibrary.org/obo/CL_0020001) [id](http://www.geneontology.org/formats/oboInOwl#id) "CL:0020001"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [V1delta gamma-delta T cell](http://purl.obolibrary.org/obo/CL_0020001) [has_related_synonym](http://www.geneontology.org/formats/oboInOwl#hasRelatedSynonym) "V&delta;1+ &gamma;&delta; T Cell"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [has_synonym_type](http://www.geneontology.org/formats/oboInOwl#hasSynonymType) [OMO_0003000](http://purl.obolibrary.org/obo/OMO_0003000) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:24895997"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [V1delta gamma-delta T cell](http://purl.obolibrary.org/obo/CL_0020001) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A mature gamma-delta T Cell with T cell receptors consisting of the V&delta;1 segment paired with various V gamma chains. This cell is enriched in the gut, skin (Davey et al., 2018) and the female reproductive tract (Wu et al., 2022). It exhibits potent cytotoxic capabilities through perforin/granzyme and death receptor pathways."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:37989744"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:29680462"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:35637401"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:25991474"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [V1delta gamma-delta T cell](http://purl.obolibrary.org/obo/CL_0020001) [has_related_synonym](http://www.geneontology.org/formats/oboInOwl#hasRelatedSynonym) "V&delta;1+ Gamma-Delta T cell"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [has_synonym_type](http://www.geneontology.org/formats/oboInOwl#hasSynonymType) [OMO_0003000](http://purl.obolibrary.org/obo/OMO_0003000) 

- [V1delta gamma-delta T cell](http://purl.obolibrary.org/obo/CL_0020001) [label](http://www.w3.org/2000/01/rdf-schema#label) "V1delta gamma-delta T cell"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [V1delta gamma-delta T cell](http://purl.obolibrary.org/obo/CL_0020001) SubClassOf [mature gamma-delta T cell](http://purl.obolibrary.org/obo/CL_0000800) 


### VIP GABAergic interneuron `http://purl.obolibrary.org/obo/CL_4023016`
#### Removed
- [VIP GABAergic interneuron](http://purl.obolibrary.org/obo/CL_4023016) [IAO_0000028](http://purl.obolibrary.org/obo/IAO_0000028) "VIP cortical interneuron"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [VIP GABAergic interneuron](http://purl.obolibrary.org/obo/CL_4023016) [description](http://purl.org/dc/terms/description) "This neuronal type is distinguished by its heterogeneous morphology and electrophysiological characteristics  (Pronneke et al., 2015,2020; Apicella and Marchionni, 2022). Variations in morphology and firing patterns are observed in human cortical VIP interneurons. In a study by Lee et al., 2023, four distinct subtypes of VIP neurons were identified, each exhibiting unique morphologies, soma locations, and electrophysiological properties. These subtypes are defined according to their transcriptomic markers: LBH-VIP and CCD184-VIP, both of which display a bipolar morphology with axons and dendrites extending across cortical layers; and SY16-VIP and ADAMTSL1-VIP, characterized by a dendritic arbor that spreads horizontally within the same layer as the soma (Lee et al., 2023). Disruption or malfunction of VIP cortical interneuron is implicated in neurodevelopmental disorders such as Rett Syndrome and Dravet Syndrome (Goff and Goldberg 2021)."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:26420784"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:31210267"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:35221922"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:33794534"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [VIP GABAergic interneuron](http://purl.obolibrary.org/obo/CL_4023016) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A transcriptomically distinct cortical GABAergic neuron that expresses the vasocactive intestinal polypeptide and that has its soma located in the cerebral cortex. The standard transcriptomic reference data for this cell type can be found on the CellxGene census under the collection: 'Transcriptomic cytoarchitecture reveals principles of human neocortex organization', dataset: 'Supercluster: CGE-derived interneurons', Author Categories: 'CrossArea_subclass', cluster Vip."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "https://cellxgene.cziscience.com/e/e4ddac12-f48f-4455-8e8d-c2a48a683437.cxg/"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:33186530"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:37824655"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [VIP GABAergic interneuron](http://purl.obolibrary.org/obo/CL_4023016) [has_related_synonym](http://www.geneontology.org/formats/oboInOwl#hasRelatedSynonym) "cortical VIP neuron"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:36384143"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [VIP GABAergic interneuron](http://purl.obolibrary.org/obo/CL_4023016) [label](http://www.w3.org/2000/01/rdf-schema#label) "VIP GABAergic cortical interneuron"^^[string](http://www.w3.org/2001/XMLSchema#string) 

#### Added
- [VIP GABAergic interneuron](http://purl.obolibrary.org/obo/CL_4023016) [IAO_0000028](http://purl.obolibrary.org/obo/IAO_0000028) "VIP interneuron"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [VIP GABAergic interneuron](http://purl.obolibrary.org/obo/CL_4023016) [in_subset](http://www.geneontology.org/formats/oboInOwl#inSubset) [human_reference_atlas](http://purl.obolibrary.org/obo/cl/cl-base#human_reference_atlas) 

- [VIP GABAergic interneuron](http://purl.obolibrary.org/obo/CL_4023016) [description](http://purl.org/dc/terms/description) "The vip GABAergic cortical interneurons are a specific subset of inhibitory neurons found in the mammalian cerebral cortex. Named after the neuropeptide they typically release, vasoactive intestinal peptide, these neurons are also characterized by producing and releasing gamma-aminobutyric acid (GABA), which is the primary inhibitory neurotransmitter in the mammalian central nervous system. This specific subclass of GABAergic cortical interneurons are among five subclasses recently defined for their marker gene expression, which include Sst, Pvalb, Vip, Lamp5, and Sncg. These interneurons play a critical role in shaping the dynamics of neural circuitry and modulating the balance between excitation and inhibition in neuronal networks. 
Regarding their function, vip GABAergic cortical interneurons serve as integral regulators of cortical activity. They primarily target other inhibitory interneuron populations such as the somatostatin-expressing sncg GABAergic cortical interneuron and the parvalbumin-expressing pvalb GABAergic cortical interneuron. By inhibiting these neuron populations, vip GABAergic cortical interneurons indirectly promote the activity of excitatory neurons, thereby serving as key facilitators of cortical activation and plasticity. They become particularly engaged during complex cognitive tasks and sensory experiences, highlighting their role in information processing and adaptation to environmental changes.
Furthermore, vip GABAergic interneurons have a unique feature of long-range axonal projections, allowing their influence to spread over wide cortical areas. Additionally, these cells display diversity in morphological, electrophysiological and molecular properties, indicating their multifunctional role in cortical functioning. Disruptions in the function of vip GABAergic interneurons have been associated with various neurological and psychiatric disorders, including schizophrenia and autism spectrum disorder, underscoring the importance of these cells for normal brain function.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.celrep.2023.112628"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1093/cercor/bhy227"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/s41467-018-07162-5"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.3389/fncel.2022.811484"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/s41586-018-0654-5"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [VIP GABAergic interneuron](http://purl.obolibrary.org/obo/CL_4023016) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A transcriptomically distinct GABAergic neuron derived from the CGE and that expresses the vasoactive intestinal polypeptide. Its soma is located in the forebrain."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "https://cellxgene.cziscience.com/e/e4ddac12-f48f-4455-8e8d-c2a48a683437.cxg/"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:33186530"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:37824655"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [VIP GABAergic interneuron](http://purl.obolibrary.org/obo/CL_4023016) [has_related_synonym](http://www.geneontology.org/formats/oboInOwl#hasRelatedSynonym) "VIP neuron"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:36384143"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [VIP GABAergic interneuron](http://purl.obolibrary.org/obo/CL_4023016) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_4023016"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "VIP GABAergic interneuron on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [VIP GABAergic interneuron](http://purl.obolibrary.org/obo/CL_4023016) [label](http://www.w3.org/2000/01/rdf-schema#label) "VIP GABAergic interneuron"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### VIP GABAergic interneuron (Homo sapiens) `http://purl.obolibrary.org/obo/CL_4072032`

#### Added
- Class: [VIP GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072032) 

- [VIP GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072032) [IAO_0000028](http://purl.obolibrary.org/obo/IAO_0000028) "VIP interneuron (Homo sapiens)"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [VIP GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072032) [RO_0002175](http://purl.obolibrary.org/obo/RO_0002175) [NCBITaxon_9606](http://purl.obolibrary.org/obo/NCBITaxon_9606) 

- [VIP GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072032) [contributor](http://purl.org/dc/terms/contributor) "https://orcid.org/0000-0002-5507-2103"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [VIP GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072032) [date](http://purl.org/dc/terms/date) "2025-09-04T10:52:00Z"^^[dateTime](http://www.w3.org/2001/XMLSchema#dateTime) 

- [VIP GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072032) [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "ILX:0770151"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [VIP GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072032) [id](http://www.geneontology.org/formats/oboInOwl#id) "CL:4072032"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [VIP GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072032) [in_subset](http://www.geneontology.org/formats/oboInOwl#inSubset) [BDS_subset](http://purl.obolibrary.org/obo/cl/cl-base#BDS_subset) 

- [VIP GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072032) [has_related_synonym](http://www.geneontology.org/formats/oboInOwl#hasRelatedSynonym) "Vip (Homo sapiens)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "ILX:0770151"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [VIP GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072032) [description](http://purl.org/dc/terms/description) "This neuronal type is distinguished by its heterogeneous morphology and electrophysiological characteristics  (Pronneke et al., 2015,2020; Apicella and Marchionni, 2022). Variations in morphology and firing patterns are observed in human VIP interneurons. In a study by Lee et al., 2023, four distinct subtypes of VIP neurons were identified, each exhibiting unique morphologies, soma locations, and electrophysiological properties. These subtypes are defined according to their transcriptomic markers: LBH-VIP and CCD184-VIP, both of which display a bipolar morphology with axons and dendrites extending across cortical layers; and SY16-VIP and ADAMTSL1-VIP, characterized by a dendritic arbor that spreads horizontally within the same layer as the soma (Lee et al., 2023). Disruption or malfunction of VIP interneuron is implicated in neurodevelopmental disorders such as Rett Syndrome and Dravet Syndrome (Goff and Goldberg 2021)."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:26420784"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:31210267"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:35221922"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:33794534"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [VIP GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072032) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A transcriptomically distinct GABAergic neuron that expresses the vasoactive intestinal polypeptide and that has its soma located in the forebrain. The standard transcriptomic reference data for this cell type can be found on the CellxGene census under the collection: 'Transcriptomic cytoarchitecture reveals principles of human neocortex organization', dataset: 'Supercluster: CGE-derived interneurons', Author Categories: 'CrossArea_subclass', cluster Vip."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "https://cellxgene.cziscience.com/e/e4ddac12-f48f-4455-8e8d-c2a48a683437.cxg/"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:33186530"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:37824655"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [VIP GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072032) [has_related_synonym](http://www.geneontology.org/formats/oboInOwl#hasRelatedSynonym) "VIP-IN (Homo sapiens)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:33794534"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [VIP GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072032) [has_related_synonym](http://www.geneontology.org/formats/oboInOwl#hasRelatedSynonym) "VIP neuron (Homo sapiens)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:36384143"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [VIP GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072032) [has_related_synonym](http://www.geneontology.org/formats/oboInOwl#hasRelatedSynonym) "VIP type (Homo sapiens)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:37824655"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [VIP GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072032) [comment](http://www.w3.org/2000/01/rdf-schema#comment) "The marker set VIP can identify the Human cell type VIP GABAergic interneuron in the neocortex with a confidence of 0.855531344 (NS-Forest FBeta value)."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "https://doi.org/10.5281/zenodo.11165918"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [VIP GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072032) [label](http://www.w3.org/2000/01/rdf-schema#label) "VIP GABAergic interneuron (Homo sapiens)"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [VIP GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072032) SubClassOf [VIP GABAergic interneuron](http://purl.obolibrary.org/obo/CL_4023016) 

- [VIP GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072032) SubClassOf [RO_0002162](http://purl.obolibrary.org/obo/RO_0002162) some [NCBITaxon_9606](http://purl.obolibrary.org/obo/NCBITaxon_9606) 

- [VIP GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072032) SubClassOf [RO_0002292](http://purl.obolibrary.org/obo/RO_0002292) some [PR_P01282](http://purl.obolibrary.org/obo/PR_P01282) 


### Vgamma1.1-positive, Vdelta6.3-negative thymocyte `http://purl.obolibrary.org/obo/CL_0002411`
#### Removed
- [Vgamma1.1-positive, Vdelta6.3-negative thymocyte](http://purl.obolibrary.org/obo/CL_0002411) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A gamma-delta receptor that expresses Vgamma1.1 but does not express Vdelta6.3 chains in the T-cell receptor."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "GOC:tfm"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "http://www.immgen.org/index_content.html"^^[string](http://www.w3.org/2001/XMLSchema#string) 

#### Added
- [Vgamma1.1-positive, Vdelta6.3-negative thymocyte](http://purl.obolibrary.org/obo/CL_0002411) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A gamma-delta receptor that expresses Vgamma1.1 but does not express Vdelta6.3 chains in the T-cell receptor."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:35757696"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:32979762"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "GOC:tfm"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "http://www.immgen.org/index_content.html"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [Vgamma1.1-positive, Vdelta6.3-negative thymocyte](http://purl.obolibrary.org/obo/CL_0002411) SubClassOf [RO_0002162](http://purl.obolibrary.org/obo/RO_0002162) some [NCBITaxon_10090](http://purl.obolibrary.org/obo/NCBITaxon_10090) 


### Vgamma1.1-positive, Vdelta6.3-positive thymocyte `http://purl.obolibrary.org/obo/CL_0002412`
#### Removed
- [Vgamma1.1-positive, Vdelta6.3-positive thymocyte](http://purl.obolibrary.org/obo/CL_0002412) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A gamma-delta receptor that expresses Vgamma1.1-Vdelta6.3 chains in the T-cell receptor."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "GOC:tfm"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "http://www.immgen.org/index_content.html"^^[string](http://www.w3.org/2001/XMLSchema#string) 

#### Added
- [Vgamma1.1-positive, Vdelta6.3-positive thymocyte](http://purl.obolibrary.org/obo/CL_0002412) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A gamma-delta receptor that expresses Vgamma1.1-Vdelta6.3 chains in the T-cell receptor."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:35757696"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:32979762"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "GOC:tfm"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "http://www.immgen.org/index_content.html"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [Vgamma1.1-positive, Vdelta6.3-positive thymocyte](http://purl.obolibrary.org/obo/CL_0002412) SubClassOf [RO_0002162](http://purl.obolibrary.org/obo/RO_0002162) some [NCBITaxon_10090](http://purl.obolibrary.org/obo/NCBITaxon_10090) 


### acinar cell of salivary gland `http://purl.obolibrary.org/obo/CL_0002623`
#### Removed
- [acinar cell of salivary gland](http://purl.obolibrary.org/obo/CL_0002623) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "An acinar cell of salivary gland."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "GOC:tfm"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [acinar cell of salivary gland](http://purl.obolibrary.org/obo/CL_0002623) EquivalentTo [acinar cell](http://purl.obolibrary.org/obo/CL_0000622) and ([BFO_0000050](http://purl.obolibrary.org/obo/BFO_0000050) some [UBERON_0001044](http://purl.obolibrary.org/obo/UBERON_0001044)) 

#### Added
- [acinar cell of salivary gland](http://purl.obolibrary.org/obo/CL_0002623) [description](http://purl.org/dc/terms/description) "Acinar cells of the salivary gland are a critical part of the oral physiology and the digestive mechanism. These specialized cells are concentrated towards the terminal ends of the salivary glands, where they form sac-like acini or clusters. The primary function of these cells is to produce and secrete saliva. This substance, comprised mainly of water, electrolytes, mucins, and enzymes, is vital for the initial stages of digestion and for the regular maintenance of oral health. 
Acinar cells produce the two main types of saliva: serous and mucinous. Serous saliva contains the enzyme amylase responsible for the initial breakdown of complex carbohydrates and is predominantly produced by the parotid gland, the largest of salivary glands. The mucinous type, more common in the submandibular and sublingual glands, contains mucin that aids in lubrication, facilitating the process of mastication and bolus formation.
Moreover, acinar cells also express and respond to various endocrine, paracrine, and autocrine hormones that govern the quantity and composition of the saliva produced. This implies sensitivity and modulation according to the internal and external environment, dietary habits, and even circadian rhythms. 
Impairment of acinar cells, whether due to damage or related to a systemic condition such as Sj&ouml;gren's Syndrome, can disrupt the optimal functioning of the oral and digestive systems. In conclusion, acinar cells of the salivary gland have a substantial role in oral health and overall homeostasis.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "https://www.ncbi.nlm.nih.gov/books/NBK538325/"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/S0140-6736(05)66990-5"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1111/prd.12116"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1152/physrev.00011.2011"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/sj.bdj.4807861"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [acinar cell of salivary gland](http://purl.obolibrary.org/obo/CL_0002623) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "An acinar cell that is part of the salivary gland epithelium, responsible for synthesizing and secreting the primary saliva essential for oral digestion, antimicrobial defense, and mucosal immunity. It features apical-basal polarity with apical secretory granules, and tight junctions that maintain polarity and regulate calcium-dependent saliva exocytosis triggered by parasympathetic and sympathetic signals (Ambudkar, 2015). This cell functions in coordination with contractile myoepithelial cells, which aid saliva flow through the ducts into the oral cavity (Khan et al., 2022). In mice and humans, acinar cell development relies on the transcription factor SOX2 (Emmerson et al., 2017), with self-renewal achieved primarily through acinar cell self-duplication and supported by progenitor populations during injury and regeneration (Aure et al., 2016)."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:24646566"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:25843887"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:28623666"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:34989394"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "GOC:tfm"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:35343828"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [acinar cell of salivary gland](http://purl.obolibrary.org/obo/CL_0002623) [has_exact_synonym](http://www.geneontology.org/formats/oboInOwl#hasExactSynonym) "salivary acinar cell"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:24646566"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [acinar cell of salivary gland](http://purl.obolibrary.org/obo/CL_0002623) [has_exact_synonym](http://www.geneontology.org/formats/oboInOwl#hasExactSynonym) "salivary gland acinar cell"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:24646566"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [acinar cell of salivary gland](http://purl.obolibrary.org/obo/CL_0002623) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0002623"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "acinar cell of salivary gland on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [acinar cell of salivary gland](http://purl.obolibrary.org/obo/CL_0002623) EquivalentTo [acinar cell](http://purl.obolibrary.org/obo/CL_0000622) and ([BFO_0000050](http://purl.obolibrary.org/obo/BFO_0000050) some [UBERON_0004809](http://purl.obolibrary.org/obo/UBERON_0004809)) 

- [acinar cell of salivary gland](http://purl.obolibrary.org/obo/CL_0002623) SubClassOf [BFO_0000051](http://purl.obolibrary.org/obo/BFO_0000051) some [GO_0030141](http://purl.obolibrary.org/obo/GO_0030141) 

- [acinar cell of salivary gland](http://purl.obolibrary.org/obo/CL_0002623) SubClassOf [RO_0002215](http://purl.obolibrary.org/obo/RO_0002215) some [GO_0017156](http://purl.obolibrary.org/obo/GO_0017156) 

- [acinar cell of salivary gland](http://purl.obolibrary.org/obo/CL_0002623) SubClassOf [RO_0002215](http://purl.obolibrary.org/obo/RO_0002215) some [GO_0045087](http://purl.obolibrary.org/obo/GO_0045087) 

- [acinar cell of salivary gland](http://purl.obolibrary.org/obo/CL_0002623) SubClassOf [RO_0002215](http://purl.obolibrary.org/obo/RO_0002215) some [GO_0046541](http://purl.obolibrary.org/obo/GO_0046541) 


### adipocyte `http://purl.obolibrary.org/obo/CL_0000136`

#### Added
- [adipocyte](http://purl.obolibrary.org/obo/CL_0000136) [description](http://purl.org/dc/terms/description) "A fat cell, also known as an adipocyte, is a specialized type of connective tissue cell responsible for the storage of fat in the body. These cells can be found throughout the body, but are most densely located in adipose tissue, such as the subcutaneous tissue beneath the skin and around organs, and act as the body&rsquo;s energy storage units.
The primary function of fat cells is to store energy in the form of fat, also known as triglycerides, for use during periods of caloric deficit. When we consume more calories than we use for energy, our bodies convert the excess energy into fat and store it in adipose tissue. 
Fat cells also play a role in regulating energy balance by secreting hormones, such as leptin and adiponectin, which help to regulate appetite and metabolism. Fat calls have also been implicated in the development of obesity-related diseases, such as insulin resistance and type 2 diabetes. 
Finally, fat cells have been found to play a role in the immune system by secreting cytokines, which can either promote or inhibit inflammation depending on the type of cytokine produced.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1007/s10753-021-01559-z"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1098/rstb.2006.1859"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1091/mbc.e15-10-0749"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [adipocyte](http://purl.obolibrary.org/obo/CL_0000136) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0000136"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "adipocyte on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### adipocyte of breast `http://purl.obolibrary.org/obo/CL_0002617`

#### Added
- [adipocyte of breast](http://purl.obolibrary.org/obo/CL_0002617) [description](http://purl.org/dc/terms/description) "Adipocytes of the breast, commonly known as breast fat cells, serve crucial structural, protective, and metabolic roles. These specialized cells, ubiquitously found within the breast's adipose tissue, primarily contribute to the organ's overall shape and cushioning, providing physical support to surrounding structures. 
Adipocytes are highly active metabolically and play a key role in lipid homeostasis, acting as principal sites for energy storage and release. These cells store excess energy in the form of triglycerides during times of energy surplus, releasing it during periods of energy deficit as free fatty acids and glycerol into the bloodstream. Therefore, regulation of breast adipocytes' size and number is critical to maintain overall energy balance. Moreover, during lactation, these fat cells assist in providing lipid molecules necessary for the production of breast milk.
Moreover, breast adipocytes notably function in the endocrine system, secreting a wide array of hormones, growth factors, and cytokines, collectively referred to as adipokines. While adipokines have a wide range of effects, some are believed to exert crucial influence on breast cancer physiology. Indeed, excessive adipose tissue, leading to a state of chronic low-grade inflammation, is associated with an increased risk of several types of cancer, including breast cancer.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/s41467-018-05911-0"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.phrs.2020.104741"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/s41523-017-0015-9"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1007/s11154-021-09633-5"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.3390/ijms21165760"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [adipocyte of breast](http://purl.obolibrary.org/obo/CL_0002617) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0002617"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "adipocyte of breast on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### adipocyte of epicardial fat of left ventricle `http://purl.obolibrary.org/obo/CL_1000311`

#### Added
- [adipocyte of epicardial fat of left ventricle](http://purl.obolibrary.org/obo/CL_1000311) [description](http://purl.org/dc/terms/description) "Adipocytes of the epicardial fat of the left ventricle are specialized cells localized within the epicardial adipose tissue, which is found enveloping the heart's outer layer, particularly the left ventricle. The primary function of these adipocytes is the storage and release of energy in the form of lipids, which facilitates efficient energy transfer to the myocardium when necessary. They are mesenchymal-derived cells filled with lipid droplets, giving them a unique and distinct appearance when compared to other cell types.
The secondary, yet critically important function of these adipocytes revolves around their role as endocrine cells. They're involved in the production and secretion of various bioactive molecules, such as adiponectin, resistin, leptin, cytokines, and other adipocytokines, which play crucial roles in local and systemic metabolism, inflammatory responses, and myocardial protection. Furthermore, these adipocytes serve as a thermal insulation layer for the heart, protecting it from hypothermia and potentially mitigating the risk of arrhythmias caused by abrupt temperature changes.
Dysregulation of the adipocytes of the epicardial fat of the left ventricle in their functions can lead to pathological conditions such as obesity, diabetes, and cardiovascular diseases like coronary artery disease and heart failure. For instance, it has been observed that increased volume of epicardial adipose tissue could be linked to the development of coronary atherosclerosis.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.3390/ijms24076838"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1093/eurheartj/ehab724.1266"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/ncpcardio0319"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/s41598-023-41779-x"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1186/s12933-017-0528-4"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [adipocyte of epicardial fat of left ventricle](http://purl.obolibrary.org/obo/CL_1000311) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_1000311"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "adipocyte of epicardial fat of left ventricle on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### agouti-related protein expressing neuron `http://purl.obolibrary.org/obo/CL_4072017`

#### Added
- Class: [agouti-related protein expressing neuron](http://purl.obolibrary.org/obo/CL_4072017) 

- [agouti-related protein expressing neuron](http://purl.obolibrary.org/obo/CL_4072017) [date](http://purl.org/dc/terms/date) "2025-07-07T09:55:43Z"^^[dateTime](http://www.w3.org/2001/XMLSchema#dateTime) 

- [agouti-related protein expressing neuron](http://purl.obolibrary.org/obo/CL_4072017) [id](http://www.geneontology.org/formats/oboInOwl#id) "CL:4072017"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [agouti-related protein expressing neuron](http://purl.obolibrary.org/obo/CL_4072017) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A GABAergic neuron located in the arcuate nucleus of the hypothalamus that expresses agouti-related protein (AgRP). This neuron can coexpress AgRP, GABA, and neuropeptide Y (NPY) in mice, and plays a key role in integrating metabolic signals. It is activated by hunger-related hormones such as ghrelin and inhibited by satiety signals like leptin."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:37957320"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:39719709"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:34469623"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [agouti-related protein expressing neuron](http://purl.obolibrary.org/obo/CL_4072017) [has_related_synonym](http://www.geneontology.org/formats/oboInOwl#hasRelatedSynonym) "AgRP neuron"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [has_synonym_type](http://www.geneontology.org/formats/oboInOwl#hasSynonymType) [OMO_0003000](http://purl.obolibrary.org/obo/OMO_0003000) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:39719709"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [agouti-related protein expressing neuron](http://purl.obolibrary.org/obo/CL_4072017) [label](http://www.w3.org/2000/01/rdf-schema#label) "agouti-related protein expressing neuron"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [agouti-related protein expressing neuron](http://purl.obolibrary.org/obo/CL_4072017) SubClassOf [BFO_0000002](http://purl.obolibrary.org/obo/BFO_0000002) 

- [agouti-related protein expressing neuron](http://purl.obolibrary.org/obo/CL_4072017) SubClassOf [GABAergic neuron](http://purl.obolibrary.org/obo/CL_0000617) 

- [agouti-related protein expressing neuron](http://purl.obolibrary.org/obo/CL_4072017) SubClassOf [neuron of the forebrain](http://purl.obolibrary.org/obo/CL_0012001) 

- [agouti-related protein expressing neuron](http://purl.obolibrary.org/obo/CL_4072017) SubClassOf [RO_0002100](http://purl.obolibrary.org/obo/RO_0002100) some [UBERON_0001932](http://purl.obolibrary.org/obo/UBERON_0001932) 

- [agouti-related protein expressing neuron](http://purl.obolibrary.org/obo/CL_4072017) SubClassOf [RO_0002292](http://purl.obolibrary.org/obo/RO_0002292) some [PR_000003846](http://purl.obolibrary.org/obo/PR_000003846) 

- [agouti-related protein expressing neuron](http://purl.obolibrary.org/obo/CL_4072017) SubClassOf [RO_0002292](http://purl.obolibrary.org/obo/RO_0002292) some [PR_000011387](http://purl.obolibrary.org/obo/PR_000011387) 


### alpha7 GABAergic cortical interneuron (Mmus) `http://purl.obolibrary.org/obo/CL_4023010`

#### Added
- [alpha7 GABAergic cortical interneuron (Mmus)](http://purl.obolibrary.org/obo/CL_4023010) SubClassOf [BFO_0000002](http://purl.obolibrary.org/obo/BFO_0000002) 


### antibody secreting cell `http://purl.obolibrary.org/obo/CL_0000946`

#### Added
- [antibody secreting cell](http://purl.obolibrary.org/obo/CL_0000946) [description](http://purl.org/dc/terms/description) "Antibody secreting cells, such as short-lived plasmablasts and long-lived plasma cells, form a key component of the adaptive immune system. These cells originate from B-lymphocytes upon exposure to antigens. They are primarily characterized by their capacity to produce and secrete large amounts of antibodies (immunoglobulins) that help to neutralize the pathogens, defining the primary role of these cells in the humoral immune response. 
Antibody secreting cells are found predominantly within the lymph nodes, spleen, and mucosa-associated lymphoid tissues. They are characterized by an extensive rough endoplasmic reticulum, reflecting the high antibody production. The interesting dynamics of these cells, switching from a nonsecreting, antigen-specific B-cell state to a high-rate immunoglobulin-secreting cell state, is a major factor in sustaining an effective adaptive immune system: Derived from a single clone, the secreted immunoglobulins have a unique antigen specificity inherited from the parent B cell. They are able to recognize specific extracellular antigens, aiding in marking harmful invaders such as viruses and bacteria for elimination, either by direct neutralization or by recruiting other immune cells for assistance.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1002/eji.1830161018"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/nri3795"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [antibody secreting cell](http://purl.obolibrary.org/obo/CL_0000946) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0000946"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "antibody secreting cell on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### astrocyte of the cerebellum `http://purl.obolibrary.org/obo/CL_0002603`

#### Added
- [astrocyte of the cerebellum](http://purl.obolibrary.org/obo/CL_0002603) [description](http://purl.org/dc/terms/description) "Astrocytes of the cerebellum, also known as cerebellar astrocytes, constitute a major subtype of glial cells in the cerebellum, an integral region of the brain responsible for regulating motor movements and cognitive processes. These astrocytes are unique to the cerebellum, differing in morphological and functional characteristics from astrocytess located in other parts of the nervous system. They play a crucial role in managing the brain's complex microenvironment by maintaining neuronal health, supporting biochemical pathways, and contributing to targeted neural communication.
Astrocytes of the cerebellum operate through an interactive network with neurons, blood vessels, and other glial cells. They provide physical and nutritional support for neurons, helping preserve neuronal survival and function. These cells effectively manage the extracellular ion and neurotransmitter balance, pivotal for synaptic transmission. They also contribute to the maintenance of the blood-brain barrier, a semi-permeable membrane that safeguards the brain from harmful substances in the bloodstream, ensuring cerebral homeostasis.
Furthermore, cerebellar astrocytes are involved in the processing and modulation of synapses, the junctions where neurons exchange information. They regulate the release and uptake of neurotransmitters, which helps in synaptic plasticity, a process that is integral in learning and memory. Moreover, these astrocytes produce and secrete various neurotrophic factors, including brain-derived neurotrophic factor (BDNF), which is crucial for neurogenesis and synaptic plasticity. Therefore, astrocytes of the cerebellum not only execute supportive roles but are also actively involved in modifying neuronal function and enabling neuroplasticity.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1007/s12311-019-01046-0"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1155/2014/232105"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.brainres.2006.06.008"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.pneurobio.2013.08.001"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/nrn1824"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [astrocyte of the cerebellum](http://purl.obolibrary.org/obo/CL_0002603) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0002603"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "astrocyte of the cerebellum on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### basal cell of epidermis `http://purl.obolibrary.org/obo/CL_0002187`

#### Added
- [basal cell of epidermis](http://purl.obolibrary.org/obo/CL_0002187) [description](http://purl.org/dc/terms/description) "The basal cells of the epidermis make up the innermost layer of the skin, providing an essential structural component to the body's largest organ. These cuboidal-shaped cells are critical to the skin's renewal process, dividing to form new keratinocytes that gradually make their way to the skin&rsquo;s surface. The migration of these cells creates a protective barrier that impedes damage from environmental factors like sunlight, radiation, and harmful pathogens.
Basal cells have an additional, pivotal role beyond formation and migration of keratinocytes: they are also directly involved in the production of various types of skin appendages, including hair follicles, sebaceous glands, and sweat glands. Coupled with their prolific regenerative properties, these functions make basal cells vital for maintaining the skin's overall health and vitality.
Changes and mutations in the formation and growth of basal cells of the epidermis can lead to cancerous lesions. Though usually not life-threatening, early identification is critical to minimizing further tissue damage.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1186/s41038-016-0064-6"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1172/JCI58779"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.7748/ns.27.3.35.s52"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1007/s12015-021-10256-1"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [basal cell of epidermis](http://purl.obolibrary.org/obo/CL_0002187) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0002187"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "basal cell of epidermis on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### basal cell of prostate epithelium `http://purl.obolibrary.org/obo/CL_0002341`

#### Added
- [basal cell of prostate epithelium](http://purl.obolibrary.org/obo/CL_0002341) [description](http://purl.org/dc/terms/description) "Basal cells of the prostate epithelium are one of the major cell types found in the glandular part of the prostate, which is a small, walnut-shaped gland that produces seminal fluid in males. These cells are located at the base of the prostatic acinar epithelial layer and form a contiguous layer separating the acinar lumen, where the prostatic secretions coalesce, and the underlying connective tissue. In contrast to the luminal cells, basal cells lack secretory granules. In common with basal epithelial cells in other epithelia, prostatic basal cells express markers such as cytokeratin 14 and p63. 
A key function of the basal cells of the prostate epithelium is to provide structural support to the secretory luminal cells and contribute to maintaining the integrity of the epithelial layer. They also have unique regenerative properties that aid in the recovery of the glandular epithelium after injury. Under normal conditions, these cells remain quiescent, but upon injury, they proliferate to replace the damaged secretory luminal cells, indicating a stem/progenitor cell-like behavior. They also play an important role in serving as a reserve cell population helping in maintaining tissue homeostasis. 
Basal cells of prostate epithelium are also central to understanding the pathobiology of prostate diseases. They are typically absent in prostatic adenocarcinomas, the most common form of prostate cancer, but they are thought to play a critical yet complex role in oncogenesis and therapeutic resistance mechanisms of prostate cancer, and in benign prostatic hyperplasia. They also have the ability to differentiate into neuroendocrine cells or luminal cells under certain conditions, contributing to disease heterogeneity.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4219311/"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1242/dev.01384"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.celrep.2018.11.086"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1073/pnas.1320565110"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1080/15384101.2023.2206502"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [basal cell of prostate epithelium](http://purl.obolibrary.org/obo/CL_0002341) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0002341"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "basal cell of prostate epithelium on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### basal epithelial cell of prostatic duct `http://purl.obolibrary.org/obo/CL_0002236`

#### Added
- [basal epithelial cell of prostatic duct](http://purl.obolibrary.org/obo/CL_0002236) [description](http://purl.org/dc/terms/description) "Basal epithelial cells of the prostatic duct are unique cell types found within the prostate gland of mammals. The prostatic duct forms a significant part of the prostate's glandular architecture, facilitating the secretion and flow of prostatic fluid. These basal cells are positioned at the base of the prostatic epithelium, forming a layer beneath luminal cells and adjacent to the basement membrane. Functionally, they play a significant role in maintaining tissue homeostasis, managing secretory activities, and contributing to the overall biological functions of the prostate. 
These basal epithelial cells exhibit a high degree of differentiation. Morphologically, they are characterized by their low columnar or cuboidal shape and exhibit less cytoplasm compared to luminal epithelial cells. A notable feature is their high nucleus-to-cytoplasm ratio and the presence of prominent nucleoli. On a molecular level, these cells are known to express certain distinct markers such as cytokeratin 5 (CK5) and p63, differentiating them from other prostatic cells. 
The basal epithelial cells of the prostatic duct also serve as a reserve cell population for the regeneration of the prostate epithelium, particularly upon injury or disease, indicating their role in tissue repair and regrowth. Importantly, these cells are also considered as the cells-of-origin in prostate cancer due to their extended lifespan and genomic stability.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/B978-0-12-801238-3.64595-6"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1242/dev.01384"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1309/AJCP5ZC4GQVNWTYR"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.celrep.2018.11.086"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1002/stem.1094"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [basal epithelial cell of prostatic duct](http://purl.obolibrary.org/obo/CL_0002236) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0002236"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "basal epithelial cell of prostatic duct on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### basal-myoepithelial cell of mammary gland `http://purl.obolibrary.org/obo/CL_0002324`

#### Added
- [basal-myoepithelial cell of mammary gland](http://purl.obolibrary.org/obo/CL_0002324) [description](http://purl.org/dc/terms/description) "Myoepithelial cells of the mammary gland are a specialized type of smooth muscle cell that plays a vital role in lactation and the normal functioning of the mammary gland. Their unique elongated, stellate shape allows them to wrap around the ductal and alveolar units of the gland. 
Myoepithelial cells of the mammary gland facilitate the expulsion of milk during lactation in response to signaling by the hormone oxytocin. Due to their cytoskeletal composition of actin and myosin, which mirrors that of smooth muscle cells, these cells are able to contract and advance the milk from the alveoli into the ducts, and eventually to the nipple, enabling breastfeeding. 
Other critical roles of myoepithelial cells of the mammary gland include the maintenance of mammary gland integrity and the suppression of tumor formation. They create a mechanical barrier that separates the luminal epithelial cells from the stromal compartment and play a significant role in the suppression of carcinogenesis; loss or alteration of these cells often paves the way for invasive carcinomas. Moreover, these cell types produce extracellular matrix proteins such as fibronectin that help in the synthesis of the basement membrane, providing structural support to the gland and maintaining its overall health and integrity.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1186/bcr1286"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1007/s10911-005-9586-4"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1186/bcr459"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [basal-myoepithelial cell of mammary gland](http://purl.obolibrary.org/obo/CL_0002324) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0002324"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "basal-myoepithelial cell of mammary gland on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### beam A cell `http://purl.obolibrary.org/obo/CL_7770003`

#### Added
- Class: [beam A cell](http://purl.obolibrary.org/obo/CL_7770003) 

- [beam A cell](http://purl.obolibrary.org/obo/CL_7770003) [contributor](http://purl.org/dc/terms/contributor) [0009-0000-8480-9277](https://orcid.org/0009-0000-8480-9277) 

- [beam A cell](http://purl.obolibrary.org/obo/CL_7770003) [creation date](http://www.geneontology.org/formats/oboInOwl#creation_date) "2025-08-29T08:53:47Z"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [beam A cell](http://purl.obolibrary.org/obo/CL_7770003) [id](http://www.geneontology.org/formats/oboInOwl#id) "CL:7770003"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [beam A cell](http://purl.obolibrary.org/obo/CL_7770003) [depiction](http://xmlns.com/foaf/0.1/depiction) "http://purl.obolibrary.org/obo/cl/images/Trabecular_meshowk_cells_PMID35836364.jpg"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:35836364"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [comment](http://www.w3.org/2000/01/rdf-schema#comment) "Trabecular meshwork (TM) structure. The TM consists of uveal (blue), corneoscleral (red), and juxtacanalicular (green) regions. TM beam cells (blue/red) line the uveal and corneoscleral regions, while JCT cells (green) occupy the juxtacanalicular region."^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [license](http://purl.org/dc/elements/1.1/license) "http://creativecommons.org/licenses/by/4.0/"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [beam A cell](http://purl.obolibrary.org/obo/CL_7770003) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A beam cell within the eye's trabecular meshwork, molecularly distinguished by FABP4 expression in humans and spatially intermingled with Beam B cells throughout the uveal and corneoscleral meshwork regions. In mice, a transcriptionally analogous Beam A&ndash;like cluster (mC14) exists, sharing core TM markers (Myoc, Mgp, Pdpn, Chil1) and functional attributes."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:32341164"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:39829808"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:27443500"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [beam A cell](http://purl.obolibrary.org/obo/CL_7770003) [label](http://www.w3.org/2000/01/rdf-schema#label) "beam A cell"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [beam A cell](http://purl.obolibrary.org/obo/CL_7770003) SubClassOf [beam cell](http://purl.obolibrary.org/obo/CL_7770005) 


### beam B cell, human `http://purl.obolibrary.org/obo/CL_7770006`

#### Added
- Class: [beam B cell, human](http://purl.obolibrary.org/obo/CL_7770006) 

- [beam B cell, human](http://purl.obolibrary.org/obo/CL_7770006) [contributor](http://purl.org/dc/terms/contributor) [0009-0000-8480-9277](https://orcid.org/0009-0000-8480-9277) 

- [beam B cell, human](http://purl.obolibrary.org/obo/CL_7770006) [date](http://purl.org/dc/terms/date) "2025-09-15T08:48:01Z"^^[dateTime](http://www.w3.org/2001/XMLSchema#dateTime) 

- [beam B cell, human](http://purl.obolibrary.org/obo/CL_7770006) [id](http://www.geneontology.org/formats/oboInOwl#id) "CL:7770006"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [beam B cell, human](http://purl.obolibrary.org/obo/CL_7770006) [depiction](http://xmlns.com/foaf/0.1/depiction) "http://purl.obolibrary.org/obo/cl/images/Trabecular_meshowk_cells_PMID35836364.jpg"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:35836364"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [comment](http://www.w3.org/2000/01/rdf-schema#comment) "Trabecular meshwork (TM) structure. The TM consists of uveal (blue), corneoscleral (red), and juxtacanalicular (green) regions. TM beam cells (blue/red) line the uveal and corneoscleral regions, while JCT cells (green) occupy the juxtacanalicular region."^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [license](http://purl.org/dc/elements/1.1/license) "http://creativecommons.org/licenses/by/4.0/"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [beam B cell, human](http://purl.obolibrary.org/obo/CL_7770006) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A trabecular meshwork cell within the eye's trabecular meshwork. This cell is molecularly distinguished by TMEFF2 expression in humans. A beam B cell is spatially intermingled with beam A cells throughout the uveal and corneoscleral meshwork regions, but demonstrates a tendency to localise closer to the juxtacanalicular tissue."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:32341164"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:39829808"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:25356439"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [beam B cell, human](http://purl.obolibrary.org/obo/CL_7770006) [label](http://www.w3.org/2000/01/rdf-schema#label) "beam B cell, human"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [beam B cell, human](http://purl.obolibrary.org/obo/CL_7770006) SubClassOf [BFO_0000002](http://purl.obolibrary.org/obo/BFO_0000002) 

- [beam B cell, human](http://purl.obolibrary.org/obo/CL_7770006) SubClassOf [trabecular meshwork cell](http://purl.obolibrary.org/obo/CL_0002367) 

- [beam B cell, human](http://purl.obolibrary.org/obo/CL_7770006) SubClassOf [RO_0002162](http://purl.obolibrary.org/obo/RO_0002162) some [NCBITaxon_9606](http://purl.obolibrary.org/obo/NCBITaxon_9606) 

- [beam B cell, human](http://purl.obolibrary.org/obo/CL_7770006) SubClassOf [RO_0002292](http://purl.obolibrary.org/obo/RO_0002292) some [PR_Q9UIK5](http://purl.obolibrary.org/obo/PR_Q9UIK5) 


### beam cell `http://purl.obolibrary.org/obo/CL_7770005`

#### Added
- Class: [beam cell](http://purl.obolibrary.org/obo/CL_7770005) 

- [beam cell](http://purl.obolibrary.org/obo/CL_7770005) [contributor](http://purl.org/dc/terms/contributor) [0009-0000-8480-9277](https://orcid.org/0009-0000-8480-9277) 

- [beam cell](http://purl.obolibrary.org/obo/CL_7770005) [creation date](http://www.geneontology.org/formats/oboInOwl#creation_date) "2025-09-12T11:19:45Z"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [beam cell](http://purl.obolibrary.org/obo/CL_7770005) [id](http://www.geneontology.org/formats/oboInOwl#id) "CL:7770005"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [beam cell](http://purl.obolibrary.org/obo/CL_7770005) [depiction](http://xmlns.com/foaf/0.1/depiction) "http://purl.obolibrary.org/obo/cl/images/Trabecular_meshowk_cells_PMID35836364.jpg"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:35836364"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [comment](http://www.w3.org/2000/01/rdf-schema#comment) "Trabecular meshwork (TM) structure. The TM consists of uveal (blue), corneoscleral (red), and juxtacanalicular (green) regions. TM beam cells (blue/red) line the uveal and corneoscleral regions, while JCT cells (green) occupy the juxtacanalicular region."^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [license](http://purl.org/dc/elements/1.1/license) "http://creativecommons.org/licenses/by/4.0/"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [beam cell](http://purl.obolibrary.org/obo/CL_7770005) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A trabecular meshwork cell that is part of the eye's trabecular meshwork, residing in the uveal and corneoscleral meshwork regions and serving as the primary biological filter in the aqueous humor drainage cascade. This cell exhibits endothelial-like properties, including production of antithrombogenic substances such as tissue plasminogen activator, while also demonstrating phagocytic activity to remove cellular debris from aqueous humour before the fluid moves deeper into the less porous juxtacanalicular tissue."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:25356439"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:27443500"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [beam cell](http://purl.obolibrary.org/obo/CL_7770005) [label](http://www.w3.org/2000/01/rdf-schema#label) "beam cell"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [beam cell](http://purl.obolibrary.org/obo/CL_7770005) SubClassOf [phagocyte](http://purl.obolibrary.org/obo/CL_0000234) 

- [beam cell](http://purl.obolibrary.org/obo/CL_7770005) SubClassOf [trabecular meshwork cell](http://purl.obolibrary.org/obo/CL_0002367) 
  - [is_inferred](http://www.geneontology.org/formats/oboInOwl#is_inferred) "true"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### bipolar neuron `http://purl.obolibrary.org/obo/CL_0000103`

#### Added
- [bipolar neuron](http://purl.obolibrary.org/obo/CL_0000103) [description](http://purl.org/dc/terms/description) "The bipolar neuron is a type of nerve cell primarily found mainly in the sensory systems of the human body. These specialized cells possess two distinctive extensions or processes - one axon and one dendrite, emerging from the cell body on opposite ends, which gives them their bipolar appearance. These two extensions serve as conduits for electrical signals, with the dendrite primarily playing the role of receiving signals, and the axon transmitting them further.
Functionally, bipolar neurons are crucial in facilitating the transmission of special senses, particularly in the senses of sight, smell, hearing, and balance. For example, the bipolar neurons in the retina convey information about light intensity, color, and movement from the photoreceptors to the ganglion cells, forming part of the pathway for visual input to travel towards the brain. 
Similarly, in the olfactory system, the olfactory cells are bipolar neurons characterized by a peripheral process, called the olfactory rod, which projects fine cilia. These cilia are the sites of olfactory receptors which play a critical role in recognizing thousands of odorant molecules and transmit information about smell from the nose to the brain. 
Bipolar neurons are also involved in hearing. The vestibular ganglion contains thousands of bipolar neurons that receive sensory signals from hair cells in the inner ear. The nerve signals pass through the bipolar neurons and leave along the vestibular nerve. 
The morphology of bipolar neurons with their single dendrite and axon also subtly boosts their efficiency, maintaining a streamlined path for signal conduction. Their distinctive structuring allows for faster and more targeted relay of sensory information from the body's peripheries to the central nervous system. Loss or damage of these neurons would directly impact sensory reception and could lead to disorders related to vision, hearing, smell, or balance, underlining the significance of this cell type in the body's sensory capacity.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "https://www.sciencedirect.com/topics/neuroscience/bipolar-neuron"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.cell.2016.07.054"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/nrn3783"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.yexcr.2006.10.010"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1152/physrev.1986.66.3.772"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [bipolar neuron](http://purl.obolibrary.org/obo/CL_0000103) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0000103"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "bipolar neuron on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### blood vessel endothelial cell `http://purl.obolibrary.org/obo/CL_0000071`

#### Added
- [blood vessel endothelial cell](http://purl.obolibrary.org/obo/CL_0000071) [description](http://purl.org/dc/terms/description) "Blood vessel endothelial cells, also known as vascular endothelial cells, form the delicate inner lining of the complex circulatory system, including arteries and veins. These cells form a single, continuous monolayer inside every blood vessel, acting as a crucial biomolecular barrier between the blood, and the rest of the body tissues. They perform various fundamental biological functions, ranging from maintenance of vessel integrity to blood clotting regulation. Additionally, they control the transportation of fluids, molecules, and blood particulate matter from and into the bloodstream.
One of the primary functions of blood vessel endothelial cells is to maintain vascular homeostasis - a stable environment within the blood vessels. They accomplish this through several methods: they control vasomotor tone, i.e. the degree of contraction and dilation of blood vessels, by secreting vasoactive substances such as nitric oxide and endothelin; mediate responses to inflammatory stimuli; and they also regulate angiogenesis, the process of new blood vessel formations. Blood vessel endothelial cells play a significant role in preventing blood clot formation by generating a slick, non-thrombogenic surface that disallows platelets or coagulant proteins from adhering to it.
Blood vessel endothelial cells are sensitive to various stimuli, including shear stress caused by blood flow, and they can respond by altering their morphology, function, and gene expression. This ability to adapt enables blood vessels to remodel and ensure the adequate delivery of nutrients and oxygen to all tissues, regardless of physiological demand variations. Conversely, endothelial dysfunction, often associated with a reduction in the bioavailability of endothelium-derived nitric oxide, can lead to the development of pathological conditions such as atherosclerosis, hypertension, diabetes mellitus, and even aging-related diseases.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1007/s00441-021-03471-2"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1152/physrev.00001.2017"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.3390/ijms20184411"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.biomaterials.2007.01.039"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/s41569-022-00770-1"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [blood vessel endothelial cell](http://purl.obolibrary.org/obo/CL_0000071) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0000071"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "blood vessel endothelial cell on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### bone marrow hematopoietic cell `http://purl.obolibrary.org/obo/CL_1001610`

#### Added
- [bone marrow hematopoietic cell](http://purl.obolibrary.org/obo/CL_1001610) [description](http://purl.org/dc/terms/description) "Bone marrow hematopoietic cells are a category of blood-forming stem cells located within the bone marrow, the soft, spongy tissue that fills the cavities of bones. The bone marrow contains specialized environments, referred to as the stem cell niche, to nurture and regulate the development of these cells. These cells are vital for the production and development of new blood cells, functioning as the primary source of cellular regeneration throughout a person's life. They are capable of self-renewal and differentiation into every type of mature blood cell.
Hematopoietic stem cells differentiate into two main lineages: myeloid and lymphoid cells. Myeloid cells can further differentiate into red blood cells, which carry oxygen throughout the body, platelets that contribute to clot formation, and certain types of white blood cells such as neutrophils, eosinophils, basophils, and monocytes. These white blood cells play a crucial role in the innate immune response, helping to fight infection and disease. Lymphoid cells develop into other types of white blood cells: T-cells, B-cells, and innate lymphoid cells, which contribute to the cellular and humoral immune response.
Furthermore, the bone marrow hematopoietic cells have a unique ability to replace any blood cell in the body that has been destroyed or lost due to illness or injury. Through this regenerative functionality, they are vitally important to the body's defense mechanisms, disease response, and homeostasis. These cells are used in bone marrow transplants and are of interest in regenerative medicine applications.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.semcdb.2020.05.014"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1182/blood-2009-11-253534"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/B978-0-323-04404-2.10002-8"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [bone marrow hematopoietic cell](http://purl.obolibrary.org/obo/CL_1001610) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_1001610"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "bone marrow hematopoietic cell on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### branchiomotor neuron `http://purl.obolibrary.org/obo/CL_0005023`
#### Removed
- [branchiomotor neuron](http://purl.obolibrary.org/obo/CL_0005023) SubClassOf [BFO_0000002](http://purl.obolibrary.org/obo/BFO_0000002) 



### brown adipocyte `http://purl.obolibrary.org/obo/CL_0000449`

#### Added
- [brown adipocyte](http://purl.obolibrary.org/obo/CL_0000449) [description](http://purl.org/dc/terms/description) "Brown fat cells, also known as brown adipocytes, form a type of adipose tissue that regulates energy by thermogenesis (heat production). These cells are rich in mitochondria, which lends them their characteristic brown colouring. Mitochondria are crucial to the brown fat cell's function since they contain an integral protein called uncoupling protein 1 (UCP1). This protein permits protons to leak across the mitochondrial inner membrane, disrupting the usual pathway of ATP production. Instead of generating ATP, the energy derived from glucose and fatty acids is dissipated as heat. This unique function makes brown fat cells an important biological tool for regulating body temperature, especially in infants and hibernating animals.
While historically believed to be primarily relevant in newborns, recent findings highlight a role of brown fat cells in adults as well. They provide significant health benefits by metabolising excess glucose in the body and burning unwanted fat, and thereby raising the possibilities of using brown fat cells therapeutically for conditions like obesity or diabetes. Furthermore, stimulation of brown fat activity in adults is linked with improved overall metabolic health and lower risk of metabolic syndrome.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1097/MED.0b013e328337a81f"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1042/BJ20130457"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1074/jbc.M000547200"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1007/164_2018_118"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.4161/adip.18951"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [brown adipocyte](http://purl.obolibrary.org/obo/CL_0000449) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0000449"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "brown adipocyte on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### brown preadipocyte `http://purl.obolibrary.org/obo/CL_0002335`

#### Added
- [brown preadipocyte](http://purl.obolibrary.org/obo/CL_0002335) [description](http://purl.org/dc/terms/description) "Brown preadipocytes are a type of precursor cell vital in the development and function of brown adipose tissue (BAT). These immature cells are capable of proliferating and differentiating into mature brown adipocytes. Brown preadipocytes are distinct in their thermogenic potential due to their capacity to become brown adipocytes, known for heat generation, which is especially crucial in maintaining body temperature in cold environments. These cells are primarily found in mammalian organisms, particularly those that need to adapt to lower temperatures. 
Functionally, brown preadipocytes play an instrumental role in the process of adipogenesis, which is the formation of adipose (fat-storing) tissue. When subjected to specific stimuli or under the appropriate conditions, these cells can differentiate into mature brown adipocytes. These adipocytes contain a high volume of mitochondria and have the unique ability to burn energy and release it as heat, a process known as thermogenesis. This unique energy expenditure process is chiefly due to the presence of a mitochondrial protein called uncoupling protein 1 (UCP1).
The study of brown preadipocytes and their subsequent conversion into mature brown adipocytes is of significant interest in tackling obesity and related metabolic disorders. Since their thermogenic activity contributes to energy expenditure, they provide an attractive target for approaches looking to increase energy consumption in the body. Emerging research suggests that enhancing the number or activity of brown adipocytes could be a possible avenue for therapeutic interventions against obesity and type 2 diabetes.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.3109/07853890.2010.535557"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.3389/fphys.2015.00004"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1152/physrev.00015.2003"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/nrendo.2016.136"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [brown preadipocyte](http://purl.obolibrary.org/obo/CL_0002335) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0002335"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "brown preadipocyte on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### brush cell of bronchus `http://purl.obolibrary.org/obo/CL_0002208`

#### Added
- [brush cell of bronchus](http://purl.obolibrary.org/obo/CL_0002208) [description](http://purl.org/dc/terms/description) "Brush cells of the bronchus, also known as bronchial brush cells or tuft cells, are specialized epithelial cells found in the respiratory tract, specifically within the bronchi. These are called 'brush cells' due to their distinctive appearance, characterized by their apical surface being enveloped by a dense collection of long, thin, rigid microvilli, often likened to bristles of a brush. Some sources also describe them as having a bottle-like shape, with an enlarged middle section that narrows towards the base.
The primary function of brush cells in the respiratory tract is chemosensation. They have the built-in capacity to sense changes in their chemical environments. Brush cells of the bronchi are equipped with a host of sensory receptors, including bitter-taste receptors, which allow them to detect and respond to a wide array of airborne substances. Several studies also indicate the potential roles these cells play in initiating signal transduction pathways that regulate local airway behaviors such as bronchoconstriction, mucus secretion and immune responses.
Recent research also suggests that brush cells of the bronchus could potentially play a role in immune surveillance. They might can react to harmful pathogenic invasions in the airway by releasing certain compounds that aid in attracting immune cells to the site of infection. An increase in the number of brush cells has also been reported in certain lung diseases, suggesting a possible involvement in pathological processes. Although the precise role of bronchial brush cells continue to be the subject of ongoing research, they contribute to maintaining the overall integrity and function of the bronchial epithelium.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1186/s12931-023-02570-8"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1172/JCI161439"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1164/rccm.200502-203WS"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [brush cell of bronchus](http://purl.obolibrary.org/obo/CL_0002208) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0002208"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "brush cell of bronchus on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### brush cell of trachea `http://purl.obolibrary.org/obo/CL_0002207`

#### Added
- [brush cell of trachea](http://purl.obolibrary.org/obo/CL_0002207) [description](http://purl.org/dc/terms/description) "The brush cell of the trachea is a specialized cell type located predominantly in the epithelial lining of the trachea. Brush cells, also called tuft cells, are named for their distinctive morphology - a broad base leading to a narrower apex, adorned with a tuft of rigid, long microvilli resembling a brush. The cells are columnar and among the smallest in the respiratory tract, yet their appearance sets them apart from the typical ciliated cells that make up the majority of the tracheal epithelium. 
Brush cells are believed to serve a role as chemosensory sentinels in the trachea, monitoring the biochemical environment. The &lsquo;brush&rsquo; or &lsquo;tuft&rsquo; serves a critical purpose in heightening the cell's sense of contact with its surroundings. Brush cells of the trachea are equipped with taste receptors and a taste transduction machinery, such as the cation channel TRPM5 (transient receptor potential melastatin 5). Following activation through bitter-taste receptors, brush cells initiate downstream signaling to sensory neurons which then leads to protective respiratory nerve reflexes. 
Recent research has also shown that tracheal brush cells not only detect pathogenic substances but also contribute to the antimicrobial host defense by releasing neuropeptides that ultimately direct an influx of innate immune cells.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1172/JCI161439"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1164/rccm.200502-203WS"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [brush cell of trachea](http://purl.obolibrary.org/obo/CL_0002207) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0002207"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "brush cell of trachea on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### brush cell of tracheobronchial tree `http://purl.obolibrary.org/obo/CL_0002075`

#### Added
- [brush cell of tracheobronchial tree](http://purl.obolibrary.org/obo/CL_0002075) [description](http://purl.org/dc/terms/description) "The brush cell of the tracheobronchial tree is a specialized type of epithelial cell situated in the lung's airways system. Characterized by their visually distinguishable brush-like apical surface, brush cells (also called tuft cells) are given their name due to the tuft of microvilli protruding into the lumen of the respiratory tract. The high electron density and presence of a prominent tubulovesicular system in the apical cytoplasm are other defining microscopic features. 
The primary function of brush cells is chemosensation within the tracheobronchial system, including the trachea, bronchi and bronchioles. They have been identified as the pulmonary equivalent of taste receptor cells, acting as solitary chemosensory cells that serve as the primary line of defense and alarm signal against hazardous inhaled substances. Brush cells in the tracheobronchial tree express taste receptors and a taste transduction machinery which allow them to detect specific chemical signals from bacteria or environmental toxins and signal a downstream response such as protective reflexes like coughing and increased mucus secretion.
Brush cells also act as physical barriers to help filtrate and clean the inhaled air, thus making an essential contribution to setting up the so-called mucociliary escalator. The tubulovesicular system within brush cells likely plays a role in transcytosis, which involves the movement of various substances across the epithelial barrier. Overall, the brush or tuft cells form an integral part of the airway's innate immune system, helping maintain a healthy and functional tracheobronchial tree.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1186/s12931-023-02570-8"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1073/pnas.101941810"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1164/rccm.200502-203WS"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [brush cell of tracheobronchial tree](http://purl.obolibrary.org/obo/CL_0002075) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0002075"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "brush cell of tracheobronchial tree on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### capillary endothelial cell `http://purl.obolibrary.org/obo/CL_0002144`

#### Added
- [capillary endothelial cell](http://purl.obolibrary.org/obo/CL_0002144) [description](http://purl.org/dc/terms/description) "Capillary endothelial cells are particularly essential in the formation and function of capillaries - the smallest and most widespread blood vessels in the body. They play two primary roles: barrier function and transport function. Acting as barriers, they strictly control the passage of nutrients, gases, biological molecules, and cells between the blood and the surrounding tissues. This function not only provides a protective shield to internal organs against potentially harmful substances but also regulates the fluid and solute exchange. In terms of transport, these cells facilitate the diffusion of oxygen and essential nutrients from the bloodstream into the tissues. Concurrently, they allow waste products like carbon dioxide to pass from tissues into the bloodstream for removal, making these cells exceptionally instrumental in maintaining the body&rsquo;s homeostasis.
Moreover, capillary endothelial cells are also actively involved in numerous other physiological processes such as inflammation, blood clotting, and angiogenesis. During inflammation, they can specifically express and secrete molecules to attract white blood cells to sites of damage. Similarly, in blood clotting, they produce substances that prevent blood cells from sticking to the vessel wall and deter the formation of unnecessary clots. In angiogenesis, they proliferate and organize the formation of new blood vessels, crucial for wound healing and embryonic development. The disruption of their function contributes to pathological conditions like edema, thrombosis, and several cardiovascular diseases.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1097/01.CCM.0000057847.32590.C1"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.3390/ijms20184411"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1152/ajpheart.00704.2007"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [capillary endothelial cell](http://purl.obolibrary.org/obo/CL_0002144) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0002144"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "capillary endothelial cell on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### cardiac blood vessel endothelial cell `http://purl.obolibrary.org/obo/CL_0010006`

#### Added
- [cardiac blood vessel endothelial cell](http://purl.obolibrary.org/obo/CL_0010006) [description](http://purl.org/dc/terms/description) "Cardiac blood vessel endothelial cells are a specialized type of endothelial cells that constitute the innermost layer, or endothelium, of blood vessels within the heart, namely the arteries, veins and capillaries. They function as a semi-permeable barrier, regulating the exchange of nutrients, ions, and small molecules between the blood and surrounding cardiac tissues. Additionally, they control the trafficking of white blood cells into the myocardium, playing a vital role in immune surveillance and response to pathogens.
An important feature of cardiac blood vessel endothelial cells is their contribution to vessel tone and blood pressure regulation. Endothelial cells release nitric oxide, a potent vasodilator, which helps to modulate blood vessel diameter as per the cardiac demands and maintain homeostasis. Moreover, these cells are active participants in the blood clotting process by expressing a variety of proteins and receptors that interact with platelets and plasma clotting factors.
In pathological conditions, the function of cardiac blood vessel endothelial cells can change dramatically, leading to various cardiovascular diseases. In response to injury or disease, these cells can proliferate and migrate contributing to vessel repair or aberrant remodelling, promote inflammation, and contribute to the formation of atherosclerotic plaques. Their ability to switch from a quiescent, anticoagulant phenotype to a more active, procoagulant state is a principal factor in thrombosis.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1146/annurev.physiol.68.040104.124629"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1161/circulationaha.105.602532"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1161/ATVBAHA.117.310549"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/s41569-022-00770-1"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.ajpath.2020.10.001"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [cardiac blood vessel endothelial cell](http://purl.obolibrary.org/obo/CL_0010006) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0010006"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "cardiac blood vessel endothelial cell on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### cardiac neuron `http://purl.obolibrary.org/obo/CL_0010022`

#### Added
- [cardiac neuron](http://purl.obolibrary.org/obo/CL_0010022) [description](http://purl.org/dc/terms/description) "Cardiac neurons are a specialized type of neuron found within the heart tissue. They are the primary constituents of the intrinsic cardiac nervous system, also known as the cardiac plexus or intrinsic cardiac ganglia, which is responsible for the regulation of the heart's rhythmic contractions and overall functioning. This group of neurons is integral to the autonomic control of the heart and plays a crucial role in modulating cardiac activity, thereby playing an essential role in maintaining cardiovascular homeostasis.
These neurons are responsible for both the sympathetic and parasympathetic regulation of the heart&rsquo;s function. In terms of sympathetic control, cardiac neurons respond to the release of norepinephrine, which increases heart rate, strength of contractions, and blood pressure. Conversely, with regards to parasympathetic control, these neurons respond to the neurotransmitter acetylcholine, which decreases heart rate, blood pressure, and the strength of contractions. Thus, the balance between these sympathetic and parasympathetic inputs regulated by cardiac neurons is essential for normal heart function.
Cardiac neurons also partake in the transmission of pain signals during conditions such as myocardial ischemia, contributing to the symptomatology of ischemic heart disease. Additionally, abnormal function or pathological alterations in these neurons could lead to various cardiac dysfunctions, including arrhythmias and heart failure.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.3390/jcdd7040054"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.autneu.2016.08.006"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1152/ajpregu.00183.2004"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1002/cphy.c150046"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1161/01.RES.84.8.921"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [cardiac neuron](http://purl.obolibrary.org/obo/CL_0010022) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0010022"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "cardiac neuron on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### caudal ganglionic eminence derived interneuron `http://purl.obolibrary.org/obo/CL_4023064`
#### Removed
- [caudal ganglionic eminence derived interneuron](http://purl.obolibrary.org/obo/CL_4023064) [has_exact_synonym](http://www.geneontology.org/formats/oboInOwl#hasExactSynonym) "caudal ganglionic eminence derived GABAergic cortical interneuron"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:20130169"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [caudal ganglionic eminence derived interneuron](http://purl.obolibrary.org/obo/CL_4023064) [has_related_synonym](http://www.geneontology.org/formats/oboInOwl#hasRelatedSynonym) "CGE cortical interneuron"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [has_synonym_type](http://www.geneontology.org/formats/oboInOwl#hasSynonymType) [OMO_0003000](http://purl.obolibrary.org/obo/OMO_0003000) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:20130169"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [caudal ganglionic eminence derived interneuron](http://purl.obolibrary.org/obo/CL_4023064) [label](http://www.w3.org/2000/01/rdf-schema#label) "caudal ganglionic eminence derived cortical interneuron"^^[string](http://www.w3.org/2001/XMLSchema#string) 

#### Added
- [caudal ganglionic eminence derived interneuron](http://purl.obolibrary.org/obo/CL_4023064) [has_exact_synonym](http://www.geneontology.org/formats/oboInOwl#hasExactSynonym) "caudal ganglionic eminence derived GABAergic interneuron"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:20130169"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [caudal ganglionic eminence derived interneuron](http://purl.obolibrary.org/obo/CL_4023064) [has_related_synonym](http://www.geneontology.org/formats/oboInOwl#hasRelatedSynonym) "CGE interneuron"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [has_synonym_type](http://www.geneontology.org/formats/oboInOwl#hasSynonymType) [OMO_0003000](http://purl.obolibrary.org/obo/OMO_0003000) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:20130169"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [caudal ganglionic eminence derived interneuron](http://purl.obolibrary.org/obo/CL_4023064) [label](http://www.w3.org/2000/01/rdf-schema#label) "caudal ganglionic eminence derived interneuron"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### central nervous system neuron `http://purl.obolibrary.org/obo/CL_2000029`

#### Added
- [central nervous system neuron](http://purl.obolibrary.org/obo/CL_2000029) [RO_0002175](http://purl.obolibrary.org/obo/RO_0002175) [NCBITaxon_9606](http://purl.obolibrary.org/obo/NCBITaxon_9606) 

- [central nervous system neuron](http://purl.obolibrary.org/obo/CL_2000029) [in_subset](http://www.geneontology.org/formats/oboInOwl#inSubset) [human_reference_atlas](http://purl.obolibrary.org/obo/cl/cl-base#human_reference_atlas) 


### centrilobular region hepatocyte `http://purl.obolibrary.org/obo/CL_0019029`
#### Removed
- [centrilobular region hepatocyte](http://purl.obolibrary.org/obo/CL_0019029) EquivalentTo [hepatocyte](http://purl.obolibrary.org/obo/CL_0000182) and ([BFO_0000050](http://purl.obolibrary.org/obo/BFO_0000050) some [UBERON_8400008](http://purl.obolibrary.org/obo/UBERON_8400008)) 

- [centrilobular region hepatocyte](http://purl.obolibrary.org/obo/CL_0019029) SubClassOf [BFO_0000050](http://purl.obolibrary.org/obo/BFO_0000050) some [UBERON_8400008](http://purl.obolibrary.org/obo/UBERON_8400008) 

#### Added
- [centrilobular region hepatocyte](http://purl.obolibrary.org/obo/CL_0019029) [description](http://purl.org/dc/terms/description) "Centrilobular region hepatocytes, also referred to as perivenous or pericentral hepatocytes, are specialized cells that constitute the largest component of the liver. These cells are strategically arranged around the central vein of a liver lobule, forming the zone 3 region. This unique location is critically significant for their distinctive functional capabilities, making them different from periportal hepatocytes of zone 1 and midzonal hepatocytes of zone 2.
One of the primary functions of centrilobular region hepatocytes is linked to the body&rsquo;s metabolic detoxification processes. These cells are predominantly involved in xenobiotic biotransformation, assisting with the metabolism and excretion of foreign substances, such as drugs or harmful toxins. This role is facilitated by their ability to express high levels of Cytochrome P450 enzymes, which are pivotal in phase I detoxification reactions. Additionally, centrilobular region hepatocytes also play a crucial role in bile acid synthesis and the biotransformation of endogenous compounds such as steroids, fatty acids, and bilirubin.
Being situated in zone 3, centrilobular hepatocytes are more susceptible to hypoxic injury than other hepatocytes due to the lower oxygen supply. Furthermore, due to their central role in detoxification, they are particularly prone to drug-induced hepatic damage.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:/10.3389/fphys.2021.732929/full"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/s41575-019-0134-x"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1007/s10620-006-3174-3"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1002/hep.510310201"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1042/bj3290017"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [centrilobular region hepatocyte](http://purl.obolibrary.org/obo/CL_0019029) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0019029"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "centrilobular region hepatocyte on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [centrilobular region hepatocyte](http://purl.obolibrary.org/obo/CL_0019029) EquivalentTo [hepatocyte](http://purl.obolibrary.org/obo/CL_0000182) and ([BFO_0000050](http://purl.obolibrary.org/obo/BFO_0000050) some [UBERON_8400007](http://purl.obolibrary.org/obo/UBERON_8400007)) 

- [centrilobular region hepatocyte](http://purl.obolibrary.org/obo/CL_0019029) SubClassOf [BFO_0000050](http://purl.obolibrary.org/obo/BFO_0000050) some [UBERON_8400007](http://purl.obolibrary.org/obo/UBERON_8400007) 


### cerebellar Golgi cell `http://purl.obolibrary.org/obo/CL_0000119`

#### Added
- [cerebellar Golgi cell](http://purl.obolibrary.org/obo/CL_0000119) [description](http://purl.org/dc/terms/description) "Cerebellar Golgi cells are a specific type of inhibitory interneuron found within the granular layer of the cerebellum, a major part of the vertebrate brain responsible for motor control, learning, and cognitive functions. Named after the Italian scientist Camillo Golgi, these cells play a unique and important role in modulating electrical signals in the complex cerebellar neural network.
The distinctive feature of the cerebellar Golgi cells is their dendritic arbor that extends throughout the molecular layer of the cerebellum, in line with the axons of the granule cells. The axons of Golgi cells primarily synapse with the dendrites of granule cells, generating a synaptic feedback loop that plays a vital role in regulating the intensity and timing of signal transmission. They also form synapses with mossy fiber terminals, which provide primary input to the cerebellar cortex.
Functionally, cerebellar Golgi cells serve as &quot;gatekeepers&quot; of the cerebellar cortex. They have a strong inhibitory influence on granule cells, modulating the information flow to the Purkinje cells, the main output neurons of the cerebellar cortex. Thus, Golgi cells play a critical role in controlling the input-output relationship within the cerebellum, shaping the timing and strength of signals originating from various sensory and motor areas of the cerebral cortex. This regulatory function is instrumental for the fine-tuning of motor coordination, timing and learning. Thus, alterations in Golgi cell functions can contribute to a range of neurological and neuropsychiatric disorders such as ataxias and cognitive deficits.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1113/jphysiol.2010.189605"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.neuron.2011.10.030"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.3389/fncir.2013.00093"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1126/science.1215101"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/S0896-6273(00)81126-5"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [cerebellar Golgi cell](http://purl.obolibrary.org/obo/CL_0000119) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0000119"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "cerebellar Golgi cell on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### cerebellar granule cell `http://purl.obolibrary.org/obo/CL_0001031`

#### Added
- [cerebellar granule cell](http://purl.obolibrary.org/obo/CL_0001031) [description](http://purl.org/dc/terms/description) "The cerebellar granule cell is a predominant type of neuron located in the granular layer of the cerebellum. These cells are among the smallest neurons in the brain, yet they constitute the bulk of the neural cells in the cerebellum, making cerebellar granule cells the most populous type of neuron in mammals. The granule cells of the cerebellum play an essential role in the functioning of the cerebellar cortex by participating in a complex neuronal network that results in coordination and fine-tuning of motor activities.
Initiating and controlling movement is the primary functional role of cerebellar granule cells. They achieve this through highly sophisticated neuronal networking. Each granule cell gives rise to an axon that ascends into the molecular layer of the cerebellar cortex, where it bifurcates into T-shape to form parallel fibers that run perpendicular to the Purkinje cell dendrites. This structure allows a single granule cell to make synaptic contact with hundreds of Purkinje cells and thereby influence their output. Cerebellar granule cells contribute to the generation of movement by transforming unstructured input from mossy fibers into organized output to the Purkinje cells.
Understanding the role of cerebellar granule cells in neural plasticity is an emerging area of study. Recent research suggests that cerebellar granule cells may be involved in learning and memory processes, particularly those related to motor learning. They display unique synaptic plasticity, equipping them with the ability to adjust their response over time, which aids in the learning and storage of motor patterns.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/nn.4531"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/B978-0-444-63356-9.00002-9"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1152/jn.1998.80.5.2521"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/nrn3312"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.3389/fncir.2020.611841"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [cerebellar granule cell](http://purl.obolibrary.org/obo/CL_0001031) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0001031"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "cerebellar granule cell on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### cerebral cortex GABAergic interneuron `http://purl.obolibrary.org/obo/CL_0010011`

#### Added
- [cerebral cortex GABAergic interneuron](http://purl.obolibrary.org/obo/CL_0010011) [description](http://purl.org/dc/terms/description) "The cerebral cortex GABAergic interneuron is a type of neuron found primarily in the brain's cerebral cortex. These neurons are distinguished by their primary inhibitory neurotransmitter, gamma-aminobutyric acid (GABA) which plays a crucial role in managing the excitability of neurons. Interneurons, in general, function as primary regulators of data flow in the nervous system. They manage the communication between sensory and motor neurons, influencing our responses to environmental stimuli.
The cerebral cortex GABAergic interneurons are particularly essential for the proper functioning of cerebral cortical networks. These neurons generate diverse inhibition patterns, creating a balance between excitation and inhibition in neuronal networks. This balance is fundamental in ensuring normal cortical processes such as sensory perception, memory, and cognitive functions. The diverse range of GABAergic interneurons contributes to generating distinct cortical rhythms which are linked with specific behavioral states, mental health, and neurological diseases.
Alterations or disruptions in the function of cerebral cortex GABAergic interneurons can have serious implications, and have been associated with various neurological and psychiatric conditions such as epilepsy, schizophrenia, and autism. The heterogeneity of cerebral cortex GABAergic interneurons, in terms of their morphology, connectivity, and physiology, contribute greatly to the functional diversity of the cerebral cortex.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.conb.2023.102703"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/nrn2402"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/nrn3155"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.neuron.2013.01.011"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.neuron.2018.10.009"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [cerebral cortex GABAergic interneuron](http://purl.obolibrary.org/obo/CL_0010011) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0010011"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "cerebral cortex GABAergic interneuron on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### cerebral cortex neuron `http://purl.obolibrary.org/obo/CL_0010012`
#### Removed
- [cerebral cortex neuron](http://purl.obolibrary.org/obo/CL_0010012) SubClassOf [BFO_0000002](http://purl.obolibrary.org/obo/BFO_0000002) 

#### Added
- [cerebral cortex neuron](http://purl.obolibrary.org/obo/CL_0010012) [description](http://purl.org/dc/terms/description) "A cerebral cortex neuron, also known as a cortical neuron, is a unique type of cell found in the cerebral cortex, which is the outer covering of the brain. This region is critical to cognitive functions, sensory input processes, memory formation, attention, and consciousness. These neurons are one of the most abundant cell type in the cortex and allow for the intricate capabilities of the human brain due to their diverse characteristics and complex connectivity.
The primary function of cortical neurons is to manage and interpret the sensory input being received by the brain, generating suitable motor output responses. They play a critical role in processing information and communication within the brain. These neurons are involved in the exchange of direct synaptic communication with thousands of other neurons. Due to the wide extent of their dendritic and axonal arbors, these neurons present a broad receptive field for incoming information and an expansive projecting field to convey integrated information.
Cortical neurons further define the functional structure of the cerebral cortex by their type and connection. They can be broken down into two categories: excitatory projection neurons and inhibitory interneurons. The excitatory projection neurons are the principal neurons of the cerebral cortex and utilize glutamate as a neurotransmitter. They form the connections between the cerebral cortex and other brain regions, hence their tag as &quot;projection&quot; neurons. Conversely, inhibitory interneurons are local to the cerebral cortex and use gamma-aminobutyric acid (GABA) as a neurotransmitter, serving to regulate the activity of the projection neurons. Through these intricate interconnections, cerebral cortex neurons play a significant role in the higher cognitive functions of the human brain.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.tics.2022.08.012"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1186/s13064-018-0104-y"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/nrn2151"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/s41586-021-04230-7"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.neuron.2011.01.015"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [cerebral cortex neuron](http://purl.obolibrary.org/obo/CL_0010012) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0010012"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "cerebral cortex neuron on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### chandelier pvalb GABAergic interneuron `http://purl.obolibrary.org/obo/CL_4023036`
#### Removed
- [chandelier pvalb GABAergic interneuron](http://purl.obolibrary.org/obo/CL_4023036) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A transcriptomically distinct pvalb GABAergic cortical interneuron that is recognizable by the straight terminal axonal 'cartridges' of vertically oriented strings of synaptic boutons. Chandelier PV cells' boutons target exclusively the axon initial segment (AIS) of pyramidal cells, with a single cell innervating hundreds of pyramidal cells in a clustered manner. The standard transcriptomic reference data for this cell type can be found on the CellxGene census under the collection: 'Transcriptomic cytoarchitecture reveals principles of human neocortex organization', dataset: 'Supercluster: CGE-derived interneurons', Author Categories: 'CrossArea_subclass', clusters Chandelier."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:27477017"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "https://cellxgene.cziscience.com/e/9c63201d-bfd9-41a8-bbbc-18d947556f3d.cxg/"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [chandelier pvalb GABAergic interneuron](http://purl.obolibrary.org/obo/CL_4023036) [label](http://www.w3.org/2000/01/rdf-schema#label) "chandelier pvalb GABAergic cortical interneuron"^^[string](http://www.w3.org/2001/XMLSchema#string) 

#### Added
- [chandelier pvalb GABAergic interneuron](http://purl.obolibrary.org/obo/CL_4023036) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A transcriptomically distinct pvalb GABAergic interneuron that is recognizable by the straight terminal axonal 'cartridges' of vertically oriented strings of synaptic boutons. Chandelier PV cells' boutons target exclusively the axon initial segment (AIS) of pyramidal cells, with a single cell innervating hundreds of pyramidal cells in a clustered manner."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:27477017"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "https://cellxgene.cziscience.com/e/9c63201d-bfd9-41a8-bbbc-18d947556f3d.cxg/"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [chandelier pvalb GABAergic interneuron](http://purl.obolibrary.org/obo/CL_4023036) [label](http://www.w3.org/2000/01/rdf-schema#label) "chandelier pvalb GABAergic interneuron"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### chandelier pvalb GABAergic interneuron (Homo sapiens) `http://purl.obolibrary.org/obo/CL_4072046`

#### Added
- Class: [chandelier pvalb GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072046) 

- [chandelier pvalb GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072046) [IAO_0000028](http://purl.obolibrary.org/obo/IAO_0000028) "chandelier PV interneuron (Homo sapiens)"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [chandelier pvalb GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072046) [RO_0002175](http://purl.obolibrary.org/obo/RO_0002175) [NCBITaxon_9606](http://purl.obolibrary.org/obo/NCBITaxon_9606) 

- [chandelier pvalb GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072046) [contributor](http://purl.org/dc/terms/contributor) "https://orcid.org/0000-0002-5507-2103"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [chandelier pvalb GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072046) [date](http://purl.org/dc/terms/date) "2025-09-29T10:07:47Z"^^[dateTime](http://www.w3.org/2001/XMLSchema#dateTime) 

- [chandelier pvalb GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072046) [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "ILX:0107356"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [chandelier pvalb GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072046) [has_exact_synonym](http://www.geneontology.org/formats/oboInOwl#hasExactSynonym) "Chandelier PV (Homo sapiens)"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [chandelier pvalb GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072046) [id](http://www.geneontology.org/formats/oboInOwl#id) "CL:4072046"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [chandelier pvalb GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072046) [in_subset](http://www.geneontology.org/formats/oboInOwl#inSubset) [BDS_subset](http://purl.obolibrary.org/obo/cl/cl-base#BDS_subset) 

- [chandelier pvalb GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072046) [in_subset](http://www.geneontology.org/formats/oboInOwl#inSubset) [cellxgene_subset](http://purl.obolibrary.org/obo/cl/cl-base#cellxgene_subset) 

- [chandelier pvalb GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072046) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A transcriptomically definined chandelierl pvalb GABAergic interneuron in humans. The standard transcriptomic reference data for this cell type can be found on the CellxGene census under the collection: 'Transcriptomic cytoarchitecture reveals principles of human neocortex organization', dataset: 'Supercluster: CGE-derived interneurons', Author Categories: 'CrossArea_subclass', clusters Chandelier."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:27477017"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "https://cellxgene.cziscience.com/e/9c63201d-bfd9-41a8-bbbc-18d947556f3d.cxg/"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [chandelier pvalb GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072046) [comment](http://www.w3.org/2000/01/rdf-schema#comment) "The marker set GPR149, CA8 can identify the Human cell type chandelier pvalb GABAergic cortical interneuron in the neocortex with a confidence of 0.799748111 (NS-Forest FBeta value)."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "https://doi.org/10.5281/zenodo.11165918"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [chandelier pvalb GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072046) [label](http://www.w3.org/2000/01/rdf-schema#label) "chandelier pvalb GABAergic interneuron (Homo sapiens)"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [chandelier pvalb GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072046) SubClassOf [chandelier pvalb GABAergic interneuron](http://purl.obolibrary.org/obo/CL_4023036) 

- [chandelier pvalb GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072046) SubClassOf [RO_0002162](http://purl.obolibrary.org/obo/RO_0002162) some [NCBITaxon_9606](http://purl.obolibrary.org/obo/NCBITaxon_9606) 


### cholinergic neuron `http://purl.obolibrary.org/obo/CL_0000108`

#### Added
- [cholinergic neuron](http://purl.obolibrary.org/obo/CL_0000108) [RO_0002175](http://purl.obolibrary.org/obo/RO_0002175) [NCBITaxon_9606](http://purl.obolibrary.org/obo/NCBITaxon_9606) 

- [cholinergic neuron](http://purl.obolibrary.org/obo/CL_0000108) [in_subset](http://www.geneontology.org/formats/oboInOwl#inSubset) [human_reference_atlas](http://purl.obolibrary.org/obo/cl/cl-base#human_reference_atlas) 


### choroid plexus epithelial cell `http://purl.obolibrary.org/obo/CL_0000706`

#### Added
- [choroid plexus epithelial cell](http://purl.obolibrary.org/obo/CL_0000706) [description](http://purl.org/dc/terms/description) "The choroid plexus epithelial cell is one of the crucial cell types found in the choroid plexus, a capillary-rich tissue located in the brain's ventricles. These cells are distinctly cuboidal or columnar, organized in a monolayer pattern with tight junctions that form a barrier between the cerebrospinal fluid inside the ventricles and the blood circulating in the choroid plexus. The polarity of these cells is highly specialized. The apical (ventricular) side, facing the cerebrospinal fluid, is characterized by numerous microvilli and cilia, while the basolateral (vascular) side is located adjacent to the choroidal interstitium. 
The main function of the choroid plexus epithelial cells is to produce cerebrospinal fluid, an essential fluid that provides mechanical protection, nutrient delivery, and waste removal to and from the brain and spinal cord. These cells actively transport ions, glucose, and other molecules from the blood into the ventricles, driving the osmotic movement of water and the formation of cerebrospinal fluid. The cerebrospinal fluid turnover rate is remarkably high, with the choroid plexus epithelial cells producing approximately 500 mL each day.
Another noteworthy function of choroid plexus epithelial cells is their role as a constituent of the blood-cerebrospinal fluid barrier. Their organized pattern and tightly connected structure play a key role in strictly controlling the entry of substances from the blood to the  cerebrospinal fluid, thus protecting the central nervous system from potentially harmful substances and fluctuations in blood composition. As such, these cells also serve in the transport of certain molecules into the central nervous system and in the clearance of waste products from the  cerebrospinal fluid. Emerging evidence also shows that these cells contribute to brain immune responses acting as immunological sentinels.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1093/jnen/nlv030"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "https://www.ncbi.nlm.nih.gov/books/NBK538156/#:~:text=The%20choroid%20plexus%20has%20a,canal%20of%20the%20spinal%20cord."^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/nrn3921"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1523/JNEUROSCI.3258-13.2013"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1152/physrev.00004.2013"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [choroid plexus epithelial cell](http://purl.obolibrary.org/obo/CL_0000706) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0000706"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "choroid plexus epithelial cell on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### choroidal cell of the eye `http://purl.obolibrary.org/obo/CL_0000348`
#### Removed
- [choroidal cell of the eye](http://purl.obolibrary.org/obo/CL_0000348) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A structural cell that is part of optic choroid."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "GOC:add"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [choroidal cell of the eye](http://purl.obolibrary.org/obo/CL_0000348) EquivalentTo [obsolete structural cell](http://purl.obolibrary.org/obo/CL_0000293) and ([BFO_0000050](http://purl.obolibrary.org/obo/BFO_0000050) some [UBERON_0001776](http://purl.obolibrary.org/obo/UBERON_0001776)) 

- [choroidal cell of the eye](http://purl.obolibrary.org/obo/CL_0000348) SubClassOf [obsolete structural cell](http://purl.obolibrary.org/obo/CL_0000293) 

#### Added
- [choroidal cell of the eye](http://purl.obolibrary.org/obo/CL_0000348) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A cell that is part of optic choroid."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "GOC:add"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [choroidal cell of the eye](http://purl.obolibrary.org/obo/CL_0000348) EquivalentTo [cell](http://purl.obolibrary.org/obo/CL_0000000) and ([BFO_0000050](http://purl.obolibrary.org/obo/BFO_0000050) some [UBERON_0001776](http://purl.obolibrary.org/obo/UBERON_0001776)) 


### circulating fibrocyte `http://purl.obolibrary.org/obo/CL_0000135`
#### Removed
- [circulating fibrocyte](http://purl.obolibrary.org/obo/CL_0000135) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "An inactive fibroblast; cytoplasm is sparse, endoplasmic reticulum is scanty with flattened nucleus. Term used by some histologists; when fibroblasts become relatively inactive in fiber formation. However, this cell has the potential for fibrogenesis in quiescent connective tissue of the adult, as well as during development, other histologists prefer to use the term fibroblast in all circumstances. These cells represent ~0.5% of peripheral blood leukocytes."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "http://en.wikipedia.org/wiki/Fibrocyte"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:15010326"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:17607298"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:11641248"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "GOC:dsd"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:18222966"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "ISBN:0412046911"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "ISBN:0517223651"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "GOC:tfm"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:9551999"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:8790603"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [circulating fibrocyte](http://purl.obolibrary.org/obo/CL_0000135) [label](http://www.w3.org/2000/01/rdf-schema#label) "fibrocyte"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [circulating fibrocyte](http://purl.obolibrary.org/obo/CL_0000135) EquivalentTo [stromal cell](http://purl.obolibrary.org/obo/CL_0000499) and ([RO_0002215](http://purl.obolibrary.org/obo/RO_0002215) some [GO_0002495](http://purl.obolibrary.org/obo/GO_0002495)) and ([RO_0002215](http://purl.obolibrary.org/obo/RO_0002215) some [GO_0045766](http://purl.obolibrary.org/obo/GO_0045766)) 

- [circulating fibrocyte](http://purl.obolibrary.org/obo/CL_0000135) SubClassOf [RO_0002202](http://purl.obolibrary.org/obo/RO_0002202) some [fibroblast](http://purl.obolibrary.org/obo/CL_0000057) 

#### Added
- [circulating fibrocyte](http://purl.obolibrary.org/obo/CL_0000135) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A bone marrow-derived cell that predominantly develops from myeloid lineage-restricted progenitor cells and circulates in peripheral blood, characterized by co-expression of hematopoietic markers CD45 and CD34 along with stromal markers including collagen type I in both humans and mice (Chesney et al., 1997; Blakaj and Bucala, 2012). This spindle-shaped cell exhibits a unique dual identity, functionally bridging immune and stromal compartments with characteristics of both monocytes and fibroblasts. Predominantly arising from circulating monocytes, it migrates to injury sites via chemokine receptors and participates in wound repair through extracellular matrix deposition, cytokine secretion, and antigen presentation via MHC class II. As mesenchymal progenitors, fibrocytes can differentiate into fibroblasts, myofibroblasts, and adipocytes, contributing to both physiological repair and pathological fibrosis (Blakaj and Bucala, 2012)."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:31473260"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "doi:/10.1186/1755-1536-5-S1-S6"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:32084275"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:29286323"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "GOC:dsd"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "GOC:tfm"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:20303382"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:9177213"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:20305780"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [circulating fibrocyte](http://purl.obolibrary.org/obo/CL_0000135) [has_narrow_synonym](http://www.geneontology.org/formats/oboInOwl#hasNarrowSynonym) "monocyte-derived fibrocyte"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:30826294"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [circulating fibrocyte](http://purl.obolibrary.org/obo/CL_0000135) [label](http://www.w3.org/2000/01/rdf-schema#label) "circulating fibrocyte"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [circulating fibrocyte](http://purl.obolibrary.org/obo/CL_0000135) SubClassOf [progenitor cell](http://purl.obolibrary.org/obo/CL_0011026) 

- [circulating fibrocyte](http://purl.obolibrary.org/obo/CL_0000135) SubClassOf [RO_0002202](http://purl.obolibrary.org/obo/RO_0002202) some [myeloid lineage restricted progenitor cell](http://purl.obolibrary.org/obo/CL_0000839) 

- [circulating fibrocyte](http://purl.obolibrary.org/obo/CL_0000135) SubClassOf [RO_0002215](http://purl.obolibrary.org/obo/RO_0002215) some [GO_0042060](http://purl.obolibrary.org/obo/GO_0042060) 


### club cell `http://purl.obolibrary.org/obo/CL_0000158`

#### Added
- [club cell](http://purl.obolibrary.org/obo/CL_0000158) [description](http://purl.org/dc/terms/description) "Club cells, originally termed Clara cells, are specialized nonciliated epithelial cells primarily located in the bronchioles, or small airways, of the lungs. Compared to other cells in the lung epithelium, they are distinguished by their characteristic dome shape often marked by short, microvilli-like projections (but no cilia). Underneath their dome-like apex, they possess a large amount of cytoplasm containing numerous granules, and they have a round, centrally located nucleus. Their name 'club' was adopted to signify their appearance, which resembles a club or goblet.
Club cells perform various crucial roles in maintaining the homeostasis of the respiratory tract. Among their functions is the production and secretion of a variety of proteins, including the club cell secretory protein (CCSP), which plays a role in the immune response of the lungs, particularly, in mitigating inflammation. The secretion of surfactant proteins by these cells also contributes to reducing surface tension in the alveoli, thus facilitating the process of gas exchange. Besides helping to defend against oxidative stress and inflammatory response, surfactants also aid in preventing the air sacs in the lungs from collapsing.
Another vital role of club cells is their capacity to self-proliferate and differentiate into ciliated cells or basal cells, highlighting their regenerative capacity. This quality is especially critical for maintaining the integrity of the epithelial layer in the lung following any injury or inflammation, aiding the lung tissue to recover and restore its function. Furthermore, it is speculated that club cells might minimize the adverse impact of harmful substances in the air we inhale, thanks to a range of cytochrome P450 enzymes present in them. These enzymes assist in detoxifying harmful substances, a protective mechanism that helps preserve the overall health of the lungs.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/s41467-022-32052-2"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.2147/JIR.S284800"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1152/ajplung.00192.2022"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1378/chest.14-3171"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/s41385-020-00370-7"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [club cell](http://purl.obolibrary.org/obo/CL_0000158) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0000158"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "club cell on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### common lymphoid progenitor `http://purl.obolibrary.org/obo/CL_0000051`

#### Added
- [common lymphoid progenitor](http://purl.obolibrary.org/obo/CL_0000051) [description](http://purl.org/dc/terms/description) "The common lymphoid progenitor (CLP) arises from the multipotential hematopoietic stem cell in the bone marrow. Hematopoietic stem cells have the unique capability of developing into any type of blood cell depending on the body's requirements. In contrast, common lymphoid progenitors are already committed to the lymphoid lineage but are still oligopotent with the capability to further differentiate into various types of lymphoid cells. 
In mice, cells in the common lymphoid progenitor population in the bone marrow can be identified by their Lin- IL7r&alpha;+ c-Kit+ Sca-1+ cell surface phenotype. The IL-7 receptor is first expressed on common lymphoid progenitor cells and distinguishes them from primitive common myeloid progenitors, and IL-7 is thought to play a critical role in the downstream development and homeostasis of immune cells.
CLPs give rise to B lymphocytes, T lymphocytes, dendritic cells, and natural killer cells, all of which play a crucial and distinct role in immune responses. B and T lymphocytes are responsible for adaptive immunity, allowing the body to recognize and remember specific pathogens to fight them off more effectively in the future. B lymphocytes are responsible for producing antibodies, while T lymphocytes either help regulate immune responses or directly kill infected cells. Natural Killer cells are part of the body&rsquo;s innate immune response, providing immediate, nonspecific defense against pathogens. They are able to interact with infected or malignant cells and induce controlled cell death, or apoptosis, effectively eliminating the threat to bodily health. Dendritic cells link the innate and adaptive immune systems by presenting antigens to T cells, promoting an adaptive response.
Thus, common lymphoid progenitors provide the basis for the body's strategic and layered defense against varied and evolving external threats, making them an essential part of the immune system.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1182/blood-2009-08-236398"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/s0092-8674(00)80453-5"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1111/j.1600-065X.2010.00963.x"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/ni.3168"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.3389/fimmu.2021.747324"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [common lymphoid progenitor](http://purl.obolibrary.org/obo/CL_0000051) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0000051"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "common lymphoid progenitor on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### common myeloid progenitor `http://purl.obolibrary.org/obo/CL_0000049`

#### Added
- [common myeloid progenitor](http://purl.obolibrary.org/obo/CL_0000049) [description](http://purl.org/dc/terms/description) "The common myeloid progenitor (CMP) is a multipotent cell that serves as a crucial intermediary within the overall framework of hematopoietic cell differentiation, which is the process of blood cell production. This part of the hematopoiesis process leads to the formation of myeloid lineage cells, which includes granulocytes (neutrophils, eosinophils, and basophils), monocytes, erythrocytes (red blood cells), and megakaryocytes (platelet-forming cells). 
The primary function of common myeloid progenitors is differentiation and proliferation into various types of mature blood cells, thereby providing a critical contribution to maintaining homeostasis within the body's immune and circulatory systems. Single-cell analyses have shown that the CMP cell population is heterogeneous with at least two distinct subpopulations, characterized by different cell surface markers and likely already committed to downstream differentiation into either the granulocyte/macrophage lineage or the megakaryocyte/erythrocyte lineage-restricted progenitor. 
The destiny of a CMP is influenced by numerous cytokines and growth factors in its environment that promote differentiation along specific lineages. For example, IL-3 and GM-CSF stimulate the production of eosinophils, which are essential for the body&rsquo;s defense against allergens and help fight fungal and parasitic infections. 
By being able to differentiate into a wide range of different immune cells, common myeloid progenitors play a crucial role in the body's ability to respond to injuries, infections, and other physiological challenges.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/35004599"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1111/j.1600-065X.2010.00963.x"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/ni.3412"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.7554/eLife.60939"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "https://www.sciencedirect.com/topics/engineering/myeloid-progenitor-cell"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [common myeloid progenitor](http://purl.obolibrary.org/obo/CL_0000049) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0000049"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "common myeloid progenitor on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### conjunctival epithelial cell `http://purl.obolibrary.org/obo/CL_1000432`

#### Added
- [conjunctival epithelial cell](http://purl.obolibrary.org/obo/CL_1000432) [description](http://purl.org/dc/terms/description) "Conjunctival epithelial cells are located in the outermost layer of the conjunctiva, the transparent mucous membrane that covers the front surface of the eye (sclera) and lines the inside of the eyelids. Depending on the location, the conjunctival epithelium varies from a stratified squamous non-keratinizing epithelium (close to the lid margin) to a stratified columnar epithelium (bulbar). It consists of between two and seven layers of epithelial cells, which are organized into three main types: basal, intermediate and superficial; amongst them are goblet cells, which produce the mucous layer of the tear film, 
The conjunctival cells play a vital role in preserving the health and functionality of the eye, contributing significantly to ocular defense mechanisms. They also participate in maintaining the ocular surface's physiological environment. Specifically, conjunctival epithelial cells serve as a physical barrier against environmental threats, such as dust, allergens, and microbes. They prevent foreign bodies from entering the eye, thanks to their tightly packed arrangement, which is integral to the eye's first line of defense. 
Apart from this physical protection, these conjunctival epithelial cells are also involved in immune defense. They secrete microbicidal and immunomodulatory peptides and cytokines and can recognize and respond to pathogens through their Toll- and NOD-like receptor signaling pathways and modulate DC and lymphocyte function. 
Additionally, conjunctival goblet cells, assist in maintaining ocular surface moisture by secreting soluble mucins for tear film production, a thin fluid layer that keeps the cornea and conjunctiva moist. This tear film lubricates the eye, protects against infections, and ensures uninterrupted, clear vision.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1111/imm.12716"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.exer.2013.10.008"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1167/iovs.17-23623"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.celrep.2021.108715"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "https://www.sciencedirect.com/topics/neuroscience/conjunctiva-epithelium"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [conjunctival epithelial cell](http://purl.obolibrary.org/obo/CL_1000432) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_1000432"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "conjunctival epithelial cell on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### connective tissue cell `http://purl.obolibrary.org/obo/CL_0002320`
#### Removed
- [connective tissue cell](http://purl.obolibrary.org/obo/CL_0002320) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A cell of the supporting or framework tissue of the body, arising chiefly from the embryonic mesoderm and including adipose tissue, cartilage, and bone."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "ISBN:0618947256"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "GOC:tfm"^^[string](http://www.w3.org/2001/XMLSchema#string) 

#### Added
- [connective tissue cell](http://purl.obolibrary.org/obo/CL_0002320) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A cell of the supporting or framework tissue of the body, which includes adipose tissue, cartilage, and bone. In vertebrates, this cell arises chiefly from the embryonic mesoderm and, in cranial regions, from neural crest."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:31194383"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "ISBN:0618947256"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:26043871"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "GOC:tfm"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:19198652"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### conventional dendritic cell `http://purl.obolibrary.org/obo/CL_0000990`

#### Added
- [conventional dendritic cell](http://purl.obolibrary.org/obo/CL_0000990) [description](http://purl.org/dc/terms/description) "Conventional dendritic cells (cDCs) are primarily responsible for processing antigens and presenting them to T cells to initiate an immune response. Also known as classical dendritic cells, cDCs originate from hematopoietic stem cells in the bone marrow, maturing in peripheral tissues. They are characterized by their distinct morphology, which features a central body with numerous branching extensions known as dendrites.
These cells have a pivotal role in adaptive immunity, bridging the gap between the innate and adaptive immune responses. They achieve this by capturing, processing, and presenting antigens to naive T-cells in the secondary lymphoid organs. This antigen presentation, in conjunction with co-stimulatory signals provided by the cDCs, instructs the T cells whether to mount an immune response against the antigen. cDCs are also very effective in the cross-presentation of antigens in the context of major histocompatibility complex (MHC) I, thereby activating CD8+ T cells an initiating a cytotoxic T cell response.
In addition to their role in antigen presentation, cDCs have other important functions in maintaining immune homeostasis. They can perceive signals from the surrounding tissue microenvironment, responding to inflammation, and influencing or even polarizing T cell differentiation into various effector subtypes. Importantly, while cDCs are often associated with immunogenic responses, they also play a crucial role in immune tolerance, particularly towards self-antigens, thereby preventing autoimmune reactions. In conclusion, conventional dendritic cells serve as the sentinels of the immune system, monitoring for threat signals and coordinating the appropriate immune response.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/bs.ircmb.2019.07.004"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.3389/fimmu.2018.03176"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1146/annurev-immunol-061020-053707"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [conventional dendritic cell](http://purl.obolibrary.org/obo/CL_0000990) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0000990"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "conventional dendritic cell on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### corneal endothelial cell `http://purl.obolibrary.org/obo/CL_0000132`

#### Added
- [corneal endothelial cell](http://purl.obolibrary.org/obo/CL_0000132) [description](http://purl.org/dc/terms/description) "Corneal endothelial cells are specialized cells located on the innermost layer of the cornea, known as the endothelium. The cornea is the clear outer layer of the eye that helps to transmit and focus light into the retina. The endothelium is crucial in maintaining corneal clarity by actively pumping water out of the corneal stroma, the thick, collagen-containing middle layer of the cornea, to prevent swelling and ensure transparency.
Corneal endothelial cells are characterized by their unique hexagonal shape forming a mosaic, and they are highly metabolically active. Their primary function is to maintain corneal dehydration and thereby clarity through maintaining a barrier and an active 'pump-leak' mechanism, which regulates fluid and solute transport. This transport sustains the deturgescence (state of being non-swollen) of the corneal stroma, since stromal swelling can cause corneal opacification, hindering light transmission. Importantly, human corneal endothelial cells have limited regenerative capacity, so damage to them, through disease, aging, or surgical trauma may result in vision loss.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1097/j.jcrs.0000000000000650"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.exer.2011.06.004"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.4103/ijo.IJO_646_17"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/s41598-020-64311-x"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [corneal endothelial cell](http://purl.obolibrary.org/obo/CL_0000132) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0000132"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "corneal endothelial cell on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### corneal epithelial cell `http://purl.obolibrary.org/obo/CL_0000575`

#### Added
- [corneal epithelial cell](http://purl.obolibrary.org/obo/CL_0000575) [description](http://purl.org/dc/terms/description) "Corneal epithelial cells are a specialized cell type that form the outermost layer of the cornea, the transparent dome-shaped tissue that covers the front of the eye. The corneal epithelium is 5-7 cells thick and is comprised of inner basal cells, middle wing cells and superficial squamous cells. 
The corneal epithelium plays a crucial role in the maintenance of corneal transparency, protection against external insults, and the refractive power of the cornea. This cell type is responsible for the formation of tight junctions, desmosomes, and hemidesmosomes, which provide structural support to the corneal epithelium and prevent the infiltration of foreign particles and microorganisms. 
Corneal epithelial cells also have a unique and highly organized cytoskeleton that confers mechanical integrity and limits the penetration of foreign substances. Additionally, corneal epithelial cells secrete a variety of proteins and lipids that help to maintain the hydration and clarity of the cornea. These include mucins, which form a lubricating layer that facilitates blinking and limits dryness, and crystallins, which appear to protect against UV- and oxidative stress-induced apoptosis.
Dysfunction of corneal epithelial cells due to infections, inflammation, or injury can lead to corneal opacity, scarring, and impaired vision.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/S1350-9462(98)00001-9"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.semcdb.2007.09.015"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1242/jcs.198119"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.3390/cells10092302"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1097/ACI.0b013e32830e6b04"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [corneal epithelial cell](http://purl.obolibrary.org/obo/CL_0000575) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0000575"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "corneal epithelial cell on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### cortical cell of adrenal gland `http://purl.obolibrary.org/obo/CL_0002097`

#### Added
- [cortical cell of adrenal gland](http://purl.obolibrary.org/obo/CL_0002097) [description](http://purl.org/dc/terms/description) "Cortical cells of the adrenal gland, or adrenal cortical cells, are essential components of the endocrine system found in the adrenal cortex, the outer region of the adrenal gland. They are responsible for the synthesis and secretion of vital hormones, including glucocorticoids, mineralocorticoids, and small amounts of adrenal androgens. 
The adrenal cortical cells are further subdivided into three groups based on their position and function: zona glomerulosa, zona fasciculata, and zona reticularis. The zona glomerulosa cells are scattered and primarily secrete mineralocorticoids such as aldosterone; the zona fasciculata contains large adrenal cortical cells replete with liquids (&ldquo;clear cells&rdquo;) that release glucocorticoids; and the zona reticularis contains &ldquo;compact&rdquo; cells containing lipofuscin that are involved in the secretion of adrenal androgens. Each layer plays a distinct and vital role in maintaining a range of body functions, from metabolic processes to stress response mechanisms.
The primary function of these cells corresponds to the hormones they produce. Glucocorticoids, such as cortisol, play a crucial role in glucose metabolism, immune response modulation, and stress response. Mineralocorticoids, primarily aldosterone, actively participate in maintaining the body's electrolyte and water balance by regulating sodium and potassium levels. Meanwhile, adrenal androgens serve as precursors for other sex hormones in both males and females. Therefore, adrenal cortical cells are a central part of the endocrine system, coordinating multiple vital physiological processes.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.steroids.2009.09.002"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1152/ajpendo.00100.2011"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "https://www.ncbi.nlm.nih.gov/books/NBK278945/"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/s41574-021-00491-4"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "https://www.ncbi.nlm.nih.gov/books/NBK537260/"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [cortical cell of adrenal gland](http://purl.obolibrary.org/obo/CL_0002097) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0002097"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "cortical cell of adrenal gland on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### cortical interneuron `http://purl.obolibrary.org/obo/CL_0008031`

#### Added
- [cortical interneuron](http://purl.obolibrary.org/obo/CL_0008031) [description](http://purl.org/dc/terms/description) "Cortical interneurons are a specialized type of neuron predominantly found in the cerebral cortex, the brain's outer layer responsible for higher cognitive functions. They constitute approximately 20-30% of all cortical neurons and are characterized by their crucial role in inhibiting the activity of other neurons in the brain, adding a critical dimension of control to neural activity and information processing.
These cortical interneurons are significantly implicated in maintaining the balance of excitation and inhibition in the cerebral cortex, which is essential for normal brain function. This balance allows the brain to respond appropriately to stimuli, modifying behavior and facilitating motor, sensory, and cognitive processes. Furthermore, cortical interneurons are instrumental in the generation of brain oscillations, rhythmic patterns of neuronal activity believed to contribute to various aspects of cognitive processes, such as attention, memory, and consciousness.
Their unique function and morphology, along with the numerous subtypes distinguished by differences in physiological properties, molecular expression profiles, and connectivity patterns, highlight the complexity and diversity of cortical interneurons. Dysfunction or abnormal development of these cells has been linked to various neurological disorders, including epilepsy, schizophrenia, and autism.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.conb.2023.102703"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1242/dev.132852"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/nrn3155"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.conb.2017.08.018"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/ncomms6333"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [cortical interneuron](http://purl.obolibrary.org/obo/CL_0008031) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0008031"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "cortical interneuron on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### corticothalamic-projecting glutamatergic cortical neuron `http://purl.obolibrary.org/obo/CL_4023013`

#### Added
- [corticothalamic-projecting glutamatergic cortical neuron](http://purl.obolibrary.org/obo/CL_4023013) [description](http://purl.org/dc/terms/description) "The corticothalamic-projecting glutamatergic cortical neuron is a specific glutamatergic neuron found primarily in the cerebral cortex of the brain. These neurons are named for their synthesis of glutamate, the most abundant excitatory neurotransmitter in the nervous system. They play a pivotal role in supporting fundamental neural processing and communication by transmitting nerve signals to the thalamus from the cortex. These signals contribute to various high-level functions such as sensation, cognition, and motor commands.
Corticothalamic-projecting glutamatergic cortical neurons are significant components of a larger corticothalamic circuit. They receive sensory information relayed from peripheral organs and integrate the data to ensure a coordinated and appropriate behavioral response. These types of neurons project their axons to different regions of the thalamus, creating a network that is crucial for sensory processing.
The functionality of these neurons is central to maintaining the brain's rhythmic activity, as their critical role in signal transmission affects almost all sensory experiences except for smell. Malfunction of these neurons can lead to disruption in information integration and processing, which is seen in various cognitive, neurological, and psychiatric disorders such as schizophrenia, Parkinson's disease, epilepsy, and Alzheimer&rsquo;s disease.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1523/JNEUROSCI.4023-14.2015"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.brainres.2017.09.014"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.conb.2008.09.002"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.3389/fnins.2012.00053"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1152/jn.2001.86.1.1"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [corticothalamic-projecting glutamatergic cortical neuron](http://purl.obolibrary.org/obo/CL_4023013) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_4023013"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "corticothalamic-projecting glutamatergic cortical neuron on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### corticotropin-releasing neuron `http://purl.obolibrary.org/obo/CL_4072021`

#### Added
- Class: [corticotropin-releasing neuron](http://purl.obolibrary.org/obo/CL_4072021) 

- [corticotropin-releasing neuron](http://purl.obolibrary.org/obo/CL_4072021) [date](http://purl.org/dc/terms/date) "2025-07-14T12:17:35Z"^^[dateTime](http://www.w3.org/2001/XMLSchema#dateTime) 

- [corticotropin-releasing neuron](http://purl.obolibrary.org/obo/CL_4072021) [id](http://www.geneontology.org/formats/oboInOwl#id) "CL:4072021"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [corticotropin-releasing neuron](http://purl.obolibrary.org/obo/CL_4072021) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A specialised type of interneuron defined by the secretion of corticotropin-releasing hormone (CRH), a key peptide in activating the hypothalamic&ndash;pituitary&ndash;adrenal (HPA) axis during stress. It is primarily located in the paraventricular nucleus (PVN) of the hypothalamus, with additional presence in the hippocampus, piriform area, central amygdalar nucleus, Barrington&rsquo;s nucleus, and inferior olivary complex (Peng et al., 2017)."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:33158870"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:9662138"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:21871477"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:28790896"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:2785663"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [corticotropin-releasing neuron](http://purl.obolibrary.org/obo/CL_4072021) [has_related_synonym](http://www.geneontology.org/formats/oboInOwl#hasRelatedSynonym) "CRH neuron"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [has_synonym_type](http://www.geneontology.org/formats/oboInOwl#hasSynonymType) [OMO_0003000](http://purl.obolibrary.org/obo/OMO_0003000) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:21871477"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [corticotropin-releasing neuron](http://purl.obolibrary.org/obo/CL_4072021) [has_related_synonym](http://www.geneontology.org/formats/oboInOwl#hasRelatedSynonym) "CRF neuron"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:33158870"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [has_synonym_type](http://www.geneontology.org/formats/oboInOwl#hasSynonymType) [OMO_0003000](http://purl.obolibrary.org/obo/OMO_0003000) 

- [corticotropin-releasing neuron](http://purl.obolibrary.org/obo/CL_4072021) [label](http://www.w3.org/2000/01/rdf-schema#label) "corticotropin-releasing neuron"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [corticotropin-releasing neuron](http://purl.obolibrary.org/obo/CL_4072021) SubClassOf [interneuron](http://purl.obolibrary.org/obo/CL_0000099) 

- [corticotropin-releasing neuron](http://purl.obolibrary.org/obo/CL_4072021) SubClassOf [neuroendocrine cell](http://purl.obolibrary.org/obo/CL_0000165) 

- [corticotropin-releasing neuron](http://purl.obolibrary.org/obo/CL_4072021) SubClassOf [adrenocorticotropic hormone secreting cell](http://purl.obolibrary.org/obo/CL_0000467) 


### dendritic cell `http://purl.obolibrary.org/obo/CL_0000451`

#### Added
- [dendritic cell](http://purl.obolibrary.org/obo/CL_0000451) [description](http://purl.org/dc/terms/description) "Dendritic cells, named for their tree-like extensions called dendrites, are a type of immune cell that form a critical bridge between the innate and adaptive immune systems. They are derived from hematopoietic bone marrow progenitor cells and can be found throughout the body poised as sentinels in locations such as the skin, lungs, or liver, as well as immune organs such as the thymus, lymph nodes, or spleen. They can also be found in an immature state in the blood. Dendritic cells serve as antigen-presenting cells, with a primary function of processing and presenting antigens to T-cells to facilitate an immune response.
Upon encountering an antigen, dendritic cells engulf the pathogen via phagocytosis and subsequently process the pathogen into antigens. These antigens are then presented on the surface of the dendritic cell in the context of major histocompatibility complex (MHC) molecules. The dendritic cell migrates to the lymph nodes, containing naive T-cells, and presents the antigens to them. This process triggers the activation of cognate T-cells into effector T-cells, which can then recognize and eradicate cells bearing the same antigen.
Additionally, the dendritic cells can influence the type of immune response the body will mount. They communicate with T-cells through the use of co-stimulatory molecules and cytokines. Depending on the type of co-stimulatory molecules used and the set of cytokines released, dendritic cells can promote the development of a cellular immune response, dominated by cytotoxic T cells effective against intracellular pathogens such as viruses, a humoral response, mediated by B cells and effective against extracellular bacteria and toxins, or a tolerogenic response. Thus, dendritic cells play a pivotal role in immunity, helping the body to effectively combat infections, prevent autoimmune diseases, and maintain tolerance to healthy self-tissues.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/nri.2016.116"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/bs.ircmb.2019.07.004"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1146/annurev-immunol-100311-102839"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1146/annurev-immunol-061020-053707"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1111/imm.12888"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [dendritic cell](http://purl.obolibrary.org/obo/CL_0000451) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0000451"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "dendritic cell on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### dental papilla cell `http://purl.obolibrary.org/obo/CL_0000345`

#### Added
- [dental papilla cell](http://purl.obolibrary.org/obo/CL_0000345) SubClassOf [eukaryotic cell](http://purl.obolibrary.org/obo/CL_0000255) 


### direct pathway medium spiny neuron `http://purl.obolibrary.org/obo/CL_4023026`
#### Removed
- [direct pathway medium spiny neuron](http://purl.obolibrary.org/obo/CL_4023026) SubClassOf [BFO_0000002](http://purl.obolibrary.org/obo/BFO_0000002) 



### dopaminergic neuron `http://purl.obolibrary.org/obo/CL_0000700`

#### Added
- [dopaminergic neuron](http://purl.obolibrary.org/obo/CL_0000700) [RO_0002175](http://purl.obolibrary.org/obo/RO_0002175) [NCBITaxon_9606](http://purl.obolibrary.org/obo/NCBITaxon_9606) 

- [dopaminergic neuron](http://purl.obolibrary.org/obo/CL_0000700) [in_subset](http://www.geneontology.org/formats/oboInOwl#inSubset) [human_reference_atlas](http://purl.obolibrary.org/obo/cl/cl-base#human_reference_atlas) 


### dual-feature fallopian tube progenitor cell `http://purl.obolibrary.org/obo/CL_4052070`

#### Added
- Class: [dual-feature fallopian tube progenitor cell](http://purl.obolibrary.org/obo/CL_4052070) 

- [dual-feature fallopian tube progenitor cell](http://purl.obolibrary.org/obo/CL_4052070) [contributor](http://purl.org/dc/terms/contributor) [0009-0000-8480-9277](https://orcid.org/0009-0000-8480-9277) 

- [dual-feature fallopian tube progenitor cell](http://purl.obolibrary.org/obo/CL_4052070) [date](http://purl.org/dc/terms/date) "2025-08-12T16:05:47Z"^^[dateTime](http://www.w3.org/2001/XMLSchema#dateTime) 

- [dual-feature fallopian tube progenitor cell](http://purl.obolibrary.org/obo/CL_4052070) [id](http://www.geneontology.org/formats/oboInOwl#id) "CL:4052070"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [dual-feature fallopian tube progenitor cell](http://purl.obolibrary.org/obo/CL_4052070) [has_narrow_synonym](http://www.geneontology.org/formats/oboInOwl#hasNarrowSynonym) "non-ciliated secretory epithelial cell 2-1 (NCSE2-1 cells)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:35320732"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [dual-feature fallopian tube progenitor cell](http://purl.obolibrary.org/obo/CL_4052070) [has_narrow_synonym](http://www.geneontology.org/formats/oboInOwl#hasNarrowSynonym) "non-ciliated secretory epithelial cell 2-2 (NCSE2-2 cells)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:35320732"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [dual-feature fallopian tube progenitor cell](http://purl.obolibrary.org/obo/CL_4052070) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A bipotent progenitor cell within the human fallopian tube epithelium, characterized by the concurrent expression of epithelial (e.g., EpCAM/CD326) and endothelial (e.g., PECAM1/CD31) markers at both the cell surface and transcript levels. This cell has the capacity to differentiate into ciliated and secretory epithelial cells, as well as potentially endothelial/stromal lineages. Positioned at the apex of lineage bifurcation, the cell exhibits stem-like and endothelial features, representing an intermediate developmental state between undifferentiated progenitors and lineage-committed epithelial cells."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:40475517"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [dual-feature fallopian tube progenitor cell](http://purl.obolibrary.org/obo/CL_4052070) [has_exact_synonym](http://www.geneontology.org/formats/oboInOwl#hasExactSynonym) "unclassified fallopian tube progenitor"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:40475517"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [dual-feature fallopian tube progenitor cell](http://purl.obolibrary.org/obo/CL_4052070) [label](http://www.w3.org/2000/01/rdf-schema#label) "dual-feature fallopian tube progenitor cell"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [dual-feature fallopian tube progenitor cell](http://purl.obolibrary.org/obo/CL_4052070) SubClassOf [progenitor cell](http://purl.obolibrary.org/obo/CL_0011026) 

- [dual-feature fallopian tube progenitor cell](http://purl.obolibrary.org/obo/CL_4052070) SubClassOf [BFO_0000050](http://purl.obolibrary.org/obo/BFO_0000050) some [UBERON_8600124](http://purl.obolibrary.org/obo/UBERON_8600124) 

- [dual-feature fallopian tube progenitor cell](http://purl.obolibrary.org/obo/CL_4052070) SubClassOf [RO_0002162](http://purl.obolibrary.org/obo/RO_0002162) some [NCBITaxon_9606](http://purl.obolibrary.org/obo/NCBITaxon_9606) 


### duct epithelial cell `http://purl.obolibrary.org/obo/CL_0000068`

#### Added
- [duct epithelial cell](http://purl.obolibrary.org/obo/CL_0000068) [description](http://purl.org/dc/terms/description) "Duct epithelial cells are distinctly specialized cells that predominantly form the lining of glandular ducts. These cells play a crucial role in the secretion, absorption, and transport of fluids depending on the organ they inhabit. They are typically organized in a tubular structure, forming collectively the conduit for the transport of substances. These cells exhibit tight junctions, enabling the careful regulation of the passage of substances and ensuring that no unwanted particles seep through. They are predominantly characterized by their polarized nature, having distinct apical and basolateral surfaces that support its functions of secretion and absorption. The apical surface, exposed to the duct lumen, is specialized for secretion and interaction with the luminal environment, while the basolateral surface is oriented towards the underlying tissue, facilitating interactions with neighboring cells and the extracellular matrix.
The ducts serve as the bridge between the gland and the site of secretion, allowing the substances produced by the gland to exit into the body. Thus, duct epithelial cells have a sweeping role in various physiological functions, ranging from defense against pathogens in the respiratory and digestive tracts, to secretion of digestive enzymes, to milk production in mammary glands.
As a result of their specialized function, any impairment in the function of duct epithelial cells to function properly often leads to certain diseases or disorders. Studies have identified potential correlation between malfunctioning duct epithelial cells and conditions such as cystic fibrosis, chronic obstructive pulmonary disease, and certain cancers, particularly breast and pancreatic.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.4161/tisb.25718"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/s41467-018-04334-1"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "https://www.sciencedirect.com/topics/medicine-and-dentistry/mammary-epithelium"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1513/AnnalsATS.201402-049AW"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.3389/fphar.2019.00074"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [duct epithelial cell](http://purl.obolibrary.org/obo/CL_0000068) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0000068"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "duct epithelial cell on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### endothelial cell of coronary artery `http://purl.obolibrary.org/obo/CL_2000018`

#### Added
- [endothelial cell of coronary artery](http://purl.obolibrary.org/obo/CL_2000018) [description](http://purl.org/dc/terms/description) "Endothelial cells of the coronary artery are specialized cells that line the inner surface of the coronary arteries, which supply oxygen-rich blood to the myocardium (heart muscle). These cells play key roles in vascular biology including the regulation of vascular tone, blood coagulation, leukocyte adhesion, and platelet adhesion. They are also responsible for maintaining the integrity of the vascular wall.
To maintain cardiovascular homeostasis, the endothelial cells of the coronary artery produce substances such as endothelin, a powerful vasoconstrictor,  and prostacyclin, which has vasodilatory, anti-inflammatory, and anti-thrombotic properties. They also produce nitric oxide, which has a protective role in the vasculature, preventing inflammation, thrombosis, and platelet aggregation, which are major risk factors for atherosclerosis, leading to coronary heart disease. 
Dysfunction or impairment of endothelial cell function in coronary arteries is often considered an early prognostic marker in the development of atherosclerosis. With decreased nitric oxide production or availability, the arterial wall becomes more prone to inflammation, thrombosis, and constriction, all of which lead to plaque formation and eventual blockage of the artery.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1055/s-0031-1283220"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1097/00041433-200108000-00003"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1097/MCA.0000000000000178"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1161/01.CIR.0000153339.27064.14"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [endothelial cell of coronary artery](http://purl.obolibrary.org/obo/CL_2000018) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_2000018"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "endothelial cell of coronary artery on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### endothelial tip cell `http://purl.obolibrary.org/obo/CL_0000704`

#### Added
- [endothelial tip cell](http://purl.obolibrary.org/obo/CL_0000704) [description](http://purl.org/dc/terms/description) "Endothelial tip cells are a specialized type of cell that play a crucial role in angiogenesis, the formation of new blood vessels from the existing vasculature. These cells are located at the ends of growing capillary sprouts, their position giving them the term &quot;tip&quot; cells. 
Endothelial tip cells are characterized by a number of attributes which facilitate them to lead the way in angiogenesis: They have a highly dynamic, large, and polarized shape, and they extend numerous long and mobile filopodia, or finger-like projections, which continuously probe the extracellular microenvironment for cues that control the growth and movement of the cells. The filopodia are equipped with several receptors, including Vascular Endothelial Growth Factor Receptor (VEGFR) -2 and -3, which respond to guidance signals during angiogenesis. Their distinctive morphology with a broad leading edge and small cell body trailing behind helps in efficiently moving in the right direction responding to cues.
The function of endothelial tip cells is fundamental to the angiogenesis process. Guiding the sprouting vessel, they direct the subsequent proliferation and migration of the stalk cells, which follow the tip cells and form the body of the sprout. These tip cells do not proliferate, but instead, dictate the division of stalk cells, leading to the elongation of the sprout. They also help in lumen formation. The guidance and sprouting decisions of endothelial tip cells are critical for creating a functional and efficient network of vessels that can appropriately supply oxygen and nutrients to tissues. Notably, tip cells play a role in pathological conditions such as tumor angiogenesis and eye diseases, making them an important area of research in cell biology and medicine.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/nrclinonc.2009.64"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.devcel.2021.06.021"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1101/cshperspect.a006569"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.semcdb.2014.04.009"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [endothelial tip cell](http://purl.obolibrary.org/obo/CL_0000704) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0000704"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "endothelial tip cell on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### enteric neuron `http://purl.obolibrary.org/obo/CL_0007011`

#### Added
- [enteric neuron](http://purl.obolibrary.org/obo/CL_0007011) [description](http://purl.org/dc/terms/description) "Enteric neurons, also known as the neurons of the enteric nervous system (ENS), form a vast and complex network within the walls of the gastrointestinal tract. These unique cells constitute the principal component of the ENS, which is often referred to as the &quot;second brain&quot; or the gut's brain, due to its autonomy from the central nervous system (CNS). While the ENS does communicate with the CNS, it can function independently, thereby playing an essential role in maintaining the body's homeostasis.
Enteric neurons are incredibly diverse in function and morphology. Integral to essential processes like peristalsis, secretion, and blood flow regulation, these neurons are further classified into different types based on their roles. The organization of the ENS is characterized by two main ganglionated layers: the inner submucosal (Meissner's) plexus, involved primarily in regulating gastrointestinal blood flow and epithelial cell function, and the outer myenteric (Auerbach's) plexus, which predominantly manages gut motility.
Enteric neurons play a significant role in sensing and responding to changes within the gut environment. First, they detect physical and chemical changes, for instance, the arrival of food, and respond accordingly by adjusting gut motility and secretions. Second, they communicate information about the state of the gut to the CNS; however, much of the routine detailed management of the digestive system is carried out within the ENS itself. In addition, they also interact with the gut's large microbial population and the immune system, playing a pivotal role in health and disease. Dysfunctions or alterations in the enteric neurons may contribute to various gastrointestinal disorders, such as functional dyspepsia or irritable bowel syndrome, and neurodegenerative diseases such as Parkinson's.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/s41575-020-0271-2"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/nrgastro.2016.107"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "https://www.ncbi.nlm.nih.gov/books/NBK551559/"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.immuni.2017.05.011"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/nrgastro.2012.32"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [enteric neuron](http://purl.obolibrary.org/obo/CL_0007011) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0007011"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "enteric neuron on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### enterocyte of epithelium of large intestine `http://purl.obolibrary.org/obo/CL_0002071`
#### Removed
- [enterocyte of epithelium of large intestine](http://purl.obolibrary.org/obo/CL_0002071) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "Columnar cell which populate the epithelium of large intestine and absorb water. This cell is the most numerous of the epithelial cell types in the large intestine; bear apical microvilli, contain secretory granules in their apical cytoplasm; secretion appears to be largely mucins, but is also rich in antibodies of the IgA type."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "ISBN:0517223651"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "GOC:tfm"^^[string](http://www.w3.org/2001/XMLSchema#string) 

#### Added
- [enterocyte of epithelium of large intestine](http://purl.obolibrary.org/obo/CL_0002071) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A columnar absorptive epithelial cell found in the epithelium of large intestine, specialized for water and electrolyte absorption. Unlike small intestinal enterocytes, those in the large intestine have fewer apical microvilli."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:35626748"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "ISBN:0517223651"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:22288583"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "GOC:tfm"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:38872928"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### eosinophil `http://purl.obolibrary.org/obo/CL_0000771`

#### Added
- [eosinophil](http://purl.obolibrary.org/obo/CL_0000771) [description](http://purl.org/dc/terms/description) "Eosinophils are a type of granulocyte derived from myeloid progenitors in the bone marrow; they are a part of the body's immune system. Named for their affinity to eosin, a red acidic dye used in staining procedures, eosinophils contribute to the immune system's response by migrating to inflammation sites, particularly those produced by allergic reactions and infections due to multicellular parasites. They comprise ~1-5% of the white blood cells in circulation.
Eosinophils play crucial roles in mediating inflammatory responses. They carry granules containing a variety of enzymes and proteins, including major basic protein (MBP), eosinophil peroxidase (EPO), eosinophil cationic protein (ECP), and eosinophil-derived neurotoxin (EDN). These enzymes and proteins are effective in killing bacteria, parasites, and some cancer cells. When exposed to an antigen, eosinophils degranulate and release these mediators to the site of infection, resulting in the destruction of invading microorganisms. Specifically, in parasitic infections, eosinophils attach themselves to the parasites and release these cytotoxic granules, causing a lethal rupture in the parasite's surface. Eosinphils may also release cytokines to attract other immune cells to sites of infection and to skew the response towards type 2 immunity.
Eosinophils are also involved in conditions such as asthma and allergic reactions. They secrete chemical mediators such as leukotrienes and prostaglandins that trigger contraction of smooth muscles, causing symptoms such as bronchoconstriction observed in asthma. However, these multifunctional cells, while useful in defending the body, may also inadvertently cause tissue damage if excessively activated. Increased numbers of eosinophils, a condition known as eosinophilia, can indicate a variety of diseases including allergies, skin disorders, parasitic and fungal infections, autoimmune diseases, certain types of cancer, and asthma.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1146/annurev-pathmechdis-012419-032756"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.mayocp.2021.04.025"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/nri3341"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1182/blood-2012-06-330845"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1128/microbiolspec.mchd-0020-2015"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [eosinophil](http://purl.obolibrary.org/obo/CL_0000771) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0000771"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "eosinophil on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### ependymal cell `http://purl.obolibrary.org/obo/CL_0000065`

#### Added
- [ependymal cell](http://purl.obolibrary.org/obo/CL_0000065) [description](http://purl.org/dc/terms/description) "Ependymal cells are ciliated neuroepithelial cells lining the ventricles of the brain and the central canal of the spinal cord, forming a single-layered epithelium, known as the ependyma. Together with astrocytes, microglial cells, and oligodendrocytes, they are one of the four main types of neuroglia found in the nervous system. 
Ependymal cells are equipped with multiple functions that support the brain and spinal cord. Key amongst these functions are the secretion, circulation, and absorption of cerebrospinal fluid, which cushions the brain, distributes neuroendocrine factors, and disposes of neural waste. The cilia on the apical surface of ependymal cells vigorously beat in a coordinated fashion to circulate the cerebrospinal fluid. Additionally, these cells may also function in aiding the migration of neuroblasts during brain development.
Ependymal cells form a barrier between the cerebrospinal fluid and the adjacent nervous tissue in the central nervous system, which appears to be particularly important during development. In response to injury, ependymal cells have been observed to proliferate and have the potential to transform into astrocytes or oligodendrocytes, suggesting stem cell-like functions.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.4161/tisb.28426"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.14336/AD.2022.0826-1"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1007/s00401-009-0624-y"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [ependymal cell](http://purl.obolibrary.org/obo/CL_0000065) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0000065"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "ependymal cell on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### epicardial adipocyte `http://purl.obolibrary.org/obo/CL_1000309`

#### Added
- [epicardial adipocyte](http://purl.obolibrary.org/obo/CL_1000309) [description](http://purl.org/dc/terms/description) "Epicardial adipocytes are a type of specialized cell found in the epicardial adipose tissue (EAT). This specific adipose tissue is located between the myocardium (the heart muscle) and the visceral pericardium (the innermost layer of the pericardium). Epicardial adipocytes store energy in the form of lipids; specifically, triglyceride droplets. These cells can rapidly mobilize and release these fatty acid molecules, when required, providing an effective and efficient energy source to the body. This energy is particularly important to the heart, which is one of the highest energy-demanding organs. The proximity of these specialized cells to the myocardium allows for direct and rapid transmission of these fatty acid molecules to meet the heart&rsquo;s energy needs. 
Beyond their primary role in energy storage and regulation, epicardial adipocytes also play a pivotal role in providing a protective cushioning to the heart, releasing several beneficial adipokines which impacts cardiac function, along with acting as a source of stem cells. Adipocytes also contribute significantly to cardiac health and function in other ways. They have been found to secrete multiple bioactive molecules, known as adipokines, which can influence local cardiac function and overall cardiovascular health. Adipokines such as adiponectin, leptin and resistin regulate aspects of cardiac metabolism, inflammation, and contractility. Additionally, studies have also pointed towards epicardial adipocytes as a potential reservoir of adult epicardial progenitor cells, which can give rise to smooth muscle cells, endothelial cells and more adipocytes, thus contributing to cardiac repair and regeneration. Despite these important functions, an excess of epicardial adipose tissue has been associated with cardiovascular disease, indicating the importance of understanding and managing the balance of these unique cells in the body.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.tem.2011.07.003"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.3390/biology11030355"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/ncpcardio0319"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.3978/j.issn.2223-3652.2014.11.05"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/s41569-022-00679-9"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [epicardial adipocyte](http://purl.obolibrary.org/obo/CL_1000309) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_1000309"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "epicardial adipocyte on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### epidermal cell `http://purl.obolibrary.org/obo/CL_0000362`

#### Added
- [epidermal cell](http://purl.obolibrary.org/obo/CL_0000362) [description](http://purl.org/dc/terms/description) "Epidermal cells constitute the outermost layer of the skin in animals and have a multifaceted role in maintaining the body's overall homeostasis. They originate from the ectoderm and form the epidermis, a stratified squamous epithelium that mostly consists of keratinocytes in various stages of differentiation. The primary role of these cells is to act as a physical barrier, shielding the internal organs, tissues, and system from dehydration, infection, and other external influences such as UV radiation. The thickness and stratification of the epidermis vary with its location in the body, depending on the specific demand for protective function.
One notable characteristic of epidermal cells is their continuous cycle of renewal. The cells are produced in the deepest layer of the epidermis, known as the basal or germinative layer, where they undergo cell division. As new cells form, they gradually push the older cells towards the skin surface. While ascending, these cells undergo a process known as keratinization, where they become filled with keratin, a protein that provides strength and rigidity. By the time they reach the surface, they are essentially dead and filled with keratin, forming a tough, resistant layer that is regularly shed and replaced by newer cells beneath. This entire process, from cell generation to shedding, usually takes around a month.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1146/annurev.cellbio.22.010305.104357"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.tcb.2018.05.002"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "https://www.ncbi.nlm.nih.gov/books/NBK513299/#:~:text=The%20stratum%20corneum%20is%20the%20outermost%20layer%20of%20the%20epidermis,potential%20and%20undergo%20programmed%20destruction."^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [epidermal cell](http://purl.obolibrary.org/obo/CL_0000362) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0000362"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "epidermal cell on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### epithelial cell of prostate `http://purl.obolibrary.org/obo/CL_0002231`

#### Added
- [epithelial cell of prostate](http://purl.obolibrary.org/obo/CL_0002231) [description](http://purl.org/dc/terms/description) "Epithelial cells of the prostate, also known as prostatic epithelial cells, are specialized cells found in the prostate gland, which surrounds the male urethra beneath the bladder. They play a crucial role in forming the acini and ducts of the gland's structure. The prostate's epithelium has multiple layers, including basal, luminal, and neuroendocrine cells; the luminal layer is composed of columnar epithelial cells, and the basal layer is composed of cuboidal epithelial cells.
Prostatic epithelial cells primarily aid in the production and secretion of prostatic fluid, a critical component of semen. This fluid, which accounts for around 20-30% of semen volume, serves several key functions related to fertility. The prostatic fluid is alkaline in nature that aids in neutralizing acidic vaginal secretions, promoting the survival of sperm in the female reproductive tract. It also contains proteolytic enzymes that, upon ejaculation, lead to the liquefaction of semen, enhancing the motility of sperm. 
Additionally, prostate epithelial cells are involved in the secretion of prostate-specific antigen (PSA), a kallikrein-like serine protease, which is a normal component of the seminal plasma further aids in semen liquefaction.
Prostatic epithelial cells are also linked to conditions like benign prostatic hyperplasia, which involves abnormal cell growth, and prostate cancer, which originates from prostatic epithelial cell transformations, especially in the peripheral zone. Consequently, these cells are key focal points in understanding and treating prostatic conditions and malignancies.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/s41598-019-40820-2"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1159/000496289"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/onc.2012.541"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.celrep.2018.11.086"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [epithelial cell of prostate](http://purl.obolibrary.org/obo/CL_0002231) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0002231"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "epithelial cell of prostate on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### epithelial cell of proximal tubule `http://purl.obolibrary.org/obo/CL_0002306`

#### Added
- [epithelial cell of proximal tubule](http://purl.obolibrary.org/obo/CL_0002306) [description](http://purl.org/dc/terms/description) "An epithelial cell of the proximal tubule, commonly referred to as a proximal tubular epithelial cell, is a specialized brush border cell found lining the proximal tubule within the functional unit of the kidney, the nephron. These cells play a critical role in the resorption process in the kidney and are responsible for maintaining homeostasis, including blood filtration, electrolyte balance, and acid-base balance. Epithelial cells of the proximal tubule have a cuboidal shape and microvilli that increase surface area, facilitating effective reabsorption.
The primary function of epithelial cells of the proximal tubule revolves around their ability to reabsorb approximately 60-70% of the glomerular filtrate, retaining essential substances such as sodium, glucose, amino acids, bicarbonate, and certain ions that are filtered out by the glomerulus. Additionally, these cells are also responsible for the secretion of organic acids, bases, and the hydrolysis of small proteins that are reabsorbed after filtration. 
Proximal tubule epithelial cells are highly susceptible to apoptosis, and damage to these cells by exposure to toxins, pathogens, or obstructive conditions may result in acute kidney injury.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/ki.2011.120"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1007/s11033-020-05977-4"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.2215/CJN.02440314"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1073/pnas.0405179101"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1002/cphy.c110061"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [epithelial cell of proximal tubule](http://purl.obolibrary.org/obo/CL_0002306) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0002306"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "epithelial cell of proximal tubule on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### epithelial cell of stratum germinativum of esophagus `http://purl.obolibrary.org/obo/CL_1000447`

#### Added
- [epithelial cell of stratum germinativum of esophagus](http://purl.obolibrary.org/obo/CL_1000447) [description](http://purl.org/dc/terms/description) "The epithelial cell in the stratum germinativum of the esophagus, commonly referred to as a basal cell, is a crucial component of the body's mechanism for tissue homeostasis and regeneration. Situated in the deepest layer of the esophageal lining, these cells are pivotal in preserving the integrity of the esophagus, which faces exposure to various physical and chemical irritants like stomach acid, food particles, and pathogens.
These cells are renowned for their proliferative potential. When the superficial layers of the esophageal epithelium are damaged, these basal cells are triggered to multiply, differentiate, and migrate upwards to restore the integrity and function of the affected tissue. This process, known as regeneration or repair, ensures that the esophageal wall remains continuous and functional, capable of performing its primary task: the conveyance of food and drink from the mouth to the stomach.
However, it's crucial to recognize that the continuous cycle of damage and repair in the esophageal epithelium, driven by the epithelial cells of the stratum germinativum, can result in pathological conditions like Barrett's esophagus and esophageal cancer. Under certain conditions, these basal cells may deviate from their normal differentiation into squamous cells and instead transform into abnormal cell types, such as intestinal columnar cells, contributing to the development of these diseases.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1136/gut.2005.089144"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.tcb.2018.04.007"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/nrc2773"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [epithelial cell of stratum germinativum of esophagus](http://purl.obolibrary.org/obo/CL_1000447) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_1000447"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "epithelial cell of stratum germinativum of esophagus on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### epithelial cell of sweat gland `http://purl.obolibrary.org/obo/CL_1000448`

#### Added
- [epithelial cell of sweat gland](http://purl.obolibrary.org/obo/CL_1000448) [description](http://purl.org/dc/terms/description) "Epithelial cells of sweat glands, also known as sudoriferous glandular cells, play a crucial role in the body's thermoregulation, waste excretion, and hydration balance. They form the main structure of the sweat glands, which are effectively long, coiled tubes located in the dermis. There are two types of sweat glands &ndash; eccrine glands, which typically open directly onto the skin surface (e.g. on palms and soles), and apocrine glands, which open onto associated hair follicles (e.g. in arm pits).  
Eccrine glands consist of a tube-shaped duct that ends in a coiled, secretory unit made up of three types of epithelial cells &ndash; dark, clear and myoepithelial. The dark cells contain basophilic granules to secrete sialomucin, while clear cells secrete glycogen-containing aqueous material; the myoepithelial cells contract in response to stimuli to assist with secretion. Apocrine glands have a similar structure, but their secretory component is larger. 
The primary function of the epithelial cells of these glands is to produce sweat, a bodily fluid made mostly of water, but also containing trace amounts of salts, glucose, urea, and other substances. This process begins with the secretion of a primary sweat fluid - an isotonic solution derived from the blood plasma - into the glandular lumen. Subsequently, as this primary sweat passes through the gland's duct, reabsorption of sodium ions occurs, leading to a hypotonic final sweat that is excreted onto the skin's surface. This sweat evaporates, removing heat from the body and thus, cooling it down. 
In addition, epithelial cells of sweat glands contribute to the body's immune defense system. As part of the innate skin barrier function, they secrete antimicrobial peptides and proteins, such as dermcidin, which can inhibit the growth of harmful microorganisms on the skin surface. They also aid in maintaining skin pH and contribute to fluid balance by regulating water resorption. Consequently, any dysfunction in these cells can lead to conditions like hyperhidrosis, anhidrosis, and certain skin infections. Thus, these epithelial cells of sweat glands play a multifaceted role in health and disease.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "https://www.ncbi.nlm.nih.gov/books/NBK513244/"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "https://www.ncbi.nlm.nih.gov/books/NBK482278/"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "https://doi.org/10.1038/s41419-019-1485-5"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1080/23328940.2019.1632145"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1111/exd.12773"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [epithelial cell of sweat gland](http://purl.obolibrary.org/obo/CL_1000448) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_1000448"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "epithelial cell of sweat gland on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### epithelial fate stem cell `http://purl.obolibrary.org/obo/CL_0000036`

#### Added
- [epithelial fate stem cell](http://purl.obolibrary.org/obo/CL_0000036) [description](http://purl.org/dc/terms/description) "Epithelial fate stem cells, also known as epithelial stem cells, possess remarkable abilities for self-renewal and the generation of diverse cell types constituting epithelial tissues. The epithelium, derived from these cells, acts as the body's frontline defense against external threats and facilitates the exchange of nutrients, gases, and waste materials within the internal environment.
Epithelial fate stem cells contribute to tissue development, homeostasis, and repair, maintaining the structural and functional stability of epithelial tissues throughout the human body, including the skin, organs, and glands. Through a delicate balance of quiescence, proliferation, and differentiation, these cells ensure the continuous turnover of epithelial cells.
Furthermore, these cells serve a critical role in cellular differentiation, where initially unspecialized stem cells mature into specialized entities that perpetuate the functionality of epithelial tissues. This process is intricately regulated by a network of gene expression patterns and external signaling cues to guarantee a coordinated and balanced cellular turnover. Hence, epithelial fate stem cells, with their unique attributes of self-renewal and differentiation, are central to the physiological and reparative functions performed by epithelial tissues in the human body.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.cell.2007.01.014"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.stem.2020.09.011"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.celrep.2022.111451"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/nrm2766"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [epithelial fate stem cell](http://purl.obolibrary.org/obo/CL_0000036) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0000036"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "epithelial fate stem cell on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### erythrocyte `http://purl.obolibrary.org/obo/CL_0000232`

#### Added
- [erythrocyte](http://purl.obolibrary.org/obo/CL_0000232) [description](http://purl.org/dc/terms/description) "Erythrocytes, also known as red blood cells, are a type of specialized cell that make up the largest percentage of cells in the human body. Erythrocytes are biconcave in shape and lack nuclei and organelles, allowing them to carry oxygen and carbon dioxide through the bloodstream more efficiently. The main function of erythrocytes is to transport oxygen from the lungs to the body's tissues and organs, and to transport carbon dioxide from the body's tissues to the lungs, where it can be exhaled out of the body. Hemoglobin, a protein located inside erythrocytes, is responsible for binding to oxygen. 
Erythrocytes are produced in the bone marrow and have a lifespan of approximately 120 days before they are broken down and recycled by the body. Erythrocytes are regulated by the hormone erythropoietin, which is produced by the kidneys and liver in response to low oxygen levels in the body, thereby stimulating erythrocyte production .
In addition to their role in oxygen transport, erythrocytes also play a key role in regulating the body's pH levels. Without erythrocytes, oxygen would not be able to be transported efficiently throughout the body and tissues would not receive the necessary oxygen to function properly. Overall, erythrocytes are essential for maintaining the health and homeostasis of the body.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.3389/fphys.2021.655393"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1056/NEJM199801223380407"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.blre.2012.12.003"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [erythrocyte](http://purl.obolibrary.org/obo/CL_0000232) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0000232"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "erythrocyte on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### erythroid progenitor cell `http://purl.obolibrary.org/obo/CL_0000038`

#### Added
- [erythroid progenitor cell](http://purl.obolibrary.org/obo/CL_0000038) [description](http://purl.org/dc/terms/description) "Erythroid progenitor cells play a key role in the hematopoietic system, specifically in the process of erythropoiesis &ndash; the production of red blood cells (erythrocytes). Found within the bone marrow, these cells are dedicated precursors to erythrocytes, and ensue in the hematopoietic lineage from multipotent progenitors. They represent a transitional cell type that bridges the gap between the early-stage erythroid precursors and the maturing red blood cell population.
The earliest committed erythroid progenitor cell is thought to be the burst-forming unit-erythroid (BFU-E) cell, which after several cell divisions gives rise to another erythroid progenitor, the colony-forming unit-erythroid (CFU-E) cell. 
Erythroid progenitor cells are primarily responsible for responding to the demand for new red blood cells during conditions of physiological stress like infection, blood loss, or illness. This responsive behavior is regulated by a glycoprotein hormone known as erythropoietin, which is produced by the kidneys. Erythropoietin interacts with the receptors on the erythroid progenitor cells, prompting them to proliferate and differentiate into erythroblasts. The erythroblasts then further mature, eventually producing new, functional red cells.
The role of erythroid progenitor cells extends beyond their main function of erythrocyte production, as they also participate in maintaining the balance of iron in the body. Upon erythroid differentiation, they increase their uptake of transferrin-bound iron for hemoglobin synthesis, a vital component of red blood cells which allows them to transport oxygen. A deficiency or excess of erythroid progenitor cells can lead to extensive health issues such as anemia or polycythemia respectively, emphasizing their crucial role in human health.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.exphem.2015.10.007"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1097/MOH.0000000000000328"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1182/blood-2014-07-588806"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1111/bjh.13938"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.3389/fphys.2019.00304"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [erythroid progenitor cell](http://purl.obolibrary.org/obo/CL_0000038) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0000038"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "erythroid progenitor cell on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### erythroid progenitor cell, mammalian `http://purl.obolibrary.org/obo/CL_0001066`

#### Added
- [erythroid progenitor cell, mammalian](http://purl.obolibrary.org/obo/CL_0001066) [description](http://purl.org/dc/terms/description) "Erythroid progenitor cells serve as precursors for mature erythrocytes. These cells sit at a critical juncture in the erythroid lineage, representing the stage of commitment to producing red blood cells. As such, erythroid progenitor cells are a central element in hematopoiesis, the process by which the body replenishes its supply of blood cells. This production occurs primarily in the bone marrow, where erythroid progenitor cells multiply and differentiate into mature erythrocytes.
Hematopoietic stem cells give rise to the multipotent progenitor (MPP), followed by the common myeloid progenitor (CMP), which further commits to the megakaryocyte-erythroid progenitor (MEP). Commitment to the erythroid lineage proceeds with the burst-forming unit-erythroid (BFU-E), which further differentiates into the colony-forming unit-erythroid (CFU-E), also referred to as the erythroid progenitor cell.
Erythroid progenitor cells embark on a journey of progressive maturation marked by cellular changes such as reduced cell size, increased cell density, condensed chromatin structure, and hemoglobin accumulation. The final stages of this maturation process leads to loss of the cell nucleus and the acquisition of the characteristic biconcave shape. This process is tightly regulated through a combination of various growth factors, cytokines, and hormones, the most critical of which is erythropoietin. Aberrations in the differentiation and maturation of these progenitor cells can result in blood diseases such as anemia and polycythemia.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1097/MOH.0000000000000328"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1002/stem.2834"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/B978-0-323-04404-2.10002-8"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1083/jcb.96.2.386"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [erythroid progenitor cell, mammalian](http://purl.obolibrary.org/obo/CL_0001066) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0001066"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "erythroid progenitor cell, mammalian on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### excretory duct cell of salivary gland `http://purl.obolibrary.org/obo/CL_4052069`

#### Added
- Class: [excretory duct cell of salivary gland](http://purl.obolibrary.org/obo/CL_4052069) 

- [excretory duct cell of salivary gland](http://purl.obolibrary.org/obo/CL_4052069) [contributor](http://purl.org/dc/terms/contributor) [0009-0000-8480-9277](https://orcid.org/0009-0000-8480-9277) 

- [excretory duct cell of salivary gland](http://purl.obolibrary.org/obo/CL_4052069) [date](http://purl.org/dc/terms/date) "2025-08-08T11:20:00Z"^^[dateTime](http://www.w3.org/2001/XMLSchema#dateTime) 

- [excretory duct cell of salivary gland](http://purl.obolibrary.org/obo/CL_4052069) [id](http://www.geneontology.org/formats/oboInOwl#id) "CL:4052069"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [excretory duct cell of salivary gland](http://purl.obolibrary.org/obo/CL_4052069) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A duct epithelial cell that is part of the excretory duct of the salivary gland, which transports saliva from the intralobular ductal system to the oral cavity. While primarily serving as a conduit, this cell may play a role in the saliva modification, contributing to its hypotonicity. In humans, this cell has a pseudostratified columnar shape."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:28192873"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "doi:/10.1007/978-3-030-35784-9_2"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "doi:10.5772/intechopen.75538"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:31194408"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [excretory duct cell of salivary gland](http://purl.obolibrary.org/obo/CL_4052069) [has_exact_synonym](http://www.geneontology.org/formats/oboInOwl#hasExactSynonym) "salivary excretory duct cell"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "doi:/10.1007/978-3-030-35784-9_2"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [excretory duct cell of salivary gland](http://purl.obolibrary.org/obo/CL_4052069) [label](http://www.w3.org/2000/01/rdf-schema#label) "excretory duct cell of salivary gland"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [excretory duct cell of salivary gland](http://purl.obolibrary.org/obo/CL_4052069) SubClassOf [duct epithelial cell](http://purl.obolibrary.org/obo/CL_0000068) 

- [excretory duct cell of salivary gland](http://purl.obolibrary.org/obo/CL_4052069) SubClassOf [salivary gland glandular cell](http://purl.obolibrary.org/obo/CL_1001596) 

- [excretory duct cell of salivary gland](http://purl.obolibrary.org/obo/CL_4052069) SubClassOf [BFO_0000050](http://purl.obolibrary.org/obo/BFO_0000050) some [UBERON_0035049](http://purl.obolibrary.org/obo/UBERON_0035049) 


### externa theca cell `http://purl.obolibrary.org/obo/CL_4052013`

#### Added
- [externa theca cell](http://purl.obolibrary.org/obo/CL_4052013) [has_related_synonym](http://www.geneontology.org/formats/oboInOwl#hasRelatedSynonym) "structural theca cell"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:36599970"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### eye photoreceptor cell `http://purl.obolibrary.org/obo/CL_0000287`

#### Added
- [eye photoreceptor cell](http://purl.obolibrary.org/obo/CL_0000287) [description](http://purl.org/dc/terms/description) "Eye photoreceptor cells, also commonly known as photoreceptors, are specialized types of cells found in the retina, the light-sensitive layer at the back of the eye. They are integral to the complex process of vision, as they are directly involved in converting light into electrical signals. This process is known as phototransduction. There are two primary classes of photoreceptor cells; rods, which function mainly in dim light and provide black-and-white vision, and cones, which function in brighter light and allow for the perception of color.
Rods and cones carry out their roles through sophisticated biochemical reactions. The main component of these reactions is a protein called opsin, which binds to a small molecule known as retinal. In the presence of light, retinal changes its shape, leading to a series of chemical reactions that result in changes to the cell&rsquo;s electrical potential, thus creating an electrical signal that travels to the optic nerve. Rod cells possess only one kind of opsin, called rhodopsin, which only allows them to distinguish between light and dark. Conversely, cone cells have three types of opsins, allowing for the differentiation of various colors.
The spatial distribution of these cell types throughout the retina also contributes to their function. Rod cells are more numerous and are distributed more at the periphery of the retina, contributing to peripheral and night vision. On the other hand, cone cells are concentrated in the macula, a tiny central region in the retina, and with the highest density in the fovea, a small pit in the macula. This distribution of cones lies at the base of fine detail and color vision in the central visual field. In summary, eye photoreceptor cells carry out the primary function of transforming light into visual signals, a role that is crucial for the overall process of vision.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1098/rstb.2009.0051"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1146/annurev.biophys.31.082901.134348"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1186/gb-2005-6-3-213"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/eye.2016.107"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1002/cne.902920402"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [eye photoreceptor cell](http://purl.obolibrary.org/obo/CL_0000287) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0000287"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "eye photoreceptor cell on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### fallopian tube secretory epithelial cell `http://purl.obolibrary.org/obo/CL_4030006`

#### Added
- [fallopian tube secretory epithelial cell](http://purl.obolibrary.org/obo/CL_4030006) [description](http://purl.org/dc/terms/description) "Fallopian tube secretory epithelial cells are non-ciliated cells that form an integral part of the epithelial lining of the fallopian tubes, the delicate conduits leading from the ovaries to the uterus in the female reproductive system. These cells are specialized to produce and secrete various biochemical substances necessary for gamete transportation, fertilization, and early embryonic development.
A key function of the different subtypes of secretory epithelial cells is to release various substances, including proteins, glycoproteins, and ions that create an environment conducive to the journey of the ova from the ovaries to the uterus. The secretions aid in multiple ways, such as nourishing and protecting the oocytes, facilitating their transportation, and preparing them for potential fertilization. Some of these secretions, like tubal fluid, provide the essential medium for the oocyte's transit and the sperm's locomotion, thereby playing a critical role in successful fertilization.
In addition to their secretory function, fallopian tube secretory epithelial cells also contribute significantly to the formation of ciliated cells through differentiation, ensuring a functional fallopian epithelium and, subsequently, successful reproduction. They act as a frontline barrier against ascending infections by maintaining a robust immune environment. Recent studies also indicate their potential relevance in ovarian epithelial cancers, as the transformation of these cells might contribute to the development of this type of neoplasm.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/S0143-4004(99)80008-3"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1530/rep.0.1210339"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.ccr.2013.10.013"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1186/1471-2121-14-43"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.devcel.2022.02.017"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [fallopian tube secretory epithelial cell](http://purl.obolibrary.org/obo/CL_4030006) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_4030006"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "fallopian tube secretory epithelial cell on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### fibroblast of cardiac tissue `http://purl.obolibrary.org/obo/CL_0002548`

#### Added
- [fibroblast of cardiac tissue](http://purl.obolibrary.org/obo/CL_0002548) [description](http://purl.org/dc/terms/description) "Fibroblasts of cardiac tissue, or cardiac fibroblasts, are critical cells involved in the normal function and repair of the heart. They are essentially a type of non-muscle cell that are integral to the connective tissue and structural integrity of the heart. The primary role of cardiac fibroblasts is the synthesis and deposition of extracellular matrix (ECM) components. They synthesize a variety of ECM proteins, such as collagen, fibronectin, and elastin, forming a distinctive three-dimensional network that provides both mechanical and biochemical support for the cardiomyocytes.
Cardiac fibroblasts also participate in crucial cellular communication within the myocardium - the muscular tissue of the heart responsible for pumping blood. They engage in intense cross-talk with cardiomyocytes and endothelial cells, influencing cell survival, growth, differentiation, and tissue repair processes. Particularly notable is their role in cardiac response to injury, such as myocardial infarction, where they proliferate intensely, and subsequently differentiate into myofibroblasts, contributing to scar formation and remodeling of cardiac tissue post-injury.
Moreover, during pathological conditions of the heart, including cardiac fibrosis and heart failure, these fibroblasts can undergo phenotypic transformation into myofibroblasts, leading to an overproduction of ECM components and resulting in stiffening of the cardiac tissue. Over time, this diminishes the heart's ability to pump efficiently, sometimes leading to heart failure. Recent studies have also implicated abnormal behavior of cardiac fibroblasts in arrhythmogenesis.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1253/circj.CJ-16-1003"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1186/1755-1536-5-15"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1161/CIRCRESAHA.109.209809"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1093/cvr/cvaa324"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1126/science.adh9925"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [fibroblast of cardiac tissue](http://purl.obolibrary.org/obo/CL_0002548) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0002548"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "fibroblast of cardiac tissue on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### fibroblast of connective tissue of glandular part of prostate `http://purl.obolibrary.org/obo/CL_1000305`

#### Added
- [fibroblast of connective tissue of glandular part of prostate](http://purl.obolibrary.org/obo/CL_1000305) [description](http://purl.org/dc/terms/description) "Fibroblasts of the connective tissue of the glandular part of the prostate are fibroblasts found within the connective tissue encapsulating and supporting the glandular tissues in the prostate. These cells are responsible for producing and maintaining the extracellular matrix (ECM), a critical component of any connective tissue. The ECM provides a crucial scaffold or architecture for the tissue and plays a pivotal role in cell behavior and function. 
Fibroblast cells in the prostate, in particular, are involved in the regulation of the glandular activities, contributing to the homeostasis of this organ, primarily through their ability to secrete various growth factors, cytokines, and enzymes. They help regulate the growth and differentiation of prostatic epithelial cells, thus playing a role in maintaining proper glandular structure and function. Furthermore, these cells serve as a key conduit of intercellular communication due to their reciprocal interaction with the surrounding epithelial and stromal cells, influencing various aspects of cellular function such as proliferation, differentiation, and migration. 
Additionally, these cells have a significant part in pathological conditions that hit the prostate including prostate cancer and benign prostatic hyperplasia (BPH). In particular, when the prostate is afflicted by cancer, the fibroblasts are known to undergo a reactive change, transforming into myofibroblasts. These altered fibroblasts then exacerbate the disease by encouraging cancer cell growth and invasion, and promoting angiogenesis. Conversely, in BPH, the fibroblasts contribute to the excessive growth of prostatic tissue.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.diff.2011.05.007"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.ucl.2016.04.012"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.3390/cancers12071887"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1101/cshperspect.a030510"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [fibroblast of connective tissue of glandular part of prostate](http://purl.obolibrary.org/obo/CL_1000305) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_1000305"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "fibroblast of connective tissue of glandular part of prostate on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### fibroblast of connective tissue of nonglandular part of prostate `http://purl.obolibrary.org/obo/CL_1000304`

#### Added
- [fibroblast of connective tissue of nonglandular part of prostate](http://purl.obolibrary.org/obo/CL_1000304) [description](http://purl.org/dc/terms/description) "Fibroblasts of connective tissue of nonglandular part of the prostate, also referred to as prostatic stromal fibroblasts, are specialized cells that are instrumental in maintaining the structure and function of the prostate. These cells constitute a crucial part of the non-glandular or stromal compartment of the prostate, which is composed largely of connective tissue and smooth muscle fibers. Being the most abundant cell type in the prostate stroma, fibroblasts are responsible for the production and maintenance of the extracellular matrix (ECM) via synthesis of collagen and elastin, structural proteins that provides crucial supportive and adhesive framework for the glandular elements of the prostate. These proteins provide tensile strength and elasticity, respectively, contributing to the healthy architecture of the prostate tissue. These cells also play a significant role in coordinating intercellular communication within the prostate by interacting with adjacent epithelial cells, endothelial cells, immune cells, and nerve cells, affecting cell growth, differentiation, and function.
Prostatic stromal fibroblasts are actively involved in modulating local immune responses and tissue repair. Acting as local immunomodulatory cells, they can secrete a variety of cytokines and chemokines, influencing the functions of immune cells in the prostate microenvironment. Similarly, under pathological conditions, such as tissue injury or inflammation, these fibroblasts can transition into a more activated state, known as myofibroblasts, driving wound healing and tissue remodeling processes. It's worth noting that aberrant activation and proliferation of these fibroblasts can contribute to pathological situations, such as prostatic hyperplasia and prostate cancer, underscoring their importance in both health and disease.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.ajpath.2013.11.015"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.3892/ijmm.18.5.941"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/s41416-022-02085-x"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1210/me.2010-0340"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1101/cshperspect.a030510"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [fibroblast of connective tissue of nonglandular part of prostate](http://purl.obolibrary.org/obo/CL_1000304) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_1000304"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "fibroblast of connective tissue of nonglandular part of prostate on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### fibroblast of connective tissue of prostate `http://purl.obolibrary.org/obo/CL_1000299`

#### Added
- [fibroblast of connective tissue of prostate](http://purl.obolibrary.org/obo/CL_1000299) [description](http://purl.org/dc/terms/description) "The fibroblasts of connective tissue of the prostate are responsible for the production and maintenance of the extracellular matrix in the stromal region of the prostate gland. Fibroblasts, in general, are ubiquitous in the body's connective tissues, playing vital roles in wound healing and tissue repair. However, within the prostate, these cells exhibit unique characteristics targeted at the organ's health and function, permitting dynamic interactions with other cell types and fostering tissue architecture.
Fibroblasts found in the prostate contribute significantly to the organ's function, particularly in maintaining tissue health and structure. These cells synthesize major structural proteins, such as collagen, elastin, and glycosaminoglycans&mdash;components of the extracellular matrix that provide structural support to the prostate tissue. This ensures the prostate's smooth muscle maintains its capability to contract and expand, thereby enabling proper function during urination and ejaculation. Furthermore, as the prostate gland undergoes adaptive alterations in response to ageing, fibroblasts play an integral role in coordinating these changes through their ability to modulate the extracellular matrix.
Beyond maintaining tissue structure, the fibroblast of connective tissue of the prostate also participates actively in paracrine signaling&mdash;communication with other cell types within the prostate. Such cross-talk is especially significant in settings of inflammation, tissue injury, and in the pathogenesis of diseases such as benign prostatic hyperplasia and prostate cancer. In disease states, these fibroblasts can transition to a reactive phenotype characterized by increased proliferation and secretion of growth factors and pro-inflammatory cytokines. This transition excessively modulates the extracellular matrix, contributing to the diseased state.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1002/pros.23867"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/s41416-022-02085-x"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.3390/jcm7040068"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1530/ERC-12-0085"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1101/cshperspect.a030510"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [fibroblast of connective tissue of prostate](http://purl.obolibrary.org/obo/CL_1000299) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_1000299"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "fibroblast of connective tissue of prostate on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### fibroblast of mammary gland `http://purl.obolibrary.org/obo/CL_0002555`

#### Added
- [fibroblast of mammary gland](http://purl.obolibrary.org/obo/CL_0002555) [description](http://purl.org/dc/terms/description) "Fibroblasts of the mammary gland, often referred to as mammary fibroblasts, are a specialized type of cell located within the connective tissue of the mammary glands. These cells are integral to the structure and composition of the gland and are one of its primary cell types, specializing in the production of the extracellular matrix and collagen. They are involved in tissue remodeling, shaping the structure and maintaining the integrity of the mammary tissue by providing necessary rigidity and elasticity through their production of structural proteins. 
These cells also play a crucial role in mammary gland development and lactation. During mammary gland development, fibroblasts play a significant role in the induction of branching morphogenesis, which is vital for establishing proper ductal architecture. They also support the immune response within the mammary gland by producing and secreting cytokines and growth factors. 
Importantly, mammary fibroblasts are important in the context of breast cancer. Changes in the characteristics and behavior of these fibroblasts, often spurred by interactions with tumor cells, can promote cancer progression and metastasis.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1089/jir.2014.0026"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.3390/jcm11030799"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/onc.2017.65"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.bbadis.2012.10.013"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.canlet.2012.06.014"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [fibroblast of mammary gland](http://purl.obolibrary.org/obo/CL_0002555) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0002555"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "fibroblast of mammary gland on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### follicular dendritic cell `http://purl.obolibrary.org/obo/CL_0000442`

#### Added
- [follicular dendritic cell](http://purl.obolibrary.org/obo/CL_0000442) [description](http://purl.org/dc/terms/description) "Follicular dendritic cells (FDCs) are specialized cells found in B cell follicles in secondary lymphoid organs such as lymph nodes and spleen. FDCs play a key role in the immune response by presenting antigens to B-cells, which are responsible for producing antibodies. They accomplish this by acquiring antigens via multiple pathways and retaining them in non-degradative endosomal vesicles which cycle to the cell surface, allowing B-cells to encounter and recognize native antigens. In addition, FDCs secrete various cytokines and chemokines that direct immune cells to sites of infection or inflammation.
FDCs are unique among immune cells in that they do not originate from hematopoietic stem cells, but rather from stromal cells in the lymphoid tissues. They are characterized by their distinctive shape, with long, branching processes that form a network within the follicle. FDCs are also distinguished by the presence of complement receptors, Fc receptors, and other molecules that enable them to capture and present antigens to B-cells.
Because of their important role in antigen presentation and immune regulation, FDCs have been studied extensively in the context of infectious diseases, autoimmunity, and cancer. Notably, FDCs appear to function as a reservoir for HIV, contribute to autoimmune pathology by presenting autoantigens, and drive FDC sarcoma.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/nri3689"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1002/jcp.30662"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.3389/fimmu.2016.00225"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [follicular dendritic cell](http://purl.obolibrary.org/obo/CL_0000442) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0000442"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "follicular dendritic cell on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### forebrain radial glial cell `http://purl.obolibrary.org/obo/CL_0013000`

#### Added
- [forebrain radial glial cell](http://purl.obolibrary.org/obo/CL_0013000) [description](http://purl.org/dc/terms/description) "Forebrain radial glial cells are specialized cells in the central nervous system known for their essential role in the formation of the brain during neurodevelopment. These cells are primarily located in the ventricular zone of the developing forebrain, which is composed of the cerebrum and the diencephalon. Radial glial cells are named for their unique morphology, specifically their elongated cell bodies, which extend radially from the ventricular surface to the pial surface, acting as a scaffold for newborn neurons to migrate along during the development of the brain.
Functionally, forebrain radial glial cells serve as both progenitor cells and guide cells for neurons. They proliferate to increase their population and produce neurons or glial cells through asymmetric division, contributing to the complexity and diversity of cell types in the brain. During neurogenesis, an important process in brain development, radial glial cells provide physical and biochemical guidance to newborn neurons, helping them migrate from their birthplace in the ventricular zone to their final destination in the cortical plate.
Furthermore, forebrain radial glial cells play a critical role in the formation of the cerebral cortex layers through their interaction with newborn neurons. They are involved in the establishment of the six-layered architecture of the cerebral cortex, crucial to higher cognitive functions, sensory processing, and control of motor behavior. After completing their roles in neurogenesis and the formation of cortical layers, most radial glial cells transform into astrocytes or oligodendrocytes, the main glial cell types in the mature brain. Thus, from their contribution to the proliferation and differentiation of neurons to the establishment of cortical layering, forebrain radial glial cells perform several fundamental roles in brain development.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.neuron.2016.09.005"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/S0070-2153(05)69004-7"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.biocel.2013.11.013"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/35055553"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1002/glia.10244"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [forebrain radial glial cell](http://purl.obolibrary.org/obo/CL_0013000) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0013000"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "forebrain radial glial cell on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### fraction A pre-pro B cell `http://purl.obolibrary.org/obo/CL_0002045`
#### Removed
- [fraction A pre-pro B cell](http://purl.obolibrary.org/obo/CL_0002045) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A pro-B cell that is CD45R/B220-positive, CD43-positive, HSA-low, BP-1-negative and Ly6c-negative. This cell type is also described as being lin-negative, AA4-positive, Kit-positive, IL7Ra-positive and CD45R-positive."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "GOC:ak"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "GOC:tfm"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "http://www.immgen.org/index_content.html"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:17582343"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:12633665"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [fraction A pre-pro B cell](http://purl.obolibrary.org/obo/CL_0002045) EquivalentTo [pro-B cell](http://purl.obolibrary.org/obo/CL_0000826) and ([lacks_plasma_membrane_part](http://purl.obolibrary.org/obo/CL_4030046) some [PR_000002039](http://purl.obolibrary.org/obo/PR_000002039)) and ([lacks_plasma_membrane_part](http://purl.obolibrary.org/obo/CL_4030046) some [PR_000002980](http://purl.obolibrary.org/obo/PR_000002980)) and ([RO_0002104](http://purl.obolibrary.org/obo/RO_0002104) some [PR_000001015](http://purl.obolibrary.org/obo/PR_000001015)) and ([RO_0002104](http://purl.obolibrary.org/obo/RO_0002104) some [PR_000001879](http://purl.obolibrary.org/obo/PR_000001879)) and ([RO_0002104](http://purl.obolibrary.org/obo/RO_0002104) some [PR_000002037](http://purl.obolibrary.org/obo/PR_000002037)) and ([RO_0002104](http://purl.obolibrary.org/obo/RO_0002104) some [PR_000002065](http://purl.obolibrary.org/obo/PR_000002065)) and ([RO_0015016](http://purl.obolibrary.org/obo/RO_0015016) some [PR_000001932](http://purl.obolibrary.org/obo/PR_000001932)) 

#### Added
- [fraction A pre-pro B cell](http://purl.obolibrary.org/obo/CL_0002045) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A pro-B cell that is CD45R/B220-positive, CD43-positive, HSA-low, BP-1-negative and Ly6c-negative. This cell type is also described as being lin-negative, AA4-positive, Kit-positive, IL7Ra-positive and CD45R-positive."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "GOC:ak"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "GOC:tfm"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:37442242"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "http://www.immgen.org/index_content.html"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:17582343"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:12633665"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [fraction A pre-pro B cell](http://purl.obolibrary.org/obo/CL_0002045) EquivalentTo [pro-B cell](http://purl.obolibrary.org/obo/CL_0000826) and ([lacks_plasma_membrane_part](http://purl.obolibrary.org/obo/CL_4030046) some [PR_000002039](http://purl.obolibrary.org/obo/PR_000002039)) and ([lacks_plasma_membrane_part](http://purl.obolibrary.org/obo/CL_4030046) some [PR_000002980](http://purl.obolibrary.org/obo/PR_000002980)) and ([RO_0002104](http://purl.obolibrary.org/obo/RO_0002104) some [PR_000001015](http://purl.obolibrary.org/obo/PR_000001015)) and ([RO_0002104](http://purl.obolibrary.org/obo/RO_0002104) some [PR_000001879](http://purl.obolibrary.org/obo/PR_000001879)) and ([RO_0002104](http://purl.obolibrary.org/obo/RO_0002104) some [PR_000002037](http://purl.obolibrary.org/obo/PR_000002037)) and ([RO_0002104](http://purl.obolibrary.org/obo/RO_0002104) some [PR_000002065](http://purl.obolibrary.org/obo/PR_000002065)) and ([RO_0002162](http://purl.obolibrary.org/obo/RO_0002162) some [NCBITaxon_10090](http://purl.obolibrary.org/obo/NCBITaxon_10090)) and ([RO_0015016](http://purl.obolibrary.org/obo/RO_0015016) some [PR_000001932](http://purl.obolibrary.org/obo/PR_000001932)) 

- [fraction A pre-pro B cell](http://purl.obolibrary.org/obo/CL_0002045) SubClassOf [RO_0002162](http://purl.obolibrary.org/obo/RO_0002162) some [NCBITaxon_10090](http://purl.obolibrary.org/obo/NCBITaxon_10090) 


### fraction B precursor B cell `http://purl.obolibrary.org/obo/CL_0002047`
#### Removed
- [fraction B precursor B cell](http://purl.obolibrary.org/obo/CL_0002047) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A precursor B cell that is CD45RA-positive, CD43-positive, CD24-positive and BP-1-negative."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "GOC:ak"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "GOC:tfm"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:18432934"^^[string](http://www.w3.org/2001/XMLSchema#string) 

#### Added
- [fraction B precursor B cell](http://purl.obolibrary.org/obo/CL_0002047) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A precursor B cell that is CD45RA-positive, CD43-positive, CD24-positive and BP-1-negative."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "GOC:ak"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "GOC:tfm"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:37442242"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:18432934"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### fraction B/C precursor B cell `http://purl.obolibrary.org/obo/CL_0002400`
#### Removed
- [fraction B/C precursor B cell](http://purl.obolibrary.org/obo/CL_0002400) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A precursor B cell that is AA4-positive, IgM-negative, CD19-positive, CD43-positive and HSA-positive."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "GOC:tfm"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "http://www.immgen.org/index_content.html"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [fraction B/C precursor B cell](http://purl.obolibrary.org/obo/CL_0002400) EquivalentTo [precursor B cell](http://purl.obolibrary.org/obo/CL_0000817) and ([lacks_plasma_membrane_part](http://purl.obolibrary.org/obo/CL_4030046) some [GO_0071753](http://purl.obolibrary.org/obo/GO_0071753)) and ([RO_0002104](http://purl.obolibrary.org/obo/RO_0002104) some [PR_000001002](http://purl.obolibrary.org/obo/PR_000001002)) and ([RO_0002104](http://purl.obolibrary.org/obo/RO_0002104) some [PR_000001879](http://purl.obolibrary.org/obo/PR_000001879)) and ([RO_0002104](http://purl.obolibrary.org/obo/RO_0002104) some [PR_000001932](http://purl.obolibrary.org/obo/PR_000001932)) and ([RO_0002104](http://purl.obolibrary.org/obo/RO_0002104) some [PR_000002037](http://purl.obolibrary.org/obo/PR_000002037)) 

#### Added
- [fraction B/C precursor B cell](http://purl.obolibrary.org/obo/CL_0002400) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A precursor B cell that is AA4-positive, IgM-negative, CD19-positive, CD43-positive and HSA-positive."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "GOC:tfm"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:37442242"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "http://www.immgen.org/index_content.html"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [fraction B/C precursor B cell](http://purl.obolibrary.org/obo/CL_0002400) EquivalentTo [precursor B cell](http://purl.obolibrary.org/obo/CL_0000817) and ([lacks_plasma_membrane_part](http://purl.obolibrary.org/obo/CL_4030046) some [GO_0071753](http://purl.obolibrary.org/obo/GO_0071753)) and ([RO_0002104](http://purl.obolibrary.org/obo/RO_0002104) some [PR_000001002](http://purl.obolibrary.org/obo/PR_000001002)) and ([RO_0002104](http://purl.obolibrary.org/obo/RO_0002104) some [PR_000001879](http://purl.obolibrary.org/obo/PR_000001879)) and ([RO_0002104](http://purl.obolibrary.org/obo/RO_0002104) some [PR_000001932](http://purl.obolibrary.org/obo/PR_000001932)) and ([RO_0002104](http://purl.obolibrary.org/obo/RO_0002104) some [PR_000002037](http://purl.obolibrary.org/obo/PR_000002037)) and ([RO_0002162](http://purl.obolibrary.org/obo/RO_0002162) some [NCBITaxon_10090](http://purl.obolibrary.org/obo/NCBITaxon_10090)) 

- [fraction B/C precursor B cell](http://purl.obolibrary.org/obo/CL_0002400) SubClassOf [RO_0002162](http://purl.obolibrary.org/obo/RO_0002162) some [NCBITaxon_10090](http://purl.obolibrary.org/obo/NCBITaxon_10090) 


### fraction C precursor B cell `http://purl.obolibrary.org/obo/CL_0002049`
#### Removed
- [fraction C precursor B cell](http://purl.obolibrary.org/obo/CL_0002049) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A precursor B cell that is CD45R-positive, CD43-positive, CD24-positive, and BP-positive. Intracellularly expression of surrogate light chain, Rag1 and Rag2, TdT, occurs while there is no expression of mu heavy chain."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "GOC:tfm"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:17582343"^^[string](http://www.w3.org/2001/XMLSchema#string) 

#### Added
- [fraction C precursor B cell](http://purl.obolibrary.org/obo/CL_0002049) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A precursor B cell that is CD45R-positive, CD43-positive, CD24-positive, and BP-positive. Intracellularly expression of surrogate light chain, Rag1 and Rag2, TdT, occurs while there is no expression of mu heavy chain."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "GOC:tfm"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:37442242"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:17582343"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### fraction C' precursor B cell `http://purl.obolibrary.org/obo/CL_0002050`
#### Removed
- [fraction C' precursor B cell](http://purl.obolibrary.org/obo/CL_0002050) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A pre-BCR-positive precursor B cell that is CD24-high, CD25-positive, CD43-positive, CD45R-positive and BP-positive."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "GOC:tfm"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:17582343"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [fraction C' precursor B cell](http://purl.obolibrary.org/obo/CL_0002050) EquivalentTo [precursor B cell](http://purl.obolibrary.org/obo/CL_0000817) and ([RO_0002104](http://purl.obolibrary.org/obo/RO_0002104) some [GO_0035369](http://purl.obolibrary.org/obo/GO_0035369)) and ([RO_0002104](http://purl.obolibrary.org/obo/RO_0002104) some [PR_000001014](http://purl.obolibrary.org/obo/PR_000001014)) and ([RO_0002104](http://purl.obolibrary.org/obo/RO_0002104) some [PR_000001380](http://purl.obolibrary.org/obo/PR_000001380)) and ([RO_0002104](http://purl.obolibrary.org/obo/RO_0002104) some [PR_000001879](http://purl.obolibrary.org/obo/PR_000001879)) and ([RO_0002104](http://purl.obolibrary.org/obo/RO_0002104) some [PR_000002039](http://purl.obolibrary.org/obo/PR_000002039)) and ([RO_0015015](http://purl.obolibrary.org/obo/RO_0015015) some [PR_000001932](http://purl.obolibrary.org/obo/PR_000001932)) 

#### Added
- [fraction C' precursor B cell](http://purl.obolibrary.org/obo/CL_0002050) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A pre-BCR-positive precursor B cell that is CD24-high, CD25-positive, CD43-positive, CD45R-positive and BP-positive."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "GOC:tfm"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:37442242"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:17582343"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [fraction C' precursor B cell](http://purl.obolibrary.org/obo/CL_0002050) EquivalentTo [precursor B cell](http://purl.obolibrary.org/obo/CL_0000817) and ([RO_0002104](http://purl.obolibrary.org/obo/RO_0002104) some [GO_0035369](http://purl.obolibrary.org/obo/GO_0035369)) and ([RO_0002104](http://purl.obolibrary.org/obo/RO_0002104) some [PR_000001014](http://purl.obolibrary.org/obo/PR_000001014)) and ([RO_0002104](http://purl.obolibrary.org/obo/RO_0002104) some [PR_000001380](http://purl.obolibrary.org/obo/PR_000001380)) and ([RO_0002104](http://purl.obolibrary.org/obo/RO_0002104) some [PR_000001879](http://purl.obolibrary.org/obo/PR_000001879)) and ([RO_0002104](http://purl.obolibrary.org/obo/RO_0002104) some [PR_000002039](http://purl.obolibrary.org/obo/PR_000002039)) and ([RO_0002162](http://purl.obolibrary.org/obo/RO_0002162) some [NCBITaxon_10090](http://purl.obolibrary.org/obo/NCBITaxon_10090)) and ([RO_0015015](http://purl.obolibrary.org/obo/RO_0015015) some [PR_000001932](http://purl.obolibrary.org/obo/PR_000001932)) 


### fraction D precursor B cell `http://purl.obolibrary.org/obo/CL_0002052`
#### Removed
- [fraction D precursor B cell](http://purl.obolibrary.org/obo/CL_0002052) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A pre-B cell that is pre-BCR-negative, and the kappa- and lambda- light immunoglobulin light chain-negative, CD43-low, and is BP-1-positive, CD45R-positive and CD25-positive. This cell type is also described as being AA4-positive, IgM-negative, CD19-positive, CD43-low/negative, and HSA-positive."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "GOC:ak"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "GOC:dsd"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "GOC:tfm"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "http://www.immgen.org/index_content.html"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:18432934"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:17582343"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [fraction D precursor B cell](http://purl.obolibrary.org/obo/CL_0002052) EquivalentTo [precursor B cell](http://purl.obolibrary.org/obo/CL_0000817) and ([has_not_completed](http://purl.obolibrary.org/obo/CL_4030044) some [GO_0071708](http://purl.obolibrary.org/obo/GO_0071708)) and ([lacks_plasma_membrane_part](http://purl.obolibrary.org/obo/CL_4030046) some [GO_0035369](http://purl.obolibrary.org/obo/GO_0035369)) and ([RO_0002104](http://purl.obolibrary.org/obo/RO_0002104) some [PR_000001002](http://purl.obolibrary.org/obo/PR_000001002)) and ([RO_0002104](http://purl.obolibrary.org/obo/RO_0002104) some [PR_000001014](http://purl.obolibrary.org/obo/PR_000001014)) and ([RO_0002104](http://purl.obolibrary.org/obo/RO_0002104) some [PR_000001380](http://purl.obolibrary.org/obo/PR_000001380)) and ([RO_0002104](http://purl.obolibrary.org/obo/RO_0002104) some [PR_000001932](http://purl.obolibrary.org/obo/PR_000001932)) and ([RO_0002104](http://purl.obolibrary.org/obo/RO_0002104) some [PR_000002037](http://purl.obolibrary.org/obo/PR_000002037)) and ([RO_0002104](http://purl.obolibrary.org/obo/RO_0002104) some [PR_000002039](http://purl.obolibrary.org/obo/PR_000002039)) and ([RO_0015016](http://purl.obolibrary.org/obo/RO_0015016) some [PR_000001879](http://purl.obolibrary.org/obo/PR_000001879)) 

#### Added
- [fraction D precursor B cell](http://purl.obolibrary.org/obo/CL_0002052) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A pre-B cell that is pre-BCR-negative, and the kappa- and lambda- light immunoglobulin light chain-negative, CD43-low, and is BP-1-positive, CD45R-positive and CD25-positive. This cell type is also described as being AA4-positive, IgM-negative, CD19-positive, CD43-low/negative, and HSA-positive."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "GOC:ak"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "GOC:dsd"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "GOC:tfm"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:37442242"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "http://www.immgen.org/index_content.html"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:18432934"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:17582343"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [fraction D precursor B cell](http://purl.obolibrary.org/obo/CL_0002052) EquivalentTo [precursor B cell](http://purl.obolibrary.org/obo/CL_0000817) and ([has_not_completed](http://purl.obolibrary.org/obo/CL_4030044) some [GO_0071708](http://purl.obolibrary.org/obo/GO_0071708)) and ([lacks_plasma_membrane_part](http://purl.obolibrary.org/obo/CL_4030046) some [GO_0035369](http://purl.obolibrary.org/obo/GO_0035369)) and ([RO_0002104](http://purl.obolibrary.org/obo/RO_0002104) some [PR_000001002](http://purl.obolibrary.org/obo/PR_000001002)) and ([RO_0002104](http://purl.obolibrary.org/obo/RO_0002104) some [PR_000001014](http://purl.obolibrary.org/obo/PR_000001014)) and ([RO_0002104](http://purl.obolibrary.org/obo/RO_0002104) some [PR_000001380](http://purl.obolibrary.org/obo/PR_000001380)) and ([RO_0002104](http://purl.obolibrary.org/obo/RO_0002104) some [PR_000001932](http://purl.obolibrary.org/obo/PR_000001932)) and ([RO_0002104](http://purl.obolibrary.org/obo/RO_0002104) some [PR_000002037](http://purl.obolibrary.org/obo/PR_000002037)) and ([RO_0002104](http://purl.obolibrary.org/obo/RO_0002104) some [PR_000002039](http://purl.obolibrary.org/obo/PR_000002039)) and ([RO_0002162](http://purl.obolibrary.org/obo/RO_0002162) some [NCBITaxon_10090](http://purl.obolibrary.org/obo/NCBITaxon_10090)) and ([RO_0015016](http://purl.obolibrary.org/obo/RO_0015016) some [PR_000001879](http://purl.obolibrary.org/obo/PR_000001879)) 


### fraction E immature B cell `http://purl.obolibrary.org/obo/CL_0002054`
#### Removed
- [fraction E immature B cell](http://purl.obolibrary.org/obo/CL_0002054) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "An immature B cell that is IgM-positive, CD45R-positive, CD43-low, CD25-negative, and CD127-negative. This cell type has also been described as being AA4-positive, IgM-positive, CD19-positive, CD43-low/negative, and HSA-positive."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "GOC:ak"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "GOC:dsd"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "GOC:tfm"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:18432934"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:17582343"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [fraction E immature B cell](http://purl.obolibrary.org/obo/CL_0002054) EquivalentTo [immature B cell](http://purl.obolibrary.org/obo/CL_0000816) and ([lacks_plasma_membrane_part](http://purl.obolibrary.org/obo/CL_4030046) some [PR_000001380](http://purl.obolibrary.org/obo/PR_000001380)) and ([lacks_plasma_membrane_part](http://purl.obolibrary.org/obo/CL_4030046) some [PR_000001869](http://purl.obolibrary.org/obo/PR_000001869)) and ([RO_0002104](http://purl.obolibrary.org/obo/RO_0002104) some [PR_000001002](http://purl.obolibrary.org/obo/PR_000001002)) and ([RO_0002104](http://purl.obolibrary.org/obo/RO_0002104) some [PR_000001932](http://purl.obolibrary.org/obo/PR_000001932)) and ([RO_0002104](http://purl.obolibrary.org/obo/RO_0002104) some [PR_000002037](http://purl.obolibrary.org/obo/PR_000002037)) and ([RO_0015016](http://purl.obolibrary.org/obo/RO_0015016) some [PR_000001879](http://purl.obolibrary.org/obo/PR_000001879)) 

#### Added
- [fraction E immature B cell](http://purl.obolibrary.org/obo/CL_0002054) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "An immature B cell that is IgM-positive, CD45R-positive, CD43-low, CD25-negative, and CD127-negative. This cell type has also been described as being AA4-positive, IgM-positive, CD19-positive, CD43-low/negative, and HSA-positive."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "GOC:ak"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "GOC:dsd"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "GOC:tfm"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:37442242"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:18432934"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:17582343"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [fraction E immature B cell](http://purl.obolibrary.org/obo/CL_0002054) EquivalentTo [immature B cell](http://purl.obolibrary.org/obo/CL_0000816) and ([lacks_plasma_membrane_part](http://purl.obolibrary.org/obo/CL_4030046) some [PR_000001380](http://purl.obolibrary.org/obo/PR_000001380)) and ([lacks_plasma_membrane_part](http://purl.obolibrary.org/obo/CL_4030046) some [PR_000001869](http://purl.obolibrary.org/obo/PR_000001869)) and ([RO_0002104](http://purl.obolibrary.org/obo/RO_0002104) some [PR_000001002](http://purl.obolibrary.org/obo/PR_000001002)) and ([RO_0002104](http://purl.obolibrary.org/obo/RO_0002104) some [PR_000001932](http://purl.obolibrary.org/obo/PR_000001932)) and ([RO_0002104](http://purl.obolibrary.org/obo/RO_0002104) some [PR_000002037](http://purl.obolibrary.org/obo/PR_000002037)) and ([RO_0002162](http://purl.obolibrary.org/obo/RO_0002162) some [NCBITaxon_10090](http://purl.obolibrary.org/obo/NCBITaxon_10090)) and ([RO_0015016](http://purl.obolibrary.org/obo/RO_0015016) some [PR_000001879](http://purl.obolibrary.org/obo/PR_000001879)) 


### fraction F mature B cell `http://purl.obolibrary.org/obo/CL_0002056`
#### Removed
- [fraction F mature B cell](http://purl.obolibrary.org/obo/CL_0002056) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A mature B cell subset originally defined as having being CD45R-positive, IgM-positive, IgD-positive and CD43-negative. Subsequent research demonstrated being CD21-positive and CD23-negative and CD93 negative."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "GOC:tfm"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:17582343"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [fraction F mature B cell](http://purl.obolibrary.org/obo/CL_0002056) EquivalentTo [mature B cell](http://purl.obolibrary.org/obo/CL_0000785) and ([lacks_plasma_membrane_part](http://purl.obolibrary.org/obo/CL_4030046) some [PR_000001879](http://purl.obolibrary.org/obo/PR_000001879)) and ([lacks_plasma_membrane_part](http://purl.obolibrary.org/obo/CL_4030046) some [PR_000001880](http://purl.obolibrary.org/obo/PR_000001880)) and ([lacks_plasma_membrane_part](http://purl.obolibrary.org/obo/CL_4030046) some [PR_000002037](http://purl.obolibrary.org/obo/PR_000002037)) and ([RO_0002104](http://purl.obolibrary.org/obo/RO_0002104) some [GO_0071738](http://purl.obolibrary.org/obo/GO_0071738)) and ([RO_0002104](http://purl.obolibrary.org/obo/RO_0002104) some [GO_0071753](http://purl.obolibrary.org/obo/GO_0071753)) and ([RO_0002104](http://purl.obolibrary.org/obo/RO_0002104) some [PR_000001014](http://purl.obolibrary.org/obo/PR_000001014)) and ([RO_0002104](http://purl.obolibrary.org/obo/RO_0002104) some [PR_000001289](http://purl.obolibrary.org/obo/PR_000001289)) and ([RO_0002104](http://purl.obolibrary.org/obo/RO_0002104) some [PR_000001338](http://purl.obolibrary.org/obo/PR_000001338)) 

#### Added
- [fraction F mature B cell](http://purl.obolibrary.org/obo/CL_0002056) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A mature B cell subset originally defined as having being CD45R-positive, IgM-positive, IgD-positive and CD43-negative. Subsequent research demonstrated being CD21-positive and CD23-negative and CD93 negative."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "GOC:tfm"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:37442242"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:17582343"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [fraction F mature B cell](http://purl.obolibrary.org/obo/CL_0002056) EquivalentTo [mature B cell](http://purl.obolibrary.org/obo/CL_0000785) and ([lacks_plasma_membrane_part](http://purl.obolibrary.org/obo/CL_4030046) some [PR_000001879](http://purl.obolibrary.org/obo/PR_000001879)) and ([lacks_plasma_membrane_part](http://purl.obolibrary.org/obo/CL_4030046) some [PR_000001880](http://purl.obolibrary.org/obo/PR_000001880)) and ([lacks_plasma_membrane_part](http://purl.obolibrary.org/obo/CL_4030046) some [PR_000002037](http://purl.obolibrary.org/obo/PR_000002037)) and ([RO_0002104](http://purl.obolibrary.org/obo/RO_0002104) some [GO_0071738](http://purl.obolibrary.org/obo/GO_0071738)) and ([RO_0002104](http://purl.obolibrary.org/obo/RO_0002104) some [GO_0071753](http://purl.obolibrary.org/obo/GO_0071753)) and ([RO_0002104](http://purl.obolibrary.org/obo/RO_0002104) some [PR_000001014](http://purl.obolibrary.org/obo/PR_000001014)) and ([RO_0002104](http://purl.obolibrary.org/obo/RO_0002104) some [PR_000001289](http://purl.obolibrary.org/obo/PR_000001289)) and ([RO_0002104](http://purl.obolibrary.org/obo/RO_0002104) some [PR_000001338](http://purl.obolibrary.org/obo/PR_000001338)) and ([RO_0002162](http://purl.obolibrary.org/obo/RO_0002162) some [NCBITaxon_10090](http://purl.obolibrary.org/obo/NCBITaxon_10090)) 


### gamma-delta T cell `http://purl.obolibrary.org/obo/CL_0000798`

#### Added
- [gamma-delta T cell](http://purl.obolibrary.org/obo/CL_0000798) [description](http://purl.org/dc/terms/description) "Gamma-delta T cells, also referred to as &gamma;&delta; T cells, are a distinct subset of T lymphocytes that play unique roles in the immune system. They are defined by expression of heterodimeric T-cell receptors (TCRs) comprised of &gamma; and &delta; chains. They share characteristics with both innate and adaptive immune cells, and unlike alpha-beta (&alpha;&beta;) T cells, they do not require antigen processing and Major Histocompatibility Complex (MHC) presentation for the recognition of antigens. This unique response mechanism allows them to contribute to rapid immune responses, particularly in the context of tissue injury and infection.
Depending on the &gamma;&delta; TCR and the context, different subsets of gamma-delta T cells perform a variety of functions which range from regulating thermogenesis in adipose tissue to controlling the plasticity of neuronal synapses in the central nervous system. A key function is in tissue homeostasis and surveillance against infectious agents and tumors. 
They can carry out their defense role through several mechanisms, such as cytokine production, killing of target cells, and regulation of other immune cell responses. In response to pathogens, these cells produce a wide range of cytokines, including interferon-gamma (IFN-&gamma;), tumor necrosis factor-alpha (TNF-&alpha;), and interleukin-17 (IL-17), which help recruit and activate other immune cells to the site of infection. Furthermore, gamma-delta T cells can directly kill infected cells or cancerous cells, which is instrumental in preventing the spread of infection or malignancy.
Despite the broadly generalized role in immunity, the function of gamma-delta T cells can vastly vary depending on their tissue location. In the skin and mucosal tissues, they provide quick response to pathogens, while in secondary lymphoid organs, their regulatory functions shape the adaptive immune response. In recent years, their role in the context of autoimmunity and cancer has gained attention, with emerging studies suggesting their potential for immunotherapeutic interventions. Thus, gamma-delta T cells serve as vital components of the immune system, providing a versatile defense mechanism against a range of pathogenic targets.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1146/annurev-immunol-032713-120216"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "https://www.mdpi.com/journal/cells/special_issues/delta_T"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1073/pnas.1818488116"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/s41571-022-00722-1"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/s41577-020-00452-4"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [gamma-delta T cell](http://purl.obolibrary.org/obo/CL_0000798) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0000798"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "gamma-delta T cell on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### gamma-delta intraepithelial T cell `http://purl.obolibrary.org/obo/CL_0000801`

#### Added
- [gamma-delta intraepithelial T cell](http://purl.obolibrary.org/obo/CL_0000801) [description](http://purl.org/dc/terms/description) "Gamma-delta intraepithelial T cells, also known as &gamma;&delta; intraepithelial lymphocytes (IELs), are a unique subset of T cells residing in the epithelial layer of mucosal tissues such as the intestine and skin; more are found in intestinal intraepithelial tissue than in other tissues. They are characterized by their distinct T-cell receptor (TCR) that is composed of one gamma (&gamma;) and one delta (&delta;) chain, unlike the conventional alpha-beta T cells. Most &gamma;&delta; IELs also express the CD8-alpha-alpha homodimer (Cd8-alpha alpha positive, gamma-delta intraepithelial T cells), although some are lacking both co-receptors CD4 and CD8 (CD4-negative CD8-negative gamma-delta intraepithelial T cells). &gamma;&delta; IELs are though to be &lsquo;natural&rsquo; IELs as they acquire their activated phenotype in an &lsquo;alternative&rsquo; self-antigen-based maturation process in the thymus. 
Although the function of &gamma;&delta; IELs is still not fully established, current evidence indicates that they serve as  a crucial first line of defense against invading pathogens, thus making them an important class of immune cells at the epithelial barrier interfaces. &gamma;&delta; IELs respond to a variety of antigens, including those not recognized by conventional T cells. They are also thought to be involved in recruiting in immune surveillance by promoting homeostasis of host microbiota and in limiting excessive inflammation.
In addition to their primary role in immunity, &gamma;&delta; IELs also have roles in epithelial cell homeostasis and repair. They assist in maintaining the epithelial barrier by controlling the equilibrium of epithelial cell growth and death. They also contribute to the healing of lesions and the replacement of damaged epithelial cells. The regulative control exerted by &gamma;&delta; IELs over epithelial cell turnover indirectly influences immunity, underlining the complexity and versatility of these cells.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/s41385-020-0294-6"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.3390/cells9030686"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.3389/fimmu.2022.1059042"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [gamma-delta intraepithelial T cell](http://purl.obolibrary.org/obo/CL_0000801) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0000801"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "gamma-delta intraepithelial T cell on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### ganglion interneuron `http://purl.obolibrary.org/obo/CL_0000397`

#### Added
- [ganglion interneuron](http://purl.obolibrary.org/obo/CL_0000397) [description](http://purl.org/dc/terms/description) "Ganglion interneurons, sometimes classified under 'relay neurons,' are complex cellular structures prominently found in the nervous system across various organisms. These cells serve as intermediaries or connectors, conducting impulses between afferent (or sensory) neurons and efferent (or motor) neurons within the body. They have specialized roles in sensory-motor integration within the nervous system and play a pivotal part in defining the complexity and precision of the nervous system.
These highly specialized neurons are predominantly located in the spinal cord and brain, particularly abundant in the gray matter of these regions, forming complex neural networks. The increased presence of glial cells around interneurons acts as an insulator enhancing the speed of impulse transmission. Ganglion interneurons receive informational input through dendrites from sensory neurons; this information is then converted into electrical impulses, which are processed and transmitted to other neurons or effectors, such as muscles or glands, through axons.
Furthermore, these ganglion interneurons have significant roles in the formulation and execution of motor reflexes and serve as integral components of reflex arcs. They also participate in the modulation and filtration of relevant sensory information, ensuring that signals reaching the late-stage processing units (like the brain) are refined and critical to the organism's survival. Consequently, these neurons significantly influence the organism's behavior based on the incoming sensory information. Overall, they are responsible for creating connections amongst various neural circuits, making them pivotal in producing integrated responses and maintaining the proper functioning of the nervous system.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1371/journal.pcbi.1002626"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1111/j.1748-1716.1962.tb02497.x"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1152/jn.1992.68.4.1384"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1523/JNEUROSCI.5570-03.2004"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/nn.2707"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [ganglion interneuron](http://purl.obolibrary.org/obo/CL_0000397) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0000397"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "ganglion interneuron on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### glioblast `http://purl.obolibrary.org/obo/CL_0000030`

#### Added
- [glioblast](http://purl.obolibrary.org/obo/CL_0000030) [description](http://purl.org/dc/terms/description) "Glioblasts are primarily found in the central nervous system and are an integral part of the developing and mature brain tissue. They are primitive progenitor cells with the ability to self-renew and can differentiate into neuroglia, including astrocytes and oligodendrocytes, two primary cells involved in neurotransmission and protection of brain tissue. 
When glioblast multiplication is dysregulated, it can lead to harmful manifestations, most notably in the form of an aggressive brain tumor known as Glioblastoma multiforme, one of the most malignant types of neural cancers.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.4252/wjsc.v6.i2.230"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1186/s13064-016-0075-9"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.12688/f1000research.22405.1"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1073/pnas.97.12.6242"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [glioblast](http://purl.obolibrary.org/obo/CL_0000030) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0000030"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "glioblast on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### glutamatergic neuron of the basal ganglia `http://purl.obolibrary.org/obo/CL_4042040`

#### Added
- [glutamatergic neuron of the basal ganglia](http://purl.obolibrary.org/obo/CL_4042040) [RO_0002175](http://purl.obolibrary.org/obo/RO_0002175) [NCBITaxon_9606](http://purl.obolibrary.org/obo/NCBITaxon_9606) 

- [glutamatergic neuron of the basal ganglia](http://purl.obolibrary.org/obo/CL_4042040) [in_subset](http://www.geneontology.org/formats/oboInOwl#inSubset) [human_reference_atlas](http://purl.obolibrary.org/obo/cl/cl-base#human_reference_atlas) 


### glycinergic neuron `http://purl.obolibrary.org/obo/CL_1001509`

#### Added
- [glycinergic neuron](http://purl.obolibrary.org/obo/CL_1001509) [description](http://purl.org/dc/terms/description) "Glycinergic neurons are a specialized type of neuron that primarily utilizes the inhibitory neurotransmitter glycine to transmit signals within the nervous system. Originating from the term 'glycine' and 'ergic' (pertaining to activity), glycinergic neurons contain and release glycine which acts on specific receptors, known as glycine receptors, found on the surface of neuron cells. Glycinergic neurons are predominantly found in the spinal cord and brainstem, but also exist in the retina and other parts of the brain.
These neurons are critical to the inhibitory control of neuronal networks, helping to regulate excitation and maintaining neuronal stability. They play a primary role in modulating and controlling motor and sensory information within the nervous system. In general, when a glycinergic neuron fires, it releases glycine, which binds to glycine receptors on a nearby neuron. This binding causes the neuron membrane to hyperpolarize, subsequently reducing the probability of the neuron firing an action potential, which in essence dampens or inhibits the neuron from transmitting further signals. This function plays a vital role in preventing overexcitation, which may lead to neuronal damage or neurodegenerative conditions.
Moreover, glycinergic neurons are integral to certain sensory processes and motor coordination. In the auditory system, these neurons aid in the localization of sound, and in the spinal cord, they are involved in the regulation of motor reflexes and rhythms. Any dysfunction in the glycinergic system can cause serious neurological conditions, such as hyperekplexia, characterized by increased startle responses and periodic bouts of rigidity, and some forms of chronic pain.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1042/NS20160009"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1007/BF02740653"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.pneurobio.2010.04.008"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1093/brain/awx289"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1111/j.1399-6576.2005.00786.x"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [glycinergic neuron](http://purl.obolibrary.org/obo/CL_1001509) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_1001509"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "glycinergic neuron on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### granule cell `http://purl.obolibrary.org/obo/CL_0000120`

#### Added
- [granule cell](http://purl.obolibrary.org/obo/CL_0000120) [description](http://purl.org/dc/terms/description) "Granule cells are a class of small neurons found significantly in the cerebellum, olfactory bulb, and the dentate gyrus of the hippocampus in the human central nervous system. Their name derives from the presence of internal granular structures and they are one of the most abundant cell types in the entire brain, with estimates suggesting they likely comprise over half of all neurons. Due to their location, these cells play a vital role in the modulation of motor coordination, the olfactory system, and some aspects of memory and learning.
Structurally, granule cells differ depending on their location. They have unmyelinated, very thin, T-shaped axons and a few short dendrites in the cerebellum. These dendrites receive information from a variety of sources, while their axons transmit the information to other parts of the brain. In granule cells of the cerebellum, the axons form parallel fibers that run into the molecular layer and synapse onto Purkinje cells, stellate, and basket cells, setting up a neuronal circuit critical for generating precise and fluid movements.
Functionally, granule cells receive excitatory input from mossy fibers, which deliver sensory and motor information to the cerebellum, and this interaction is fundamental for the timing and coordination of body movements. In the olfactory bulb, granule cells process and modify olfactory information before passing it to the cerebral cortex, thereby aiding smell perception. Meanwhile, in the hippocampus, these cells participate in the formation and sorting of memories, particularly episodic memories. Due to their number, location, and function, the granule cells have a significant influence on cognition and behavior.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.brainresrev.2007.03.005"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/S0168-0102(97)00099-0"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.neubiorev.2016.02.021"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.12688/f1000research.15021.1"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.3389/fncir.2020.611841"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [granule cell](http://purl.obolibrary.org/obo/CL_0000120) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0000120"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "granule cell on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### group 1 innate lymphoid cell `http://purl.obolibrary.org/obo/CL_0001067`

#### Added
- [group 1 innate lymphoid cell](http://purl.obolibrary.org/obo/CL_0001067) [description](http://purl.org/dc/terms/description) "Group 1 Innate lymphoid cells (ILC1s) are one of five subsets of innate lymphoid cells (ILCs). ILC1s are largely tissue-resident cells and express the transcription factor T-bet and predominantly produce the cytokine interferon-gamma (IFN-&gamma;). ILC1s initially included natural killer (NK) cells and T-bet+ ILC1s, but a 2018 update to the nomenclature proposed to further separate these two cell types based upon differences in development and function. However, it can be difficult to distinguish between these cell types.
ILC1s are non-cytotoxic or weakly cytotoxic, expressing relatively low levels of perforin, and play an important role in the response to viruses, certain intracellular bacteria and parasites, and fungi. Their release of IFN-&gamma; and tumor necrosis factor (TNF) further shapes the adaptive immune response. Their functionality mirrors Th1 helper cells.
While ILC1s have been shown to be important in the response to infections, ILC1s have also been demonstrated to accumulate in inflammatory diseases such as Crohn&rsquo;s disease, chronic obstructive pulmonary disease, and rheumatoid arthritis.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1002/eji.201747294"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/nri.2017.86"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1101/cshperspect.a030304"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/nature14189"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.cell.2018.07.017"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [group 1 innate lymphoid cell](http://purl.obolibrary.org/obo/CL_0001067) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0001067"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "group 1 innate lymphoid cell on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### group 2 innate lymphoid cell `http://purl.obolibrary.org/obo/CL_0001069`

#### Added
- [group 2 innate lymphoid cell](http://purl.obolibrary.org/obo/CL_0001069) [description](http://purl.org/dc/terms/description) "Group 2 innate lymphoid cells, also known as ILC2 cells, are a subset of the innate lymphoid cell family - a family of immune cells crucial for host defense, immune regulation, and tissue repair especially in the context of infections and inflammatory conditions. These cells are largely tissue-resident, predominantly residing in barrier tissues such as the skin, lungs, and gastrointestinal tract. ILC2s are defined by their expression and reliance on the GATA3 transcription factor.
ILC2 cells play a significant role in the immune response, especially against parasitic infections and in allergic responses and autoimmune disorders. They predominantly produce type 2 cytokines including interleukin-5 (IL-5) and interleukin-13 (IL-13) in response to stimuli such as cytokines IL-25 and IL-33, which are often produced by epithelial cells in response to tissue damage. IL-5 is crucial for eosinophil activation, a component of the immune system that plays a key role in combating parasitic infections. Concurrently, IL-13 helps in expelling parasites from the body's system, as well as enforcing barrier integrity.
ILC2s are also instrumental in tissue repair and homeostasis. They help in maintaining body metabolism and fat storage, largely by producing a hormone known as methionine-enkephalin peptides. ILC2 cells also aid in early tissue repair processes, by the secretion of amphiregulin, a growth factor. Despite their protective functions, ILC2 cells can also contribute to pathologies such as asthma and other inflammatory diseases if their actions are not properly regulated.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1002/eji.201747294"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/nri.2017.86"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1101/cshperspect.a030304"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/nature14189"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.cell.2018.07.017"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [group 2 innate lymphoid cell](http://purl.obolibrary.org/obo/CL_0001069) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0001069"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "group 2 innate lymphoid cell on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### group 3 innate lymphoid cell `http://purl.obolibrary.org/obo/CL_0001071`

#### Added
- [group 3 innate lymphoid cell](http://purl.obolibrary.org/obo/CL_0001071) [description](http://purl.org/dc/terms/description) "Group 3 innate lymphoid cells (ILC3) belong to the family of innate lymphoid cells. They share a high degree of similarity with T lymphocytes, particularly in terms of their transcriptional program and their cytokine production profiles, but lack the antigen-specific receptors that characterize adaptive immune cells. Originating from common lymphoid progenitors, ILC3s are typically found in mucosal tissues and are especially prevalent in the gastrointestinal tract. The grouping of ILC3s originally included lymphoid tissue inducer (LTi) cells, which have more recently been categorized separately.
ILC3s play an important homeostatic role in the gastrointestinal tract, contributing to mucosal barrier integrity, lymphoid organogenesis, and controlling the balance between tolerance and immunity to commensal and pathogenic bacteria, respectively. The protective role of ILC3s primarily stems from their capacity to respond to signals of microbial invasion or tissue damage, releasing an array of cytokines such as Interleukin-22 (IL-22) and Interleukin-17 (IL-17) which are dependent upon the ROR&gamma;t transcription factor. These cytokines in turn mediate protective responses including the enhancement of epithelial barrier function, the promotion of tissue repair, and the control of bacterial infection.
However, ILC3s can also contribute to the development of inflammatory diseases such as inflammatory bowel disease (IBD) and psoriasis. Therefore, studies have also emphasized the importance of regulatory mechanisms controlling ILC3 activation, providing novel insights into the complexity of immune regulation in maintaining tissue homeostasis.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1002/eji.201747294"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/nri.2017.86"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1101/cshperspect.a030304"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/nature14189"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.cell.2018.07.017"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [group 3 innate lymphoid cell](http://purl.obolibrary.org/obo/CL_0001071) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0001071"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "group 3 innate lymphoid cell on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### gut mesenchymal lymphoid tissue organizer cell `http://purl.obolibrary.org/obo/CL_0020000`

#### Added
- Class: [gut mesenchymal lymphoid tissue organizer cell](http://purl.obolibrary.org/obo/CL_0020000) 

- [gut mesenchymal lymphoid tissue organizer cell](http://purl.obolibrary.org/obo/CL_0020000) [contributor](http://purl.org/dc/terms/contributor) "https://orcid.org/0009-0000-8480-9277"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [gut mesenchymal lymphoid tissue organizer cell](http://purl.obolibrary.org/obo/CL_0020000) [date](http://purl.org/dc/terms/date) "2025-01-27T00:00:00Z"^^[dateTime](http://www.w3.org/2001/XMLSchema#dateTime) 

- [gut mesenchymal lymphoid tissue organizer cell](http://purl.obolibrary.org/obo/CL_0020000) [id](http://www.geneontology.org/formats/oboInOwl#id) "CL:0020000"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [gut mesenchymal lymphoid tissue organizer cell](http://purl.obolibrary.org/obo/CL_0020000) [has_related_synonym](http://www.geneontology.org/formats/oboInOwl#hasRelatedSynonym) "gut mLTo cell"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [has_synonym_type](http://www.geneontology.org/formats/oboInOwl#hasSynonymType) [OMO_0003000](http://purl.obolibrary.org/obo/OMO_0003000) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:31201308"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [gut mesenchymal lymphoid tissue organizer cell](http://purl.obolibrary.org/obo/CL_0020000) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A mesenchymal stromal cell found in the gut during development, and interacts with lymphoid tissue inducer (LTi) cells to drive secondary lymphoid organ formation (Prados et al., 2021). It expresses LT&beta;R, PDGFR&alpha;, and VCAM, and upon activation secretes chemokines(e.g. CXCL13) and cytokines to organise lymphoid tissue architecture and recruit immune cells in both mice and humans (Denton el al., 2019; Elmentaite et al., 2021). As a multipotent progenitor, an mLTo cell differentiates into adult stromal subsets, including fibroblastic reticular cells (FRCs) (Prados et al., 2021), and its descendants persist in the adult gut-associated lymphoid tissues to maintain immune homeostasis(Denton el al., 2019; L&uuml;tge et al., 2021)."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:34497389"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:33707780"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:31324739"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:34046914"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [gut mesenchymal lymphoid tissue organizer cell](http://purl.obolibrary.org/obo/CL_0020000) [label](http://www.w3.org/2000/01/rdf-schema#label) "gut mesenchymal lymphoid tissue organizer cell"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [gut mesenchymal lymphoid tissue organizer cell](http://purl.obolibrary.org/obo/CL_0020000) SubClassOf [mesenchymal stem cell](http://purl.obolibrary.org/obo/CL_0000134) 

- [gut mesenchymal lymphoid tissue organizer cell](http://purl.obolibrary.org/obo/CL_0020000) SubClassOf [stromal cell](http://purl.obolibrary.org/obo/CL_0000499) 

- [gut mesenchymal lymphoid tissue organizer cell](http://purl.obolibrary.org/obo/CL_0020000) SubClassOf [progenitor cell](http://purl.obolibrary.org/obo/CL_0011026) 


### hematopoietic stem cell `http://purl.obolibrary.org/obo/CL_0000037`

#### Added
- [hematopoietic stem cell](http://purl.obolibrary.org/obo/CL_0000037) [description](http://purl.org/dc/terms/description) "Hematopoietic stem cells (HSCs) are multipotent cells, capable of differentiating into a wide variety of specialized blood cells. During embryonic development hematopoietic stem cells emerge from the hemogenic endothelium, a specialized subset of endothelial cells with hematopoietic potential. HSCs can be found in several organs, in particular in bone marrow, as well as peripheral blood and umbilical cord blood.
These stem cells have a distinctive ability to self-renew, and to differentiate into all lymphoid and myeloid cell types that together form the blood and immune systems including erythrocytes (red blood cells), leukocytes (white blood cells), and platelets. Their primary function is to ensure the continuous replenishment of the blood and immune system throughout an individual's lifespan.
In response to various signals, hematopoietic stem cells can either remain dormant or actively divide and differentiate. Under normal conditions, the majority of these stem cells exist in a quiescent (inactive) state, making them somewhat resistant to chemotherapy or radiation therapy in the treatment of various diseases such as leukemia. However, when the body encounters stress such as infection, bleeding, or disease, hematopoietic stem cells can rapidly become active in order to replenish the affected cells and maintain homeostasis.
Furthermore, hematopoietic stem cells are extremely important in the field of regenerative medicine and transplantation. Regarded as lifesaving in the field of oncology, they are often extracted, purified, and reinfused back into the same (autologous transplantation) or different (allogeneic transplantation) individuals to reestablish a healthy, functioning hematopoietic and immune system after high-dose chemotherapy or to replace dysfunctional bone marrow. Understanding and manipulating the behavior of these stem cells has immense potential in treating various blood disorders, autoimmune diseases, and cancers.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/s41580-019-0103-9"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1101/gr.192237.115"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1186/s12915-021-00955-z"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.3389/fcell.2022.664261"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [hematopoietic stem cell](http://purl.obolibrary.org/obo/CL_0000037) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0000037"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "hematopoietic stem cell on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### hepatic stellate cell `http://purl.obolibrary.org/obo/CL_0000632`

#### Added
- [hepatic stellate cell](http://purl.obolibrary.org/obo/CL_0000632) [description](http://purl.org/dc/terms/description) "Hepatic stellate cells (HSCs) are a unique cell type found in the liver, residing in the space between the sinusoidal endothelial cells and the hepatocytes, known as the space of Disse. Despite constituting only 5-8% of the total number of liver cells, they play a significant role in the maintenance of liver health. They have a star-shaped (stellate) structure, hence their name and their principal function is the storage of vitamin A in their cytoplasm as retinyl esters.
HSCs become highly active when the liver is damaged or diseased. They play a crucial role in the liver's response to injury and wound-healing process, converging into myofibroblasts, the cells responsible for scar tissue formation (fibrosis). They are also involved in maintaining the extracellular matrix balance by producing matrix metalloproteinases (MMPs) and tissue inhibitors of metalloproteinases (TIMPs), which break down and inhibit the breakdown of collagen respectively. HSCs also play a part in regulating blood flow through the liver&rsquo;s blood vessels, help fight infections, and are involved in interactions among different types of liver cells. Recent research has implicated them in regulating liver regeneration and in the development of liver cancer.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1002/hep4.1628"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/s41598-019-43759-6"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1172/JCI66369"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/S0083-6729(06)75006-3"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [hepatic stellate cell](http://purl.obolibrary.org/obo/CL_0000632) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0000632"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "hepatic stellate cell on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### hippocampal neuron `http://purl.obolibrary.org/obo/CL_0002608`

#### Added
- [hippocampal neuron](http://purl.obolibrary.org/obo/CL_0002608) SubClassOf [BFO_0000002](http://purl.obolibrary.org/obo/BFO_0000002) 


### immature NK T cell stage I, mouse `http://purl.obolibrary.org/obo/CL_0002039`
#### Removed
- [immature NK T cell stage I, mouse](http://purl.obolibrary.org/obo/CL_0002039) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A CD24-high, CD4-low, CD8-low, CD44-negative, NK1.1-negative NK T cell."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:17589542"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "GOC:tfm"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [immature NK T cell stage I, mouse](http://purl.obolibrary.org/obo/CL_0002039) [label](http://www.w3.org/2000/01/rdf-schema#label) "immature NK T cell stage I"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [immature NK T cell stage I, mouse](http://purl.obolibrary.org/obo/CL_0002039) EquivalentTo [immature NK T cell](http://purl.obolibrary.org/obo/CL_0000914) and ([lacks_plasma_membrane_part](http://purl.obolibrary.org/obo/CL_4030046) some [PR_000001307](http://purl.obolibrary.org/obo/PR_000001307)) and ([lacks_plasma_membrane_part](http://purl.obolibrary.org/obo/CL_4030046) some [PR_000001874](http://purl.obolibrary.org/obo/PR_000001874)) and ([RO_0015015](http://purl.obolibrary.org/obo/RO_0015015) some [PR_000001932](http://purl.obolibrary.org/obo/PR_000001932)) and ([RO_0015016](http://purl.obolibrary.org/obo/RO_0015016) some [PR_000001004](http://purl.obolibrary.org/obo/PR_000001004)) and ([RO_0015016](http://purl.obolibrary.org/obo/RO_0015016) some [PR_000025402](http://purl.obolibrary.org/obo/PR_000025402)) 

#### Added
- [immature NK T cell stage I, mouse](http://purl.obolibrary.org/obo/CL_0002039) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A CD24-high, CD4-low, CD8-low, CD44-negative, NK1.1-negative NK T cell."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:26802287"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:16087715"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:17589542"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:38440725"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:19423665"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "GOC:tfm"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [immature NK T cell stage I, mouse](http://purl.obolibrary.org/obo/CL_0002039) [comment](http://www.w3.org/2000/01/rdf-schema#comment) "The immature NK T cell stage&rdquo; (I&ndash;IV) represents mouse-specific stages used to define iNKT cell development. The immature NK T cells in humans typically exhibit a CD161low; CD4+ phenotype."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:17589542"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [immature NK T cell stage I, mouse](http://purl.obolibrary.org/obo/CL_0002039) [label](http://www.w3.org/2000/01/rdf-schema#label) "immature NK T cell stage I, mouse"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [immature NK T cell stage I, mouse](http://purl.obolibrary.org/obo/CL_0002039) EquivalentTo [immature NK T cell](http://purl.obolibrary.org/obo/CL_0000914) and ([lacks_plasma_membrane_part](http://purl.obolibrary.org/obo/CL_4030046) some [PR_000001307](http://purl.obolibrary.org/obo/PR_000001307)) and ([lacks_plasma_membrane_part](http://purl.obolibrary.org/obo/CL_4030046) some [PR_000001874](http://purl.obolibrary.org/obo/PR_000001874)) and ([RO_0002162](http://purl.obolibrary.org/obo/RO_0002162) some [NCBITaxon_10090](http://purl.obolibrary.org/obo/NCBITaxon_10090)) and ([RO_0015015](http://purl.obolibrary.org/obo/RO_0015015) some [PR_000001932](http://purl.obolibrary.org/obo/PR_000001932)) and ([RO_0015016](http://purl.obolibrary.org/obo/RO_0015016) some [PR_000001004](http://purl.obolibrary.org/obo/PR_000001004)) and ([RO_0015016](http://purl.obolibrary.org/obo/RO_0015016) some [PR_000025402](http://purl.obolibrary.org/obo/PR_000025402)) 

- [immature NK T cell stage I, mouse](http://purl.obolibrary.org/obo/CL_0002039) SubClassOf [RO_0002162](http://purl.obolibrary.org/obo/RO_0002162) some [NCBITaxon_10090](http://purl.obolibrary.org/obo/NCBITaxon_10090) 


### immature NK T cell stage II, mouse `http://purl.obolibrary.org/obo/CL_0002040`
#### Removed
- [immature NK T cell stage II, mouse](http://purl.obolibrary.org/obo/CL_0002040) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A CD24-low, CD44-negative, NK1.1-negative NK T cell."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:17589542"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "GOC:tfm"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "http://www.immgen.org/index_content.html"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [immature NK T cell stage II, mouse](http://purl.obolibrary.org/obo/CL_0002040) [comment](http://www.w3.org/2000/01/rdf-schema#comment) "NKT.44-NK1.1-.Th was assigned as an exact synonym based on sorting strategy displayed at immgene.org."^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [immature NK T cell stage II, mouse](http://purl.obolibrary.org/obo/CL_0002040) [label](http://www.w3.org/2000/01/rdf-schema#label) "immature NK T cell stage II"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [immature NK T cell stage II, mouse](http://purl.obolibrary.org/obo/CL_0002040) EquivalentTo [immature NK T cell](http://purl.obolibrary.org/obo/CL_0000914) and ([lacks_plasma_membrane_part](http://purl.obolibrary.org/obo/CL_4030046) some [PR_000001307](http://purl.obolibrary.org/obo/PR_000001307)) and ([lacks_plasma_membrane_part](http://purl.obolibrary.org/obo/CL_4030046) some [PR_000001874](http://purl.obolibrary.org/obo/PR_000001874)) and ([RO_0015016](http://purl.obolibrary.org/obo/RO_0015016) some [PR_000001932](http://purl.obolibrary.org/obo/PR_000001932)) 

#### Added
- [immature NK T cell stage II, mouse](http://purl.obolibrary.org/obo/CL_0002040) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A CD24-low, CD44-negative, NK1.1-negative NK T cell."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:26802287"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:16087715"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:17589542"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:38440725"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:19423665"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "GOC:tfm"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "http://www.immgen.org/index_content.html"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [immature NK T cell stage II, mouse](http://purl.obolibrary.org/obo/CL_0002040) [comment](http://www.w3.org/2000/01/rdf-schema#comment) "NKT.44-NK1.1-.Th was assigned as an exact synonym based on sorting strategy displayed at immgene.org. The immature NK T cell stage&rdquo; (I&ndash;IV) represents mouse-specific stages used to define iNKT cell development. The immature NK T cells in humans typically exhibit a CD161low; CD4+ phenotype."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:17589542"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [immature NK T cell stage II, mouse](http://purl.obolibrary.org/obo/CL_0002040) [label](http://www.w3.org/2000/01/rdf-schema#label) "immature NK T cell stage II, mouse"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [immature NK T cell stage II, mouse](http://purl.obolibrary.org/obo/CL_0002040) EquivalentTo [immature NK T cell](http://purl.obolibrary.org/obo/CL_0000914) and ([lacks_plasma_membrane_part](http://purl.obolibrary.org/obo/CL_4030046) some [PR_000001307](http://purl.obolibrary.org/obo/PR_000001307)) and ([lacks_plasma_membrane_part](http://purl.obolibrary.org/obo/CL_4030046) some [PR_000001874](http://purl.obolibrary.org/obo/PR_000001874)) and ([RO_0002162](http://purl.obolibrary.org/obo/RO_0002162) some [NCBITaxon_10090](http://purl.obolibrary.org/obo/NCBITaxon_10090)) and ([RO_0015016](http://purl.obolibrary.org/obo/RO_0015016) some [PR_000001932](http://purl.obolibrary.org/obo/PR_000001932)) 


### immature NK T cell stage III, mouse `http://purl.obolibrary.org/obo/CL_0002041`
#### Removed
- [immature NK T cell stage III, mouse](http://purl.obolibrary.org/obo/CL_0002041) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A CD24-low, CD44-positive, DX5-low, NK1.1-negative NK T cell."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:17589542"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [immature NK T cell stage III, mouse](http://purl.obolibrary.org/obo/CL_0002041) [label](http://www.w3.org/2000/01/rdf-schema#label) "immature NK T cell stage III"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [immature NK T cell stage III, mouse](http://purl.obolibrary.org/obo/CL_0002041) EquivalentTo [immature NK T cell](http://purl.obolibrary.org/obo/CL_0000914) and ([lacks_plasma_membrane_part](http://purl.obolibrary.org/obo/CL_4030046) some [PR_000001874](http://purl.obolibrary.org/obo/PR_000001874)) and ([RO_0002104](http://purl.obolibrary.org/obo/RO_0002104) some [PR_000001307](http://purl.obolibrary.org/obo/PR_000001307)) and ([RO_0015016](http://purl.obolibrary.org/obo/RO_0015016) some [PR_000001008](http://purl.obolibrary.org/obo/PR_000001008)) and ([RO_0015016](http://purl.obolibrary.org/obo/RO_0015016) some [PR_000001932](http://purl.obolibrary.org/obo/PR_000001932)) 

#### Added
- [immature NK T cell stage III, mouse](http://purl.obolibrary.org/obo/CL_0002041) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A CD24-low, CD44-positive, DX5-low, NK1.1-negative NK T cell."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:26802287"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:16087715"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:17589542"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:38440725"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:19423665"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [immature NK T cell stage III, mouse](http://purl.obolibrary.org/obo/CL_0002041) [comment](http://www.w3.org/2000/01/rdf-schema#comment) "The immature NK T cell stage&rdquo; (I&ndash;IV) represents mouse-specific stages used to define iNKT cell development. The immature NK T cells in humans typically exhibit a CD161low; CD4+ phenotype."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:17589542"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [immature NK T cell stage III, mouse](http://purl.obolibrary.org/obo/CL_0002041) [label](http://www.w3.org/2000/01/rdf-schema#label) "immature NK T cell stage III, mouse"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [immature NK T cell stage III, mouse](http://purl.obolibrary.org/obo/CL_0002041) EquivalentTo [immature NK T cell](http://purl.obolibrary.org/obo/CL_0000914) and ([lacks_plasma_membrane_part](http://purl.obolibrary.org/obo/CL_4030046) some [PR_000001874](http://purl.obolibrary.org/obo/PR_000001874)) and ([RO_0002104](http://purl.obolibrary.org/obo/RO_0002104) some [PR_000001307](http://purl.obolibrary.org/obo/PR_000001307)) and ([RO_0002162](http://purl.obolibrary.org/obo/RO_0002162) some [NCBITaxon_10090](http://purl.obolibrary.org/obo/NCBITaxon_10090)) and ([RO_0015016](http://purl.obolibrary.org/obo/RO_0015016) some [PR_000001008](http://purl.obolibrary.org/obo/PR_000001008)) and ([RO_0015016](http://purl.obolibrary.org/obo/RO_0015016) some [PR_000001932](http://purl.obolibrary.org/obo/PR_000001932)) 


### immature NK T cell stage IV, mouse `http://purl.obolibrary.org/obo/CL_0002042`
#### Removed
- [immature NK T cell stage IV, mouse](http://purl.obolibrary.org/obo/CL_0002042) [RO_0002175](http://purl.obolibrary.org/obo/RO_0002175) [NCBITaxon_9606](http://purl.obolibrary.org/obo/NCBITaxon_9606) 

- [immature NK T cell stage IV, mouse](http://purl.obolibrary.org/obo/CL_0002042) [in_subset](http://www.geneontology.org/formats/oboInOwl#inSubset) [human_reference_atlas](http://purl.obolibrary.org/obo/cl/cl-base#human_reference_atlas) 

- [immature NK T cell stage IV, mouse](http://purl.obolibrary.org/obo/CL_0002042) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A CD24-low, CD44-positive, DX5-high, NK1.1-negative NK T cell."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:17589542"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "GOC:tfm"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "http://www.immgen.org/index_content.html"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [immature NK T cell stage IV, mouse](http://purl.obolibrary.org/obo/CL_0002042) [label](http://www.w3.org/2000/01/rdf-schema#label) "immature NK T cell stage IV"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [immature NK T cell stage IV, mouse](http://purl.obolibrary.org/obo/CL_0002042) EquivalentTo [immature NK T cell](http://purl.obolibrary.org/obo/CL_0000914) and ([lacks_plasma_membrane_part](http://purl.obolibrary.org/obo/CL_4030046) some [PR_000001874](http://purl.obolibrary.org/obo/PR_000001874)) and ([RO_0002104](http://purl.obolibrary.org/obo/RO_0002104) some [PR_000001307](http://purl.obolibrary.org/obo/PR_000001307)) and ([RO_0015015](http://purl.obolibrary.org/obo/RO_0015015) some [PR_000001008](http://purl.obolibrary.org/obo/PR_000001008)) and ([RO_0015016](http://purl.obolibrary.org/obo/RO_0015016) some [PR_000001932](http://purl.obolibrary.org/obo/PR_000001932)) 

#### Added
- [immature NK T cell stage IV, mouse](http://purl.obolibrary.org/obo/CL_0002042) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A CD24-low, CD44-positive, DX5-high, NK1.1-negative NK T cell."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:26802287"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:16087715"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:17589542"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:38440725"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:19423665"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "GOC:tfm"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "http://www.immgen.org/index_content.html"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [immature NK T cell stage IV, mouse](http://purl.obolibrary.org/obo/CL_0002042) [comment](http://www.w3.org/2000/01/rdf-schema#comment) "The immature NK T cell stage&rdquo; (I&ndash;IV) represents mouse-specific stages used to define iNKT cell development. The immature NK T cells in humans typically exhibit a CD161low; CD4+ phenotype."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:17589542"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [immature NK T cell stage IV, mouse](http://purl.obolibrary.org/obo/CL_0002042) [label](http://www.w3.org/2000/01/rdf-schema#label) "immature NK T cell stage IV, mouse"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [immature NK T cell stage IV, mouse](http://purl.obolibrary.org/obo/CL_0002042) EquivalentTo [immature NK T cell](http://purl.obolibrary.org/obo/CL_0000914) and ([lacks_plasma_membrane_part](http://purl.obolibrary.org/obo/CL_4030046) some [PR_000001874](http://purl.obolibrary.org/obo/PR_000001874)) and ([RO_0002104](http://purl.obolibrary.org/obo/RO_0002104) some [PR_000001307](http://purl.obolibrary.org/obo/PR_000001307)) and ([RO_0002162](http://purl.obolibrary.org/obo/RO_0002162) some [NCBITaxon_10090](http://purl.obolibrary.org/obo/NCBITaxon_10090)) and ([RO_0015015](http://purl.obolibrary.org/obo/RO_0015015) some [PR_000001008](http://purl.obolibrary.org/obo/PR_000001008)) and ([RO_0015016](http://purl.obolibrary.org/obo/RO_0015016) some [PR_000001932](http://purl.obolibrary.org/obo/PR_000001932)) 


### immature Schwann cell `http://purl.obolibrary.org/obo/CL_0002377`

#### Added
- [immature Schwann cell](http://purl.obolibrary.org/obo/CL_0002377) [description](http://purl.org/dc/terms/description) "The immature Schwann cell is a distinctive, developmental stage in the life cycle of a Schwann cell, which is a type of glial cell that is essential for the formation and maintenance of the myelin sheaths that coat and insulate the axons of neurons, boosting their electrical conducting abilities.
The immature Schwann cell represents a transitional phase between the immature, proliferative Schwann cell progenitors and the mature, myelinating Schwann cells. The transition to a mature, fully differentiated state happens under the influence of axonal signals and involves significant changes in gene expression; immature Schwann cells are characterized by specific factors including the expression of the S100 protein, a unique marker that differentiates Schwann cells from other glial cells. 
Immature Schwann cells have the potential for extensive proliferation, and they actively interact with axons. During the period of synaptic refinement in development, they can respond to neuronal activity and participate in radial sorting of axons. At the same time, immature Schwann cells further modulate the microenvironment, which helps define axonal diameters and spaces.
While immature Schwann cells contribute to the initial development and refinement of neural circuits, their maturation to myelinating Schwann cells is vital for the efficient functioning of the mature peripheral nervous system.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1177/1073858415572361"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1101/cshperspect.a020487"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.cdev.2021.203686"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1371/journal.pone.0123278"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.4103/1673-5374.211172"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [immature Schwann cell](http://purl.obolibrary.org/obo/CL_0002377) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0002377"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "immature Schwann cell on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### immature T cell `http://purl.obolibrary.org/obo/CL_0002420`

#### Added
- [immature T cell](http://purl.obolibrary.org/obo/CL_0002420) SubClassOf [RO_0000053](http://purl.obolibrary.org/obo/RO_0000053) some [PATO_0001501](http://purl.obolibrary.org/obo/PATO_0001501) 


### immature alpha-beta T cell `http://purl.obolibrary.org/obo/CL_0000790`

#### Added
- [immature alpha-beta T cell](http://purl.obolibrary.org/obo/CL_0000790) [description](http://purl.org/dc/terms/description) "Immature alpha-beta T cells represent a subpopulation of T cells in the thymus that are at a development stage in which they are yet to attain full functional status. The term 'alpha-beta' refers to the two distinct types of chains, alpha (&alpha;) and beta (&beta;), which form the T cell receptor (TCR) located on the cell's surface. This T cell receptor enables the cell to identify and bind to specific antigens in order to initiate an immune response.
Immature alpha-beta T cells arise from lymphoid progenitor cells in the bone marrow and migrate to the thymus, where the maturation of T cells takes place. In the thymus, immature cells (thymocytes) undergo a series of division and differentiation events characterized by the rearrangement of their TCR genes. Upon successful recombination, the cells express a functional alpha-beta TCR on their surface and can then be subjected to a process of selection based on the specificity of the newly expressed receptors. These stages of development are critical to ensuring that only cells with a suitable TCR repertoire survive, ready to be deployed within the immune system. 
Multiple subsets of immature alpha-beta T cells can be distinguished, depending on the developmental and differentiation stage, including CD4+CD8+ double positive thymocytes, CD4+ or CD8+ single positive thymocytes and immature NK T cells. 
Immature alpha-beta T cells are critically important to immune function. They constitute the foundation for the adaptive immune system - a fundamental component in combating pathological agents. Once fully matured, these cells can differentiate further into effector T cell types (like helper, cytotoxic, and regulatory T cells), each having unique functional roles in an immune response. The proper development of immature alpha-beta T cells is fundamental to the overall health and efficacy of the immune system.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "https://www.sciencedirect.com/topics/immunology-and-microbiology/alpha-beta-t-cell"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.immuni.2018.01.007"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/s41392-023-01471-y"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.3389/fimmu.2022.884569"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.immuni.2020.03.019"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [immature alpha-beta T cell](http://purl.obolibrary.org/obo/CL_0000790) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0000790"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "immature alpha-beta T cell on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### immature innate lymphoid cell `http://purl.obolibrary.org/obo/CL_0001082`

#### Added
- [immature innate lymphoid cell](http://purl.obolibrary.org/obo/CL_0001082) [description](http://purl.org/dc/terms/description) "The immature innate lymphoid cell (ILC) refers to a stage in the development of the innate lymphoid cell, a type of immune cell that plays a pivotal role in the innate immune response. These cells are derived from common lymphoid progenitors and are primarily found in the bone marrow. During their immature stage, they possess the potential to differentiate into various mature innate lymphoid cell subsets that contribute to immune responses.
Immature Innate Lymphoid Cells are characterized by their lack of antigen specificity, which distinguishes them from adaptive immune cells such as B and T lymphocytes. While they do not have antigen-specific receptors, they are programmed to respond to certain groups of pathogens or cytokine signaling. Immature ILCs display limited effector functions; however, they are critical in the development of effective innate immune responses as they give rise to the diverse array of mature ILC subsets.
The ILC precursors develop from the fetal liver and the adult bone marrow and, depending upon the transcription factors involved, will give rise to unipotent ILC precursors which then give rise to mature ILC subsets. ILC subsets consist of natural killer (NK), ILC1, ILC2, ILC3, and lymphoid tissue inducer (LTi) cells, each having unique functions in immunity.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1002/eji.201747294"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/nri.2017.86"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1101/cshperspect.a030304"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/nature14189"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.cell.2018.07.017"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [immature innate lymphoid cell](http://purl.obolibrary.org/obo/CL_0001082) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0001082"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "immature innate lymphoid cell on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### immature neutrophil `http://purl.obolibrary.org/obo/CL_0000776`

#### Added
- [immature neutrophil](http://purl.obolibrary.org/obo/CL_0000776) [description](http://purl.org/dc/terms/description) "An immature neutrophil, also known as a band neutrophil or band cell, is an intermediary stage in neutrophilic granulocyte development. Immature neutrophils are formed in the bone marrow then released into the bloodstream where they eventually mature into functional, mature segmented neutrophils. They are characterized by an unsegmented, curved nucleus, a distinguishing factor that differentiates them from mature neutrophils, which possess a multi-lobed nucleus.
Immature neutrophils can be found in circulation, but their numbers may increase drastically during infection. These cells are released from the bone marrow as a result of emergency granulopoiesis. The role of these cells in infection and acute inflammation remains controversial, and it has been suggested that these cells could possess immunosuppressive or proinflammatory activity. 
An increase of immature neutrophils in the bloodstream, a condition referred to as bandemia or a left shift, can be an important clinical indicator. While low-level presence is typical, an abnormally high count can signal severe or systemic bacterial infection, inflammation, or less commonly, diseases that affect bone marrow function such as leukemia. However, not all bacterial infections will result in a left shift.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.it.2019.04.013"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.cca.2016.03.017"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.it.2019.04.012"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.3389/fimmu.2020.01259/full"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [immature neutrophil](http://purl.obolibrary.org/obo/CL_0000776) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0000776"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "immature neutrophil on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### indirect pathway medium spiny neuron `http://purl.obolibrary.org/obo/CL_4023029`
#### Removed
- [indirect pathway medium spiny neuron](http://purl.obolibrary.org/obo/CL_4023029) SubClassOf [BFO_0000002](http://purl.obolibrary.org/obo/BFO_0000002) 



### inhibitory interneuron `http://purl.obolibrary.org/obo/CL_0000498`

#### Added
- [inhibitory interneuron](http://purl.obolibrary.org/obo/CL_0000498) [description](http://purl.org/dc/terms/description) "Inhibitory interneurons, often referred to as local circuit neurons, are integral components of the central nervous system where they play key roles in refining neural circuits and shaping network activity. Known for their inhibitory nature, these neurons form connections mainly within the same region of the brain or spinal cord where they are located. This localization allows them to impact the activities of nearby excitatory neurons by means of inhibitory networks to inhibit or reduce the likelihood of the firing of these excitatory neurons.
Functionally, Inhibitory Interneurons serve as primary regulators and modulators of the output of nerve cells, orchestrating a balance between excitatory and inhibitory activities in neural networks. It is through their characteristic release of inhibitory neurotransmitters, such as gamma-aminobutyric acid (GABA) and glycine, that they manage to suppress postsynaptic potentials, preventing or reducing the propagation of action potentials. This regulation is not only essential for maintaining equilibrium within the nervous system, but also helps prevent conditions that arise from excessive neuronal activities, such as epilepsy.
Inhibitory Interneurons, while relatively small in number compared to their excitatory counterparts, exhibit considerable diversity in their structure, function, and neurochemical expression. They are broadly categorized into multiple subtypes, each with its unique influence on neural circuit behavior and role in different behaviors or cognitive processes. For instance, certain subtypes are involved with timing and synchrony of neural activity, while others participate in contrast enhancement in sensory processing or in isolating specific activity patterns.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1523/JNEUROSCI.23-02-00622.2003"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.neuropharm.2014.09.011"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1007/s12576-016-0442-7"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/nature25999"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1152/physrev.00007.2017"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [inhibitory interneuron](http://purl.obolibrary.org/obo/CL_0000498) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0000498"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "inhibitory interneuron on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### intermediate monocyte `http://purl.obolibrary.org/obo/CL_0002393`

#### Added
- [intermediate monocyte](http://purl.obolibrary.org/obo/CL_0002393) [description](http://purl.org/dc/terms/description) "Intermediate monocytes, also defined as CD14++CD16+ monocytes, constitute one of the three subsets of human monocytes which also include classical and non-classical monocytes. These cells play a crucial role in the physiological function of the immune system and are particularly significant in the human body's response to inflammation and infection. With a diverse array of functions, intermediate monocytes uniquely contribute to the immune response potency by bridging the gap between the classical and non-classical monocyte subsets.
Intermediate monocytes exhibit a combined functionality of the two other subsets in terms of their phenotypic characteristics, functional capabilities and gene expression profiles. They express high levels of CD14 (like classical monocytes) and also express CD16 (like non-classical monocytes). These monocytes generally release lower amounts of cytokines compared to classical and non-classical monocytes, but produce the most reactive oxygen species. Intermediate monocytes may also induce T cell proliferation and express high levels of major histocompatibility complex molecules for antigen presentation.
The relative number of intermediate monocytes in the bloodstream increases under conditions of severe inflammation and infection, although their roles have not yet been clearly defined. Additionally, some studies reported findings on monocytes with a combined population of intermediate and non-classical monocytes, further complicating the understanding of the function of these cells.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1182/blood-2010-12-326355"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1182/blood-2010-02-258558"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.2147/JIR.S292513"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [intermediate monocyte](http://purl.obolibrary.org/obo/CL_0002393) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0002393"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "intermediate monocyte on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### interna theca cell `http://purl.obolibrary.org/obo/CL_4052012`

#### Added
- [interna theca cell](http://purl.obolibrary.org/obo/CL_4052012) [has_related_synonym](http://www.geneontology.org/formats/oboInOwl#hasRelatedSynonym) "androgenic theca cell"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:36599970"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [interna theca cell](http://purl.obolibrary.org/obo/CL_4052012) [has_related_synonym](http://www.geneontology.org/formats/oboInOwl#hasRelatedSynonym) "steroidogenic theca interna cell"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:36599970"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### internal globus pallidus core projecting neuron `http://purl.obolibrary.org/obo/CL_0020003`

#### Added
- Class: [internal globus pallidus core projecting neuron](http://purl.obolibrary.org/obo/CL_0020003) 

- [internal globus pallidus core projecting neuron](http://purl.obolibrary.org/obo/CL_0020003) [contributor](http://purl.org/dc/terms/contributor) [0000-0002-5507-2103](https://orcid.org/0000-0002-5507-2103) 

- [internal globus pallidus core projecting neuron](http://purl.obolibrary.org/obo/CL_0020003) [date](http://purl.org/dc/terms/date) "2025-01-31T00:00:00Z"^^[dateTime](http://www.w3.org/2001/XMLSchema#dateTime) 

- [internal globus pallidus core projecting neuron](http://purl.obolibrary.org/obo/CL_0020003) [id](http://www.geneontology.org/formats/oboInOwl#id) "CL:0020003"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [internal globus pallidus core projecting neuron](http://purl.obolibrary.org/obo/CL_0020003) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A projection neuron that has its soma located in the internal segment of the globus pallidus (GPi) of primates, in the central &ldquo;core&rdquo; region. It expresses parvalbumin, is primarily GABAergic, and projects to motor thalamic nuclei including the ventral anterior, ventrolateral, and ventromedial nuclei. This neuron corresponds to the sensorimotor output population of the GPi, as demonstrated in human tissue and cross-species comparative studies (Wallace et al., 2017) and by classic tracing in primates (Parent &amp; De Bellefeuille, 1982; Parent et al., 2001)."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:28949036"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:7127069"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:28384468"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:11596046"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [internal globus pallidus core projecting neuron](http://purl.obolibrary.org/obo/CL_0020003) [has_related_synonym](http://www.geneontology.org/formats/oboInOwl#hasRelatedSynonym) "GPi core"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [has_synonym_type](http://www.geneontology.org/formats/oboInOwl#hasSynonymType) [OMO_0003000](http://purl.obolibrary.org/obo/OMO_0003000) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:7127069"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [internal globus pallidus core projecting neuron](http://purl.obolibrary.org/obo/CL_0020003) [label](http://www.w3.org/2000/01/rdf-schema#label) "internal globus pallidus core projecting neuron"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [internal globus pallidus core projecting neuron](http://purl.obolibrary.org/obo/CL_0020003) SubClassOf [BFO_0000002](http://purl.obolibrary.org/obo/BFO_0000002) 

- [internal globus pallidus core projecting neuron](http://purl.obolibrary.org/obo/CL_0020003) SubClassOf [GABAergic neuron](http://purl.obolibrary.org/obo/CL_0000617) 

- [internal globus pallidus core projecting neuron](http://purl.obolibrary.org/obo/CL_0020003) SubClassOf [neuron of the forebrain](http://purl.obolibrary.org/obo/CL_0012001) 

- [internal globus pallidus core projecting neuron](http://purl.obolibrary.org/obo/CL_0020003) SubClassOf [RO_0002100](http://purl.obolibrary.org/obo/RO_0002100) some [UBERON_0002477](http://purl.obolibrary.org/obo/UBERON_0002477) 

- [internal globus pallidus core projecting neuron](http://purl.obolibrary.org/obo/CL_0020003) SubClassOf [RO_0002292](http://purl.obolibrary.org/obo/RO_0002292) some [PR_000013502](http://purl.obolibrary.org/obo/PR_000013502) 


### internal globus pallidus shell projection neuron `http://purl.obolibrary.org/obo/CL_0020004`

#### Added
- Class: [internal globus pallidus shell projection neuron](http://purl.obolibrary.org/obo/CL_0020004) 

- [internal globus pallidus shell projection neuron](http://purl.obolibrary.org/obo/CL_0020004) [contributor](http://purl.org/dc/terms/contributor) [0000-0002-5507-2103](https://orcid.org/0000-0002-5507-2103) 

- [internal globus pallidus shell projection neuron](http://purl.obolibrary.org/obo/CL_0020004) [creation date](http://www.geneontology.org/formats/oboInOwl#creation_date) "2025-09-30T14:21:41Z"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [internal globus pallidus shell projection neuron](http://purl.obolibrary.org/obo/CL_0020004) [id](http://www.geneontology.org/formats/oboInOwl#id) "CL:0020004"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [internal globus pallidus shell projection neuron](http://purl.obolibrary.org/obo/CL_0020004) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A projection neuron that has its soma located in the internal segment of the globus pallidus (GPi), in the peripheral 'shell' region. In mice and primates, it expresses somatostatin and shows dual expression of GABAergic and glutamatergic markers, enabling co-release of GABA and glutamate (Wallace et al., 2017). It projects primarily to the lateral habenula (Parent &amp; De Bellefeuille, 1982; Wallace et al., 2017), forming the limbic/aversive output stream of the GPi."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:30755721"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:33787995"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:7127069"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:28384468"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:11596046"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "doi:10.1101/2022.03.16.484460"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [internal globus pallidus shell projection neuron](http://purl.obolibrary.org/obo/CL_0020004) [has_related_synonym](http://www.geneontology.org/formats/oboInOwl#hasRelatedSynonym) "GPi shell"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [has_synonym_type](http://www.geneontology.org/formats/oboInOwl#hasSynonymType) [OMO_0003000](http://purl.obolibrary.org/obo/OMO_0003000) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:28384468"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [internal globus pallidus shell projection neuron](http://purl.obolibrary.org/obo/CL_0020004) [label](http://www.w3.org/2000/01/rdf-schema#label) "internal globus pallidus shell projection neuron"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [internal globus pallidus shell projection neuron](http://purl.obolibrary.org/obo/CL_0020004) SubClassOf [BFO_0000002](http://purl.obolibrary.org/obo/BFO_0000002) 

- [internal globus pallidus shell projection neuron](http://purl.obolibrary.org/obo/CL_0020004) SubClassOf [GABAergic neuron](http://purl.obolibrary.org/obo/CL_0000617) 

- [internal globus pallidus shell projection neuron](http://purl.obolibrary.org/obo/CL_0020004) SubClassOf [neuron of the forebrain](http://purl.obolibrary.org/obo/CL_0012001) 

- [internal globus pallidus shell projection neuron](http://purl.obolibrary.org/obo/CL_0020004) SubClassOf [RO_0002100](http://purl.obolibrary.org/obo/RO_0002100) some [UBERON_0002477](http://purl.obolibrary.org/obo/UBERON_0002477) 

- [internal globus pallidus shell projection neuron](http://purl.obolibrary.org/obo/CL_0020004) SubClassOf [RO_0002292](http://purl.obolibrary.org/obo/RO_0002292) some [PR_000015665](http://purl.obolibrary.org/obo/PR_000015665) 


### interneuron `http://purl.obolibrary.org/obo/CL_0000099`
#### Removed
- [interneuron](http://purl.obolibrary.org/obo/CL_0000099) SubClassOf [RO_0002215](http://purl.obolibrary.org/obo/RO_0002215) some [GO_0014051](http://purl.obolibrary.org/obo/GO_0014051) 

#### Added
- [interneuron](http://purl.obolibrary.org/obo/CL_0000099) SubClassOf [BFO_0000050](http://purl.obolibrary.org/obo/BFO_0000050) some [UBERON_0001017](http://purl.obolibrary.org/obo/UBERON_0001017) 


### interstitial cell of Cajal `http://purl.obolibrary.org/obo/CL_0002088`

#### Added
- [interstitial cell of Cajal](http://purl.obolibrary.org/obo/CL_0002088) [description](http://purl.org/dc/terms/description) "Interstitial cells of Cajal are unique cells located in the gastrointestinal tract, specifically within the muscle layers of the entire digestive system. They have a distinctive stellate or spindle-like shape and were named after the Spanish histologist, Ram&oacute;n y Cajal, who first discovered them.
The intricate interplay of interstitial cells of Cajal with smooth muscle cells and enteric neurons orchestrates the rhythmic contractions (known as peristalsis) in the gastrointestinal tract. They generate and propagate the slow waves of electrical activity, providing the fundamental pacing signal. Interstitial cells of Cajal are capable of responding to stretch and other mechanical stimuli, which helps in adapting the frequency and strength of peristalsis as needed. They also play an essential role in relaxations of gastrointestinal tract sphincters, ensuring proper control over the passage of material.
Moreover, disruption or loss of ICCs contribute to several gastrointestinal motility disorders, underscoring their vital role in digestive health. Lack of or reduced ICCs are reported in various conditions such as achalasia, slow transit constipation, and diabetic gastroparesis.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/nrgastro.2016.161"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1152/ajpgi.00264.2021"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/S0165-6147(97)01108-5"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [interstitial cell of Cajal](http://purl.obolibrary.org/obo/CL_0002088) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0002088"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "interstitial cell of Cajal on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### intestinal tuft cell `http://purl.obolibrary.org/obo/CL_0019032`

#### Added
- [intestinal tuft cell](http://purl.obolibrary.org/obo/CL_0019032) [description](http://purl.org/dc/terms/description) "Intestinal tuft cells, also known as brush cells, are specialized epithelial cells found in the lining of the intestines. Their name stems from their distinctive morphology, characterized by a thick &lsquo;tuft&rsquo;, or brush-like, peak of long, tapering microvilli which project into the intestinal lumen. These cells impart a structural characteristic to the epithelium of the intestine, aiding in its function and interaction with the surrounding environment. Their abundance among the gastrointestinal tract varies but they are most commonly found in the small intestine.
The primary function of intestinal tuft cells is to act as chemosensory sentinels, assisting in monitoring and responding to luminal factors. They possess a highly potent sensing mechanism, facilitated by a diverse set of receptors, such as the bitter taste receptor family and succinate receptor, and a complex signaling apparatus. These sensory capabilities enable them to initiate crucial immune responses to a variety of physiological stimuli, including parasites and bacteria. Evidence suggests that tuft cells perform a critical role in maintaining gut homeostasis and in initiating type 2 immunity against parasite infection.
Moreover, intestinal tuft cells are also involved in regulating intestinal epithelial cell homeostasis and contributing to the regeneration and repair of epithelial tissue following damage. They produce several growth factors and cytokines which play essential roles in these processes. Despite being one of the least common cell types in the gut, the role of tuft cells in sensing, immunity and tissue regulation underlines their importance in maintaining gastrointestinal health.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.3389/fimmu.2022.822867"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1371/journal.ppat.1010318"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.4049/jimmunol.1801069"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/nature24489"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [intestinal tuft cell](http://purl.obolibrary.org/obo/CL_0019032) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0019032"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "intestinal tuft cell on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### intraepithelial lymphocyte `http://purl.obolibrary.org/obo/CL_0002496`

#### Added
- [intraepithelial lymphocyte](http://purl.obolibrary.org/obo/CL_0002496) [description](http://purl.org/dc/terms/description) "Intraepithelial lymphocytes (IELs) are a large and diverse population of mainly T cells that reside within the epithelial cell layer of various mucosal tissues in the body, including the skin, intestine, and respiratory tract. These cells represent a substantial amount of the total lymphocyte pool and are most numerous in the intestine. They have distinct phenotypes and functional characteristics compared to lymphocytes residing in other tissues. 
Most intraepithelial lymphocytes express heterodimeric T-cell receptors (TCRs) consisting of &alpha;lpha and beta chains (TCR&alpha;&beta;), or less commonly gamma and delta chains (TCR&gamma;&delta;), together with the CD8 co-receptor (&alpha;&beta; or &alpha;&alpha;); there are also smaller subpopulations of IELs that areCD4+, or negative for both CD4 and CD8. Intraepithelial T cells typically express high levels of activation markers, tissue-homing integrins, and receptors and molecules characteristic of natural killer cells and cytotoxic T lymphocytes. TCR&alpha;&beta;+CD8+ IEL are able to recognize both transformed self antigens via natural killer receptors and foreign antigen via the TCR.
The primary function of IELs is to maintain the integrity of mucosal barriers against pathogens. Given their strategic location within the epithelial layer, they are often the first immune cells that encounter invading infectious agents, such as bacteria, fungi, or viruses. Upon encountering pathogens, IELs activate and quickly respond by either directly killing the infected epithelial cells or alerting other immune cells. This rapid response, initiated by IELs, helps constrain the infection at the point of entry and is vital to maintaining the integrity of mucosal barriers. 
In addition to their central role in maintaining the integrity of epithelial barriers, IELs also play a crucial role in maintaining the physiological health of the epithelia by promoting repair and regeneration after injury. Furthermore, IELs are pivotal in supporting oral tolerance to dietary antigens and gut microbiota. An imbalance in the number and functionality of IELs is often associated with various human diseases including celiac disease, inflammatory bowel disease, metabolic syndrome, and colorectal cancer.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/s41385-020-0294-6"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.it.2017.11.003"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/s41385-018-0016-5"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "https://www.sciencedirect.com/topics/immunology-and-microbiology/intraepithelial-lymphocyte"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/nri3007"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [intraepithelial lymphocyte](http://purl.obolibrary.org/obo/CL_0002496) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0002496"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "intraepithelial lymphocyte on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### juxtacanalicular tissue cell `http://purl.obolibrary.org/obo/CL_7770002`

#### Added
- Class: [juxtacanalicular tissue cell](http://purl.obolibrary.org/obo/CL_7770002) 

- [juxtacanalicular tissue cell](http://purl.obolibrary.org/obo/CL_7770002) [IAO_0000233](http://purl.obolibrary.org/obo/IAO_0000233) "https://github.com/obophenotype/cell-ontology/issues/3275"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [juxtacanalicular tissue cell](http://purl.obolibrary.org/obo/CL_7770002) [contributor](http://purl.org/dc/terms/contributor) [0009-0000-8480-9277](https://orcid.org/0009-0000-8480-9277) 

- [juxtacanalicular tissue cell](http://purl.obolibrary.org/obo/CL_7770002) [creation date](http://www.geneontology.org/formats/oboInOwl#creation_date) "2025-01-29T08:52:00Z"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [juxtacanalicular tissue cell](http://purl.obolibrary.org/obo/CL_7770002) [id](http://www.geneontology.org/formats/oboInOwl#id) "CL:7770002"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [juxtacanalicular tissue cell](http://purl.obolibrary.org/obo/CL_7770002) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A trabecular meshwork cell of the juxtacanalicular tissue (JCT), characterized by a spindle-shaped, fibroblast-like morphology within a loose extracellular matrix immediately adjacent to Schlemm's canal. It expresses CHI3L1 (human and mouse) and ANGPTL7 (human), as well as smooth muscle actin for contractility. Unlike other trabecular meshwork cells, it does not form monolayers but exists in a loose network, regulating aqueous humor outflow resistance through continuous ECM remodelling, mechanotransduction, and formation of intracellular pores historically called giant vacuoles."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:32341164"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:39829808"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:25356439"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [juxtacanalicular tissue cell](http://purl.obolibrary.org/obo/CL_7770002) [has_related_synonym](http://www.geneontology.org/formats/oboInOwl#hasRelatedSynonym) "JCT cell"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [has_synonym_type](http://www.geneontology.org/formats/oboInOwl#hasSynonymType) [OMO_0003000](http://purl.obolibrary.org/obo/OMO_0003000) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:32341164"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [juxtacanalicular tissue cell](http://purl.obolibrary.org/obo/CL_7770002) [label](http://www.w3.org/2000/01/rdf-schema#label) "juxtacanalicular tissue cell"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [juxtacanalicular tissue cell](http://purl.obolibrary.org/obo/CL_7770002) SubClassOf [secretory cell](http://purl.obolibrary.org/obo/CL_0000151) 

- [juxtacanalicular tissue cell](http://purl.obolibrary.org/obo/CL_7770002) SubClassOf [extracellular matrix secreting cell](http://purl.obolibrary.org/obo/CL_0000327) 

- [juxtacanalicular tissue cell](http://purl.obolibrary.org/obo/CL_7770002) SubClassOf [trabecular meshwork cell](http://purl.obolibrary.org/obo/CL_0002367) 

- [juxtacanalicular tissue cell](http://purl.obolibrary.org/obo/CL_7770002) SubClassOf [RO_0002215](http://purl.obolibrary.org/obo/RO_0002215) some [GO_0070278](http://purl.obolibrary.org/obo/GO_0070278) 


### keratinized cell of hair follicle `http://purl.obolibrary.org/obo/CL_0010000`
#### Removed
- [keratinized cell of hair follicle](http://purl.obolibrary.org/obo/CL_0010000) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A hair follicle matrix region cell which synthesizes keratin and undergoes a characteristic change as it moves upward from the hair bulb to become the hair medulla cortex and hair root sheath."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "https://github.com/obophenotype/cell-ontology/issues/417"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [keratinized cell of hair follicle](http://purl.obolibrary.org/obo/CL_0010000) SubClassOf [BFO_0000050](http://purl.obolibrary.org/obo/BFO_0000050) some [UBERON_0005933](http://purl.obolibrary.org/obo/UBERON_0005933) 

#### Added
- [keratinized cell of hair follicle](http://purl.obolibrary.org/obo/CL_0010000) [date](http://purl.org/dc/terms/date) "2025-01-24T12:00:00Z"^^[dateTime](http://www.w3.org/2001/XMLSchema#dateTime) 

- [keratinized cell of hair follicle](http://purl.obolibrary.org/obo/CL_0010000) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A differentiating matrix keratinocyte that has initiated terminal differentiation and keratin synthesis. This cell undergoes a characteristic change while migrating upward from the hair follicle bulb, representing a committed progenitor cell destined to form the hair medulla, cortex and hair root sheath."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:2478566"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:30420661"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:28445731"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "https://github.com/obophenotype/cell-ontology/issues/417"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [keratinized cell of hair follicle](http://purl.obolibrary.org/obo/CL_0010000) SubClassOf [BFO_0000050](http://purl.obolibrary.org/obo/BFO_0000050) some [UBERON_0005932](http://purl.obolibrary.org/obo/UBERON_0005932) 


### keratinocyte stem cell `http://purl.obolibrary.org/obo/CL_0002337`

#### Added
- [keratinocyte stem cell](http://purl.obolibrary.org/obo/CL_0002337) [description](http://purl.org/dc/terms/description) "Keratinocyte stem cells are a subpopulation of cells located in the basal layer, or stratum basale, of the epidermis, in the bulge region of hair follicles, and in sebaceous glands. The primary function of keratinocyte stem cells is to renew the skin tissue, the primary barrier that protects our bodies from external environmental factors. To perform this function, keratinocyte stem cells possess the ability to self-renew, maintaining their population, and to produce transient amplifying (TA) cells, which replicate rapidly and differentiate into keratinocytes.
The differentiation of keratinocyte stem cells into mature keratinocytes is crucial for skin integrity and health. Specifically, mature keratinocytes give rise to the stratum corneum, the outermost barrier layer of the skin. Throughout this process, keratinocytes produce several types of keratin&mdash;structural proteins that contribute to the mechanical resistance and elasticity of the skin. Anomalies in this procedure might lead to a variety of skin disorders, such as psoriasis and skin cancer, highlighting the critical role of keratinocyte stem cells in skin homeostasis.
In addition to their role in skin homeostasis and wound healing, recent research suggests that keratinocyte stem cells have potential in regenerative medicine, specifically in the treatment of skin injuries and diseases.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/s41467-019-12559-x"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.3389/fimmu.2022.786859"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1002/jcp.22275"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [keratinocyte stem cell](http://purl.obolibrary.org/obo/CL_0002337) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0002337"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "keratinocyte stem cell on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### keratocyte `http://purl.obolibrary.org/obo/CL_0002363`

#### Added
- [keratocyte](http://purl.obolibrary.org/obo/CL_0002363) [description](http://purl.org/dc/terms/description) "Keratocytes, also referred to as corneal fibroblasts, are specific cell types that primarily reside in the stromal layer of the cornea. Highly specialized, these cells play an essential role in the maintenance and repair of corneal tissue, thereby contributing to clear, distortion-free vision. As fibroblasts, their function often involves synthesis and secretion of the extracellular matrix, which gives the corneal stroma its structural integrity.
Keratocytes are primarily responsible for the synthesis of collagen and proteoglycans, the principal components of the stromal extracellular matrix (ECM). The high optical clarity of the cornea is largely attributed to the unique arrangement of collagen fibrils, and keratocytes play a fundamental role in maintaining this arrangement. In response to corneal wounds or injuries, the keratocytes near the wound site become activated and undergo a transformation into a specialized phenotype, often referred to as repair keratocytes or activated keratocytes.
During corneal repair, these activated keratocytes proliferate and migrate to the wound site, where they differentiate into myofibroblasts. Myofibroblasts play a significant role in wound contraction and synthesis of an alternative ECM required for wound closure. However, the overzealous activity of myofibroblasts can lead to corneal haze, a condition that impairs vision, and numerous studies suggest that optimal wound healing involves a delicate balance between repair and the pro-scar actions of myofibroblasts.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1111/j.1463-5224.2009.00742.x"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.exer.2020.108228"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/s41598-019-53123-3"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.biocel.2006.03.010"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [keratocyte](http://purl.obolibrary.org/obo/CL_0002363) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0002363"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "keratocyte on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### kidney capillary endothelial cell `http://purl.obolibrary.org/obo/CL_1000892`

#### Added
- [kidney capillary endothelial cell](http://purl.obolibrary.org/obo/CL_1000892) [description](http://purl.org/dc/terms/description) "Kidney capillary endothelial cells form the innermost lining of the capillaries and microvessels within kidneys, including those lining the glomerular capillaries and the vasa recta. Each cell exhibits a unique array of adhesion molecules, receptors, and secreted mediators which interact with circulating blood cells and solutes. 
One of the key functions of kidney capillary endothelial cells is enabling filtration, reabsorption, secretion, and excretion &ndash; the crucial steps in the formation of urine. In the glomerulus, these cells form a collaborative relationship with the podocytes to constitute the glomerular filtration barrier; here, the ultra-thin endothelial cells create a wall through which water and small solutes pass while retaining larger proteins and molecules within the bloodstream. This permits the rapid and selective filtration of plasma, a primary aspect of the kidney's role in waste clearance and fluid, electrolyte, and acid-base homeostasis.
Kidney capillary endothelial cells also conduct multiple homeostatic roles, such as vasomotor control, fibrinolysis, platelet adhesion, inflammation, and coagulation. They also play a crucial role in responding to renal injury by participating in fibrogenesis, an initial response following acute kidney injury. The endothelial cells spearhead angiogenesis (new blood vessel formation) and the repair and regeneration of renal tissue, thereby enhancing the survival function of the kidney itself. Furthermore, factors secreted by kidney capillary endothelial cells also help control the growth and differentiation of other types of renal cells.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.biocel.2010.05.015"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/s41581-021-00411-9"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1111/j.1523-1755.2005.00260.x"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/sj.ki.5002312"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [kidney capillary endothelial cell](http://purl.obolibrary.org/obo/CL_1000892) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_1000892"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "kidney capillary endothelial cell on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### kidney interstitial fibroblast `http://purl.obolibrary.org/obo/CL_1000692`

#### Added
- [kidney interstitial fibroblast](http://purl.obolibrary.org/obo/CL_1000692) [description](http://purl.org/dc/terms/description) "The kidney interstitial fibroblasts are crucial cells found predominantly in the interstitium of the kidneys. These cells play a significant role in maintaining kidney structure and orchestrating key physiological procedures related to kidney function. The interstitium forms a complex network with other cells, capillaries, and the renal tubules, which are crucial for maintaining kidney functions.
Functionally, these cells participate in renal homeostasis by synthesizing and secreting vital extracellular matrix (ECM) components, such as collagen, proteoglycan, and fibronectin, which contribute to the structural integrity of the kidneys. Additionally, these fibroblasts provide supportive tissues to nephrons (structural and functional units of kidneys), help in regulating fluid and electrolyte balance, along with providing an arena for immune response in case of any infection or damage. They are also integral to the regulation of kidney blood flow and maintenance of tubular structures by stabilizing capillaries through synthesis of angiogenic factors.
Under pathological conditions, such as renal fibrosis, kidney interstitial fibroblasts can transform into myofibroblasts, which speed up the production and deposition of extracellular matrix proteins. This overproduction can lead to scar formation and affect the kidney's functionality, often leading to a chronic and progressive loss of renal function.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1111/j.1365-2613.2011.00764.x"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/s41598-019-38691-8"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1007/s00418-008-0452-5"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1152/ajprenal.00284.2023"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.3389/fphys.2016.00061"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [kidney interstitial fibroblast](http://purl.obolibrary.org/obo/CL_1000692) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_1000692"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "kidney interstitial fibroblast on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### lamp5 GABAergic interneuron `http://purl.obolibrary.org/obo/CL_4023011`
#### Removed
- [lamp5 GABAergic interneuron](http://purl.obolibrary.org/obo/CL_4023011) [IAO_0000028](http://purl.obolibrary.org/obo/IAO_0000028) "Lamp5 cortical interneuron"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [lamp5 GABAergic interneuron](http://purl.obolibrary.org/obo/CL_4023011) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A transcriptomically distinct GABAergic neuron located in the cerebral cortex that expresses Lamp5. The standard transcriptomic reference data for this cell type can be found on the CellxGene census under the collection: 'Transcriptomic cytoarchitecture reveals principles of human neocortex organization', dataset: 'Supercluster: CGE-derived interneurons', Author Categories: 'CrossArea_subclass', clusters Lamp5."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "https://cellxgene.cziscience.com/e/e4ddac12-f48f-4455-8e8d-c2a48a683437.cxg/"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:30382198"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:37824655"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [lamp5 GABAergic interneuron](http://purl.obolibrary.org/obo/CL_4023011) [label](http://www.w3.org/2000/01/rdf-schema#label) "lamp5 GABAergic cortical interneuron"^^[string](http://www.w3.org/2001/XMLSchema#string) 

#### Added
- [lamp5 GABAergic interneuron](http://purl.obolibrary.org/obo/CL_4023011) [IAO_0000028](http://purl.obolibrary.org/obo/IAO_0000028) "Lamp5 interneuron"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [lamp5 GABAergic interneuron](http://purl.obolibrary.org/obo/CL_4023011) [in_subset](http://www.geneontology.org/formats/oboInOwl#inSubset) [human_reference_atlas](http://purl.obolibrary.org/obo/cl/cl-base#human_reference_atlas) 

- [lamp5 GABAergic interneuron](http://purl.obolibrary.org/obo/CL_4023011) [description](http://purl.org/dc/terms/description) "The lamp5 GABAergic cortical interneuron, also known as Lamp5 interneuron, is a specialized subtype of cortical interneurons predominantly found in cortical layers of the brain. These cells are principally identified by the expression of the lamp5 gene. This specific subclass of GABAergic cortical interneurons are among five subclasses recently defined for their marker gene expression, which include Sst, Pvalb, Vip, Lamp5, and Sncg. In addition, these interneurons synthesize and release gamma-aminobutyric acid (GABA), an inhibitory neurotransmitter, which gives them their GABAergic characteristic.
Within the cortical microcircuitry, the lamp5 GABAergic cortical interneurons play crucial roles in modulating neuronal excitability and synaptic transmission. GABA release from these interneurons results in the inhibition of post-synaptic neurons, effectively controlling their activity rates and preventing unchecked excitatory responses, ultimately helping in regulating the balance between excitation and inhibition. This balance is vital for maintaining normal brain functions and its disruption often leads to neuropathological conditions, such as epilepsy and Alzheimer's disease.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1371/journal.pone.0157052"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.cell.2020.09.057"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/s41467-020-18752-7"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1007/s00401-022-02457-w"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [lamp5 GABAergic interneuron](http://purl.obolibrary.org/obo/CL_4023011) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A transcriptomically distinct GABAergic neuron located in the cerebral cortex that expresses Lamp5."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "https://cellxgene.cziscience.com/e/e4ddac12-f48f-4455-8e8d-c2a48a683437.cxg/"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:30382198"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:37824655"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [lamp5 GABAergic interneuron](http://purl.obolibrary.org/obo/CL_4023011) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_4023011"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "lamp5 GABAergic interneuron on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [lamp5 GABAergic interneuron](http://purl.obolibrary.org/obo/CL_4023011) [label](http://www.w3.org/2000/01/rdf-schema#label) "lamp5 GABAergic interneuron"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### lamp5 GABAergic interneuron (Homo sapiens) `http://purl.obolibrary.org/obo/CL_4072034`

#### Added
- Class: [lamp5 GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072034) 

- [lamp5 GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072034) [IAO_0000028](http://purl.obolibrary.org/obo/IAO_0000028) "Lamp5 interneuron (Homo sapiens)"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [lamp5 GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072034) [RO_0002175](http://purl.obolibrary.org/obo/RO_0002175) [NCBITaxon_9606](http://purl.obolibrary.org/obo/NCBITaxon_9606) 

- [lamp5 GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072034) [contributor](http://purl.org/dc/terms/contributor) "https://orcid.org/0000-0002-5507-2103"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [lamp5 GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072034) [date](http://purl.org/dc/terms/date) "2025-09-04T14:48:07Z"^^[dateTime](http://www.w3.org/2001/XMLSchema#dateTime) 

- [lamp5 GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072034) [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "ILX:0770149"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [lamp5 GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072034) [id](http://www.geneontology.org/formats/oboInOwl#id) "CL:4072034"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [lamp5 GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072034) [in_subset](http://www.geneontology.org/formats/oboInOwl#inSubset) [BDS_subset](http://purl.obolibrary.org/obo/cl/cl-base#BDS_subset) 

- [lamp5 GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072034) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A transcriptomically distinct GABAergic neuron located in the cerebral cortex that expresses Lamp5. The standard transcriptomic reference data for this cell type can be found on the CellxGene census under the collection: 'Transcriptomic cytoarchitecture reveals principles of human neocortex organization', dataset: 'Supercluster: CGE-derived interneurons', Author Categories: 'CrossArea_subclass', clusters Lamp5."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "https://cellxgene.cziscience.com/e/e4ddac12-f48f-4455-8e8d-c2a48a683437.cxg/"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:30382198"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:37824655"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [lamp5 GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072034) [comment](http://www.w3.org/2000/01/rdf-schema#comment) "The marker set LINC00298, KIT can identify the Human cell type lamp5 GABAergic interneuron in the neocortex with a confidence of 0.814687838 (NS-Forest FBeta value)."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "https://doi.org/10.5281/zenodo.11165918"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [lamp5 GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072034) [label](http://www.w3.org/2000/01/rdf-schema#label) "lamp5 GABAergic interneuron (Homo sapiens)"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [lamp5 GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072034) SubClassOf [lamp5 GABAergic interneuron](http://purl.obolibrary.org/obo/CL_4023011) 

- [lamp5 GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072034) SubClassOf [RO_0002162](http://purl.obolibrary.org/obo/RO_0002162) some [NCBITaxon_9606](http://purl.obolibrary.org/obo/NCBITaxon_9606) 


### lateral ganglionic eminence derived neuron `http://purl.obolibrary.org/obo/CL_4023035`

#### Added
- [lateral ganglionic eminence derived neuron](http://purl.obolibrary.org/obo/CL_4023035) [RO_0002175](http://purl.obolibrary.org/obo/RO_0002175) [NCBITaxon_9606](http://purl.obolibrary.org/obo/NCBITaxon_9606) 

- [lateral ganglionic eminence derived neuron](http://purl.obolibrary.org/obo/CL_4023035) [in_subset](http://www.geneontology.org/formats/oboInOwl#inSubset) [human_reference_atlas](http://purl.obolibrary.org/obo/cl/cl-base#human_reference_atlas) 


### lens epithelial cell `http://purl.obolibrary.org/obo/CL_0002224`

#### Added
- [lens epithelial cell](http://purl.obolibrary.org/obo/CL_0002224) [description](http://purl.org/dc/terms/description) "Lens epithelial cells are specialized cells located in the anterior portion of the eye's lens, a crucial part of the visual system. These cells form a monolayer of cuboidal cells under the lens capsule known as the lens epithelium, which plays several key roles in maintaining the health and functionality of the ocular lens. 
Lens epithelial cells are responsible for the synthesis, metabolism, and transport of various substances that make up the lens fibers. These cells ensure the lens maintains its transparency and refractive properties, crucial to the eye's capacity to focus light onto the retina. Lens epithelial cells are also able to divide and differentiate into lens fiber cells and therefore play a fundamental role in lens growth and development. 
The lenses of some species are able to regenerate; the regeneration ability of lenses in higher mammals is relatively weak, but there has been some progress in using lens epithelial cells as endogenous stem cells to achieve the partial regeneration of lens tissue. 
In pathological conditions such as cataracts, the function and behavior of lens epithelial cells can be disrupted, which can lead to cloudy or opaque areas in the lens that interfere with vision.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.biocel.2007.10.034"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1242/dev.107953"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.isci.2023.106921"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.3389/fcell.2022.983178"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.preteyeres.2022.101112"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [lens epithelial cell](http://purl.obolibrary.org/obo/CL_0002224) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0002224"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "lens epithelial cell on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### liver dendritic cell `http://purl.obolibrary.org/obo/CL_2000055`

#### Added
- [liver dendritic cell](http://purl.obolibrary.org/obo/CL_2000055) [description](http://purl.org/dc/terms/description) "Liver dendritic cells (DCs) refer to all dendritic cells found in the liver. Liver DCs, much like other DCs, play a critical role in initiating and directing immune responses. However, they show unique characteristics and functions due to their location in the liver, an organ with an important role in metabolism and immunity.
The liver contains both conventional or classical and plasmacytoid DCs. Liver DCs generally have an immature phenotype and are thought to aid in the maintenance of hepatic tolerance. The liver is constantly exposed to dietary and gut-derived antigens, which requires the liver to strike a balance to avoid excessive immune response against these substances. Liver dendritic cells play an essential role in this balance through various mechanisms, such as promoting the development of regulatory T cells. In some cases, liver DCs have also been demonstrated to exhibit stimulatory activity.
Liver DCs play an important role in liver transplantation outcomes and hepatic diseases including viral infections, carcinomas, and primary biliary cirrhosis.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/bs.ircmb.2019.07.004"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1002/hep.21974"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1136/gut.52.2.307"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1046/j.0818-9641.2001.01058.x"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.bbadis.2013.01.005"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [liver dendritic cell](http://purl.obolibrary.org/obo/CL_2000055) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_2000055"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "liver dendritic cell on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### lower motor neuron `http://purl.obolibrary.org/obo/CL_0008039`

#### Added
- [lower motor neuron](http://purl.obolibrary.org/obo/CL_0008039) SubClassOf [BFO_0000002](http://purl.obolibrary.org/obo/BFO_0000002) 

- [lower motor neuron](http://purl.obolibrary.org/obo/CL_0008039) SubClassOf [RO_0002162](http://purl.obolibrary.org/obo/RO_0002162) some [NCBITaxon_7742](http://purl.obolibrary.org/obo/NCBITaxon_7742) 


### luminal cell of prostate epithelium `http://purl.obolibrary.org/obo/CL_0002340`

#### Added
- [luminal cell of prostate epithelium](http://purl.obolibrary.org/obo/CL_0002340) [description](http://purl.org/dc/terms/description) "Luminal cells of the prostate epithelium are secretory cells in the prostate gland, which is located around the urethra at the base of the bladder. In addition to luminal epithelial cells, the adult prostate contains basal epithelial cells and rare neuroendocrine cells, organized as a pseudostratified epithelium surrounded by stromal cells. The main function of the prostate is to nourish and protect sperm by secreting a slightly alkaline fluid which makes up approximately 30% of the semen. 
The luminal epithelial cells form a single-layer lining in the prostate's acini. They have a polarized structure with the basal side attached to the basal lamina and the apical side facing the lumen. Specialized microvilli-like structures on the apical side aid in product secretion. Luminal cells express androgen receptors, making them sensitive to androgenic stimulation, which is crucial for regulating prostatic secretions and overall gland function.
The prostate's Luminal cells are responsible for producing and releasing prostatic fluid, which supports sperm post-ejaculation. They synthesize key proteins like prostate-specific antigen (PSA) and prostatic acid phosphatase. 
Luminal cells are also a focus of prostate cancer research as they are the putative cells-of-origin for the majority of prostate cancers. Overproliferation and persistent damage can lead to the initiation of carcinogenic events within these cells. The PSA produced by luminal cells is also a key biomarker for diagnosing, monitoring, and treating prostate cancer. As such, understanding the health and disease-related functions of these cells is essential for managing prostate conditions.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.celrep.2014.08.002"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.trecan.2018.09.003"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.cell.2014.08.017"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [luminal cell of prostate epithelium](http://purl.obolibrary.org/obo/CL_0002340) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0002340"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "luminal cell of prostate epithelium on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### luminal epithelial cell of mammary gland `http://purl.obolibrary.org/obo/CL_0002326`

#### Added
- [luminal epithelial cell of mammary gland](http://purl.obolibrary.org/obo/CL_0002326) [description](http://purl.org/dc/terms/description) "The luminal epithelial cells of the mammary gland are a key constituent of the glandular tissue in the mammalian breast. Packed into complex structures known as mammary ducts and lobules, these specialized cell types derive their nomenclature from their location within these units - specifically against the inner surface or 'lumen'. 
The main physiological functions they perform are the synthesis, storage, and secretion of milk. During lactation, these cells synthesize the various macromolecular constituents of milk, such as lipids, proteins, and lactose, which are then secreted into the ductal lumen. An intricately coordinated mix of hormonal activation and signaling pathways, involving prolactin and oxytocin amongst others, regulate this process. Luminal cells also exhibit receptivity towards estrogen, and their proliferation is influenced by estrogen and progesterone.
Luminal epithelial cells are also engaged in several other key aspects of mammary gland biology. They carry a role in the formation of the ductal tree during the developmental process and are involved in glandular remodeling during pregnancy and post-weaning involution. Furthermore, luminal epithelial cells, just like other cell types in the breast, are susceptible to oncogenic transformation. In fact, the majority of breast cancers, specifically those classified as ductal carcinomas, are believed to originate from this cell population, underlining the pathophysiological significance of these cells.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1101/gad.242511.114"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/s41467-021-27895-0"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1186/bcr1411"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1152/physrev.00040.2018"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [luminal epithelial cell of mammary gland](http://purl.obolibrary.org/obo/CL_0002326) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0002326"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "luminal epithelial cell of mammary gland on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### macrophage `http://purl.obolibrary.org/obo/CL_0000235`

#### Added
- [macrophage](http://purl.obolibrary.org/obo/CL_0000235) [description](http://purl.org/dc/terms/description) "Macrophages are specialized white blood cells involved in immune responses, inflammation, and tissue repair. Tissue-resident macrophages can be self-renewing and arise from embryo-derived populations. Macrophages can also originate from circulating monocytes that differentiate into macrophages upon migration into body tissues. 
Macrophages are involved in the detection, phagocytosis, and destruction of bacteria and other pathogens. They are also responsible for engulfing and removing dead cells and debris. Macrophages are also one of the cells that present pathogenic antigens to T cells so that the pathogens may be recognized and killed, or so that an antibody response may be mounted. They can also secrete cytokines, chemokines, and enzymes that further orchestrate the immune response.
Tissue-resident macrophages are diverse and heterogenous, likely as a result of the signals received from the tissue environment. The functions of these macrophages are specific to each tissue and contribute to tissue homeostasis. Dysregulation of macrophage functioning may also contribute to the development of various disease processes, such as fibrosis, chronic inflammation, or cancer.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1093/intimm/dxy054"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.3389/fimmu.2021.708186"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/ni.2705"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.immuni.2016.02.015"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1146/annurev-immunol-032414-112220"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [macrophage](http://purl.obolibrary.org/obo/CL_0000235) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0000235"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "macrophage on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### mammary gland epithelial cell `http://purl.obolibrary.org/obo/CL_0002327`

#### Added
- [mammary gland epithelial cell](http://purl.obolibrary.org/obo/CL_0002327) [description](http://purl.org/dc/terms/description) "Mammary gland epithelial cells, also known as mammary epithelial cells, are essential cell types in the tissue of mammary glands, which are milk-producing glands in female mammals. They can be categorized into two types: luminal epithelial cells and myoepithelial cells, each with their distinct functions in milk processing and delivery.
Luminal epithelial cells, which are located in the inner layer of the mammary gland ducts and alveoli, play a vital role in the secretion of milk proteins and lipids to form milk during lactation. These cells contain well-developed secretory apparatus, including rough endoplasmic reticulum and Golgi complex, facilitating the synthesis and secretion of milk proteins. Luminal epithelial cells express markers, like cytokeratin 8 and 18, and hormone receptors, such as estrogen and progesterone receptors, which play a crucial role in regulating the cell proliferation during the mammary gland development and lactation.
Myoepithelial cells, on the other hand, are contractile cells situated between the luminal epithelial cells and the basal lamina in the ducts and alveoli of the mammary gland. They express markers such as smooth muscle actin and p63. One of their chief functions is the contraction during lactation, which aids in the expulsion of milk from the alveoli into the ductal network. Additionally, myoepithelial cells play a role in shaping the mammary gland ductal system and initiating the involution process after weaning.
The homeostasis and functionality of the mammary epithelial cells are sensitive to hormonal regulation and surrounding microenvironment cues. Any abnormalities in these two types of mammary epithelial cells could disrupt the mammary gland function and potentially prompt the development of breast diseases, including cancer.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1101/gad.242511.114"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.3390/ijms23073883"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.isci.2021.102413"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/s41574-022-00742-y"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1083/jcb.200107065"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [mammary gland epithelial cell](http://purl.obolibrary.org/obo/CL_0002327) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0002327"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "mammary gland epithelial cell on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### mast cell `http://purl.obolibrary.org/obo/CL_0000097`

#### Added
- [mast cell](http://purl.obolibrary.org/obo/CL_0000097) [description](http://purl.org/dc/terms/description) "Mast cells, also known as mastocytes, are a type of innate immune cell. They originate from multipotent hematopoietic progenitor cells within the bone marrow, after which they migrate into nearly all tissues throughout the body. In these tissues, mast cells undergo complete maturation, with the exception of the blood. They are most densely located in the interfaces between the internal and external environments, such as the skin, respiratory system, and gastrointestinal mucosa.
Mast cells are crucial players against pathogens and serve as first-line defense components. They contain numerous granules rich in histamine and heparin and are known primarily for their role in allergy and anaphylaxis. Upon encountering an allergen, pathogen, or even certain drugs and toxins, mast cells release these mediators via degranulation. Histamine induces vasodilation and vascular permeability, allowing for immune cells to rapidly reach the site of infection, while heparin prevents clotting in these areas to facilitate immune cell recruitment. Mast cells may also selectively secrete mediators without degranulation.
These cells also play other functions beyond immune defense. They are involved in wound healing processes and maintenance of tissue integrity, angiogenesis, and even regulation of physiological functions in the gut. Consequently, because of their wide-ranging impacts, abnormalities in mast cell function can result in various disorders, including those related to allergies, inflammation, and autoimmune conditions.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1007/s12016-019-08729-w"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.bbadis.2010.12.014"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1056/NEJMra1409760"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1007/s11894-010-0132-1"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1146/annurev-immunol-071719-094903"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [mast cell](http://purl.obolibrary.org/obo/CL_0000097) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0000097"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "mast cell on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### matrix D1 medium spiny neuron `http://purl.obolibrary.org/obo/CL_4030043`

#### Added
- [matrix D1 medium spiny neuron](http://purl.obolibrary.org/obo/CL_4030043) [RO_0002175](http://purl.obolibrary.org/obo/RO_0002175) [NCBITaxon_9606](http://purl.obolibrary.org/obo/NCBITaxon_9606) 

- [matrix D1 medium spiny neuron](http://purl.obolibrary.org/obo/CL_4030043) [in_subset](http://www.geneontology.org/formats/oboInOwl#inSubset) [human_reference_atlas](http://purl.obolibrary.org/obo/cl/cl-base#human_reference_atlas) 


### matrix D2 medium spiny neuron `http://purl.obolibrary.org/obo/CL_4030047`

#### Added
- [matrix D2 medium spiny neuron](http://purl.obolibrary.org/obo/CL_4030047) [RO_0002175](http://purl.obolibrary.org/obo/RO_0002175) [NCBITaxon_9606](http://purl.obolibrary.org/obo/NCBITaxon_9606) 

- [matrix D2 medium spiny neuron](http://purl.obolibrary.org/obo/CL_4030047) [in_subset](http://www.geneontology.org/formats/oboInOwl#inSubset) [human_reference_atlas](http://purl.obolibrary.org/obo/cl/cl-base#human_reference_atlas) 


### mature NK T cell `http://purl.obolibrary.org/obo/CL_0000814`
#### Removed
- [mature NK T cell](http://purl.obolibrary.org/obo/CL_0000814) SubClassOf [RO_0002202](http://purl.obolibrary.org/obo/RO_0002202) some [immature NK T cell stage IV, mouse](http://purl.obolibrary.org/obo/CL_0002042) 

#### Added
- [mature NK T cell](http://purl.obolibrary.org/obo/CL_0000814) [description](http://purl.org/dc/terms/description) "Mature NK T cells, or natural killer T cells, are a unique subset of alpha-beta T cells that serve as mediators in the immune response. They are named for their distinct ability to express markers typical of natural killer (NK) cells (such as NK1.1) as well as a T cell receptor specific for a limited set of ligands.
Similar to mature effector T cells, mature NK T cells represent a heterogeneous group of effector cells, which express different transcription factors and cytokines and hence have different effector functions in both innate and adaptive immune responses. They are CD1d-restricted T lymphocytes: they recognize endogenous and exogenous lipid antigens presented by the MHC-like molecule CD1d. They have been classified into two subpopulations based on their TCR usage and specificity for lipid antigen: Type I NK T cells have an invariant TCR&alpha;-chain while type II NK T cells have a more diverse TCR repertoire. 
Mature NKT cells have been shown to have both pro- and anti-inflammatory functions and modulate immune responses, including cancer, autoimmunity, allergy, infection and graft-versus-host disease. The primary function of mature NK T cells is to rapidly respond to infections by releasing large amounts of cytokines to aid cell-to-cell communication in immune responses. They also play a role in recognizing and eliminating certain types of cancer cells. Upon activation via their TCRs, mature NK T cells can promptly release a plethora of cytokines including interferon-&gamma; (IFN-&gamma;) and interleukin-4 (IL-4). These cytokines subsequently stimulate other immune cells, such as B cells, T cells and macrophages, to respond to the infection or abnormal cell growth.
In addition to antibody-mediated responses, NK T cells are capable of cytotoxic responses, a crucial function in managing viral infections. Mature NK T cells can bind to infected cells, inducing cell death and thereby preventing the spread of the infection. Furthermore, they have been found to play an instrumental role in autoimmunity and allergic diseases, highlighting their key part in maintaining the body&rsquo;s overall immune balance. Continuous research on mature NK T cells informs therapeutic efforts in immunological disorders, infections, and cancer.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "https://www.ncbi.nlm.nih.gov/books/NBK532332/"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "https://www.sciencedirect.com/topics/medicine-and-dentistry/natural-killer-t-cell"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.3389/fimmu.2018.00367"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1128/cmr.00232-20"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.3389/fncel.2019.00245"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [mature NK T cell](http://purl.obolibrary.org/obo/CL_0000814) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0000814"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "mature NK T cell on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [mature NK T cell](http://purl.obolibrary.org/obo/CL_0000814) SubClassOf [RO_0002202](http://purl.obolibrary.org/obo/RO_0002202) some [immature NK T cell](http://purl.obolibrary.org/obo/CL_0000914) 


### mature T cell `http://purl.obolibrary.org/obo/CL_0002419`

#### Added
- [mature T cell](http://purl.obolibrary.org/obo/CL_0002419) SubClassOf [RO_0000053](http://purl.obolibrary.org/obo/RO_0000053) some [PATO_0001701](http://purl.obolibrary.org/obo/PATO_0001701) 


### mature alpha-beta T cell `http://purl.obolibrary.org/obo/CL_0000791`

#### Added
- [mature alpha-beta T cell](http://purl.obolibrary.org/obo/CL_0000791) [description](http://purl.org/dc/terms/description) "Mature alpha-beta T cells, also known as &alpha;&beta; T cells, are critical components of the adaptive immune system, responsible for a mediated immune response to pathogens and infected body cells. They derive their name from the alpha and beta chains that make up their characteristic T-cell receptor (TCR), a protein complex that binds to antigens. These cells undergo a maturation process in the thymus, where they learn to recognize self-antigens, thus achieving self-tolerance.
Maturation of alpha-beta T cells takes place in the thymus and is tightly controlled by various transcriptional and epigenetic regulators. They develop through several stages from progenitors to mature T cells, from CD4-CD8- double negative cells to CD4+CD8+ double positive cells, until they eventually become CD4+ or CD8+ single positive cells.
Once matured, alpha-beta T cells primarily serve as regulators and effectors in an immune response. They are adept at recognizing fragments of antigens, often viruses or bacteria, presented on the surface of other cells via major histocompatibility complex (MHC) molecules. Once they encounter and recognize an antigen, they become activated, proliferate and differentiate into distinct subsets of effector T cells, such as cytotoxic T cells, helper T cells, and Natural Killer (NK) cells, each with a specific role in immune response.
Cytotoxic T cells, the effector cells for cell-mediated immunity, are adept at killing cells that have been infected, particularly with viruses. They do this by binding to infected cells and releasing cytotoxic substances which induce apoptosis, or cell death, thus eliminating the infected cells. The main function of NK cells is to kill virally infected cells, and to detect early signs of cancer. Helper T cells do not kill infected cells directly but play a key role in modulating or orchestrating the immune response by releasing cytokines that recruit and regulate other immunity cells such as B cells and macrophages. Therefore, the versatile roles of mature alpha-beta T cells are vital for maintaining the body's successful adaptive immune defenses.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "https://www.sciencedirect.com/topics/immunology-and-microbiology/alpha-beta-t-cell"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.immuni.2018.01.007"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/s41392-023-01471-y"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.3389/fimmu.2022.884569"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [mature alpha-beta T cell](http://purl.obolibrary.org/obo/CL_0000791) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0000791"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "mature alpha-beta T cell on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### mature astrocyte `http://purl.obolibrary.org/obo/CL_0002627`

#### Added
- [mature astrocyte](http://purl.obolibrary.org/obo/CL_0002627) [description](http://purl.org/dc/terms/description) "Mature astrocytes are specialized glial cells found primarily in the central nervous system (CNS), which consists of the brain and the spinal cord. They are named aftertheir star-like shape. Mature astrocytes are derived from radial glia, with the transition from radial glia to immature astrocytes, followed by maturation into the final functionally specialized form. They are further classified into two basic types: protoplasmic astrocytes present mainly in the grey matter and fibrous astrocytes in the white matter.
Functionally, mature astrocytes play a central role in both the structure and function of the CNS through regulation of neurons. By secreting molecules, such as cytokines, that influence the formation, functioning, and elimination of synapses, they actively participate in the formation and modulation of synaptic connections between neurons. Additionally, they also possess a unique capability known as gliotransmission, referring to the astrocytes' ability to respond to and release neurotransmitters, which impacts neuronal signaling.
The mature astrocytes also maintain the brain's homeostasis, which includes controlling the chemical environment, providing nutrients to nervous tissues, regulating blood flow, responding to injury, and maintaining the blood-brain barrier, a crucial defense mechanism of the brain. They metabolize neurotransmitters like glutamate to prevent neurotoxicity and help in the detoxification process. Furthermore, these cells play a vital role in neuroinflammation and reactive gliosis, a response to trauma or neurodegenerative diseases, and assist in repairing damaged neural tissues. Thus, the versatile mature astrocytes play multiple roles and significantly contribute to the correct functioning and reparative process of the CNS.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1002/glia.23908"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1002/glia.22836"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1177/1073858409342593"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.neuron.2017.09.056"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1002/jnr.24922"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [mature astrocyte](http://purl.obolibrary.org/obo/CL_0002627) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0002627"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "mature astrocyte on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### mature gamma-delta T cell `http://purl.obolibrary.org/obo/CL_0000800`

#### Added
- [mature gamma-delta T cell](http://purl.obolibrary.org/obo/CL_0000800) [description](http://purl.org/dc/terms/description) "Mature gamma-delta (&gamma;&delta;) T-cells represent a distinct subgroup within the diverse T-cell lineage. They are rare in secondary lymphoid organs but enriched in many peripheral tissues, such as the intestines, lungs and skin, where they play key immunological roles in the protection against infections and tumors, tissue healing, and immune regulation. They are &lsquo;unconventional&rsquo; T cells defined by the expression of T-cell receptors (TCRs) that are heterodimers  composed of &gamma; and &delta; chains.
Unlike their alpha-beta (&alpha;&beta;) T-cell counterparts, which recognize antigens via Major Histocompatibility Complex (MHC) molecules, &gamma;&delta; T-cells are activated in an MHC-independent manner perceive their targets in a more direct manner, reacting to stress-induced self molecules or unprocessed antigens, providing a more rapid immune response.
Mature &gamma;&delta; T-cells derive from double-negative CD4&minus; CD8&minus; (DN) thymocytes, which can give rise to either &alpha;&beta; T or &gamma;&delta; T cell lineages. &gamma;&delta; T cells then develop from this precursor following a single selection step mediated by the &gamma;&delta; TCR; unlike cells in the &alpha;&beta; T cell lineage, they do not progress through to a double positive stage. Instead, most &gamma;&delta; T cells remain double-negative and develop into mature &gamma;&delta; T cells before they egress from the thymus.
Upon maturation, &gamma;&delta; T-cells characteristics of both innate and adaptive immunity and can have both pro- and anti-inflammatory functions. Mature &gamma;&delta; T-cells fall into distinct IFN&gamma;- and IL-17A-producing cell populations that can be distinguished by CD27 and CCR6 expression, respectively. The mature &gamma;&delta; T-cell population also includes &gamma;&delta; intraepithelial T-cells and dendritic epidermal T cells (DETCs), which are only present in the epidermis of rodents. 
These phenotypically different mature &gamma;&delta; T-cells are functionally versatile. In addition to their role in immunosurveillance against microbes and tumor cells, aiding in early defense against invaders. They produce a cytokine and chemokine response and exert cytolysis of infected or transformed target cells (involving perforin and granzymes). They can assist with tissue repair in certain organs like the skin and muscle due to their ability to produce growth factors. In humans, specific &gamma;&delta; T-cells are able to serves as professional antigen presenting cells. Finally, because these cells are able to recognize and lyse tumors in an MHC-unrestricted manner, they have a huge potential for pan-population tumor immunotherapy.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.smim.2022.101662"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.3389/fimmu.2020.00042"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.3389/fimmu.2023.1106652"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [mature gamma-delta T cell](http://purl.obolibrary.org/obo/CL_0000800) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0000800"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "mature gamma-delta T cell on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### mature natural killer cell `http://purl.obolibrary.org/obo/CL_0000824`

#### Added
- [mature natural killer cell](http://purl.obolibrary.org/obo/CL_0000824) EquivalentTo [natural killer cell](http://purl.obolibrary.org/obo/CL_0000623) and ([RO_0000053](http://purl.obolibrary.org/obo/RO_0000053) some [PATO_0001701](http://purl.obolibrary.org/obo/PATO_0001701)) 

- [mature natural killer cell](http://purl.obolibrary.org/obo/CL_0000824) SubClassOf [RO_0000053](http://purl.obolibrary.org/obo/RO_0000053) some [PATO_0001701](http://purl.obolibrary.org/obo/PATO_0001701) 


### medial ganglionic eminence derived GABAergic interneuron `http://purl.obolibrary.org/obo/CL_4023069`
#### Removed
- [medial ganglionic eminence derived GABAergic interneuron](http://purl.obolibrary.org/obo/CL_4023069) [IAO_0000028](http://purl.obolibrary.org/obo/IAO_0000028) "MGE cortical gaba interneuron"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [medial ganglionic eminence derived GABAergic interneuron](http://purl.obolibrary.org/obo/CL_4023069) [has_exact_synonym](http://www.geneontology.org/formats/oboInOwl#hasExactSynonym) "MGE cortical interneuron"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [medial ganglionic eminence derived GABAergic interneuron](http://purl.obolibrary.org/obo/CL_4023069) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A GABAergic cortical interneuron that develops from the medial ganglionic eminence and has migrated to the cerebral cortex."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:12637172"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [medial ganglionic eminence derived GABAergic interneuron](http://purl.obolibrary.org/obo/CL_4023069) [label](http://www.w3.org/2000/01/rdf-schema#label) "medial ganglionic eminence derived GABAergic cortical interneuron"^^[string](http://www.w3.org/2001/XMLSchema#string) 

#### Added
- [medial ganglionic eminence derived GABAergic interneuron](http://purl.obolibrary.org/obo/CL_4023069) [IAO_0000028](http://purl.obolibrary.org/obo/IAO_0000028) "MGE gaba interneuron"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [medial ganglionic eminence derived GABAergic interneuron](http://purl.obolibrary.org/obo/CL_4023069) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A GABAergic interneuron that develops from the medial ganglionic eminence and has migrated to the cerebral cortex."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:12637172"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [medial ganglionic eminence derived GABAergic interneuron](http://purl.obolibrary.org/obo/CL_4023069) [label](http://www.w3.org/2000/01/rdf-schema#label) "medial ganglionic eminence derived GABAergic interneuron"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### medium spiny neuron `http://purl.obolibrary.org/obo/CL_1001474`
#### Removed
- [medium spiny neuron](http://purl.obolibrary.org/obo/CL_1001474) SubClassOf [central nervous system neuron](http://purl.obolibrary.org/obo/CL_2000029) 

#### Added
- [medium spiny neuron](http://purl.obolibrary.org/obo/CL_1001474) [description](http://purl.org/dc/terms/description) "Medium spiny neurons, constituting the main type of neurons found in the striatum of the brain, are key players in various pathways that facilitate a multitude of cognitive functions. Projecting dendrites laden with spines, these cells are equipped to receive a wide range of synaptic input, mainly from the cerebral cortex and the thalamus. The &quot;medium&quot; and &quot;spiny&quot; descriptive parts of the name stem from the neurons' size, which is intermediate when compared to other neurons, and the presence of numerous spines on the dendrites, which serve as critical points for synaptic connections.
The primary function of medium spiny neurons lies within motor control and reward pathways, which are fundamental aspects of the basal ganglia circuitry. They act by receiving and integrating various excitatory and inhibitory inputs to generate a resultant inhibitory output, generally towards other basal ganglia structures. This inhibitory output is crucial to the balance within the basal ganglia system, which regulates voluntary movements, habitual behaviors, and cognitive and emotional functionalities. The medium spiny neurons act through the use of the neurotransmitter gamma-aminobutyric acid (GABA) and express high levels of dopamine D1 or D2 receptors, which facilitate communication within the brain's dopamine system.
Disruptions or malfunctions in medium spiny neurons have been associated with several neurological and psychiatric disorders. For instance, in Huntington's disease, these neurons display a specific vulnerability, leading to motor control issues, cognitive decline, and psychiatric symptoms. Similarly, these neurons play critical roles in substance addiction, as the neurons' reward pathways can become sensitized or desensitized due to drug abuse.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1007/s13311-012-0112-2"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.tins.2007.03.008"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1523/JNEUROSCI.3833-10.2011"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1007/978-3-7091-0932-8_18"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.3389/fnbeh.2011.00047"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [medium spiny neuron](http://purl.obolibrary.org/obo/CL_1001474) [comment](http://www.w3.org/2000/01/rdf-schema#comment) "Direct genetic lineage tracing in mice, including Dlx-based and Sp9-Cre fate mapping, demonstrates that medium spiny neurons (MSNs) originate from progenitors in the lateral ganglionic eminence (LGE), with labeled cells migrating from LGE to the striatum and expressing MSN markers such as FOXP1 and CTIP2."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "doi:10.1038/s41420-022-01088-0"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "doi:10.1523/JNEUROSCI.20-16-06063.2000"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [medium spiny neuron](http://purl.obolibrary.org/obo/CL_1001474) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_1001474"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "medium spiny neuron on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [medium spiny neuron](http://purl.obolibrary.org/obo/CL_1001474) SubClassOf [lateral ganglionic eminence derived neuron](http://purl.obolibrary.org/obo/CL_4023035) 


### medulla-projecting glutamatergic neuron of the primary motor cortex `http://purl.obolibrary.org/obo/CL_4023045`

#### Added
- [medulla-projecting glutamatergic neuron of the primary motor cortex](http://purl.obolibrary.org/obo/CL_4023045) SubClassOf [BFO_0000002](http://purl.obolibrary.org/obo/BFO_0000002) 


### megakaryocyte `http://purl.obolibrary.org/obo/CL_0000556`

#### Added
- [megakaryocyte](http://purl.obolibrary.org/obo/CL_0000556) [description](http://purl.org/dc/terms/description) "A megakaryocyte is a large, unique, and complex cell type found primarily in the bone marrow, the spongy, semi-solid tissue located in the medullary cavities of certain bones. The term &lsquo;megakaryocyte&rsquo; is derived from the Greek 'mega', meaning great, and 'karyon', referring to the nucleus. Megakaryocytes are characterized by their large size, typically with a diameter of 50-100 &micro;m in humans, and the presence of a multilobulated nucleus; they are the largest hematopoietic cells in the bone marrow but also the rarest cells. 
The primary function of a megakaryocyte is the production of platelets, also known as thrombocytes. Platelets are small, anucleate cell fragments that play a crucial role in the process of hemostasis, the arrest of blood loss from a damaged blood vessel. The process begins when the megakaryocyte extends long, branching processes called proplatelets, into the sinusoidal blood vessels present in the bone marrow. These proplatelets then fragment into the discoid structures recognized as platelets. Each megakaryocyte can produce thousands of platelets during its lifespan. 
Apart from platelet production, megakaryocytes are involved in several other functions. They express immune-sensing receptors such as Fc-&gamma; receptor and Toll-like receptors and produce factors such as platelet factor 4 and platelet-derived growth factor that can stimulate the immune system. Also, in certain pathological conditions, megakaryocytes release TGF-&beta; and PDGF and contribute to bone marrow fibrosis, a condition characterized by excessive accumulation of fibrous tissue in the bone marrow leading to anemia. Recent studies have also highlighted the role of megakaryocytes in regulating hematopoiesis, the process of blood cell production. They can influence the bone marrow microenvironment and control the behavior of hematopoietic stem cells, the progenitors of all types of blood cells.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1083/jcb.201304054"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "https://www.sciencedirect.com/topics/agricultural-and-biological-sciences/megakaryocyte"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1186/s12967-023-04393-z"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1002/JLB.MR0718-261RR"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1161/ATVBAHA.119.312129"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [megakaryocyte](http://purl.obolibrary.org/obo/CL_0000556) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0000556"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "megakaryocyte on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### megakaryocyte-erythroid progenitor cell `http://purl.obolibrary.org/obo/CL_0000050`

#### Added
- [megakaryocyte-erythroid progenitor cell](http://purl.obolibrary.org/obo/CL_0000050) [description](http://purl.org/dc/terms/description) "The megakaryocyte-erythroid progenitor (MEP) cell belongs to the multipotent hematopoietic progenitor cells, which can give rise to multiple different cell types. They are found in the bone marrow, the primary organ where new blood cells are generated. MEPs are specially programmed to develop into two distinct blood cell lineages, namely megakaryocytes, which go on to produce platelets for blood clotting, and the erythroid lineage which consists of red blood cells responsible for oxygen transport in the body.
The MEP cells are the result of a decision-making process within the hematopoietic stem cell population. Hematopoietic stem cell can differentiate into multiple types of progenitor cells, including myeloid progenitor cells. A further subdivision within this myeloid lineage generates the MEP cells. Therefore, these megakaryocyte-erythroid progenitors represent a key juncture between hematopoietic stem cells and mature blood cell types in the myeloid lineage.
Megakaryocyte-erythroid progenitor cells play a crucial role in the homeostasis of some of the most crucial constituents of blood. The megakaryocytes produced from these bipotent progenitor cells contribute to hemostasis by producing platelets - small anucleated fragments that are pivotal for blood clotting and preventing bleeding. Similarly, the erythroid lineage from MEP cells gives rise to erythrocytes, which harbor hemoglobin, the molecule that transports oxygen from lungs to every nook and corner of the body. An imbalance or malfunction in MEP function thus can significantly impact human health, leading to diseases like thrombocytopenia or anemia.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.celrep.2018.10.084"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1097/MOH.0000000000000625"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1002/stem.2834"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1182/blood-2016-01-693705"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1161/ATVBAHA.119.312129"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [megakaryocyte-erythroid progenitor cell](http://purl.obolibrary.org/obo/CL_0000050) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0000050"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "megakaryocyte-erythroid progenitor cell on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### meis2 expressing cortical GABAergic cell `http://purl.obolibrary.org/obo/CL_4023065`

#### Added
- [meis2 expressing cortical GABAergic cell](http://purl.obolibrary.org/obo/CL_4023065) [description](http://purl.org/dc/terms/description) "The meis2 expressing cortical GABAergic cell is a unique and specialized type of neuron found within the complex architecture of the cerebral cortex. Specifically, these neurons originate from a region bordering the pallium and subpallium. Meis2 refers to the gene that codes for the Meis homeobox2 protein, which is a crucial transcription factor that regulates neural development. Most importantly, the GABAergic characteristic of these cells refers to their function in relation to the release of the neurotransmitter gamma-aminobutyric acid (GABA), which carries inhibitory signals to help maintain stable and controlled communication between neurons.
One of the essential roles that the meis2 expressing cortical GABAergic cell plays in the organism is in establishing a balance between excitation and inhibition in neuronal signaling. The primary neurotransmitter that this cell synthesizes and releases is GABA, the main inhibitory neurotransmitter in the mammalian central nervous system. It acts to reduce neuronal excitability and prevents over-firing of neurons, effectively preventing excessive neuronal activity that could lead to seizures or overstimulation. By countering excessive excitatory signals, GABAergic cells work to stabilize the intricate dynamics of the brain and maintain physiological homeostasis.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1242/dev.200035"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/ncomms14219"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/s41598-022-21496-7"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.stemcr.2018.03.010"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/s41586-018-0654-5"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [meis2 expressing cortical GABAergic cell](http://purl.obolibrary.org/obo/CL_4023065) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_4023065"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "meis2 expressing cortical GABAergic cell on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [meis2 expressing cortical GABAergic cell](http://purl.obolibrary.org/obo/CL_4023065) SubClassOf [BFO_0000002](http://purl.obolibrary.org/obo/BFO_0000002) 


### mesangial cell `http://purl.obolibrary.org/obo/CL_0000650`

#### Added
- [mesangial cell](http://purl.obolibrary.org/obo/CL_0000650) [description](http://purl.org/dc/terms/description) "Mesangial cells are specialized cells that primarily reside in the middle layer, or mesangium, of the renal glomerulus within the kidney. The primary function of mesangial cells is to maintain the structural stability and function of the glomerular capillary tuft. They achieve this by providing structural support to the capillary loops, controlling the surface area available for filtration in response to physiological alterations in blood pressure, and removing trapped material and proteins from the basement membrane, which otherwise could cause damage. Moreover, mesangial cells regulate the glomerular blood flow by contracting and relaxing, similar to smooth muscle cells. They also synthesize extracellular matrix components, including collagen and fibronectin, that form the structural framework of the glomerulus.
Beyond their structural and regulatory roles, mesangial cells are involved in several immunologic functions. They express a variety of receptors that can recognize immune complexes and subsequently mediate an inflammatory reaction. Their role in the immune response includes the production of a variety of cytokines and growth factors, which make them a key cellular component in the progression of several renal diseases.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1007/BF00180754"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1681/ASN.2019070724"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.3389/fmed.2021.740527"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/s41581-021-00474-8"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.3389/fimmu.2022.1063497"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [mesangial cell](http://purl.obolibrary.org/obo/CL_0000650) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0000650"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "mesangial cell on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### mesenchymal cell `http://purl.obolibrary.org/obo/CL_0008019`
#### Removed
- [mesenchymal cell](http://purl.obolibrary.org/obo/CL_0008019) SubClassOf [eukaryotic cell](http://purl.obolibrary.org/obo/CL_0000255) 

#### Added
- [mesenchymal cell](http://purl.obolibrary.org/obo/CL_0008019) [description](http://purl.org/dc/terms/description) "Mesenchymal cells are a type of undifferentiated cell that possess the ability to develop into a wide variety of other cells. These cells originate from the mesoderm during embryonic development, which gives rise to tissues such as muscles, the skeletal system, and connective and vascular structures. Mesenchymal cells primarily function as progenitor cells with exceptional renewal capabilities and are characterized by their ability to migrate, aiding in tissue repair and regeneration.
Mesenchymal cells play a critical role in the development and maintenance of various biological systems throughout the body due to their versatility and adaptability. They can differentiate into more specialized cells, including osteoblasts, adipocytes, and chondrocytes, which are essential for bone, fat, and cartilage tissues, respectively. Furthermore, they contribute to the vascular and lymphatic systems through the formation of pericytes and lymphatic endothelial cells, securing the integrity of the circulatory system.
In the field of regenerative medicine, mesenchymal cells, particularly mesenchymal stem cells (MSCs), are of great interest due to their unique therapeutic potential. MSCs, for instance, have been implicated in wound healing and tissue repair because of their capacity to modulate the immune response. They can secrete a range of proteins and growth factors that aid in angiogenesis, the formation of new blood vessels, and the stimulation of resident progenitor cells to aid in tissue restoration.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1155/2016/1314709"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1089/scd.2016.0109"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/S0076-6879(06)20016-8"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/s41536-019-0083-6"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.21037/sci.2019.08.11"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [mesenchymal cell](http://purl.obolibrary.org/obo/CL_0008019) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0008019"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "mesenchymal cell on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### mesenchymal stem cell `http://purl.obolibrary.org/obo/CL_0000134`

#### Added
- [mesenchymal stem cell](http://purl.obolibrary.org/obo/CL_0000134) [description](http://purl.org/dc/terms/description) "Mesenchymal stem cells (MSCs) are multipotent stromal cells that can differentiate into a variety of cell types, including osteoblasts, chondrocytes, myocytes, and adipocytes. These cells originate mainly from the mesoderm of the embryo, which forms connective tissues, muscle, and the circulatory and urinary systems. However, in adults, MSCs are found in multiple tissues, including bone marrow, adipose tissue, the umbilical cord, and dental tissues. 
The primary function of MSCs is to maintain and repair the tissues in which they are found. When damage occurs, the MSCs are able to migrate to the site of injury, where they aid in regenerating the damaged tissue by differentiating into the required cell type and by secreting growth factors that enhance tissue repair and reduce inflammation. Furthermore, MSCs can also act as immunomodulators, suppressing immune reactions and reducing inflammation, both locally and systemically.
Apart from their role in tissue maintenance and repair, MSCs are integral to the field of regenerative medicine and are being investigated for their therapeutic potential in various clinical settings. Owing to their multipotent nature, immunomodulatory activity, and the relative ease of isolation, these cells can be engineered and translated into therapies to treat a variety of diseases, including bone and cartilage defects, liver diseases, heart disorders, and autoimmune diseases, amongst others. They have also been used as vectors for anticancer agents and in cell and gene therapy applications.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/nri2395"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/cdd.2013.158"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/s41536-019-0083-6"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.3727/096368910X"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [mesenchymal stem cell](http://purl.obolibrary.org/obo/CL_0000134) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0000134"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "mesenchymal stem cell on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### mesothelial cell `http://purl.obolibrary.org/obo/CL_0000077`

#### Added
- [mesothelial cell](http://purl.obolibrary.org/obo/CL_0000077) [description](http://purl.org/dc/terms/description) "Mesothelial cells are crucial components of the mesothelium, a protective layer that lines the body's internal cavities, including pleural cavity (around the lungs), the pericardial cavity (around the heart), and the peritoneal cavity (within the abdomen). Derived from the embryonic mesoderm layer, these cells exhibit a squamous-like epithelial morphology, usually appearing as a single layer of flat, pavement-like cells in their healthy state.
Mesothelial cells play a vital role in various physiological processes, such as intraperitoneal immunity, fibrinolysis, tissue repair, and tumor suppression. Their primary function is to provide a slippery, non-adhesive and protective surface to reduce friction and eases the movements of the organs within the pleural, pericardial, and peritoneal cavities, and they participate in the transport and movement of fluid and various molecules across the so-called serosal cavities. 
These cells have adaptive capabilities that facilitate their role in many pathological processes. In response to injury or inflammation, mesothelial cells can undergo morphological and functional changes, such as mesothelial-mesenchymal transitions, aiding in wound healing and tissue repair. Dysfunctions of mesothelial cells, however, can lead to serious health conditions, including adhesions, fibrosis, and malignancies such as peritoneal and pleural mesotheliomas.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/s41467-020-16893-3"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1046/j.1440-1843.2002.00404.x"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.3389/fphar.2015.00113"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/S1357-2725(03)00242-5"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [mesothelial cell](http://purl.obolibrary.org/obo/CL_0000077) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0000077"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "mesothelial cell on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### mesothelial fibroblast `http://purl.obolibrary.org/obo/CL_4023054`

#### Added
- [mesothelial fibroblast](http://purl.obolibrary.org/obo/CL_4023054) [description](http://purl.org/dc/terms/description) "Mesothelial fibroblasts primarily originate from mesothelial cells, which form a protective layer known as mesothelium covering certain internal organs such as the heart, lungs, and abdominal cavity. Functionally, mesothelial fibroblasts excel in their capacity to synthesize and secrete extracellular matrix (ECM), a complex network of proteins that provides structural and biochemical support to surrounding cells. The ECM produced by mesothelial fibroblasts acts as a scaffold for tissue growth and remodeling, which is particularly important in wound healing and tissue repair processes. Additionally, these cells participate in immune response modulation within the mesothelium layer, interacting with various immune cells and mediating inflammatory processes. However, in certain diseases like fibrosis, these cells can excessively proliferate and produce ECM, leading to tissue thickening and organ malfunction. They also play a role in the progression of mesothelioma, a type of cancer that affects the mesothelium.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.3747/pdi.2014.00188"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/s42003-018-0180-x"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.ccell.2022.04.011"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.3389/fphar.2015.00113"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.cell.2018.06.021"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [mesothelial fibroblast](http://purl.obolibrary.org/obo/CL_4023054) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_4023054"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "mesothelial fibroblast on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### mesothelial fibroblast of the leptomeninx `http://purl.obolibrary.org/obo/CL_4023058`

#### Added
- [mesothelial fibroblast of the leptomeninx](http://purl.obolibrary.org/obo/CL_4023058) [description](http://purl.org/dc/terms/description) "The mesothelial fibroblast of the leptomeninx is a specialized type of cell typically located in the leptomeninx or leptomeninges, a portion of meninges consisting of the arachnoid mater and pia mater. The meninges are protective layers surrounding the brain and spinal cord, with the leptomeninx being the delicate inner two layers. These fibroblasts play a critical role within the central nervous system, contributing to the structure, maintenance, and overall functionality of the leptomeninx.
Generally, mesothelial fibroblasts are characterized by their spindle-shaped morphology and their function, which pertains primarily to the production of the extracellular matrix (ECM). The ECM provides structural support, contributes to tissue repair, and influences various cellular activities such as growth and differentiation. Mesothelial fibroblasts are highly involved in collagen production, which is an integral protein in the ECM. Consequently, these cells play a vital role in physical and biological scaffolding and in maintaining the stability and resilience of the leptomeninges.
Under pathological conditions, mesothelial fibroblasts of the leptomeninx may undergo changes and contribute to disease progression. For example, in cases of meningitis, these cells can mount an inflammatory response to help fight off the infection. When the ECM is damaged due to trauma or disease, mesothelial fibroblasts of the leptomeninx can also become activated, migrating to the site of injury to aid in tissue repair and remodeling. Therefore, the mesothelial fibroblasts of the leptomeninx reflect a crucial multidimensional role in both normal physiological and disease-related processes within the leptomeninges of the central nervous system.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1371/journal.pone.0080463"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1148/rg.230039"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1126/science.adc8810"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.3892/mmr.2016.5411"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.neuron.2023.09.002"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [mesothelial fibroblast of the leptomeninx](http://purl.obolibrary.org/obo/CL_4023058) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_4023058"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "mesothelial fibroblast of the leptomeninx on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### midbrain dopaminergic neuron `http://purl.obolibrary.org/obo/CL_2000097`

#### Added
- [midbrain dopaminergic neuron](http://purl.obolibrary.org/obo/CL_2000097) [RO_0002175](http://purl.obolibrary.org/obo/RO_0002175) [NCBITaxon_9606](http://purl.obolibrary.org/obo/NCBITaxon_9606) 

- [midbrain dopaminergic neuron](http://purl.obolibrary.org/obo/CL_2000097) [in_subset](http://www.geneontology.org/formats/oboInOwl#inSubset) [human_reference_atlas](http://purl.obolibrary.org/obo/cl/cl-base#human_reference_atlas) 


### midzonal region hepatocyte `http://purl.obolibrary.org/obo/CL_0019028`
#### Removed
- [midzonal region hepatocyte](http://purl.obolibrary.org/obo/CL_0019028) EquivalentTo [hepatocyte](http://purl.obolibrary.org/obo/CL_0000182) and ([BFO_0000050](http://purl.obolibrary.org/obo/BFO_0000050) some [UBERON_8400007](http://purl.obolibrary.org/obo/UBERON_8400007)) 

- [midzonal region hepatocyte](http://purl.obolibrary.org/obo/CL_0019028) SubClassOf [BFO_0000050](http://purl.obolibrary.org/obo/BFO_0000050) some [UBERON_8400007](http://purl.obolibrary.org/obo/UBERON_8400007) 

#### Added
- [midzonal region hepatocyte](http://purl.obolibrary.org/obo/CL_0019028) EquivalentTo [hepatocyte](http://purl.obolibrary.org/obo/CL_0000182) and ([BFO_0000050](http://purl.obolibrary.org/obo/BFO_0000050) some [UBERON_8400008](http://purl.obolibrary.org/obo/UBERON_8400008)) 

- [midzonal region hepatocyte](http://purl.obolibrary.org/obo/CL_0019028) SubClassOf [BFO_0000050](http://purl.obolibrary.org/obo/BFO_0000050) some [UBERON_8400008](http://purl.obolibrary.org/obo/UBERON_8400008) 


### migratory dendritic cell `http://purl.obolibrary.org/obo/CL_4047054`
#### Removed
- [migratory dendritic cell](http://purl.obolibrary.org/obo/CL_4047054) [label](http://www.w3.org/2000/01/rdf-schema#label) "migratory dendtritic cell"^^[string](http://www.w3.org/2001/XMLSchema#string) 

#### Added
- [migratory dendritic cell](http://purl.obolibrary.org/obo/CL_4047054) [label](http://www.w3.org/2000/01/rdf-schema#label) "migratory dendritic cell"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### monocyte `http://purl.obolibrary.org/obo/CL_0000576`

#### Added
- [monocyte](http://purl.obolibrary.org/obo/CL_0000576) [description](http://purl.org/dc/terms/description) "Monocytes are a type of white blood cell that constitute part of the mononuclear phagocyte system. They feature prominently in the human body's first line of defense against foreign substances as a part of the innate immune system. Monocytes are the largest of all circulating leukocytes and are identified by their bean-shaped nucleus. They are produced in the bone marrow and circulate in the bloodstream; in some conditions, they may enter tissues and differentiate into macrophages or dendritic cells or a distinct monocyte-derived cell.
Monocytes function effectively as phagocytes, devouring harmful pathogens in the body. They engulf and destroy microbes, foreign substances, cancer cells, and other materials detrimental to the body's well-being. This process, known as phagocytosis, is vital for the body's immune response. Monocytes are also equipped to identify harmful pathogens and alert other immune cells to their presence. This is achieved by producing proinflammatory cytokines that recruit other immune cells to the affected area, amplifying the immune response.
Beyond their role in fighting infections, monocytes play an important role in the repair and healing of tissues. After an infection or inflammation subsides, monocytes can transition into different types of cells and assist in healing by clearing dead cells and debris and facilitating tissue regeneration. They are crucial in wound healing, control of microbial growth, and tissue development.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1146/annurev.immunol.021908.132557"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1126/science.1178331"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.immuni.2018.10.005"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [monocyte](http://purl.obolibrary.org/obo/CL_0000576) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0000576"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "monocyte on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### motor neuron `http://purl.obolibrary.org/obo/CL_0000100`

#### Added
- [motor neuron](http://purl.obolibrary.org/obo/CL_0000100) [description](http://purl.org/dc/terms/description) "Motor neurons, also known as motoneurons, are a type of neural cell that is fundamental to voluntary movement of muscles. Located within the central nervous system (CNS), motor neurons in the motor cortex travel to the brainstem and spinal cord. Motor neurons in the brainstem and spinal cord extend their lengthy axons out to muscles throughout the body. These specialized neurons are an inherent part of the so-called somatic division of the nervous system which mediates conscious control of skeletal muscles.
The primary function of motor neurons is to relay signals from the CNS to the body's many muscles, controlling both their contraction and relaxation. In response to impulses transmitted from the brain or spinal cord, it creates an electrical signal (action potential), which travels along its axon to cause the controlled movement of specific muscles. Motor neurons can connect with skeletal muscles, cardiac muscles, or glands, leading to three respective types: somatic motor neurons, autonomic motor neurons, and the neurons of the enteric nervous system.
Key components of motor neurons include the cell body (soma), dendrites, and axons. The cell body houses the nucleus and is the site of the neuron's intrinsic functions. Dendrites are thin structures that branch from the cell body and receive messages (electrical signals) from other neurons. The axon, a long, single tail-like extension, carries these signals away from the cell body towards the targeted muscle. If a motor neuron is damaged or dies, the muscle or gland it controls can lose function, which can result in conditions like motor neuron disease, leading to muscle weakness and atrophy.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.3389/fncel.2014.00293"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1146/annurev.neuro.051508.135722"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1249/JES.0000000000000032"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.semcdb.2017.11.016"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [motor neuron](http://purl.obolibrary.org/obo/CL_0000100) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0000100"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "motor neuron on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### mucosal-associated invariant T cell `http://purl.obolibrary.org/obo/CL_0000940`
#### Removed
- [mucosal-associated invariant T cell](http://purl.obolibrary.org/obo/CL_0000940) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "An alpha-beta T cell that is found in the lamina propria of mucosal tissues and is restricted by the MR-1 molecule."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "GOC:add"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "GO_REF:0000031"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "GOC:pam"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:19416870"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "GOC:tfm"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [mucosal-associated invariant T cell](http://purl.obolibrary.org/obo/CL_0000940) [label](http://www.w3.org/2000/01/rdf-schema#label) "mucosal invariant T cell"^^[string](http://www.w3.org/2001/XMLSchema#string) 

#### Added
- [mucosal-associated invariant T cell](http://purl.obolibrary.org/obo/CL_0000940) [has_exact_synonym](http://www.geneontology.org/formats/oboInOwl#hasExactSynonym) "mucosal invariant T cell"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [mucosal-associated invariant T cell](http://purl.obolibrary.org/obo/CL_0000940) [has_exact_synonym](http://www.geneontology.org/formats/oboInOwl#hasExactSynonym) "mucosal-associated invariant T lymphocyte"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [mucosal-associated invariant T cell](http://purl.obolibrary.org/obo/CL_0000940) [has_exact_synonym](http://www.geneontology.org/formats/oboInOwl#hasExactSynonym) "mucosal-associated invariant T-cell"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [mucosal-associated invariant T cell](http://purl.obolibrary.org/obo/CL_0000940) [has_exact_synonym](http://www.geneontology.org/formats/oboInOwl#hasExactSynonym) "mucosal-associated invariant T-lymphocyte"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [mucosal-associated invariant T cell](http://purl.obolibrary.org/obo/CL_0000940) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "An alpha-beta T cell that is found in the lamina propria of mucosal tissues and is restricted by the MR-1 molecule. This cell develops from double-positive, alpha-beta thymocyte."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:33352411"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "GOC:add"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "GO_REF:0000031"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "GOC:pam"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:19416870"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "GOC:tfm"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [mucosal-associated invariant T cell](http://purl.obolibrary.org/obo/CL_0000940) [has_related_synonym](http://www.geneontology.org/formats/oboInOwl#hasRelatedSynonym) "MAIT"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [has_synonym_type](http://www.geneontology.org/formats/oboInOwl#hasSynonymType) [OMO_0003000](http://purl.obolibrary.org/obo/OMO_0003000) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:40781543"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [mucosal-associated invariant T cell](http://purl.obolibrary.org/obo/CL_0000940) [label](http://www.w3.org/2000/01/rdf-schema#label) "mucosal-associated invariant T cell"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [mucosal-associated invariant T cell](http://purl.obolibrary.org/obo/CL_0000940) SubClassOf [RO_0002202](http://purl.obolibrary.org/obo/RO_0002202) some [double-positive, alpha-beta thymocyte](http://purl.obolibrary.org/obo/CL_0000809) 


### mucosal-associated invariant T cell, human `http://purl.obolibrary.org/obo/CL_0020002`

#### Added
- Class: [mucosal-associated invariant T cell, human](http://purl.obolibrary.org/obo/CL_0020002) 

- [mucosal-associated invariant T cell, human](http://purl.obolibrary.org/obo/CL_0020002) [contributor](http://purl.org/dc/terms/contributor) [0000-0001-5742-4697](https://orcid.org/0000-0001-5742-4697) 

- [mucosal-associated invariant T cell, human](http://purl.obolibrary.org/obo/CL_0020002) [contributor](http://purl.org/dc/terms/contributor) [0009-0000-8480-9277](https://orcid.org/0009-0000-8480-9277) 

- [mucosal-associated invariant T cell, human](http://purl.obolibrary.org/obo/CL_0020002) [date](http://purl.org/dc/terms/date) "2025-10-07T00:00:00Z"^^[dateTime](http://www.w3.org/2001/XMLSchema#dateTime) 

- [mucosal-associated invariant T cell, human](http://purl.obolibrary.org/obo/CL_0020002) [has_exact_synonym](http://www.geneontology.org/formats/oboInOwl#hasExactSynonym) "mucosal-associated invariant T lymphocyte, human"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [mucosal-associated invariant T cell, human](http://purl.obolibrary.org/obo/CL_0020002) [has_exact_synonym](http://www.geneontology.org/formats/oboInOwl#hasExactSynonym) "mucosal-associated invariant T-cell, human"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [mucosal-associated invariant T cell, human](http://purl.obolibrary.org/obo/CL_0020002) [has_exact_synonym](http://www.geneontology.org/formats/oboInOwl#hasExactSynonym) "mucosal-associated invariant T-lymphocyte, human"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [mucosal-associated invariant T cell, human](http://purl.obolibrary.org/obo/CL_0020002) [id](http://www.geneontology.org/formats/oboInOwl#id) "CL:0020002"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [mucosal-associated invariant T cell, human](http://purl.obolibrary.org/obo/CL_0020002) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A mucosal-associated invariant T (MAIT) cell found in various tissues including human blood, liver, gut, and lung, characterised by a semi-invariant T cell receptor (TCR) comprising TRAV1-2 (V&alpha;7.2) predominantly paired with TRAJ33, and high expression of CD161. Upon activation, MAIT cells produce inflammatory cytokines, including IFN-&gamma;, TNF-&alpha;, and IL-17."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:33352411"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:35641678"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:26382249"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:40781543"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [mucosal-associated invariant T cell, human](http://purl.obolibrary.org/obo/CL_0020002) [has_related_synonym](http://www.geneontology.org/formats/oboInOwl#hasRelatedSynonym) "MAIT, human"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [has_synonym_type](http://www.geneontology.org/formats/oboInOwl#hasSynonymType) [OMO_0003000](http://purl.obolibrary.org/obo/OMO_0003000) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:40781543"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [mucosal-associated invariant T cell, human](http://purl.obolibrary.org/obo/CL_0020002) [label](http://www.w3.org/2000/01/rdf-schema#label) "mucosal-associated invariant T cell, human"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [mucosal-associated invariant T cell, human](http://purl.obolibrary.org/obo/CL_0020002) EquivalentTo [mucosal-associated invariant T cell](http://purl.obolibrary.org/obo/CL_0000940) and ([lacks_plasma_membrane_part](http://purl.obolibrary.org/obo/CL_4030046) some [PR_Q12918](http://purl.obolibrary.org/obo/PR_Q12918)) and ([RO_0002104](http://purl.obolibrary.org/obo/RO_0002104) some [PR_A0A0B4J238](http://purl.obolibrary.org/obo/PR_A0A0B4J238)) and ([RO_0002162](http://purl.obolibrary.org/obo/RO_0002162) some [NCBITaxon_9606](http://purl.obolibrary.org/obo/NCBITaxon_9606)) 

- [mucosal-associated invariant T cell, human](http://purl.obolibrary.org/obo/CL_0020002) SubClassOf [mucosal-associated invariant T cell](http://purl.obolibrary.org/obo/CL_0000940) 

- [mucosal-associated invariant T cell, human](http://purl.obolibrary.org/obo/CL_0020002) SubClassOf [lacks_plasma_membrane_part](http://purl.obolibrary.org/obo/CL_4030046) some [PR_Q12918](http://purl.obolibrary.org/obo/PR_Q12918) 

- [mucosal-associated invariant T cell, human](http://purl.obolibrary.org/obo/CL_0020002) SubClassOf [RO_0002104](http://purl.obolibrary.org/obo/RO_0002104) some [PR_A0A0B4J238](http://purl.obolibrary.org/obo/PR_A0A0B4J238) 


### mucous acinar cell of salivary gland `http://purl.obolibrary.org/obo/CL_4052066`

#### Added
- Class: [mucous acinar cell of salivary gland](http://purl.obolibrary.org/obo/CL_4052066) 

- [mucous acinar cell of salivary gland](http://purl.obolibrary.org/obo/CL_4052066) [contributor](http://purl.org/dc/terms/contributor) [0009-0000-8480-9277](https://orcid.org/0009-0000-8480-9277) 

- [mucous acinar cell of salivary gland](http://purl.obolibrary.org/obo/CL_4052066) [date](http://purl.org/dc/terms/date) "2025-08-05T15:33:46Z"^^[dateTime](http://www.w3.org/2001/XMLSchema#dateTime) 

- [mucous acinar cell of salivary gland](http://purl.obolibrary.org/obo/CL_4052066) [id](http://www.geneontology.org/formats/oboInOwl#id) "CL:4052066"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [mucous acinar cell of salivary gland](http://purl.obolibrary.org/obo/CL_4052066) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "An acinar cell of the salivary gland, responsible for producing a viscous, mucin‚Äêrich solution, thereby facilitating lubrication and protection of the oral mucosa. This cell is characterized by an elongated shape, clear cytoplasm filled with mucin granules, and a basally compressed nucleus (Barrows et al., 2020). In humans and mice, the mucous acinar cell predominates in the sublingual gland and minor salivary glands, while also being present in mixed glands such as the submandibular gland, where it coexists with serous acinar cells (Maruyama et al., 2019)."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:35343828"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:32829674"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:29383862"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [mucous acinar cell of salivary gland](http://purl.obolibrary.org/obo/CL_4052066) [label](http://www.w3.org/2000/01/rdf-schema#label) "mucous acinar cell of salivary gland"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [mucous acinar cell of salivary gland](http://purl.obolibrary.org/obo/CL_4052066) EquivalentTo [acinar cell of salivary gland](http://purl.obolibrary.org/obo/CL_0002623) and ([BFO_0000050](http://purl.obolibrary.org/obo/BFO_0000050) some [UBERON_0014717](http://purl.obolibrary.org/obo/UBERON_0014717)) 

- [mucous acinar cell of salivary gland](http://purl.obolibrary.org/obo/CL_4052066) SubClassOf [mucus secreting cell](http://purl.obolibrary.org/obo/CL_0000319) 

- [mucous acinar cell of salivary gland](http://purl.obolibrary.org/obo/CL_4052066) SubClassOf [acinar cell of salivary gland](http://purl.obolibrary.org/obo/CL_0002623) 

- [mucous acinar cell of salivary gland](http://purl.obolibrary.org/obo/CL_4052066) SubClassOf [BFO_0000050](http://purl.obolibrary.org/obo/BFO_0000050) some [UBERON_0014717](http://purl.obolibrary.org/obo/UBERON_0014717) 

- [mucous acinar cell of salivary gland](http://purl.obolibrary.org/obo/CL_4052066) SubClassOf [BFO_0000051](http://purl.obolibrary.org/obo/BFO_0000051) some [GO_0098594](http://purl.obolibrary.org/obo/GO_0098594) 


### multiciliated epithelial cell of the bronchus `http://purl.obolibrary.org/obo/CL_0002332`

#### Added
- [multiciliated epithelial cell of the bronchus](http://purl.obolibrary.org/obo/CL_0002332) [description](http://purl.org/dc/terms/description) "The ciliated cell of the bronchus is a specialized type of epitheliali cell found in the lining of the bronchus, which is part of the tracheobronchial tree. These cells are characterized by the presence of hair-like structures called cilia on their surface. The primary function of these cilia is the active transport of mucus and trapped particles out of the respiratory tract.
Ciliated cells of the bronchus form a crucial part of the body's innate defensive mechanism: They function as a barrier and play a key role in the mucociliary clearance mechanism - the coordinated, wave-like movement of the cilia that sweeps pathogens and particles trapped within the mucus in the airways, upward and out of the lungs, either to be swallowed or expelled by coughing. By doing so, these cells are crucial for preventing the lodging of pathogens deep in the lungs and hence reduce the risk of respiratory infections.
In conditions affecting the respiratory tract, including chronic obstructive pulmonary disease (COPD) and asthma, the normal functioning of the ciliated cells may be compromised. For instance, the cilia may become impaired or destroyed due to exposure to harmful substances such as cigarette smoke, which can hinder the process of mucus removal, leading to a build-up of mucus and increased susceptibility to infection.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1146/annurev-physiol-021014-071931"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1111/resp.12542"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1101/cshperspect.a028241"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [multiciliated epithelial cell of the bronchus](http://purl.obolibrary.org/obo/CL_0002332) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0002332"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "multiciliated epithelial cell of the bronchus on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### multipolar neuron `http://purl.obolibrary.org/obo/CL_0000104`

#### Added
- [multipolar neuron](http://purl.obolibrary.org/obo/CL_0000104) [description](http://purl.org/dc/terms/description) "A multipolar neuron is a specialized type of neuron that comprises one axon and multiple dendrites, distinguishing it from other varieties in the neuronal family. This cell type is distinctively known for its structural architecture and extraordinary capabilities in information processing. The multipolar configuration enables a wide branching network that supports the reception, integration, and transmission of signals over varying distances, translating into complex functionalities like thought, action, and perception.
The primary function of a multipolar neuron is to process and transmit information in the form of electrical impulses or action potentials. Each of the multiple dendrites acts as a receiver, collecting synaptic inputs from other neurons and transferring these signals to the neuron's cell body. Here, the information is assembled and, upon reaching a particular threshold, will trigger the generation of an action potential along the axon, the singular elongated projection that results in the neuron's output. This signal propagation, facilitated by the axon's myelination and nodes of Ranvier, leads to highly efficient and fast information transmission to other neurons, muscle cells, gland cells, and other organs.
The capacity of multipolar neurons to establish vast connections with other neurons makes them the primary constituents of the central nervous system. They play indispensable roles in both sensory and motor pathways, as well as in higher cognitive functions such as memory and learning. These neurons also enrich the autonomic system, contributing to both sympathetic and parasympathetic pathways. Hence, multipolar neurons play an essential role in performing and regulating a plethora of activities, ranging from basic reflexes to intricate cognitive processes.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1126/science.3059497"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.autneu.2006.05.003"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1101/cshperspect.a001644"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1523/JNEUROSCI.1610-15.2015"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [multipolar neuron](http://purl.obolibrary.org/obo/CL_0000104) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0000104"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "multipolar neuron on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### muscle fibroblast `http://purl.obolibrary.org/obo/CL_1001609`

#### Added
- [muscle fibroblast](http://purl.obolibrary.org/obo/CL_1001609) [description](http://purl.org/dc/terms/description) "Muscle fibroblasts contribute indispensably to the maintenance, repair, and regeneration processes in the muscles tissue of the body. They are a subtype of fibroblasts, which are generally known as the 'builders' of the animal body's extracellular matrix (ECM) and are crucial players in wound healing. Occupying the connective tissues of muscles, muscle fibroblasts partake in the synthesis of the ECM components, including collagen, fibronectin, and laminin, that give the muscle its structure and flexibility.
Interestingly, muscle fibroblasts metamorphose functionally during the process of muscle repair following injury. In this circumstance, they react to signals from injured muscle fibers and inflammatory cells, transforming into myofibroblasts. These newly-transformed myofibroblasts showcase contractile properties similar to smooth muscle cells and play a significant role in wound contraction during the healing process while continuing to produce ECM components to ensure reparative scaffolding. Furthermore, muscle fibroblasts secrete various growth factors, primarily those belonging to the 'Transforming Growth Factor-&beta;' (TGF-&beta;) family, to stimulate muscle repair and regulate immune responses.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1113/JP274403"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.3389/fphys.2020.00253"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1002/cbin.10725"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.3389/fimmu.2022.918223"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.diff.2016.05.007"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [muscle fibroblast](http://purl.obolibrary.org/obo/CL_1001609) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_1001609"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "muscle fibroblast on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### muscle precursor cell `http://purl.obolibrary.org/obo/CL_0000680`

#### Added
- [muscle precursor cell](http://purl.obolibrary.org/obo/CL_0000680) [description](http://purl.org/dc/terms/description) "Muscle precursor cells, commonly referred to as myogenic progenitor cells or myoblasts, play a pivotal role in the intricate processes of muscle regeneration and development. As undifferentiated cells, they harbor the potential to evolve into distinct specialized cells, specifically the muscle cell or myocyte. Their crucial contribution lies in guiding the transformation of the mesodermal layer, one of the three primary germ cell layers in the early embryo, into fully developed skeletal muscle tissues during embryonic development.
Muscle precursor cells are characterized by their proliferative capacity and rapid differentiation into mature myocytes. They fuse into myotubes, culminating in the formation of muscle fibers. Their responsiveness to signals such as insulin-like growth factors, transforming growth factor-&beta;, and neural inputs enables precise control over their proliferation and differentiation. This regenerative capacity equips them with the ability to repair injured or damaged muscle tissue, a critical function in preserving muscle homeostasis and overall function. These cells also actively contribute to adaptive responses, such as muscle growth and strengthening, in reaction to factors like training or aging.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "https://www.sciencedirect.com/topics/engineering/muscle-precursor-cell"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1152/physrev.00061.2017"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1089/ars.2007.1792"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [muscle precursor cell](http://purl.obolibrary.org/obo/CL_0000680) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0000680"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "muscle precursor cell on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### myelocyte `http://purl.obolibrary.org/obo/CL_0002193`

#### Added
- [myelocyte](http://purl.obolibrary.org/obo/CL_0002193) [description](http://purl.org/dc/terms/description) "A myelocyte is a type of immature granulocyte found in the bone marrow during the production of blood cells, a process known as hematopoiesis. Myelocytes represent an intermediate stage in the development of granulocytes, which are a type of white blood cell essential for the immune system. The three kinds of granulocytes that derive from myelocytes are neutrophils, eosinophils, and basophils, each with distinct properties and functions. Myelocytes possess some functionalities of a mature cell such as the presence of specific granules.
The main function of myelocytes is to differentiate into mature granulocytes. The process starts with a hematopoietic stem cell, which progressively matures into a myeloblast, and further into a promyelocyte before eventually becoming a myelocyte. At this stage, the cell begins producing granules that carry specific enzymes and other substances used to fight infections. The most common type of myelocyte is the neutrophilic myelocyte, which eventually develops into a neutrophil, the most common type of white blood cell.
Neutrophilic myelocytes contribute significantly to the body's primary immune response as mature neutrophils are often the first cells to arrive at a site of infection. They ingest and destroy bacteria and other harmful agents to prevent further infection. Eosinophilic and basophilic myelocytes develop into eosinophils and basophils, respectively, playing their role in managing allergic reactions and parasitic infections. Dysfunction in myelocyte development can give rise to various diseases, including different forms of leukemia, where myelocytes undergo uncontrolled proliferation.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/myelocyte"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1182/blood.V71.3.640.640"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/s41375-023-01989-8"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "https://www.sciencedirect.com/topics/medicine-and-dentistry/granulopoiesis"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [myelocyte](http://purl.obolibrary.org/obo/CL_0002193) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0002193"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "myelocyte on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### myeloid dendritic cell `http://purl.obolibrary.org/obo/CL_0000782`

#### Added
- [myeloid dendritic cell](http://purl.obolibrary.org/obo/CL_0000782) [description](http://purl.org/dc/terms/description) "Myeloid dendritic cells are a subset of dendritic cells derived from the common myeloid progenitor cell. &quot;Conventional&quot; and &quot;myeloid&quot; DCs are often used interchangeably. In contrast, dendritic cells may also arise from the common lymphoid progenitor. Monocyte-derived dendritic cells may also be induced by inflammatory states.
Dendritic cells function as antigen-presenting cells and are able to activate na&iuml;ve T cells to induce the adaptive immune response. Dendritic cells capture, process, and present antigens derived from pathogens to mount a specific immune response against the invading antigen. Dendritic cells also produce a range of cytokines that help in modulating and recruiting other immune cells during an immune response. 
Dendritic cells can contribute to immune tolerance by deleting self-reactive T cells or by inducing regulatory T-cells, thereby preventing autoimmunity. Dendritic cells thus are important in orchestrating the adaptive immune response and are pivotal in combating infections and maintaining tolerance to self.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.imbio.2014.12.010"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1002/jlb.66.2.205"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.3389/fimmu.2018.03176"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.immuni.2007.06.006"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1111/imm.12888"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [myeloid dendritic cell](http://purl.obolibrary.org/obo/CL_0000782) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0000782"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "myeloid dendritic cell on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### myeloid leukocyte `http://purl.obolibrary.org/obo/CL_0000766`

#### Added
- [myeloid leukocyte](http://purl.obolibrary.org/obo/CL_0000766) [description](http://purl.org/dc/terms/description) "Myeloid leukocytes are a group of immune cells descended from the myeloid lineage. The myeloid lineage constitutes a significant portion of the body's defense system, particularly against microbial pathogens and foreign substances. Myeloid leukocytes originate from hematopoietic stem cells located in the bone marrow. The commitment of a pluripotent hematopoietic stem cell into a myeloid progenitor cell represents the initiation step towards the development of myeloid leukocytes. Once differentiated, these cells contribute diversely to both the innate and adaptive immune responses in the body.
Myeloid leukocytes include several distinct cell types based on functionality and structure, encompassing monocytes, macrophages, granulocytes (neutrophils, eosinophils, and basophils), and dendritic cells. Each cell type has specialized functions, occupying a unique role in the body's immune response. For instance, neutrophils are prolific phagocytes&mdash;cells that engulf and destroy pathogens&mdash;while eosinophils are instrumental in combating multicellular parasites and modulating allergic inflammatory responses. Monocytes, in addition to their phagocytic activities, can enter peripheral tissues and differentiate into macrophages or dendritic cells which are crucial for antigen presentation to T cells.
Another critical function of myeloid leukocytes is their involvement in tissue remodeling and repair. For example, macrophages not only play a key role in defending the body but also perform duties essential for wound healing, such as removing dead cells and stimulating tissue repair. In general, myeloid leukocytes participate in the removal of old and damaged cells, an essential process for maintaining good health and homeostasis. Through the orchestration of complex interactions and signal transductions, myeloid leukocytes provide the body with a robust defense system against foreign invaders and participate in maintaining the body's overall balance and function.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.biocel.2004.01.020"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "https://www.ncbi.nlm.nih.gov/books/NBK563148/"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [myeloid leukocyte](http://purl.obolibrary.org/obo/CL_0000766) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0000766"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "myeloid leukocyte on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### myocardial endocrine cell of septal division of left branch of atrioventricular bundle `http://purl.obolibrary.org/obo/CL_1000375`
#### Removed
- [myocardial endocrine cell of septal division of left branch of atrioventricular bundle](http://purl.obolibrary.org/obo/CL_1000375) SubClassOf [His-Purkinje system cell](http://purl.obolibrary.org/obo/CL_0010007) 



### myocyte of anterior internodal tract `http://purl.obolibrary.org/obo/CL_1000287`

#### Added
- [myocyte of anterior internodal tract](http://purl.obolibrary.org/obo/CL_1000287) [RO_0002175](http://purl.obolibrary.org/obo/RO_0002175) [NCBITaxon_9606](http://purl.obolibrary.org/obo/NCBITaxon_9606) 

- [myocyte of anterior internodal tract](http://purl.obolibrary.org/obo/CL_1000287) [in_subset](http://www.geneontology.org/formats/oboInOwl#inSubset) [human_reference_atlas](http://purl.obolibrary.org/obo/cl/cl-base#human_reference_atlas) 


### myocyte of middle internodal tract `http://purl.obolibrary.org/obo/CL_1000290`

#### Added
- [myocyte of middle internodal tract](http://purl.obolibrary.org/obo/CL_1000290) [RO_0002175](http://purl.obolibrary.org/obo/RO_0002175) [NCBITaxon_9606](http://purl.obolibrary.org/obo/NCBITaxon_9606) 

- [myocyte of middle internodal tract](http://purl.obolibrary.org/obo/CL_1000290) [in_subset](http://www.geneontology.org/formats/oboInOwl#inSubset) [human_reference_atlas](http://purl.obolibrary.org/obo/cl/cl-base#human_reference_atlas) 


### myocyte of posterior internodal tract `http://purl.obolibrary.org/obo/CL_1000291`

#### Added
- [myocyte of posterior internodal tract](http://purl.obolibrary.org/obo/CL_1000291) [RO_0002175](http://purl.obolibrary.org/obo/RO_0002175) [NCBITaxon_9606](http://purl.obolibrary.org/obo/NCBITaxon_9606) 

- [myocyte of posterior internodal tract](http://purl.obolibrary.org/obo/CL_1000291) [in_subset](http://www.geneontology.org/formats/oboInOwl#inSubset) [human_reference_atlas](http://purl.obolibrary.org/obo/cl/cl-base#human_reference_atlas) 


### myofibroblast cell `http://purl.obolibrary.org/obo/CL_0000186`

#### Added
- [myofibroblast cell](http://purl.obolibrary.org/obo/CL_0000186) [description](http://purl.org/dc/terms/description) "Myofibroblasts are specialized cells predominantly found within the connective tissues of the body, most notably in areas of injury or wound healing. They carry out a crucial role in many biological processes, including tissue repair and fibrosis, wound contraction, and organogenesis. Though similar to fibroblasts, myofibroblasts can be differentiated by their unique morphological features and the presence of specialized protein bundles called stress fibers.
Myofibroblasts originate from several cell types based on the physiological condition or the injury context, including local fibroblasts, epithelial cells, and endothelial cells, shedding light onto their plasticity. Upon tissue injury or wound creation, myofibroblasts activate and undergo a transformation process whereby they develop contractile features. They express a type of protein called alpha-smooth muscle actin (&alpha;-SMA), allowing them to contract and aid in wound closure. These cells also produce extracellular matrix (ECM) components, primarily collagen, to support and provide structural integrity to the healing tissue.
However, while myofibroblasts play a critical role in normal wound healing, their function can be detrimental in certain pathological conditions. For instance, in chronic diseases such as fibrosis or scarring, the extended presence or overactivity of myofibroblasts can lead to the excessive production of ECM and fibrous tissues, leading to hardened, rigid tissues and organ dysfunction.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1002/path.4104"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.7860/JCDR/2014/7820.4231"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/nrm809"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.3390/biom11081095"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.2147/CCID.S50046"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [myofibroblast cell](http://purl.obolibrary.org/obo/CL_0000186) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0000186"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "myofibroblast cell on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### myotube `http://purl.obolibrary.org/obo/CL_0002372`

#### Added
- [myotube](http://purl.obolibrary.org/obo/CL_0002372) [description](http://purl.org/dc/terms/description) "Myotubes are multinucleated cells and are categorized under the umbrella of skeletal muscle cells. They are formed through an intricate process known as myogenesis, which is a part of skeletal muscle development. It is vital to note that myotubes are not terminally differentiated but are precursors to mature muscle fiber cells.
Myotubes come to life through the fusion of myoblasts, which are undifferentiated mononucleated progenitor cells. In the early stages of embryonic muscle development, myoblasts proliferate rapidly under the influence of growth and differentiation factors. As myoblasts reach confluence, they withdraw from the cell cycle, align themselves with each other, increase expression of muscle-specific proteins, and ultimately fuse to form myotubes. The formation of myotubes heralds a critical transition, as this is when cells shift from proliferation and growth to differentiation and maturation.
Functionally, myotubes play a critical role in the formation and development of skeletal muscles. Myotubes gradually grow in size through the incorporation of additional myoblasts and eventually mature into muscle fibers, the contractile units of skeletal muscles. During this maturation process, myotubes polymerize their soluble muscle-specific proteins into myofibrils, which are essential for muscle contraction. Overall, myotubes are vital players in the complex process of muscle development and homeostasis.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "https://www.sciencedirect.com/topics/medicine-and-dentistry/myotube"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "https://www.ncbi.nlm.nih.gov/books/NBK10006"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1007/s10544-012-9709-9"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [myotube](http://purl.obolibrary.org/obo/CL_0002372) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0002372"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "myotube on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### nail bed keratinocyte `http://purl.obolibrary.org/obo/CL_4052063`

#### Added
- Class: [nail bed keratinocyte](http://purl.obolibrary.org/obo/CL_4052063) 

- [nail bed keratinocyte](http://purl.obolibrary.org/obo/CL_4052063) [contributor](http://purl.org/dc/terms/contributor) [0009-0000-8480-9277](https://orcid.org/0009-0000-8480-9277) 

- [nail bed keratinocyte](http://purl.obolibrary.org/obo/CL_4052063) [date](http://purl.org/dc/terms/date) "2024-07-22T15:30:00Z"^^[dateTime](http://www.w3.org/2001/XMLSchema#dateTime) 

- [nail bed keratinocyte](http://purl.obolibrary.org/obo/CL_4052063) [id](http://www.geneontology.org/formats/oboInOwl#id) "CL:4052063"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [nail bed keratinocyte](http://purl.obolibrary.org/obo/CL_4052063) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A keratinocyte that resides in the nail bed, responsible for tightly attaching the nail plate to the underlying dermis. Unlike typical epidermal keratinocytes, it exhibits limited proliferation, undergoes modified differentiation that skips the granular layer and conventional cornification, and primarily ensures adhesion and maintenance of the nail plate. In humans, this cell expresses characteristic markers, including the protease inhibitor SPINK6 and the differentiation-associated protein KRT16."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:34099859"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:10651700"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:39044464"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:30969555"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:33799809"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [nail bed keratinocyte](http://purl.obolibrary.org/obo/CL_4052063) [label](http://www.w3.org/2000/01/rdf-schema#label) "nail bed keratinocyte"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [nail bed keratinocyte](http://purl.obolibrary.org/obo/CL_4052063) SubClassOf [epidermal keratinocyte](http://purl.obolibrary.org/obo/CL_4052061) 

- [nail bed keratinocyte](http://purl.obolibrary.org/obo/CL_4052063) SubClassOf [BFO_0000050](http://purl.obolibrary.org/obo/BFO_0000050) some [UBERON_0005273](http://purl.obolibrary.org/obo/UBERON_0005273) 


### nail matrix keratinocyte `http://purl.obolibrary.org/obo/CL_4052064`

#### Added
- Class: [nail matrix keratinocyte](http://purl.obolibrary.org/obo/CL_4052064) 

- [nail matrix keratinocyte](http://purl.obolibrary.org/obo/CL_4052064) [contributor](http://purl.org/dc/terms/contributor) [0009-0000-8480-9277](https://orcid.org/0009-0000-8480-9277) 

- [nail matrix keratinocyte](http://purl.obolibrary.org/obo/CL_4052064) [date](http://purl.org/dc/terms/date) "2025-07-22T15:30:00Z"^^[dateTime](http://www.w3.org/2001/XMLSchema#dateTime) 

- [nail matrix keratinocyte](http://purl.obolibrary.org/obo/CL_4052064) [id](http://www.geneontology.org/formats/oboInOwl#id) "CL:4052064"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [nail matrix keratinocyte](http://purl.obolibrary.org/obo/CL_4052064) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A keratinocyte that is part of the nail matrix epithelium, distinguished by its high proliferative capacity and its role in nail plate formation via onychokeratinization, a differentiation process that uniquely bypasses the formation of a granular layer, resulting in the production of hard, compact keratin. Unlike skin epidermal keratinocytes, this cell expresses a distinctive mix of hard (hair-type) and select epidermal keratins (Kitahara and Ogawa, 1993). In humans, LGR6 and WNT6 are enriched in the basal compartment, with LGR6 marking nail stem cells in both mice and humans, and exhibiting strong expression in the human proximal nail matrix (Kim et al., 2021)."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:34099859"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:7679137"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:39044464"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:30969555"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:28098391"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [nail matrix keratinocyte](http://purl.obolibrary.org/obo/CL_4052064) [has_exact_synonym](http://www.geneontology.org/formats/oboInOwl#hasExactSynonym) "nail matrix epithelial keratinocyte"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:39044464"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [nail matrix keratinocyte](http://purl.obolibrary.org/obo/CL_4052064) [label](http://www.w3.org/2000/01/rdf-schema#label) "nail matrix keratinocyte"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [nail matrix keratinocyte](http://purl.obolibrary.org/obo/CL_4052064) EquivalentTo [keratinocyte](http://purl.obolibrary.org/obo/CL_0000312) and ([BFO_0000050](http://purl.obolibrary.org/obo/BFO_0000050) some [UBERON_0002283](http://purl.obolibrary.org/obo/UBERON_0002283)) 

- [nail matrix keratinocyte](http://purl.obolibrary.org/obo/CL_4052064) SubClassOf [epidermal keratinocyte](http://purl.obolibrary.org/obo/CL_4052061) 

- [nail matrix keratinocyte](http://purl.obolibrary.org/obo/CL_4052064) SubClassOf [BFO_0000050](http://purl.obolibrary.org/obo/BFO_0000050) some [UBERON_0002283](http://purl.obolibrary.org/obo/UBERON_0002283) 


### near-projecting glutamatergic cortical neuron `http://purl.obolibrary.org/obo/CL_4023012`

#### Added
- [near-projecting glutamatergic cortical neuron](http://purl.obolibrary.org/obo/CL_4023012) [description](http://purl.org/dc/terms/description) "The near-projecting glutamatergic cortical neuron is a critical functional unit of the mammalian cerebral cortex, predominantly found in the brain's cortical layers. This neuron employs glutamate as its principal neurotransmitter, implying its role is primarily excitatory as glutamate is the chief excitatory neurotransmitter within the mammalian central nervous system and is essential for instigating action potentials. Unlike other glutamatergic projecting neurons, these neurons project their axons over shorter distances, usually within the same region of the brain, to communicate with nearby neurons or neural circuits, giving them their 'near-projecting' nomenclature. This local propagation ability allows for fine-tuned transmission of signals, supporting the enhancement of sophisticated neural mechanisms like sensory processing, learning, and memory.
The specific functions of near-projecting glutamatergic cortical neurons are determined by which cortical layers they are located in and which neurons are their main postsynaptic targets. As a fundamental component of the complex cortical circuitry, they act interdependently with their neighboring cells, modulating the delicately balanced excitation and inhibition system. Dysfunction of these glutamatergic neurons can contribute to neurological conditions like epilepsy, Alzheimer's, or other manifestations of neurotoxicity.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.conb.2014.01.015"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/nrn3586"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.neuron.2015.11.002"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/s41586-018-0654-5"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [near-projecting glutamatergic cortical neuron](http://purl.obolibrary.org/obo/CL_4023012) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_4023012"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "near-projecting glutamatergic cortical neuron on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### neuromast hair cell `http://purl.obolibrary.org/obo/CL_0000856`
#### Removed
- [neuromast hair cell](http://purl.obolibrary.org/obo/CL_0000856) SubClassOf [BFO_0000002](http://purl.obolibrary.org/obo/BFO_0000002) 



### neutrophil `http://purl.obolibrary.org/obo/CL_0000775`

#### Added
- [neutrophil](http://purl.obolibrary.org/obo/CL_0000775) [description](http://purl.org/dc/terms/description) "Neutrophils are an essential component of the human immune system and belong to the category of white blood cells known as granulocytes. They constitute approximately 50-70% of all white blood cells, making them the most abundant type of immune cells in mammals. Neutrophils develop in the bone marrow from hematopoietic stem cells before being released into the bloodstream where they circulate before migration into tissues, thereby exhibiting a multi-step process that includes margination, rolling, adhesion, and transmigration.
Phenotypically and functionally, neutrophils represent a heterogeneous cell population. They are the immune system&rsquo;s immediate responders and are the first cells to migrate towards the site of an infection. They play a critical role in defending the body from potential pathogenic bacteria and fungi. Their primary mechanism of action involves phagocytosis, where they engulf and destroy these microbes. Neutrophils are characterized by the presence of large quantities of granules containing antimicrobial proteins and enzymes, including defensins and myeloperoxidase, which are released once the cell has recognized and attached to the microbes. The enzymes and other antimicrobial agents can also be released into the surrounding environment, a process known as degranulation.
Neutrophils also possess the unique biological ability to eliminate pathogens through a process called neutrophil extracellular trap (NET) formation, or NETosis. In this process, neutrophils release a mixture of DNA, histones, and granule proteins that form a web-like structure which traps and kills microbes. Notably, this process usually leads to the death of the neutrophil itself, referred to as cell suicide or programmed cell death. 
Neutrophils play a crucial role in the immune system by rapidly responding to infections and inflammation, excessive or dysregulated neutrophil activity can contribute to tissue damage and chronic inflammation, exacerbating various diseases such as rheumatoid arthritis, and neutrophils may contribute to tumor growth and progression.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.3389/fphys.2018.00113"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/s41577-021-00571-6"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1111/febs.15803"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1007/s10753-016-0458-4"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/s41577-019-0141-8"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [neutrophil](http://purl.obolibrary.org/obo/CL_0000775) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0000775"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "neutrophil on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### nodal myocyte `http://purl.obolibrary.org/obo/CL_0002072`

#### Added
- [nodal myocyte](http://purl.obolibrary.org/obo/CL_0002072) [RO_0002175](http://purl.obolibrary.org/obo/RO_0002175) [NCBITaxon_9606](http://purl.obolibrary.org/obo/NCBITaxon_9606) 

- [nodal myocyte](http://purl.obolibrary.org/obo/CL_0002072) [in_subset](http://www.geneontology.org/formats/oboInOwl#inSubset) [human_reference_atlas](http://purl.obolibrary.org/obo/cl/cl-base#human_reference_atlas) 


### non-classical monocyte `http://purl.obolibrary.org/obo/CL_0000875`

#### Added
- [non-classical monocyte](http://purl.obolibrary.org/obo/CL_0000875) [description](http://purl.org/dc/terms/description) "Non-classical monocytes, also known as patrolling monocytes, are a subtype of monocytes in the circulation, accounting for approximately 5-10% of total human monocytes. These monocytes are characterized by the differential expression of two surface markers, CD14 and CD16, with non-classical monocytes expressing CD14 and high levels of CD16 (CD14+ CD16++). Classical monocytes are released from the bone marrow into the circulation and can transition to intermediate followed by non-classical monocytes.
Non-classical monocytes patrol the endothelium and, upon sensing inflammation or injury, migrate across it. Non-classical monocytes are also involved in the removal of damaged cells via phagocytosis. This process enables the cells to engulf and destroy pathogens, debris, and apoptotic cells. During homeostasis, these monocytes appear to have an important function in maintaining vascular integrity.
Non-classical monocytes are also able to induce CD4+ T cell proliferation and sense viruses. They can produce and secrete a wide range of inflammatory cytokines and chemokines to reinforce and shape the immune response. Non-classical monocytes also appear to play a role in different inflammatory diseases states including atherosclerosis, multiple sclerosis, and rheumatoid arthritis.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1111/sji.12883"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.immuni.2018.10.005"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1146/annurev-immunol-042617-053119"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [non-classical monocyte](http://purl.obolibrary.org/obo/CL_0000875) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0000875"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "non-classical monocyte on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### non-pigmented ciliary epithelial cell `http://purl.obolibrary.org/obo/CL_0002304`

#### Added
- [non-pigmented ciliary epithelial cell](http://purl.obolibrary.org/obo/CL_0002304) [description](http://purl.org/dc/terms/description) "Non-pigmented ciliary epithelial cells are a specialized cell type found in the ciliary body of the eye. The ciliary body is responsible for producing the aqueous humor, which is the clear, watery fluid that fills the anterior portion of the eye. These cells, also known as the ciliary epithelium's non-pigmented layer, are located immediately beneath the pigmented ciliary epithelial cells. They play several key roles in maintaining eye health and function and contribute to the eye's intricate physiology.
Together with the endothelial cells of the iris and ciliary blood vessels, the non-pigmented ciliary epithelium form the blood aqueous barrier in the anterior part of the eye. One of the main functions of non-pigmented ciliary epithelial cells includes the production and regulation of aqueous humor, which provides nourishment and oxygen supply to the avascular lens and cornea, two vital eye components found in the anterior segment. Aqueous humor also helps to maintain intraocular pressure, which is critical for retaining the shape of the eye. Aberrations in this balance often result in ocular pathologies like glaucoma. Non-pigmented ciliary epithelial cells play a key role in the secretory processes that regulate the aqueous humor through active transport, diffusion, and ultrafiltration from the blood plasma. 
Therefore, due to their complex and multi-faceted functions sustaining eye health, non-pigmented ciliary epithelial cells represent a critical cell type in eye biology and physiological equilibrium.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/S1569-2590(05)10005-6"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1007/978-1-4615-1203-5_15"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [non-pigmented ciliary epithelial cell](http://purl.obolibrary.org/obo/CL_0002304) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0002304"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "non-pigmented ciliary epithelial cell on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### nucleus accumbens shell and olfactory tubercle D1 medium spiny neuron `http://purl.obolibrary.org/obo/CL_4030051`
#### Removed
- [nucleus accumbens shell and olfactory tubercle D1 medium spiny neuron](http://purl.obolibrary.org/obo/CL_4030051) SubClassOf [BFO_0000002](http://purl.obolibrary.org/obo/BFO_0000002) 

#### Added
- [nucleus accumbens shell and olfactory tubercle D1 medium spiny neuron](http://purl.obolibrary.org/obo/CL_4030051) [RO_0002175](http://purl.obolibrary.org/obo/RO_0002175) [NCBITaxon_9606](http://purl.obolibrary.org/obo/NCBITaxon_9606) 

- [nucleus accumbens shell and olfactory tubercle D1 medium spiny neuron](http://purl.obolibrary.org/obo/CL_4030051) [in_subset](http://www.geneontology.org/formats/oboInOwl#inSubset) [human_reference_atlas](http://purl.obolibrary.org/obo/cl/cl-base#human_reference_atlas) 


### nucleus accumbens shell and olfactory tubercle D2 medium spiny neuron `http://purl.obolibrary.org/obo/CL_4030052`
#### Removed
- [nucleus accumbens shell and olfactory tubercle D2 medium spiny neuron](http://purl.obolibrary.org/obo/CL_4030052) SubClassOf [BFO_0000002](http://purl.obolibrary.org/obo/BFO_0000002) 

#### Added
- [nucleus accumbens shell and olfactory tubercle D2 medium spiny neuron](http://purl.obolibrary.org/obo/CL_4030052) [RO_0002175](http://purl.obolibrary.org/obo/RO_0002175) [NCBITaxon_9606](http://purl.obolibrary.org/obo/NCBITaxon_9606) 

- [nucleus accumbens shell and olfactory tubercle D2 medium spiny neuron](http://purl.obolibrary.org/obo/CL_4030052) [in_subset](http://www.geneontology.org/formats/oboInOwl#inSubset) [human_reference_atlas](http://purl.obolibrary.org/obo/cl/cl-base#human_reference_atlas) 


### obsolete structural cell `http://purl.obolibrary.org/obo/CL_0000293`
#### Removed
- [obsolete structural cell](http://purl.obolibrary.org/obo/CL_0000293) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A cell whose primary function is to provide structural support, to provide strength and physical integrity to the organism."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "TAIR:sr"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [obsolete structural cell](http://purl.obolibrary.org/obo/CL_0000293) [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "ZFA:0005745"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [mapping_justification](https://w3id.org/sssom/mapping_justification) "https://w3id.org/semapv/vocab/UnspecifiedMatching"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [obsolete structural cell](http://purl.obolibrary.org/obo/CL_0000293) [label](http://www.w3.org/2000/01/rdf-schema#label) "structural cell"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [obsolete structural cell](http://purl.obolibrary.org/obo/CL_0000293) SubClassOf [cell](http://purl.obolibrary.org/obo/CL_0000000) 

#### Added
- [obsolete structural cell](http://purl.obolibrary.org/obo/CL_0000293) [IAO_0000233](http://purl.obolibrary.org/obo/IAO_0000233) [3010](https://github.com/obophenotype/cell-ontology/issues/3010) 

- [obsolete structural cell](http://purl.obolibrary.org/obo/CL_0000293) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "OBSOLETE. A cell whose primary function is to provide structural support, to provide strength and physical integrity to the organism."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "TAIR:sr"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [obsolete structural cell](http://purl.obolibrary.org/obo/CL_0000293) [deprecated](http://www.w3.org/2002/07/owl#deprecated) true 

- [obsolete structural cell](http://purl.obolibrary.org/obo/CL_0000293) [comment](http://www.w3.org/2000/01/rdf-schema#comment) "Unsustainable grouping term."^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [obsolete structural cell](http://purl.obolibrary.org/obo/CL_0000293) [label](http://www.w3.org/2000/01/rdf-schema#label) "obsolete structural cell"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### olfactory bulb interneuron `http://purl.obolibrary.org/obo/CL_1001434`
#### Removed
- [olfactory bulb interneuron](http://purl.obolibrary.org/obo/CL_1001434) SubClassOf [neuron of the forebrain](http://purl.obolibrary.org/obo/CL_0012001) 



### olfactory bulb tufted cell `http://purl.obolibrary.org/obo/CL_1001503`
#### Removed
- [olfactory bulb tufted cell](http://purl.obolibrary.org/obo/CL_1001503) SubClassOf [neuron of the forebrain](http://purl.obolibrary.org/obo/CL_0012001) 

#### Added
- [olfactory bulb tufted cell](http://purl.obolibrary.org/obo/CL_1001503) SubClassOf [central nervous system neuron](http://purl.obolibrary.org/obo/CL_2000029) 


### oligodendrocyte precursor cell `http://purl.obolibrary.org/obo/CL_0002453`

#### Added
- [oligodendrocyte precursor cell](http://purl.obolibrary.org/obo/CL_0002453) [description](http://purl.org/dc/terms/description) "An oligodendrocyte precursor cell (OPC), also commonly referred to as an oligodendrocyte progenitor cell, is an undifferentiated or immature type of cell found in the nervous system that differentiates into a fully mature oligodendrocyte. Oligodendrocytes are known for their role in creating the protective myelin sheath that covers and insulates neurons in the central nervous system, enhancing the speed and efficiency of neurological signal transmission. OPCs contribute to this vital process by providing a consistent source of new oligodendrocytes.
OPCs are unique for their remarkable flexibility and adaptability, making them a critical element in many neurological processes. They are involved in neurogenesis and reparative functions as they can proliferate and be mobilized to areas of brain injury to aid in tissue repair and regeneration. Particularly, they play a significant role in the recovery and remyelination of damaged neurons, attributing to their prolific distribution in both gray and white matter areas of the brain and spinal cord.
Recent research has revealed the broader plasticity of OPCs. Studies have shown that OPCs can also differentiate into astrocytes and neurons under certain conditions, enlarging their role from just players in myelination to contributors in maintaining the integrity and adaptability of the central nervous system. Furthermore, these cells have been found to interact directly with neurons and take part in synaptic transmission, implying a more complex role in neural network activity.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.neuron.2011.05.013"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1101/gad.344218.120"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.ceb.2021.05.003"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1111/j.1460-9568.2006.04625.x"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/S0166-2236(00)01691-X"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [oligodendrocyte precursor cell](http://purl.obolibrary.org/obo/CL_0002453) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0002453"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "oligodendrocyte precursor cell on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### optic choroid fibroblast `http://purl.obolibrary.org/obo/CL_2000070`
#### Removed
- [optic choroid fibroblast](http://purl.obolibrary.org/obo/CL_2000070) SubClassOf [fibroblast](http://purl.obolibrary.org/obo/CL_0000057) 

- [optic choroid fibroblast](http://purl.obolibrary.org/obo/CL_2000070) SubClassOf [BFO_0000050](http://purl.obolibrary.org/obo/BFO_0000050) some [UBERON_0001776](http://purl.obolibrary.org/obo/UBERON_0001776) 

#### Added
- [optic choroid fibroblast](http://purl.obolibrary.org/obo/CL_2000070) SubClassOf [neural crest derived fibroblast](http://purl.obolibrary.org/obo/CL_0000005) 

- [optic choroid fibroblast](http://purl.obolibrary.org/obo/CL_2000070) SubClassOf [choroidal cell of the eye](http://purl.obolibrary.org/obo/CL_0000348) 


### osteoblast `http://purl.obolibrary.org/obo/CL_0000062`

#### Added
- [osteoblast](http://purl.obolibrary.org/obo/CL_0000062) [description](http://purl.org/dc/terms/description) "Osteoblasts are a type of bone cell primarily responsible for bone formation. Osteoblasts are derived from mesenchymal stem cells, which differentiate into osteoblasts under the influence of various growth factors, including bone morphogenetic proteins (BMPs). These multipotent cells play a critical role in the bone formation process known as ossification.
The primary function of osteoblasts is to synthesize and deposit bone matrix, a mixture of organic substances and inorganic materials. This included producing a mix of collagen and non-collagenous proteins, which they secrete to create the organic portion of the extracellular bone matrix. The osteoblasts also initiate the mineralization process by depositing calcium and phosphate ions onto this organic matrix, generating a material that is incredibly strong and resilient, which we recognize as bone. 
Furthermore, osteoblasts are intricately involved in bone remodeling. They communicate with other specialized bone cells, like osteoclasts (involved in bone resorption) and osteocytes (derived from osteoblasts and important for bone maintenance), helping to maintain skeletal health and homeostasis. Dysregulation of osteoblast function can lead to various bone diseases, illustrating the importance of these cells in skeletal health and development.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/nrm3254"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1089/ten.teb.2016.0454"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.3390/ijms19020360"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/s41574-019-0246-y"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/s41580-020-00279-w"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [osteoblast](http://purl.obolibrary.org/obo/CL_0000062) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0000062"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "osteoblast on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### otic fibrocyte `http://purl.obolibrary.org/obo/CL_0002665`
#### Removed
- [otic fibrocyte](http://purl.obolibrary.org/obo/CL_0002665) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A fibrocyte of the cochlea that has specialized structural and molecular adaptions."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:18353863"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "GOC:tfm"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [otic fibrocyte](http://purl.obolibrary.org/obo/CL_0002665) SubClassOf [circulating fibrocyte](http://purl.obolibrary.org/obo/CL_0000135) 

#### Added
- [otic fibrocyte](http://purl.obolibrary.org/obo/CL_0002665) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A mesenchymal cell of the cochlea that has specialized structural and molecular adaptions."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:18353863"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "GOC:tfm"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:37720106"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [otic fibrocyte](http://purl.obolibrary.org/obo/CL_0002665) SubClassOf [eukaryotic cell](http://purl.obolibrary.org/obo/CL_0000255) 

- [otic fibrocyte](http://purl.obolibrary.org/obo/CL_0002665) SubClassOf [mesenchymal cell](http://purl.obolibrary.org/obo/CL_0008019) 


### ovarian surface epithelial cell `http://purl.obolibrary.org/obo/CL_2000064`

#### Added
- [ovarian surface epithelial cell](http://purl.obolibrary.org/obo/CL_2000064) [description](http://purl.org/dc/terms/description) "Ovarian surface epithelial (OSE) cells form the ovarian epithelium, a single layer of cells that covers the external surface of the ovaries, continuous with the peritoneal mesothelium. The epithelium varies morphologically from cuboidal, to simple flattened, and low pseudostratified columnar cells, depending on the cyclic changes in the underlying ovarian structure. 
The epithelial layer functions as a physical barrier that protects the ovarian stem cells and the follicular cells from mechanical stress and potential foreign pathogens, and they provide a supportive tissue microenvironment for the underlying cells of the ovaries.  The OSE is separated from the ovarian stroma by a basement membrane, to which it is only loosely attached. 
OSE cells play a crucial role in ovarian physiology and pathology. They are involved in processes associated with ovarian follicular growth, maturation, ovulation and follicular atresia. Moreover, they have a prominent role in repairing the ovarian surface following the rupture of the follicle during ovulation, essentially playing a key part in the regenerative processes post-ovulation. Shedding and regeneration of OSE cells is a dynamic process that repeats throughout each reproductive cycle.
OSE cells are also the primary origin of ovarian cancer, specifically, epithelial ovarian cancer which accounts for about 90% of all ovarian cancers. The constant rounds of rapid proliferation and repair following ovulation are thought to provide the opportunity for replication errors and hence contribute to neoplastic changes in the OSE.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1677/JME-07-0149"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1210/edrv.22.2.0422"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1073/pnas.0805012105"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.3389/fendo.2022.886533"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [ovarian surface epithelial cell](http://purl.obolibrary.org/obo/CL_2000064) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_2000064"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "ovarian surface epithelial cell on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### oxytocin receptor sst GABAergic cortical interneuron `http://purl.obolibrary.org/obo/CL_4023122`

#### Added
- [oxytocin receptor sst GABAergic cortical interneuron](http://purl.obolibrary.org/obo/CL_4023122) SubClassOf [BFO_0000002](http://purl.obolibrary.org/obo/BFO_0000002) 


### pacemaker neuron `http://purl.obolibrary.org/obo/CL_4072018`

#### Added
- Class: [pacemaker neuron](http://purl.obolibrary.org/obo/CL_4072018) 

- [pacemaker neuron](http://purl.obolibrary.org/obo/CL_4072018) [date](http://purl.org/dc/terms/date) "2025-07-07T12:37:16Z"^^[dateTime](http://www.w3.org/2001/XMLSchema#dateTime) 

- [pacemaker neuron](http://purl.obolibrary.org/obo/CL_4072018) [id](http://www.geneontology.org/formats/oboInOwl#id) "CL:4072018"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [pacemaker neuron](http://purl.obolibrary.org/obo/CL_4072018) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A neuron that generates rhythmic bursts of action potentials independently of synaptic input. This intrinsic property enables it to maintain oscillatory activity even when isolated from other neurons. It populates the brainstem, hypothalamus, basal ganglia, spinal cord, and cerebellum. It plays a crucial role in regulating circadian rhythms in the suprachiasmatic nucleus, generating respiratory rhythms in the preB&ouml;tzinger complex, and synchronizing neural networks."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:39303139"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:31982322"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [pacemaker neuron](http://purl.obolibrary.org/obo/CL_4072018) [label](http://www.w3.org/2000/01/rdf-schema#label) "pacemaker neuron"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [pacemaker neuron](http://purl.obolibrary.org/obo/CL_4072018) SubClassOf [neuron](http://purl.obolibrary.org/obo/CL_0000540) 


### pancreatic A cell `http://purl.obolibrary.org/obo/CL_0000171`

#### Added
- [pancreatic A cell](http://purl.obolibrary.org/obo/CL_0000171) [description](http://purl.org/dc/terms/description) "Pancreatic A cells, also known as alpha cells, are one of the main endocrine cell types found within the islets of Langerhans in the pancreas. Functionally, these cells are pivotal in glucose metabolism and homeostasis, accounting for about 20% of the total population of cells in the islets. Their primary role involves the synthesis, storage, and secretion of the peptide hormone glucagon, which is critical in energy regulation throughout the body. 
In response to a decrease in blood glucose levels, the pancreatic A cells are stimulated to secrete glucagon into the bloodstream. Glucagon acts on its target cells, mainly in the liver, to stimulate glycogenolysis and gluconeogenesis processes, thereby increasing blood glucose levels back to normal. In this way, pancreatic A cells play an integral role in maintaining glucose homeostasis and preventing hypoglycemia.
Pancreatic A cells also participate in the local regulation of islet activities as glucagon acts through glucagon receptors on A, B and D type cells within the islets. Recent studies have also shown that Pancreatic A cells play a role in the generation and regeneration of B type cells. Following beta cell injuries pancreatic A cells increase in numbers and produce Glucagon-like peptide-1 (GLP-1), which increases the proliferation and cytoprotection of beta cells. In response to extreme injury of B type cells pancreatic A cells can transform (transdifferentiate) into functioning B type cells.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1677/JOE-08-0290"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.semcdb.2020.01.006"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/nature08894"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.beem.2015.10.002"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1210/er.2006-0007"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [pancreatic A cell](http://purl.obolibrary.org/obo/CL_0000171) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0000171"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "pancreatic A cell on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### pancreatic D cell `http://purl.obolibrary.org/obo/CL_0000173`

#### Added
- [pancreatic D cell](http://purl.obolibrary.org/obo/CL_0000173) [description](http://purl.org/dc/terms/description) "Pancreatic D cells, also known as delta cells, are one of the main cell types found in the islets of Langerhans within the pancreas. They represent a minor fraction of the islet cells, comprising about 5% to 10% of the total islet cell population. Their main function is to produce and secrete the peptide hormone somatostatin, an inhibitory hormone that regulates the endocrine system. 
Their complex morphology, coupled with their distinctive function, sets the pancreatic D cells apart from other cells in the islets of Langerhans. They have long, neurite-like processes which allow them to make contact with other pancreatic endocrine cells at some distance from the cell body and create an extensive paracrine network. 
The somatostatin secreted by the pancreatic D cells has a variety of inhibitory effects on the secretory activity of other cells within the pancreas and the gastrointestinal system. In the pancreas, one of the principal roles of somatostatin is to inhibit both insulin and glucagon secretion, thereby playing a potent role in the regulation of blood glucose levels. The processes of pancreatic D cells can be &gt;20&mu;m long and may make contact with multiple pancreatic A and B type cells, potentially explaining the secretion of both glucagon and insulin observed in the presence of somatostatin receptor antagonists. The dual inhibitory role makes somatostatin a key player in preventing the excessive action of these hormones and maintaining metabolic balance.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1369/0022155415583535"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/s41467-019-11517-x"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/s41574-018-0020-6"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "https://www.sciencedirect.com/topics/agricultural-and-biological-sciences/delta-cell"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [pancreatic D cell](http://purl.obolibrary.org/obo/CL_0000173) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0000173"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "pancreatic D cell on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### pancreatic ductal cell `http://purl.obolibrary.org/obo/CL_0002079`

#### Added
- [pancreatic ductal cell](http://purl.obolibrary.org/obo/CL_0002079) [description](http://purl.org/dc/terms/description) "Pancreatic ductal cells are a vital structural component of the pancreas, a principal gland involved in digestion and metabolism. These cells exist as flat or cuboidal epithelial cells, primarily organizing themselves to form a network of ducts throughout the pancreas. They create an integral component of the glands' 'exocrine' component, aiding in the transportation of digestive enzymes from the acinar cells (another type of cells in the exocrine pancreas that produce the enzymes) to the small intestine. 
The primary function of the pancreatic ductal cells revolves around the secretion of bicarbonate ions, contributing to the alkaline pancreatic juice that neutralizes the acidic gastric juice in the duodenum. This action allows for a conducive environment for the activated digestive enzymes to efficiently break down food. Furthermore, the pancreatic ductal cells essentially control the amount of fluid that is released into the duodenum, thereby playing a significant role in maintaining the fluid-electrolyte balance in the body.
Pancreatic ductal cells can also act as the origin point for several pancreatic diseases. The uncontrolled proliferation of these cells often leads to the formation of pancreatic ductal adenocarcinoma, the most common type of pancreatic cancer.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/B978-0-12-385233-5.00002-7"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.3389/fphys.2011.00036"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1172/JCI57131"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.biocel.2004.07.010"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [pancreatic ductal cell](http://purl.obolibrary.org/obo/CL_0002079) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0002079"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "pancreatic ductal cell on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### pancreatic stellate cell `http://purl.obolibrary.org/obo/CL_0002410`

#### Added
- [pancreatic stellate cell](http://purl.obolibrary.org/obo/CL_0002410) [description](http://purl.org/dc/terms/description) "Pancreatic stellate cells, also known as PSCs, are a specific type of cell comprising 4-7% the pancreas, specifically the exocrine glandular tissue. In their quiescent state, these cells are characterized by a star-like appearance with long cytoplasmic processes, and store vitamin A droplets in their cytoplasm. The primary roles of pancreatic stellate cells are related to the production, degradation, and reorganization of the extracellular matrix in the pancreas. When activated by injury or stress, they transform into myofibroblast-like cells, beginning to proliferate, migrate and produce large amounts of extracellular matrix proteins that can lead to the fibrosis seen in chronic pancreatitis. Pancreatic stellate cells are also responsible for the synthesis of various growth factors, cytokines, and chemokines, each playing a significant role in the complex network of paracrine and autocrine signaling pathways within the pancreatic tissue. 
These cells have been implicated in the progression of pancreatic cancer. Activated PSCs can stimulate the growth, invasion, and metastasis of pancreatic cancer cells. They also help form a dense, fibrous 'shield' around cancer cells, known as desmoplasia, which can make it more difficult for anticancer drugs to reach their target. Thus, a lot of research is being conducted on how to control or modify the behavior of pancreatic stellate cells to treat chronic pancreatitis and pancreatic cancer.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1158/1078-0432.CCR-14-1051"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1136/gut.2008.170233"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1007/s00535-009-0013-2"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1007/s00424-017-1968-0"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/labinvest.2012.161"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [pancreatic stellate cell](http://purl.obolibrary.org/obo/CL_0002410) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0002410"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "pancreatic stellate cell on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### pericyte `http://purl.obolibrary.org/obo/CL_0000669`

#### Added
- [pericyte](http://purl.obolibrary.org/obo/CL_0000669) [description](http://purl.org/dc/terms/description) "Pericytes, also known as Rouget cells, are a type of cell embedded in the wall of small blood vessels such as capillaries and venules. Not only do these cells share a basement membrane with endothelial cells, but they also have a direct physical contact with them, forming the blood-brain and blood-retinal barrier. They are an essential component of the neurovascular unit, a structure critical for maintaining brain homeostasis and a consistent neural microenvironment.
Pericytes play a crucial role in the regulation of microvascular blood flow and permeability. They dynamically respond to changes in blood flow and can relax or contract like smooth muscle cells, thereby controlling capillary diameter and blood flow to the tissues. Moreover, pericytes contribute significantly to the regulation of endothelial cell proliferation and migration, implying their vital role in angiogenesis and the stability of blood vessels. Their ability to differentiate into multiple cell types, including fibroblasts, macrophages, and smooth muscle cells, reveals their importance in tissue repair and remodeling.
Furthermore, pericytes have demonstrated functions in regulating immune and inflammatory responses. Pericytes also collaborate with endothelial cells in preserving the integrity of the blood-brain barrier, thus protecting the central nervous system from potential damage due to harmful substances and pathogens.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.3390/biomedicines11020600"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1007/978-1-60761-938-3_2"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.cjtee.2020.02.006"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1177/0271678X15610340"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.3389/fncel.2019.00282"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [pericyte](http://purl.obolibrary.org/obo/CL_0000669) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0000669"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "pericyte on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### periportal region hepatocyte `http://purl.obolibrary.org/obo/CL_0019026`

#### Added
- [periportal region hepatocyte](http://purl.obolibrary.org/obo/CL_0019026) [description](http://purl.org/dc/terms/description) "Periportal region hepatocytes, also known as Zone 1 hepatocytes, are specialized epithelial cells located in the periportal region of the liver acinus, near the portal triad, which comprises the portal vein, hepatic artery, and bile duct. These cells form a major part of the liver's cellular architecture and play a pivotal role in detoxification, metabolism, and homeostasis processes in the human body. 
Periportal hepatocytes differ functionally from hepatocytes in the liver&rsquo;s midzone (zone 2) and the pericentral zone 3. They are exposed to higher oxygen concentrations than hepatocytes in zones 2 and 3 because they are closer to the portal triads where oxygen-rich blood enters the liver. They therefore catalyze predominantly oxidative energy metabolism. 
Periportal hepatocytes are responsible for gluconeogenesis, the process by which glucose is synthesized from smaller precursor molecules like lactate, glycerol and glucogenic amino acids, to maintain blood glucose levels, particularly during fasting periods. Additionally, they carry out cholesterol synthesis, fatty acid oxidation, urea synthesis, and the conversion of ammonia to urea, aiding in the detoxification and excretory functions necessary to maintain homeostasis. Periportal hepatocytes are also associated with bile synthesis and secretion, given their proximity to the bile duct. Bile components facilitate the digestion and absorption of dietary lipids and the removal of waste products from the liver. 
Because of their location near the portal triad periportal hepatocytes are more resistant to ischemic injury than other hepatocytes but are more susceptible to toxic damage due to their higher exposure to drugs and toxins.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1159/000469338"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1002/hep.1840010413"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/s41575-019-0134-x"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1093/jn/122.suppl_3.843"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1159/000468779"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [periportal region hepatocyte](http://purl.obolibrary.org/obo/CL_0019026) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0019026"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "periportal region hepatocyte on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### phagocyte `http://purl.obolibrary.org/obo/CL_0000234`

#### Added
- [phagocyte](http://purl.obolibrary.org/obo/CL_0000234) [description](http://purl.org/dc/terms/description) "Phagocytes are a heterogeneous group of white blood cells that have the capability to ingest, or &lsquo;phagocytose&rsquo;, foreign substances, pathogens, and cellular debris with high efficiency, thereby playing a vital role in the immune response. Phagocytes circulate throughout the body, patrolling and guarding against the invasion of pathogens. They are one among many types of cells that originate from hematopoietic stem cells in the bone marrow.
Upon detection of an invading microorganism, phagocytes are mobilized by inflammatory mediators (such as bacterial proteins or complement proteins) to the infection site. They recognize surface molecules, specifically antigens, on harmful bacteria or other foreign substances with the help of opsonins, which are proteins that bind to the pathogen and make them more susceptible to phagocytosis. The process of phagocytosis involves the extension of the phagocyte's membrane around the microorganism, completely enclosing it within a microbicidal vacuole, the phagolysosome, which contains various microbicidal chemicals and digestive enzymes such as cathepsins, proteases, lysozymes, and lipases that may be employed to degrade the pathogen.
Phagocytes are predominantly neutrophils or monocytes. Neutrophils are the most abundant type of phagocyte and are usually the first to arrive at the site of infection. Monocytes are derived from the bone marrow and can circulate to tissues and mature into local macrophages. Macrophages arrive later at sites of infection and are longer-lived than neutrophils. The process of phagocytosis must be tightly regulated to avoid host tissue damage.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/B978-0-08-046884-6.00607-2"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:0.3389/fimmu.2020.01066"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1182/blood-2007-12-077917"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1086/374747"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/B978-0-12-809633-8.90746-5"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [phagocyte](http://purl.obolibrary.org/obo/CL_0000234) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0000234"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "phagocyte on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### photoreceptor cell `http://purl.obolibrary.org/obo/CL_0000210`

#### Added
- [photoreceptor cell](http://purl.obolibrary.org/obo/CL_0000210) [description](http://purl.org/dc/terms/description) "Photoreceptor cells are specialized types of neurons found in the retina, responsible for the vital task of converting light into signals that can stimulate a visual response. These cells are called photoreceptor cells because they are light-sensitive, enabling vision in conditions ranging from extremely dim to very bright light. There are two main types of photoreceptor cells in the human eye: rods, which are highly sensitive to light and allow for vision in low-light conditions, and cones, which are responsible for color vision and function best in bright light.
The photoreceptor cells function by absorbing light rays and converting them into electrical signals. These signals are then sent to the brain via the optic nerve and are interpreted as visual images. This process begins when the photopigment (a light-sensitive compound) found in photoreceptor cells absorbs light, which triggers a chemical reaction. This reaction leads to changes in the photopigment's configuration and, consequently, alters its electrical properties. This change initiates a signal that is transmitted through the cell and onwards to connecting neurons.
Furthermore, photoreceptor cells exhibit a unique and highly specialized structure that reflects their function. Rods, named because of their elongated, rod-shaped structure, consist of an outer segment filled with disks containing rhodopsin, the photopigment responsible for detecting light. Cones, on the other hand, are more conical and have three different types of photopigments, each one sensitive to a different range of wavelengths, allowing for the perception of color. Given the importance of photoreceptor cells in vision, damage or degeneration to these cells can lead to vision impairment or blindness, highlighting their key role in visual perception.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/nrg2717"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1113/JP282058"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.neubiorev.2017.12.006"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1242/jcs.175687"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.cbpa.2008.04.600"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [photoreceptor cell](http://purl.obolibrary.org/obo/CL_0000210) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0000210"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "photoreceptor cell on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### plasmacytoid dendritic cell `http://purl.obolibrary.org/obo/CL_0000784`

#### Added
- [plasmacytoid dendritic cell](http://purl.obolibrary.org/obo/CL_0000784) [description](http://purl.org/dc/terms/description) "Plasmacytoid dendritic cells (pDCs), initially known as interferon-producing cells, are a unique subpopulation of dendritic cells. Unlike conventional dendritic cells (cDCs), plasmacytoid dendritic cells are characterized by their distinctive plasma cell-like morphology. These cells play vital roles in the innate and adaptive immune response, particularly in the antiviral response. Highly prevalent in the blood and lymphoid organs, pDCs are rare in non-lymphoid tissues, with the exception of inflamed tissues and tumors where they are often recruited.
A key role of plasmacytoid dendritic cells is their rapid and robust production of type 1 interferons in response to viral infections. This is crucial for antiviral defense as interferon signals to other immune cells to increase their antiviral defenses. pDCs achieve this by detecting viral nucleic acids in the cytosol through Toll-like receptors (TLRs), specifically TLR7 and TLR9. Upon recognition, these cells quickly produce substantial amounts of interferons, which plays a significant role in the containment of viral replication and the development of acquired antiviral immunity.
The potential of plasmacytoid dendritic cells for antigen presentations to naive T cells is relatively low under steady-state conditions. Upon activation by certain stimuli, such as viruses or CpG oligodeoxynucleotides, they have the ability to differentiate into efficient antigen-presenting cells. This transition to activated cells involves an upregulation of major histocompatibility complex (MHC) class II molecules and co-stimulatory molecules, thus allowing them to effectively present antigenic peptides to naive T cells. This crucial function in linking innate and adaptive immunity underlines the importance of pDCs to the overall immune response.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/bs.ircmb.2019.07.004"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.immuni.2018.12.027"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.immuni.2007.06.006"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/nri3865"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [plasmacytoid dendritic cell](http://purl.obolibrary.org/obo/CL_0000784) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0000784"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "plasmacytoid dendritic cell on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### platelet `http://purl.obolibrary.org/obo/CL_0000233`

#### Added
- [platelet](http://purl.obolibrary.org/obo/CL_0000233) [description](http://purl.org/dc/terms/description) "Platelets, also known as thrombocytes, are small, disc-shaped cell fragments devoid of a nucleus, yet remarkably intricate and multi-functional. They originate from the megakaryocytes in the bone marrow via a process called thrombopoiesis. Platelets are central to hemostasis, a process that prevents bleeding from broken or damaged blood vessels and facilitates wound healing. In their resting state, platelets are inactive but become quickly activated when an injury occurs.
Platelets function in the initiation and regulation of coagulation or clotting, which is a complex process involving several bioactive molecules. When a blood vessel is injured, the inner lining of the vessel, the endothelium, gets disrupted, exposing the underlying tissue matrix. This leads to platelet activation and adhesion to the injury site, initiating the clotting cascade. Platelets release granules packed with clotting factors, proteins, enzymes, and more platelets, which further propagates the clotting process. When activated, platelets also change their shape from discoid to a spherical form with protruding pseudopods that enables them to adhere more readily to the site of injury and mesh with one another to form a 'platelet plug'. 
Additionally, platelets serve other physiological roles. They release growth factors enhancing tissue regeneration and wound healing. They also play a significant role in inflammation, innate immunity, and antimicrobial host defense, as they interact with leukocytes and may sequester microbes. Platelets are involved in maintaining the integrity of the blood vessels and angiogenesis, the formation of new blood vessels. Proper functioning of platelets is critical for maintaining homeostasis, while dysregulation can lead to conditions such as thrombosis or bleeding disorders.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.3389/fimmu.2019.01731"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.blre.2014.10.003"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/s41569-018-0110-0"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1007/s10555-017-9677-x"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [platelet](http://purl.obolibrary.org/obo/CL_0000233) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0000233"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "platelet on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### pre-conventional dendritic cell `http://purl.obolibrary.org/obo/CL_0002010`

#### Added
- [pre-conventional dendritic cell](http://purl.obolibrary.org/obo/CL_0002010) [description](http://purl.org/dc/terms/description) "The pre-conventional dendritic cell (pre-cDC) are precursors of mature dendritic cells. They develop from common DC precursors in the bone marrow and exist in a semi-differentiated state. The term 'pre-conventional' denotes the stage of development of these cells prior to their maturation into conventional dendritic cells, which are critical players in immune responses. 
After leaving the bone marrow, pre-cDCs transiently circulate in the bloodstream to colonize lymphoid and non-lymphoid tissues, including skin, mucosa, spleen, and lymph nodes where they differentiate into immature conventional dendritic cells, which undergo further differentiation. In mouse and humans, the population of pre-cDCs has been shown to be heterogeneous with two subpopulations that pre-committed to differentiate into different subsets of dendritic cells (cDC1 and cDC2). 
These subsets of differentiated dendritic cells play a key role in the immune response by bringing about the regulation and initiation of T cell responses, thereby shaping the adaptive immunity. One subset is thought to be specialized in highly efficient activation of CD8+ T cells, while the other subset presents exogeneous antigens to CD4+ T cells. 
The main function of pre-conventional dendritic cells is to constantly replenish conventional dendritic cells. In response to infection additional pre-cDCs are recruited to the site of infection to meet increased tissue demand, and this pre-DC recruitment may be necessary for effective T cell responses. The molecular signals that direct pre-cDC migration remain poorly understood but recent research suggest that it involves specific chemokine receptors expressed on pre-cDCs.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:\10.1038/s41423-021-00726-4"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1084/jem.20161135"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1002/embj.201488027"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1146/annurev-immunol-061020-053707"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1111/imm.12888"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [pre-conventional dendritic cell](http://purl.obolibrary.org/obo/CL_0002010) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0002010"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "pre-conventional dendritic cell on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### preadipocyte `http://purl.obolibrary.org/obo/CL_0002334`

#### Added
- [preadipocyte](http://purl.obolibrary.org/obo/CL_0002334) [description](http://purl.org/dc/terms/description) "A preadipocyte is an immature precursor to adipocytes, which are specialized cells primarily tasked with the storage of energy in the form of lipids, making them key in metabolic regulation and overall energy homeostasis. Preadipocytes are present in the stromal vascular fraction of adipose tissues and can be identified by their fibroblastic morphology and the expression of specific markers such as Pref-1, CD24, CD34, and CD29.
The primary function of preadipocytes is to proliferate and differentiate into mature adipocytes upon receiving the appropriate physiological and hormonal stimuli. The process of adipogenesis, involving the differentiation of preadipocytes, is controlled by a complex network of transcription factors, including the peroxisome proliferator-activated receptor gamma (PPAR&gamma;) and members of the CCAAT/enhancer-binding proteins (C/EBPs) family. Through adipogenesis, preadipocytes contribute to the body's energy storage and insulation capabilities and act as a buffer to control the levels of free fatty acids in the bloodstream. When the energy balance of the body alters, for example during fasting or overnutrition, preadipocytes can respond by modulating adipogenic differentiation and lipid storage, illustrating their dynamic role in metabolic regulation.
Additionally, research has shown that preadipocytes also contribute to endocrine function. They produce and secrete several adipokines such as leptin, adiponectin, and resistin. These adipokines impact a variety of physiological processes, including appetite regulation, insulin sensitivity, inflammation, and immune responses. Dysregulation of preadipocyte function and adipogenesis is associated with obesity and related metabolic disorders, such as type 2 diabetes and cardiovascular disease.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.3389/fendo.2023.1199589"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1007/s12015-017-9774-9"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1186/s12950-018-0185-8"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1128/MCB.01344-13"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [preadipocyte](http://purl.obolibrary.org/obo/CL_0002334) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0002334"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "preadipocyte on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### primary cultured cell `http://purl.obolibrary.org/obo/CL_0000001`

#### Added
- [primary cultured cell](http://purl.obolibrary.org/obo/CL_0000001) [description](http://purl.org/dc/terms/description) "Primary cultured cells are cells directly extracted from living organisms and then cultivated in a laboratory under controlled environmental conditions. After extraction, these cells can grow and reproduce when provided with the appropriate nutrients. The establishment of primary cell cultures involves processes like tissue dissection, enzymatic or mechanical disaggregation, suspension, and plating. These cells closely mimic the biochemical and physiological conditions of cells in vivo, providing a more realistic model of biological function compared to other cell culture systems.
Retaining most physiological and morphological characteristics of the original tissue, primary cultured cells are a powerful tool for modeling diseases, drug responses, and basic biological functions such as synthesis and metabolism. They find extensive use in research fields such as cellular biology, genetics, pharmacology, oncology, and toxicology.
Despite their genetic stability, primary cultured cells have a finite lifespan and are not immortal. They undergo senescence after a specific number of population doublings, influenced by genetic, environmental, and epigenetic factors. Given their limited lifespan, primary cells are crucial for experimental replicability, and their cultivation demands meticulous care to prevent changes that might alter cell identity or function.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1007/978-0-387-89626-7_18"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/primary-cell-culture"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [primary cultured cell](http://purl.obolibrary.org/obo/CL_0000001) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0000001"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "primary cultured cell on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### primary motor cortex pyramidal cell `http://purl.obolibrary.org/obo/CL_2000049`

#### Added
- [primary motor cortex pyramidal cell](http://purl.obolibrary.org/obo/CL_2000049) [description](http://purl.org/dc/terms/description) "Primary motor cortex pyramidal cells (PMCPs) are a specific type of neuron found predominantly in the primary motor cortex of the brain, which holds a vital role in controlling voluntary movements. These cells are characterized by their distinct pyramidal - triangularly - shaped cell bodies, large size, and long branching dendrites that facilitate extensive interneuronal connectivity. PMCPs, along with other cortical pyramidal cells, are excitatory neurons, meaning they carry electrical impulses away from the cell body, facilitating communication between different areas of the brain and the spinal cord.
The principal function of PMCPs revolves around motor control. PMCPs in the primary motor cortex are responsible for mapping and planning complex movements, from the intention to move to the actual execution. For example, Betz cells are a type of PMCPs that extend their long axons, collectively known as the Corticospinal Tract (CST), which descends from the cortex to the spinal cord, synapsing with neurons in the spinal cord that control muscles throughout the body. This information transmission allows PMCPs to control precise, skillful, and independent movements of both proximal and distal muscles.
Aside from controlling voluntary movement, PMCPs contribute to motor learning and adaptation by establishing new neuronal connections and strengthening existing ones. Studies using techniques like optogenetics and two-photon imaging have revealed that PMCPs undergo significant synaptic plasticity during motor learning, indicating their role in modifying motor outputs in response to training and experience. Overall, PMCPs provide a biochemical and anatomic basis for translating thoughts into actions, an essential process for the organism's interaction with the environment.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1046/j.1469-7580.2001.19960699.x"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/s41586-021-03950-0"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1093/cercor/bhw134"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "https://www.ncbi.nlm.nih.gov/books/NBK540976"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [primary motor cortex pyramidal cell](http://purl.obolibrary.org/obo/CL_2000049) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_2000049"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "primary motor cortex pyramidal cell on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### professional antigen presenting cell `http://purl.obolibrary.org/obo/CL_0000145`

#### Added
- [professional antigen presenting cell](http://purl.obolibrary.org/obo/CL_0000145) [description](http://purl.org/dc/terms/description) "Professional antigen presenting cells, also known as APCs, are essential orchestrators of adaptive immunity. These cells possess the abilities to capture and process antigens and constitutively express MHC class II molecules to facilitate antigen presentation. These cells are able to deliver the three signals to activate na&iuml;ve T cells, thereby initiating adaptive immunity. 
Professional APCs include dendritic cells, macrophages, and B cells. Among these, dendritic cells are often considered the predominant APC as they have been demonstrated to activate na&iuml;ve T cells. Macrophages and B cells also possess antigen-processing machinery and constitutively express MHC class II molecules. Dendritic cells and macrophages obtain antigens via phagocytosis of cells and cell debris, while B cells obtain antigens via the B cell receptor.
APCs are responsible for stimulating an appropriate immune response. Following antigen presentation (signal 1), they deliver costimulatory signals (signal 2) and secrete mediators such as cytokines (signal 3) that shape the subsequent immune response. APCs can also induce tolerogenic responses to self-antigens. APCs play important roles in diseases such as cancer and autoimmune diseases.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/B978-0-12-394447-4.30111-0"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.3389/fimmu.2022.954936"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1159/000512729"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/nri3754"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [professional antigen presenting cell](http://purl.obolibrary.org/obo/CL_0000145) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0000145"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "professional antigen presenting cell on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### promonocyte `http://purl.obolibrary.org/obo/CL_0000559`

#### Added
- [promonocyte](http://purl.obolibrary.org/obo/CL_0000559) [description](http://purl.org/dc/terms/description) "Promonocytes are precursor cells in the mononuclear phagocyte system, which includes the progenitors in the bone marrow, circulating monocytes and macrophages in various tissues. During monocytopoiesis, promonocytes form after monoblasts differentiate, and precede the development of monocytes and macrophages, which play an integral role in the immune system.
Promonocytes are large, often round cells with a high nucleus to cytoplasm ratio. Their cytoplasm is laden with small azurophilic granules, and they are similar in appearance to myelocytes and metamyelocytes. 
The maturation into monocytes is the key function of promonocytes. Once matured, they navigate their way to the bloodstream and their primary role comes into effect. Monocytes circulate in the blood for a few days before migrating to tissues to become either macrophages or dendritic cells. These cells then proceed to ingest pathogens like bacteria and viruses, helping to keep the body healthy. Thus, promonocytes play a central role in the immune system by paving the way for the development of cells that fight off toxins and attacks.
Beyond their immune response role through differentiation, promonocytes may participate in processes such as wound healing and inflammation regulation. Defects or disruptions in promonocyte function have been linked to health disorders, including monocytic-lineage leukemias such as chronic myelomonocytic leukemia (CMML), although it is worth noting that morphologic distinction between promonocytes and other monocytic cells of various differentiation levels is difficult.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "https://www.sciencedirect.com/topics/agricultural-and-biological-sciences/promonocyte"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1111/ijlh.13212"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1182/blood.2022015850"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/B978-0-323-35762-3.00027-5"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [promonocyte](http://purl.obolibrary.org/obo/CL_0000559) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0000559"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "promonocyte on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### pulmonary alveolar type 1 cell `http://purl.obolibrary.org/obo/CL_0002062`

#### Added
- [pulmonary alveolar type 1 cell](http://purl.obolibrary.org/obo/CL_0002062) [description](http://purl.org/dc/terms/description) "Type I pneumocytes, also known as alveolar epithelial cells type I, are specialized cells lining the alveoli within the lungs. Their strategic positioning within the air sacs of the lungs plays a crucial role in gaseous exchange. They cover around 90-95% of the alveolar surface, yet make up only about 40% of the total number of cells in the alveolus. Their unique squamous, or flattened, shape allows for a vast surface area that enables a fast rate of diffusion of respiratory gases, oxygen and carbon dioxide.
Functionally, type I pneumocytes serve as the site for the critical exchange of gases between the air in the lungs and the blood in the capillaries. These cells are in direct contact with pulmonary capillaries and are therefore primarily responsible for oxygen passing into the bloodstream and carbon dioxide being removed from the bloodstream. It is important to note that these cells are extremely thin and fragile, a feature that promotes easy passage of gases but also renders the cells susceptible to damage.
A unique critical feature of type I pneumocytes is their inability to divide and proliferate. Should damage to these cells occur, repair and replenishment is addressed through the conversion of another cell type, known as type II pneumocytes, into type I pneumocytes. The role of type I cells is thus highly specific, and strictly aligned with enabling efficient respiration - an essential process for all aerobic organisms.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "https://www.ncbi.nlm.nih.gov/books/NBK557542/"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1242/jcs.235556"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1165/rcmb.2021-0421OC"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/B978-0-7216-0186-1.50012-0"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [pulmonary alveolar type 1 cell](http://purl.obolibrary.org/obo/CL_0002062) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0002062"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "pulmonary alveolar type 1 cell on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### pulmonary interstitial fibroblast `http://purl.obolibrary.org/obo/CL_0002241`

#### Added
- [pulmonary interstitial fibroblast](http://purl.obolibrary.org/obo/CL_0002241) [description](http://purl.org/dc/terms/description) "Pulmonary interstitial fibroblasts, or lung fibroblasts, are a specific type of cell located within the lungs. They are predominantly located in the connective tissue or interstitial space of the lung that is in close proximity to the alveolar epithelium and pulmonary capillaries. These cells play a crucial role in maintaining lung structure and function through the synthesis and remodeling of the extracellular matrix, which provides structural support to the lung tissues.
These cells ensure effective gas exchange by providing a scaffold for the epithelial structure. In addition to providing support, the extracellular matrix acts as a binding site for various molecules that can directly influence cell behavior and the local microenvironment. Fibroblasts not only secrete the extracellular matrix themselves but also have the capacity to remodel the matrix by producing matrix metalloproteinases (MMPs). This ability of synthesis and degradation makes fibroblasts essential for normal lung homeostasis and repair following lung injury.
Under pathological conditions such as pulmonary fibrosis, the function of pulmonary interstitial fibroblasts becomes deregulated. They become overly active and produce excess extracellular matrix, notably collagen, resulting in the thickening and stiffening of lung tissues that compromise its function. Furthermore, these cells can transform into myofibroblasts, cells that are more active in producing matrix proteins and contribute to scar formation or fibrosis. Understanding the biology and regulation of pulmonary interstitial fibroblasts is thus essential for devising therapeutic strategies for diseases characterized by abnormal tissue remodeling and fibrosis, such as idiopathic pulmonary fibrosis and chronic obstructive pulmonary disease (COPD).

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1002/sctm.20-0526"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.3390/ijms20061461"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.heliyon.2023.e19428"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1164/rccm.200706-929OC"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1513/AnnalsATS.201605-341AW"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [pulmonary interstitial fibroblast](http://purl.obolibrary.org/obo/CL_0002241) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0002241"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "pulmonary interstitial fibroblast on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### pvalb GABAergic interneuron `http://purl.obolibrary.org/obo/CL_4023018`
#### Removed
- [pvalb GABAergic interneuron](http://purl.obolibrary.org/obo/CL_4023018) [IAO_0000028](http://purl.obolibrary.org/obo/IAO_0000028) "pvalb cortical interneuron"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [pvalb GABAergic interneuron](http://purl.obolibrary.org/obo/CL_4023018) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A transcriptomically distinct GABAergic interneuron with a soma located in a cerebral cortex and it expresses Parvalbumin. The standard transcriptomic reference data for this cell type can be found on the CellxGene census under the collection: 'Transcriptomic cytoarchitecture reveals principles of human neocortex organization', dataset: 'Supercluster: MGE-derived interneurons', Author Categories: 'CrossArea_subclass', cluster Pvalb."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:27477017"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:37824655"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "https://cellxgene.cziscience.com/e/9c63201d-bfd9-41a8-bbbc-18d947556f3d.cxg/"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [pvalb GABAergic interneuron](http://purl.obolibrary.org/obo/CL_4023018) [label](http://www.w3.org/2000/01/rdf-schema#label) "pvalb GABAergic cortical interneuron"^^[string](http://www.w3.org/2001/XMLSchema#string) 

#### Added
- [pvalb GABAergic interneuron](http://purl.obolibrary.org/obo/CL_4023018) [IAO_0000028](http://purl.obolibrary.org/obo/IAO_0000028) "pvalb interneuron"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [pvalb GABAergic interneuron](http://purl.obolibrary.org/obo/CL_4023018) [description](http://purl.org/dc/terms/description) "The pvalb GABAergic cortical interneuron, named for its expression of parvalbumin (pvalb) and its use of the inhibitory neurotransmitter gamma-aminobutyric acid (GABA), plays an essential role in the intricate circuitry of the brain. This specific subclass of GABAergic cortical interneurons are among five subclasses recently defined for their marker gene expression, which include Sst, Pvalb, Vip, Lamp5, and Sncg. Predominantly found within the cortex, this subset of interneurons exhibits a significant influence on the behavior of the local network due to their robust inhibitory capabilities. Sampling input from the surrounding pyramidal neurons while concomitantly providing feedback inhibitory connections makes pvalb cortical interneurons a crucial constituent of cortical homeostasis.
Functionally, pvalb GABAergic cortical interneurons are responsible for shaping neuronal activity and facilitating precisely timed and reliable information transfer within the brain. They offer inhibitory control over the excitatory neurons they synapse, allowing these cells to temper and shape the overall neuronal firing pattern in the cortical regions, thereby promoting functional efficiency and reliability. Moreover, these interneurons, by rapidly and repetitively firing, contribute to generating brain rhythms, particularly gamma oscillations, essential for cognitive functions, like perception, attention, and memory.
Disruptions in the function of pvalb GABAergic cortical interneurons are linked to a broad spectrum of neurological and psychiatric disorders, including epilepsy, autism, and schizophrenia, indicating  their central role in maintaining cortical functionality. Moreover, the distribution, density, and interconnectedness of these complex neuronal networks may vary depending on the specific cortical region, further influencing their functionality and consequently the global neuronal output of the cortical area.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/s41386-020-0778-9"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.3389/fpsyt.2021.679960"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/s41380-023-02153-5"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/s41586-018-0654-5"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1126/science.1255263"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [pvalb GABAergic interneuron](http://purl.obolibrary.org/obo/CL_4023018) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A transcriptomically distinct GABAergic interneuron with a soma located in a cerebral cortex and it expresses Parvalbumin."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:27477017"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:37824655"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "https://cellxgene.cziscience.com/e/9c63201d-bfd9-41a8-bbbc-18d947556f3d.cxg/"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [pvalb GABAergic interneuron](http://purl.obolibrary.org/obo/CL_4023018) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_4023018"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "pvalb GABAergic interneuron on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [pvalb GABAergic interneuron](http://purl.obolibrary.org/obo/CL_4023018) [label](http://www.w3.org/2000/01/rdf-schema#label) "pvalb GABAergic interneuron"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### pvalb GABAergic interneuron (Homo sapiens) `http://purl.obolibrary.org/obo/CL_4072029`

#### Added
- Class: [pvalb GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072029) 

- [pvalb GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072029) [IAO_0000028](http://purl.obolibrary.org/obo/IAO_0000028) "pvalb interneuron (Homo sapiens)"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [pvalb GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072029) [RO_0002175](http://purl.obolibrary.org/obo/RO_0002175) [NCBITaxon_9606](http://purl.obolibrary.org/obo/NCBITaxon_9606) 

- [pvalb GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072029) [contributor](http://purl.org/dc/terms/contributor) "https://orcid.org/0000-0002-5507-2103"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [pvalb GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072029) [date](http://purl.org/dc/terms/date) "2025-08-12T14:31:10Z"^^[dateTime](http://www.w3.org/2001/XMLSchema#dateTime) 

- [pvalb GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072029) [id](http://www.geneontology.org/formats/oboInOwl#id) "CL:4072029"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [pvalb GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072029) [in_subset](http://www.geneontology.org/formats/oboInOwl#inSubset) [BDS_subset](http://purl.obolibrary.org/obo/cl/cl-base#BDS_subset) 

- [pvalb GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072029) [in_subset](http://www.geneontology.org/formats/oboInOwl#inSubset) [cellxgene_subset](http://purl.obolibrary.org/obo/cl/cl-base#cellxgene_subset) 

- [pvalb GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072029) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A transcriptomically distinct GABAergic interneuron with a soma located in a cerebral cortex and it expresses Parvalbumin. The standard transcriptomic reference data for this cell type can be found on the CellxGene census under the collection: 'Transcriptomic cytoarchitecture reveals principles of human neocortex organization', dataset: 'Supercluster: MGE-derived interneurons', Author Categories: 'CrossArea_subclass', cluster Pvalb."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:27477017"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:37824655"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "https://cellxgene.cziscience.com/e/9c63201d-bfd9-41a8-bbbc-18d947556f3d.cxg/"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [pvalb GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072029) [label](http://www.w3.org/2000/01/rdf-schema#label) "pvalb GABAergic interneuron (Homo sapiens)"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [pvalb GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072029) SubClassOf [pvalb GABAergic interneuron](http://purl.obolibrary.org/obo/CL_4023018) 

- [pvalb GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072029) SubClassOf [RO_0002162](http://purl.obolibrary.org/obo/RO_0002162) some [NCBITaxon_9606](http://purl.obolibrary.org/obo/NCBITaxon_9606) 


### pyramidal neuron `http://purl.obolibrary.org/obo/CL_0000598`

#### Added
- [pyramidal neuron](http://purl.obolibrary.org/obo/CL_0000598) [description](http://purl.org/dc/terms/description) "Pyramidal neurons, or pyramidal cells, are a distinctive type of neuron named for the triangular shape of their soma, or cell body. The primary characteristic of these neurons is a large apical dendrite that ascends toward the cortical surface, numerous basal dendrites, and a single axon. These neurons are found within parts of the brain including the cerebral cortex, the hippocampus, and the amygdala, playing a critical role in cortex communication, cognitive function, and the relay of information within the brain's neural network.
Functional roles of pyramidal neurons depend largely on their localization. However, all types of these neurons serve as principal cells within the grey matter of the brain, transmitting signals over long distances. Primarily, pyramidal cells function as excitatory neurons; they disseminate action potential outputs to other neurons which they are connected to across the synapse. In the cerebral cortex, pyramidal neurons are paramount for sensory perception and motor command, while those located in the hippocampus involve in memory and learning processes.
Firing properties of these neurons such as regular spiking and bursting are notable features thought to play a role in synaptic plasticity and signaling. Together, their unique anatomical structure and intricate connectivity underpin their critical function in brain neural networks, significantly contributing to human cognitive abilities. In addition, abnormalities in pyramidal neurons have been linked to a range of neurodegenerative and psychiatric disorders, emphasizing their importance in maintaining healthy neural function.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1093/cercor/bhg093"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/S0896-6273(01)00447-0"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/s41583-019-0125-5"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/nrn2286"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1002/jnr.23978"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [pyramidal neuron](http://purl.obolibrary.org/obo/CL_0000598) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0000598"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "pyramidal neuron on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### quiescent fibroblast `http://purl.obolibrary.org/obo/CL_4052071`

#### Added
- Class: [quiescent fibroblast](http://purl.obolibrary.org/obo/CL_4052071) 

- [quiescent fibroblast](http://purl.obolibrary.org/obo/CL_4052071) [contributor](http://purl.org/dc/terms/contributor) [0009-0000-8480-9277](https://orcid.org/0009-0000-8480-9277) 

- [quiescent fibroblast](http://purl.obolibrary.org/obo/CL_4052071) [date](http://purl.org/dc/terms/date) "2025-08-13T15:42:37Z"^^[dateTime](http://www.w3.org/2001/XMLSchema#dateTime) 

- [quiescent fibroblast](http://purl.obolibrary.org/obo/CL_4052071) [id](http://www.geneontology.org/formats/oboInOwl#id) "CL:4052071"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [quiescent fibroblast](http://purl.obolibrary.org/obo/CL_4052071) [comment](http://www.w3.org/2000/01/rdf-schema#comment) "Historically, quiescent fibroblasts in uninjured tissues were often called &ldquo;fibrocytes&rdquo;, but this distinction faded over time, and the term &ldquo;fibroblast&rdquo; came to be used universally, regardless of activation state. Fibrocyte has been repurposed to describe a distinct population of bone marrow&ndash;derived, circulating cells that home to sites of injury and contribute to tissue repair and fibrosis."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID: 35701396"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [quiescent fibroblast](http://purl.obolibrary.org/obo/CL_4052071) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A fibroblast in a quiescent state, characterized by a smaller, spindle-shaped morphology with a relatively small cytoplasm, modest rough endoplasmic reticulum and condensed chromatin. Despite low proliferation and contractility, it maintains high metabolic activity for extracellular-matrix homeostasis through continuous matrix protein turnover and mechanosensitive signaling. This cell can rapidly differentiate into contractile myofibroblasts under injury or inflammatory cues to drive tissue repair."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:21049082"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:35701396"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:40538750"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "doi:/10.1038/s41427-020-0226-7"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "Wikipedia:Fibroblast"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [quiescent fibroblast](http://purl.obolibrary.org/obo/CL_4052071) [has_exact_synonym](http://www.geneontology.org/formats/oboInOwl#hasExactSynonym) "inactive fibroblast"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:22529592"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [quiescent fibroblast](http://purl.obolibrary.org/obo/CL_4052071) [label](http://www.w3.org/2000/01/rdf-schema#label) "quiescent fibroblast"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [quiescent fibroblast](http://purl.obolibrary.org/obo/CL_4052071) SubClassOf [fibroblast](http://purl.obolibrary.org/obo/CL_0000057) 


### reelin GABAergic cortical interneuron `http://purl.obolibrary.org/obo/CL_4032001`

#### Added
- [reelin GABAergic cortical interneuron](http://purl.obolibrary.org/obo/CL_4032001) SubClassOf [BFO_0000002](http://purl.obolibrary.org/obo/BFO_0000002) 


### respiratory tract suprabasal cell `http://purl.obolibrary.org/obo/CL_4033048`

#### Added
- [respiratory tract suprabasal cell](http://purl.obolibrary.org/obo/CL_4033048) SubClassOf [suprabasal cell](http://purl.obolibrary.org/obo/CL_7770004) 


### retina horizontal cell `http://purl.obolibrary.org/obo/CL_0000745`

#### Added
- [retina horizontal cell](http://purl.obolibrary.org/obo/CL_0000745) [description](http://purl.org/dc/terms/description) "Retina horizontal cells are a specialized type of neuron found in the inner nuclear layer of the retina. They play a crucial role in the visual system by integrating and regulating the input from multiple photoreceptor cells, which sense and respond to light. Retina horizontal cells are so named due to their characteristic flat horizontal orientation, with their dendrites and axons coursing laterally in a plane parallel to the retinal surface.
Retina horizontal cells are synaptically connected with photoreceptor cells (i.e., cones and rods) and bipolar cells. Their primary function is to facilitate the process of lateral inhibition, where an excited neuron reduces the activity of its neighbors. This capability is central to the phenomenon of &quot;contrast enhancement&quot; in visual perception, where the borders between light and dark areas are accentuated to better resolve images. Retina horizontal cells receive inputs from these neighboring photoreceptor cells and transmit their inhibitory output to not only photoreceptor cells but also bipolar cells, fine-tuning the responsiveness to changes in light intensity and differences in wavelengths.
Moreover, retina horizontal cells play a substantial role in adaptation of retinal response to different light levels. In bright light conditions, they aid the reduction of the overall response to light stimuli, thus aiding in the prevention of photoreceptor saturation. These neurons are adept at adjusting the gain and thus the dynamic range of photoreceptors supporting the vision process under variable lighting conditions - from dim moonlight to bright sunshine. Therefore, retina horizontal cells contribute considerably to the adaptability, sharpness, and detail perception of our visual system.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1113/JP274177"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1371/journal.pbio.1001107"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1167/5.4.5"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.3389/fnana.2016.00077/full"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1073/pnas.88.8.3310"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [retina horizontal cell](http://purl.obolibrary.org/obo/CL_0000745) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0000745"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "retina horizontal cell on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### retinal bipolar neuron `http://purl.obolibrary.org/obo/CL_0000748`

#### Added
- [retinal bipolar neuron](http://purl.obolibrary.org/obo/CL_0000748) [description](http://purl.org/dc/terms/description) "The retinal bipolar neuron is an integral part of the eye, specifically located within the retina. They are secondary neurons in the visual pathway and primarily serve to transmit information from photoreceptor cells, which respond to light, to retinal ganglion cells, which carry the signal to the brain. As such, these cells play a pivotal role in our ability to interpret and perceive visual information.
The structure of retinal bipolar neurons is unique and significant to their function. They are termed &quot;bipolar&quot; due to their morphology, having two poles: a single dendrite on one end and an axon on the other. The dendrite end connects to photoreceptor cells (rods or cones), while the axon end synapses with retinal ganglion cells. In the center of the dendrite and axon lies the cell body, which houses the cell's nucleus and other vital organelles.
At the functional level, retinal bipolar neurons can be categorized into ON and OFF types based on their response to light. 'ON' bipolar cells are depolarized (electrical potential across the cell membrane decreases) under light conditions, thereby increasing their rate of neurotransmitter release onto the downstream ganglion cells. In contrast, 'OFF' bipolar cells are hyperpolarized (electrical potential increases) in light conditions, decreasing their rate of neurotransmitter release. This dual system allows for a fine-tuned response and the ability to perceive differences in light intensity and contrast, thus creating a nuanced and detailed visual world.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1126/science.201028"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.preteyeres.2008.03.003"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1017/S0952523810000209"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/nrn3783"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.ydbio.2021.09.005"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [retinal bipolar neuron](http://purl.obolibrary.org/obo/CL_0000748) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0000748"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "retinal bipolar neuron on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### retinal cone cell `http://purl.obolibrary.org/obo/CL_0000573`

#### Added
- [retinal cone cell](http://purl.obolibrary.org/obo/CL_0000573) [description](http://purl.org/dc/terms/description) "Retinal cone cells, or simply cone cells, are one out of the two types of photoreceptor cells in the retina, the other being retinal rod cells. They function by converting light into signals that can lead to a visual perception. These cells are less sensitive to light compared to rod cells, but they play a crucial role in our vision system for their ability to perceive color and provide sharp visual acuity. They are responsible for our daylight and color vision, enabling an individual to perceive fine detail and rapid changes in light levels.
Cone cells are specialized and named based on the absorption spectra of the visual pigments they contain, giving rise to three types of cone cells: short-wavelength S-cones (blue cones), middle-wavelength M-cones (green cones), and long-wavelength L-cones (red cones). The distribution of these cones across the retina differs, with greater densities at the central fovea region, contributing to why our central vision is so sharp and color-detailed compared to peripheral vision.
The outer segments of retinal cone cells house the photopigments that absorb light and initiate the process of phototransduction, a biochemical event by which the light signal is transformed into an electrical signal. When a cone cell is stimulated by light, an electrochemical reaction triggers nerve impulses, which are then transmitted to the brain by the optic nerve. There, these impulses are processed into the images that we see. Each type of cone responds to different wavelengths of light, enabling us to perceive a spectrum of color. By working in conjunction with rod cells, which are adapted for low-light vision scenarios, the cone cells provide us with a comprehensive and adaptable vision system.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.preteyeres.2010.11.001"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1113/JP282058"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1126/science.3576186"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.preteyeres.2009.05.003"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1364/JOSAA.10.002491"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [retinal cone cell](http://purl.obolibrary.org/obo/CL_0000573) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0000573"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "retinal cone cell on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### retinal ganglion cell `http://purl.obolibrary.org/obo/CL_0000740`

#### Added
- [retinal ganglion cell](http://purl.obolibrary.org/obo/CL_0000740) [description](http://purl.org/dc/terms/description) "Retinal ganglion cells (RGCs) are a type of neuron located near the inner surface of the retina, playing a critical role in the visual pathway. There are approximately 1 million RGCs in a typical human eye, each receiving inputs from multiple photoreceptor cells. As the only neurons in the retina that send information to the brain, they are the final output neurons of the vertebrate retina, and their axons constitute the majority of fibers in the optic nerve. 
The primary function of RGCs is to receive visual information from photoreceptors via intermediary bipolar and amacrine cells, process it, and transmit it to the brain through their axons. The information that RGCs relay encompasses various aspects of the visual field, including color, contrast, brightness, and motion. A diverse population of RGCs, up to 30 distinct subtypes, each contributes to a specific aspect of vision, such as object motion, edge detection, or color contrast, illustrating the complexity of visual data processing that occurs already at the retinal level.
RGCs also play a role in the regulation of circadian rhythms and the pupillary light reflex. Some RGCs contain the photopigment melanopsin and respond directly to light, separately from the retinal rod and cone cells found in the outer retina. These photosensitive RGCs are primarily involved in subconscious vision, particularly the regulation of circadian rhythms and the pupillary light reflex in response to changes in ambient light intensity. Hence, RGCs serve a highly specialized and diverse role in transmitting visual information and regulating light-dependent responses, view modalities, and key physiological processes.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.neuron.2019.07.016"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1146/annurev-neuro-071714-034120"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/nature16468"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1111/j.1444-0938.2010.00479.x"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1002/cne.10652"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [retinal ganglion cell](http://purl.obolibrary.org/obo/CL_0000740) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0000740"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "retinal ganglion cell on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### retinal ganglion cell A `http://purl.obolibrary.org/obo/CL_0004117`

#### Added
- [retinal ganglion cell A](http://purl.obolibrary.org/obo/CL_0004117) SubClassOf [RO_0002162](http://purl.obolibrary.org/obo/RO_0002162) some [NCBITaxon_10090](http://purl.obolibrary.org/obo/NCBITaxon_10090) 


### retinal ganglion cell B `http://purl.obolibrary.org/obo/CL_0004115`

#### Added
- [retinal ganglion cell B](http://purl.obolibrary.org/obo/CL_0004115) SubClassOf [RO_0002162](http://purl.obolibrary.org/obo/RO_0002162) some [NCBITaxon_10090](http://purl.obolibrary.org/obo/NCBITaxon_10090) 


### retinal ganglion cell C `http://purl.obolibrary.org/obo/CL_0004116`

#### Added
- [retinal ganglion cell C](http://purl.obolibrary.org/obo/CL_0004116) SubClassOf [RO_0002162](http://purl.obolibrary.org/obo/RO_0002162) some [NCBITaxon_10090](http://purl.obolibrary.org/obo/NCBITaxon_10090) 


### retinal ganglion cell D1 `http://purl.obolibrary.org/obo/CL_0003026`

#### Added
- [retinal ganglion cell D1](http://purl.obolibrary.org/obo/CL_0003026) SubClassOf [RO_0002162](http://purl.obolibrary.org/obo/RO_0002162) some [NCBITaxon_10090](http://purl.obolibrary.org/obo/NCBITaxon_10090) 


### retinal ganglion cell D2 `http://purl.obolibrary.org/obo/CL_0003027`

#### Added
- [retinal ganglion cell D2](http://purl.obolibrary.org/obo/CL_0003027) SubClassOf [RO_0002162](http://purl.obolibrary.org/obo/RO_0002162) some [NCBITaxon_10090](http://purl.obolibrary.org/obo/NCBITaxon_10090) 


### retinal rod cell `http://purl.obolibrary.org/obo/CL_0000604`

#### Added
- [retinal rod cell](http://purl.obolibrary.org/obo/CL_0000604) [description](http://purl.org/dc/terms/description) "Retinal rod cells, often referred to simply as &quot;rod cells&rdquo;, are a specialized type of photoreceptor cells found in the retina. Their primary function is to detect and convert light into electrochemical signals, which are then transmitted to the brain via nerve impulses. Compared to other types of photoreceptor cells, namely retinal cone cells, rod cells are more sensitive to light, and thus play a crucial role in night and peripheral vision.
Rod cells are named for their long, cylindrical shape, which is distinct from the cone-like shape of retinal cone cells. The structure of a rod cell is complex, with different sections responsible for various aspects of its function. The outer segment contains densely packed, light-sensitive membrane discs filled with a pigment called rhodopsin. When light hits rhodopsin, it prompts a chemical reaction that leads to the generation of electrochemical signals. These signals are processed in the cell's inner segment, which contains the cell's primary organelles, including the nucleus and mitochondria, and transmitted through the cell's synaptic terminal to the secondary neurons in the retina.
Rod cells are distributed across the majority of the retina, with the highest density found in the peripheral edges. This distribution, coupled with their high sensitivity to light, explains why rod cells contribute significantly to peripheral and low-light vision. However, they do not pick up color and have low spatial acuity, which means that while rod cells can detect the presence of light and movement, the finer details and colors of an image are processed mainly by cone cells. Together, rod cells and cone cells work in concert to enable a wide range of visual capabilities, making vision a complex and intricate biological process.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1002/cne.902920402"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/0968-0004(83)90235-9"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1113/JP282058"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.preteyeres.2016.06.002"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/eye.2015.236"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [retinal rod cell](http://purl.obolibrary.org/obo/CL_0000604) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0000604"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "retinal rod cell on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### rod bipolar cell `http://purl.obolibrary.org/obo/CL_0000751`

#### Added
- [rod bipolar cell](http://purl.obolibrary.org/obo/CL_0000751) [description](http://purl.org/dc/terms/description) "Rod bipolar cells constitute a significant component of the retinal neural network and play a pivotal role in signal transmission from rod photoreceptor cells to the other subsequent neuronal cells within the vision system. They belong to the category of interneurons, or connector cells, effectively conveying visual information under low light conditions. Non-mammalian and mammalian visual systems feature these cells serving as the primary conduit for scotopic (night vision) signals.
These cells harness their strategic position in the retina's inner nuclear layer to receive information from the rod photoreceptor cells and transmit it to amacrine and ganglion cells. Structurally, they are named bipolar due to their distinctive shape, with dendrites extending towards the outer retina to connect with rod cells, while their axons lead inwards to form synapses with ganglion and amacrine cells in the inner plexiform layer. Each rod bipolar cell tends to interface with several rod cells and several downstream neurons, forming intricate networks.
Bearing intricate dendritic trees, these cells can integrate responses from multiple rod cells efficiently, amplifying the retinal signal's strength. They employ glutamate, a type of neurotransmitter, to govern synaptic communication between photoreceptor cells and downstream neurons. This stratified arrangement of rod bipolar cells endows them with the capability to translate light signals into neural signals, predominantly maintaining the spatial and temporal resolution of our visual perception in dim lighting. They are key to our night vision and our ability to detect changes in our light environment.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1002/cne.23370"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1007/s004410000304"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.visres.2004.07.043"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/nrn3783"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/294592a0"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [rod bipolar cell](http://purl.obolibrary.org/obo/CL_0000751) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0000751"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "rod bipolar cell on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### rosehip neuron `http://purl.obolibrary.org/obo/CL_0008032`

#### Added
- [rosehip neuron](http://purl.obolibrary.org/obo/CL_0008032) SubClassOf [BFO_0000002](http://purl.obolibrary.org/obo/BFO_0000002) 


### scleral cell `http://purl.obolibrary.org/obo/CL_0000347`
#### Removed
- [scleral cell](http://purl.obolibrary.org/obo/CL_0000347) EquivalentTo [obsolete structural cell](http://purl.obolibrary.org/obo/CL_0000293) and ([BFO_0000050](http://purl.obolibrary.org/obo/BFO_0000050) some [UBERON_0001773](http://purl.obolibrary.org/obo/UBERON_0001773)) 

- [scleral cell](http://purl.obolibrary.org/obo/CL_0000347) SubClassOf [obsolete structural cell](http://purl.obolibrary.org/obo/CL_0000293) 

#### Added
- [scleral cell](http://purl.obolibrary.org/obo/CL_0000347) EquivalentTo [cell](http://purl.obolibrary.org/obo/CL_0000000) and ([BFO_0000050](http://purl.obolibrary.org/obo/BFO_0000050) some [UBERON_0001773](http://purl.obolibrary.org/obo/UBERON_0001773)) 


### seromucous acinar cell of salivary gland `http://purl.obolibrary.org/obo/CL_4052067`

#### Added
- Class: [seromucous acinar cell of salivary gland](http://purl.obolibrary.org/obo/CL_4052067) 

- [seromucous acinar cell of salivary gland](http://purl.obolibrary.org/obo/CL_4052067) [contributor](http://purl.org/dc/terms/contributor) [0009-0000-8480-9277](https://orcid.org/0009-0000-8480-9277) 

- [seromucous acinar cell of salivary gland](http://purl.obolibrary.org/obo/CL_4052067) [date](http://purl.org/dc/terms/date) "2025-08-06T11:26:37Z"^^[dateTime](http://www.w3.org/2001/XMLSchema#dateTime) 

- [seromucous acinar cell of salivary gland](http://purl.obolibrary.org/obo/CL_4052067) [id](http://www.geneontology.org/formats/oboInOwl#id) "CL:4052067"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [seromucous acinar cell of salivary gland](http://purl.obolibrary.org/obo/CL_4052067) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "An acinar cell of the salivary gland, with a hybrid seromucous phenotype, co-expressing serous and mucous markers and producing a mixed secretion of enzymes-containing serous fluid and mucins essential for lubrication, digestion, and antimicrobial defense. Predominant in the submandibular glands of humans and mice, its differentiation and secretory gene expression are regulated by FGFR2b&ndash;MAPK signaling, driven by paracrine FGF7 from adjacent myoepithelial cells, which provides a key niche signal for seromucous identity."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:35343828"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:36726292"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:29383862"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [seromucous acinar cell of salivary gland](http://purl.obolibrary.org/obo/CL_4052067) [label](http://www.w3.org/2000/01/rdf-schema#label) "seromucous acinar cell of salivary gland"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [seromucous acinar cell of salivary gland](http://purl.obolibrary.org/obo/CL_4052067) SubClassOf [seromucus secreting cell](http://purl.obolibrary.org/obo/CL_0000159) 

- [seromucous acinar cell of salivary gland](http://purl.obolibrary.org/obo/CL_4052067) SubClassOf [acinar cell of salivary gland](http://purl.obolibrary.org/obo/CL_0002623) 


### serous acinar cell of salivary gland `http://purl.obolibrary.org/obo/CL_4052065`

#### Added
- Class: [serous acinar cell of salivary gland](http://purl.obolibrary.org/obo/CL_4052065) 

- [serous acinar cell of salivary gland](http://purl.obolibrary.org/obo/CL_4052065) [contributor](http://purl.org/dc/terms/contributor) [0009-0000-8480-9277](https://orcid.org/0009-0000-8480-9277) 

- [serous acinar cell of salivary gland](http://purl.obolibrary.org/obo/CL_4052065) [date](http://purl.org/dc/terms/date) "2025-08-05T15:24:46Z"^^[dateTime](http://www.w3.org/2001/XMLSchema#dateTime) 

- [serous acinar cell of salivary gland](http://purl.obolibrary.org/obo/CL_4052065) [id](http://www.geneontology.org/formats/oboInOwl#id) "CL:4052065"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [serous acinar cell of salivary gland](http://purl.obolibrary.org/obo/CL_4052065) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "An acinar cell of the salivary gland, characterized by a pyramidal or triangular shape with a spherical nucleus and abundant eosinophilic zymogen granules. This cell secretes watery, enzyme‚Äêrich fluid containing &alpha;-amylase, proline-rich proteins, and secretory immunoglobulins for starch digestion and antimicrobial defence. In humans and mice, serous acinar cell predominates in the parotid gland and to a lesser extent in the submandibular gland, where it exists alongside mucous acinar cells (Maruyama et al., 2019)."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:35343828"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:32829674"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:29383862"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [serous acinar cell of salivary gland](http://purl.obolibrary.org/obo/CL_4052065) [label](http://www.w3.org/2000/01/rdf-schema#label) "serous acinar cell of salivary gland"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [serous acinar cell of salivary gland](http://purl.obolibrary.org/obo/CL_4052065) EquivalentTo [serous secreting cell](http://purl.obolibrary.org/obo/CL_0000313) and [acinar cell of salivary gland](http://purl.obolibrary.org/obo/CL_0002623) and ([BFO_0000050](http://purl.obolibrary.org/obo/BFO_0000050) some [UBERON_0013232](http://purl.obolibrary.org/obo/UBERON_0013232)) 

- [serous acinar cell of salivary gland](http://purl.obolibrary.org/obo/CL_4052065) SubClassOf [serous secreting cell](http://purl.obolibrary.org/obo/CL_0000313) 

- [serous acinar cell of salivary gland](http://purl.obolibrary.org/obo/CL_4052065) SubClassOf [acinar cell of salivary gland](http://purl.obolibrary.org/obo/CL_0002623) 

- [serous acinar cell of salivary gland](http://purl.obolibrary.org/obo/CL_4052065) SubClassOf [BFO_0000050](http://purl.obolibrary.org/obo/BFO_0000050) some [UBERON_0013232](http://purl.obolibrary.org/obo/UBERON_0013232) 

- [serous acinar cell of salivary gland](http://purl.obolibrary.org/obo/CL_4052065) SubClassOf [BFO_0000051](http://purl.obolibrary.org/obo/BFO_0000051) some [GO_0042588](http://purl.obolibrary.org/obo/GO_0042588) 

- [serous acinar cell of salivary gland](http://purl.obolibrary.org/obo/CL_4052065) SubClassOf [RO_0002215](http://purl.obolibrary.org/obo/RO_0002215) some [GO_0005983](http://purl.obolibrary.org/obo/GO_0005983) 


### single fate stem cell `http://purl.obolibrary.org/obo/CL_0000035`

#### Added
- [single fate stem cell](http://purl.obolibrary.org/obo/CL_0000035) [description](http://purl.org/dc/terms/description) "Single fate stem cells, or unipotent stem cells, constitute a distinctive subtype among the diverse range of stem cells, exhibiting significant biological roles and applications. These cells possess the inherent ability to self-renew and generate a single, specialized cell lineage exclusive to their nature, reflecting their unipotent or single fate characteristic. Engaged in the maintenance and regeneration of specific tissues or organs, these stem cells play a crucial role as pivotal contributors to the body's self-maintenance and repair mechanisms.
Single fate stem cells play a crucial role in the body's diverse tissues by ensuring the appropriate renewal and repair mechanisms specific to their respective tissues. For example, the human skin, where basal cells located at the base layer of the epidermis act as single fate stem cells. These basal cells continuously generate new keratinocytes, replacing the older ones shed from the skin surface. This perpetual cycle of renewal exemplifies the fundamental function of single fate stem cells&mdash;specialized cellular regeneration to uphold tissue homeostasis and function.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "https://training.seer.cancer.gov/melanoma/anatomy/layers.html"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.52965/001c.37498"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.cps.2017.02.020"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "https://www.sciencedirect.com/topics/immunology-and-microbiology/stem-cell"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [single fate stem cell](http://purl.obolibrary.org/obo/CL_0000035) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0000035"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "single fate stem cell on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### skin fibroblast `http://purl.obolibrary.org/obo/CL_0002620`

#### Added
- [skin fibroblast](http://purl.obolibrary.org/obo/CL_0002620) [description](http://purl.org/dc/terms/description) "Skin fibroblasts specialized cells found in the dermis, the middle layer of the skin, where they play critical roles in maintaining skin health and functionality. Characterized by an elongated, spindle-like shape, skin fibroblasts are the principal active cells in the connective tissue of the skin. The primary function of skin fibroblasts is the maintenance and regeneration of the extracellular matrix, which provides structural and nutritional support for other skin cells, particularly keratinocytes and epithelial cells. These cells contribute to the production of collagen, elastin, and glycosaminoglycans - key proteins that maintain the elasticity, strength, and resilience of the skin. Through this function, skin fibroblasts help prevent skin aging and maintain tissue integrity, contributing to the skin's overall health and appearance.
In addition, skin fibroblasts are implicated in the skin's wound healing process. Following injury, fibroblasts migrate to the wound site where they start proliferating and laying down new connective tissue. They play a vital role in the repair, regeneration, and healing of the skin; in the final stages of wound healing, skin fibroblasts help contract the healing tissue to minimize scar formation. Furthermore, recent research highlights their role in the immune response, with some studies indicating that skin fibroblasts can react to, and have a role in dealing with, pathogen invasion.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.cell.2021.06.024"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.stem.2022.07.006"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.2217/rme-2018-0073"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.3389/fimmu.2020.00648"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [skin fibroblast](http://purl.obolibrary.org/obo/CL_0002620) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0002620"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "skin fibroblast on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### small intestine goblet cell `http://purl.obolibrary.org/obo/CL_1000495`

#### Added
- [small intestine goblet cell](http://purl.obolibrary.org/obo/CL_1000495) [description](http://purl.org/dc/terms/description) "Small intestine goblet cells are specialized epithelial cells found in the lining of the small intestine. They are scattered among other columnar cells in the small intestine lining and form a vital part of the intestinal villi&mdash;tiny, finger-like projections that boost the absorption of nutrients by increasing the surface area of the intestine. The name &lsquo;goblet cell&rsquo; is derived from the cell's distinctive goblet shape, which features a broad apex and a narrow base.
The primary function of the small intestine goblet cells is to secrete mucus, a thick and lubricating substance that plays a crucial role in the body's defense mechanisms. This mucus layer, formed through the aggregation of individual mucin glycoproteins produced by goblet cells and secreted from vesicles located near the apical membrane, provides a protective lining that safeguards the epithelial cells of the intestine from corrosive substances and pathogenic organisms. Moreover, the mucin layer helps in the smooth passage of food material along the digestive tract.
Additionally, goblet cells in the small intestine contribute significantly to the maintenance of intestinal homeostasis. They actively engage in signal transduction pathways initiated by the native microflora and food antigens, modulating immune responses and preventing inflammation and autoimmunity. The mucus layer also functions as the habitat for beneficial bacteria, thereby facilitating a symbiotic relationship and promoting overall gut health. Any dysfunction or abnormality affecting these cells, such as depletion or hyperplasia, may potentially lead to severe gastrointestinal disorders such as inflammatory bowel disease.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1111/febs.15731"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/mi.2015.32"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1111/imr.12182"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/s41575-023-00761-8"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [small intestine goblet cell](http://purl.obolibrary.org/obo/CL_1000495) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_1000495"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "small intestine goblet cell on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### small intestine serosal dendritic cell `http://purl.obolibrary.org/obo/CL_0002466`
#### Removed
- [small intestine serosal dendritic cell](http://purl.obolibrary.org/obo/CL_0002466) EquivalentTo [CD11b-positive dendritic cell](http://purl.obolibrary.org/obo/CL_0002465) and ([lacks_plasma_membrane_part](http://purl.obolibrary.org/obo/CL_4030046) some [PR_000001010](http://purl.obolibrary.org/obo/PR_000001010)) and ([RO_0002104](http://purl.obolibrary.org/obo/RO_0002104) some [GO_0042613](http://purl.obolibrary.org/obo/GO_0042613)) and ([RO_0002104](http://purl.obolibrary.org/obo/RO_0002104) some [PR_000001006](http://purl.obolibrary.org/obo/PR_000001006)) and ([RO_0015016](http://purl.obolibrary.org/obo/RO_0015016) some [PR_000001013](http://purl.obolibrary.org/obo/PR_000001013)) 

#### Added
- [small intestine serosal dendritic cell](http://purl.obolibrary.org/obo/CL_0002466) EquivalentTo [CD11b-positive dendritic cell](http://purl.obolibrary.org/obo/CL_0002465) and ([BFO_0000050](http://purl.obolibrary.org/obo/BFO_0000050) some [UBERON_0001206](http://purl.obolibrary.org/obo/UBERON_0001206)) and ([lacks_plasma_membrane_part](http://purl.obolibrary.org/obo/CL_4030046) some [PR_000001010](http://purl.obolibrary.org/obo/PR_000001010)) and ([RO_0002104](http://purl.obolibrary.org/obo/RO_0002104) some [GO_0042613](http://purl.obolibrary.org/obo/GO_0042613)) and ([RO_0002104](http://purl.obolibrary.org/obo/RO_0002104) some [PR_000001006](http://purl.obolibrary.org/obo/PR_000001006)) and ([RO_0015016](http://purl.obolibrary.org/obo/RO_0015016) some [PR_000001013](http://purl.obolibrary.org/obo/PR_000001013)) 

- [small intestine serosal dendritic cell](http://purl.obolibrary.org/obo/CL_0002466) SubClassOf [BFO_0000050](http://purl.obolibrary.org/obo/BFO_0000050) some [UBERON_0001206](http://purl.obolibrary.org/obo/UBERON_0001206) 


### sncg GABAergic interneuron `http://purl.obolibrary.org/obo/CL_4023015`
#### Removed
- [sncg GABAergic interneuron](http://purl.obolibrary.org/obo/CL_4023015) [IAO_0000028](http://purl.obolibrary.org/obo/IAO_0000028) "sncg cortical interneuron"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [sncg GABAergic interneuron](http://purl.obolibrary.org/obo/CL_4023015) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A transcriptomically distinct GABAergic neuron located in the cerebral cortex that expresses Gamma-synuclein. The standard transcriptomic reference data for this cell type can be found on the CellxGene census under the collection: 'Transcriptomic cytoarchitecture reveals principles of human neocortex organization', dataset: 'Supercluster: CGE-derived interneurons', Author Categories: 'CrossArea_subclass', clusters Sncg."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "https://cellxgene.cziscience.com/e/e4ddac12-f48f-4455-8e8d-c2a48a683437.cxg/"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:33186530"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:37824655"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [sncg GABAergic interneuron](http://purl.obolibrary.org/obo/CL_4023015) [label](http://www.w3.org/2000/01/rdf-schema#label) "sncg GABAergic cortical interneuron"^^[string](http://www.w3.org/2001/XMLSchema#string) 

#### Added
- [sncg GABAergic interneuron](http://purl.obolibrary.org/obo/CL_4023015) [IAO_0000028](http://purl.obolibrary.org/obo/IAO_0000028) "sncg interneuron"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [sncg GABAergic interneuron](http://purl.obolibrary.org/obo/CL_4023015) [description](http://purl.org/dc/terms/description) "The sncg GABAergic cortical interneuron belongs to a subclass of diverse interneurons that populate and function within the cerebral cortex. Predominantly found in the mammalian brain, they play a crucial role in maintaining the balance between excitation and inhibition. The naming of these interneurons follows the sncg gene they express, which stands for synuclein gamma. This specific subclass of GABAergic cortical interneurons are among five subclasses recently defined for their marker gene expression, which include Sst, Pvalb, Vip, Lamp5, and Sncg.
Generally, GABAergic cortical interneurons utilize gamma-aminobutyric acid (GABA) as their chief neurotransmitter. GABA, a potent inhibitory neurotransmitter within the mammalian central nervous system, works by reducing neuronal excitability, thereby contributing to the overall regulation of the brain's intricate signaling network. This inhibitory function is of utmost importance, as it ensures the correct functioning of cortical networks and prevents excessive neural activity that could lead to disorders such as epilepsy.
Sncg GABAergic cortical interneurons have a highly diverse morphology and physiology. They are categorized as 'local circuit' neurons due to their limited influencing range which typically encompasses only nearby neurons. Their roles include, but are not limited to, sensory perception, encoding behavioural state, regulating neural oscillations, coordinating network synchrony, controlling the timing of cortical information processing, and shaping synaptic plasticity. Specifically, sncg GABAergic cortical interneurons are inhibited by visual stimuli.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/s41586-022-04915-7"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.cell.2020.09.057"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.neuron.2018.10.009"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/s41586-018-0654-5"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.neuron.2021.01.003"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [sncg GABAergic interneuron](http://purl.obolibrary.org/obo/CL_4023015) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A transcriptomically distinct GABAergic neuron located in the cerebral cortex that expresses Gamma-synuclein."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "https://cellxgene.cziscience.com/e/e4ddac12-f48f-4455-8e8d-c2a48a683437.cxg/"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:33186530"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:37824655"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [sncg GABAergic interneuron](http://purl.obolibrary.org/obo/CL_4023015) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_4023015"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "sncg GABAergic interneuron on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [sncg GABAergic interneuron](http://purl.obolibrary.org/obo/CL_4023015) [label](http://www.w3.org/2000/01/rdf-schema#label) "sncg GABAergic interneuron"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### sncg GABAergic interneuron (Homo sapiens) `http://purl.obolibrary.org/obo/CL_4072035`

#### Added
- Class: [sncg GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072035) 

- [sncg GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072035) [IAO_0000028](http://purl.obolibrary.org/obo/IAO_0000028) "sncg interneuron (Homo sapiens)"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [sncg GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072035) [RO_0002175](http://purl.obolibrary.org/obo/RO_0002175) [NCBITaxon_9606](http://purl.obolibrary.org/obo/NCBITaxon_9606) 

- [sncg GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072035) [contributor](http://purl.org/dc/terms/contributor) "https://orcid.org/0000-0002-5507-2103"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [sncg GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072035) [date](http://purl.org/dc/terms/date) "2025-09-05T10:49:37Z"^^[dateTime](http://www.w3.org/2001/XMLSchema#dateTime) 

- [sncg GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072035) [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "ILX:0770150"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [sncg GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072035) [id](http://www.geneontology.org/formats/oboInOwl#id) "CL:4072035"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [sncg GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072035) [in_subset](http://www.geneontology.org/formats/oboInOwl#inSubset) [BDS_subset](http://purl.obolibrary.org/obo/cl/cl-base#BDS_subset) 

- [sncg GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072035) [in_subset](http://www.geneontology.org/formats/oboInOwl#inSubset) [cellxgene_subset](http://purl.obolibrary.org/obo/cl/cl-base#cellxgene_subset) 

- [sncg GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072035) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A transcriptomically distinct GABAergic neuron located in the cerebral cortex that expresses Gamma-synuclein. The standard transcriptomic reference data for this cell type can be found on the CellxGene census under the collection: 'Transcriptomic cytoarchitecture reveals principles of human neocortex organization', dataset: 'Supercluster: CGE-derived interneurons', Author Categories: 'CrossArea_subclass', clusters Sncg."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "https://cellxgene.cziscience.com/e/e4ddac12-f48f-4455-8e8d-c2a48a683437.cxg/"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:33186530"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:37824655"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [sncg GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072035) [comment](http://www.w3.org/2000/01/rdf-schema#comment) "The marker set RXRG, THSD7B can identify the Human cell type sncg GABAergic interneuron in the neocortex with a confidence of 0.628943702 (NS-Forest FBeta value)."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "https://doi.org/10.5281/zenodo.11165918"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [sncg GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072035) [label](http://www.w3.org/2000/01/rdf-schema#label) "sncg GABAergic interneuron (Homo sapiens)"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [sncg GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072035) SubClassOf [sncg GABAergic interneuron](http://purl.obolibrary.org/obo/CL_4023015) 

- [sncg GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072035) SubClassOf [RO_0002162](http://purl.obolibrary.org/obo/RO_0002162) some [NCBITaxon_9606](http://purl.obolibrary.org/obo/NCBITaxon_9606) 

- [sncg GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072035) SubClassOf [RO_0002292](http://purl.obolibrary.org/obo/RO_0002292) some [PR_O76070](http://purl.obolibrary.org/obo/PR_O76070) 


### somatic stem cell `http://purl.obolibrary.org/obo/CL_0000723`

#### Added
- [somatic stem cell](http://purl.obolibrary.org/obo/CL_0000723) [description](http://purl.org/dc/terms/description) "Somatic stem cells, also known as adult stem cells, are undifferentiated cells found throughout the body post-embryonic development. They reside in specialized niches within tissues such as the bone marrow, brain, muscles, skin, and liver. Fundamentally, these cells possess the ability to self-renew, maintaining their numbers, and to generate differentiated cells that replace dying cells and repair damaged tissues. 
The potency of somatic stem cells varies based on their tissue of origin. For instance, hematopoietic stem cells in the bone marrow exhibit multipotency, giving rise to all blood cell types. Conversely, muscle stem cells display unipotency, generating only muscle cells.
Somatic stem cells respond to environmental signals that regulate their quiescence, activation, and differentiation states. In response to tissue injury, they proliferate and differentiate into specific mature cell types to replace lost or damaged cells. Intricate mechanisms ensure a delicate balance between self-renewal and differentiation, guiding tissue maintenance and repair without risking aberrant cell proliferation leading to tumor formation.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.15283/ijsc19127"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.stem.2011.10.004"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1186/s13287-019-1165-5"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "https://www.sciencedirect.com/topics/agricultural-and-biological-sciences/adult-stem-cell"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1159/000345615"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [somatic stem cell](http://purl.obolibrary.org/obo/CL_0000723) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0000723"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "somatic stem cell on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### somatomotor neuron `http://purl.obolibrary.org/obo/CL_0005024`
#### Removed
- [somatomotor neuron](http://purl.obolibrary.org/obo/CL_0005024) SubClassOf [BFO_0000002](http://purl.obolibrary.org/obo/BFO_0000002) 



### squamous cell of epidermis `http://purl.obolibrary.org/obo/CL_0002190`
#### Removed
- [squamous cell of epidermis](http://purl.obolibrary.org/obo/CL_0002190) SubClassOf [keratinocyte](http://purl.obolibrary.org/obo/CL_0000312) 

#### Added
- [squamous cell of epidermis](http://purl.obolibrary.org/obo/CL_0002190) SubClassOf [epidermal keratinocyte](http://purl.obolibrary.org/obo/CL_4052061) 


### sst GABAergic interneuron `http://purl.obolibrary.org/obo/CL_4023017`
#### Removed
- [sst GABAergic interneuron](http://purl.obolibrary.org/obo/CL_4023017) [IAO_0000028](http://purl.obolibrary.org/obo/IAO_0000028) "sst cortical interneuron"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [sst GABAergic interneuron](http://purl.obolibrary.org/obo/CL_4023017) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A transcriptomically distinct GABAergic neuron located in the cerebral cortex that expresses somatostatin (sst). The standard transcriptomic reference data for this cell type can be found on the CellxGene census under the collection: 'Transcriptomic cytoarchitecture reveals principles of human neocortex organization', dataset: 'Supercluster: MGE-derived interneurons', Author Categories: 'CrossArea_subclass', cluster Sst."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:27477017"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:37824655"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:33742131"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "https://cellxgene.cziscience.com/e/9c63201d-bfd9-41a8-bbbc-18d947556f3d.cxg/"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [sst GABAergic interneuron](http://purl.obolibrary.org/obo/CL_4023017) [comment](http://www.w3.org/2000/01/rdf-schema#comment) "In the mouse cortex, sst GABAergic cortical interneurons show heterogeneous electrophysiological signatures varying between low-threshold spiking (LTS), regular-spiking and fast-spiking (Song, Y. H., Yoon, J., &amp; Lee, S. H. 2021). Sst GABAergic cortical interneurons have multiple functional roles in the brain. In the mouse, they involved in the generation of slow-waves during the sleep phase of non-rapid eye-movement (NREM) (Funk, C. M. et al., 2017), they maintain synaptic plasticity during motor learning (Adler, A., Zhao, R., et al., 2021), they enhance visual perception in the V1 cortical area by suppressing the activity of pyramidal excitatory neurons (Song, YH. et al., 2020). In humans, malfunctioning of sst GABAergic cortical interneurons is associated with neurodegenerative and psychiatric disorders such as Parkinson's disease, Alzheimer's disease, major depressive disorder, bipolar disorder and schizophrenia as individuals affected by these pathologies show a decrease of expression of SST in the brain (Song YH., Yoon J. et al., 2021). The marker set SST, PLCH1 can identify the Human cell type sst GABAergic cortical interneuron in the neocortex with a confidence of 0.842367005 (NS-Forest FBeta value)."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:32494634"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "https://doi.org/10.5281/zenodo.11165918"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:30792151"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:33742131"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:28821651"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [sst GABAergic interneuron](http://purl.obolibrary.org/obo/CL_4023017) [label](http://www.w3.org/2000/01/rdf-schema#label) "sst GABAergic cortical interneuron"^^[string](http://www.w3.org/2001/XMLSchema#string) 

#### Added
- [sst GABAergic interneuron](http://purl.obolibrary.org/obo/CL_4023017) [IAO_0000028](http://purl.obolibrary.org/obo/IAO_0000028) "sst interneuron"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [sst GABAergic interneuron](http://purl.obolibrary.org/obo/CL_4023017) [in_subset](http://www.geneontology.org/formats/oboInOwl#inSubset) [human_reference_atlas](http://purl.obolibrary.org/obo/cl/cl-base#human_reference_atlas) 

- [sst GABAergic interneuron](http://purl.obolibrary.org/obo/CL_4023017) [description](http://purl.org/dc/terms/description) "Sst GABAergic cortical interneuron or somatostatin-expressing GABAergic cortical interneuron, is a type of inhibitory neuron that regulates the activity of other neurons in the cerebral cortex, specifically in the mammalian neocortex. This cell type is named after the inhibitory neurotransmitter it releases, gamma-aminobutyric acid (GABA) and the particular neuropeptide it expresses, somatostatin (sst). Sst GABAergic cortical interneurons make up around 30% of all GABAergic neurons in the cortex and play a critical role in shaping cortical network dynamics and function. This specific subclass of GABAergic cortical interneurons are among five subclasses recently defined for their marker gene expression, which include Sst, Pvalb, Vip, Lamp5, and Sncg.
Sst GABAergic cortical interneurons are known for their diverse morphology, axonal projection pattern and electrophysiological properties, but they are most characterized by sst neuropeptide expression. Sst is a hormone that helps regulate the endocrine system and affects neurotransmission and cell proliferation via interaction with G protein-coupled somatostatin receptors. These interneurons typically make synapses onto the dendrites of excitatory pyramidal cells and other interneurons, controlling the input they receive. In doing so, they are primarily responsible for inhibiting and fine-tuning the output of cortical circuits.
The function of sst GABAergic cortical interneurons is essential to various cognitive processes, as they play influential roles in controlling the timing and spatial aspects of information processing in the brain. For instance, these cells modulate oscillatory rhythms, a key component of cognition and perception, and help mediate long-range cortico-cortical communication. Interestingly, dysfunctions associated with this cell type have been implicated in several neurological and psychiatric disorders including schizophrenia, autism, and epilepsy.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/s41583-021-00443-x"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1093/schbul/sbaa184"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.neuron.2023.05.032"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/s41586-018-0654-5"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.3389/fncir.2016.00048"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [sst GABAergic interneuron](http://purl.obolibrary.org/obo/CL_4023017) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A transcriptomically distinct GABAergic neuron located in the cerebral cortex that expresses somatostatin (sst) and derived from the MGE."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:27477017"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:37824655"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:33742131"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "https://cellxgene.cziscience.com/e/9c63201d-bfd9-41a8-bbbc-18d947556f3d.cxg/"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [sst GABAergic interneuron](http://purl.obolibrary.org/obo/CL_4023017) [comment](http://www.w3.org/2000/01/rdf-schema#comment) "The marker set SST, PLCH1 can identify the Human cell type sst GABAergic cortical interneuron in the neocortex with a confidence of 0.842367005 (NS-Forest FBeta value)."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "https://doi.org/10.5281/zenodo.11165918"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [sst GABAergic interneuron](http://purl.obolibrary.org/obo/CL_4023017) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_4023017"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "sst GABAergic interneuron on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [sst GABAergic interneuron](http://purl.obolibrary.org/obo/CL_4023017) [label](http://www.w3.org/2000/01/rdf-schema#label) "sst GABAergic interneuron"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### sst GABAergic interneuron (Homo sapiens) `http://purl.obolibrary.org/obo/CL_4072027`

#### Added
- Class: [sst GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072027) 

- [sst GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072027) [IAO_0000028](http://purl.obolibrary.org/obo/IAO_0000028) "sst interneuron (Homo sapiens)"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [sst GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072027) [RO_0002175](http://purl.obolibrary.org/obo/RO_0002175) [NCBITaxon_9606](http://purl.obolibrary.org/obo/NCBITaxon_9606) 

- [sst GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072027) [contributor](http://purl.org/dc/terms/contributor) "https://orcid.org/0000-0002-5507-2103"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [sst GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072027) [date](http://purl.org/dc/terms/date) "2025-08-12T10:45:41Z"^^[dateTime](http://www.w3.org/2001/XMLSchema#dateTime) 

- [sst GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072027) [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "ILX:0770152"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [sst GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072027) [id](http://www.geneontology.org/formats/oboInOwl#id) "CL:4072027"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [sst GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072027) [in_subset](http://www.geneontology.org/formats/oboInOwl#inSubset) [BDS_subset](http://purl.obolibrary.org/obo/cl/cl-base#BDS_subset) 

- [sst GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072027) [in_subset](http://www.geneontology.org/formats/oboInOwl#inSubset) [cellxgene_subset](http://purl.obolibrary.org/obo/cl/cl-base#cellxgene_subset) 

- [sst GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072027) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A transcriptomically distinct GABAergic neuron located in the cerebral cortex that expresses somatostatin (sst). The standard transcriptomic reference data for this cell type can be found on the CellxGene census under the collection: 'Transcriptomic cytoarchitecture reveals principles of human neocortex organization', dataset: 'Supercluster: MGE-derived interneurons', Author Categories: 'CrossArea_subclass', cluster Sst."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:27477017"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:37824655"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:33742131"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "https://cellxgene.cziscience.com/e/9c63201d-bfd9-41a8-bbbc-18d947556f3d.cxg/"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [sst GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072027) [has_related_synonym](http://www.geneontology.org/formats/oboInOwl#hasRelatedSynonym) "SOM+ inhibitory interneuron (Homo sapiens)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:30001424"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [sst GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072027) [has_related_synonym](http://www.geneontology.org/formats/oboInOwl#hasRelatedSynonym) "somatostatin-expressing inhibitory interneuron (Homo sapiens)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:30001424"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [sst GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072027) [has_related_synonym](http://www.geneontology.org/formats/oboInOwl#hasRelatedSynonym) "SST+ IN (Homo sapiens)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:33742131"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [sst GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072027) [label](http://www.w3.org/2000/01/rdf-schema#label) "sst GABAergic interneuron (Homo sapiens)"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [sst GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072027) SubClassOf [sst GABAergic interneuron](http://purl.obolibrary.org/obo/CL_4023017) 

- [sst GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072027) SubClassOf [RO_0002162](http://purl.obolibrary.org/obo/RO_0002162) some [NCBITaxon_9606](http://purl.obolibrary.org/obo/NCBITaxon_9606) 


### sst chodl GABAergic interneuron `http://purl.obolibrary.org/obo/CL_4023121`
#### Removed
- [sst chodl GABAergic interneuron](http://purl.obolibrary.org/obo/CL_4023121) [IAO_0000028](http://purl.obolibrary.org/obo/IAO_0000028) "sst chodl cortical interneuron"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [sst chodl GABAergic interneuron](http://purl.obolibrary.org/obo/CL_4023121) [has_broad_synonym](http://www.geneontology.org/formats/oboInOwl#hasBroadSynonym) "long-range GABAergic interneuron"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [sst chodl GABAergic interneuron](http://purl.obolibrary.org/obo/CL_4023121) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A transcriptomically distinct sst GABAergic cortical interneuron that also expresses Chodl. These neurons are rare and correspond to the only known cortical interneurons with long-range projection. The standard transcriptomic reference data for this cell type can be found on the CellxGene census under the collection: 'Transcriptomic cytoarchitecture reveals principles of human neocortex organization', dataset: 'Supercluster: MGE-derived interneurons', Author Categories: 'CrossArea_subclass', clusters Sst Chodl."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:15845086"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:31209381"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:33372656"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:37824655"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "https://cellxgene.cziscience.com/e/9c63201d-bfd9-41a8-bbbc-18d947556f3d.cxg/"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [sst chodl GABAergic interneuron](http://purl.obolibrary.org/obo/CL_4023121) [label](http://www.w3.org/2000/01/rdf-schema#label) "sst chodl GABAergic cortical interneuron"^^[string](http://www.w3.org/2001/XMLSchema#string) 

#### Added
- [sst chodl GABAergic interneuron](http://purl.obolibrary.org/obo/CL_4023121) [IAO_0000028](http://purl.obolibrary.org/obo/IAO_0000028) "sst chodl interneuron"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [sst chodl GABAergic interneuron](http://purl.obolibrary.org/obo/CL_4023121) [in_subset](http://www.geneontology.org/formats/oboInOwl#inSubset) [human_reference_atlas](http://purl.obolibrary.org/obo/cl/cl-base#human_reference_atlas) 

- [sst chodl GABAergic interneuron](http://purl.obolibrary.org/obo/CL_4023121) [description](http://purl.org/dc/terms/description) "The sst chodl GABAergic cortical interneuron is a specific rare type of inhibitory neuron found within the cortex of the brain. Specifically, sst refers to somatostatin, which is a peptide hormone produced by this cell. Chodl on the other hand, refers to chondrolectin, a protein highly expressed by this particular subtype of interneurons. GABAergic denotes the neurons' use of gamma-aminobutyric acid (GABA), a prominent inhibitory neurotransmitter, playing a crucial part in reducing neuronal excitability throughout the nervous system.
GABAergic cortical interneurons expressing both sst and chodl markers play an instrumental role in balancing excitatory neural activities. Their inherent inhibitory polarity primarily functions to fine-tune the excitatory signals facilitated by the majority of cortical neurons, the pyramidal cells. Smooth dendrites and a propensity to innervate multiple targets allow these sst chodl GABAergic interneurons to control the output of many principal neurons simultaneously. This regulation results in a balanced state of neuronal activities, preventing any possibility of excess excitations that can potentially lead to brain disorders like epilepsy.
In a broader scope, sst expressing GABAergic cortical interneurons play a pivotal role in the higher cognitive processes. By controlling the timing and synchronization, they play an essential part in shaping the dynamic cortical network activities underlying complex brain functions, such as learning, memory, and attention. These interneurons are also likely involved in specific brain oscillatory patterns. Recent studies suggest that the dysfunction or misplacement of these interneurons can lead to neurological disorders such as schizophrenia or autism.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1093/schbul/sbaa184"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.neuron.2023.05.032"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/s41593-019-0417-0"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.3389/fncir.2016.00076"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/nn.4216"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [sst chodl GABAergic interneuron](http://purl.obolibrary.org/obo/CL_4023121) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A transcriptomically distinct sst GABAergic interneuron that also expresses Chodl. These neurons are rare and correspond to the only known interneurons with long-range projection."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:15845086"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:31209381"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:33372656"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:37824655"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "https://cellxgene.cziscience.com/e/9c63201d-bfd9-41a8-bbbc-18d947556f3d.cxg/"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [sst chodl GABAergic interneuron](http://purl.obolibrary.org/obo/CL_4023121) [has_broad_synonym](http://www.geneontology.org/formats/oboInOwl#hasBroadSynonym) "long-range GABAergic interneuron"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:33372656"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [sst chodl GABAergic interneuron](http://purl.obolibrary.org/obo/CL_4023121) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_4023121"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "sst chodl GABAergic interneuron on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [sst chodl GABAergic interneuron](http://purl.obolibrary.org/obo/CL_4023121) [label](http://www.w3.org/2000/01/rdf-schema#label) "sst chodl GABAergic interneuron"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### sst chodl GABAergic interneuron (Homo sapiens) `http://purl.obolibrary.org/obo/CL_4072031`

#### Added
- Class: [sst chodl GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072031) 

- [sst chodl GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072031) [IAO_0000028](http://purl.obolibrary.org/obo/IAO_0000028) "sst chodl interneuron (Homo sapiens)"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [sst chodl GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072031) [RO_0002175](http://purl.obolibrary.org/obo/RO_0002175) [NCBITaxon_9606](http://purl.obolibrary.org/obo/NCBITaxon_9606) 

- [sst chodl GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072031) [contributor](http://purl.org/dc/terms/contributor) "https://orcid.org/0000-0002-5507-2103"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [sst chodl GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072031) [date](http://purl.org/dc/terms/date) "2025-09-04T10:20:07Z"^^[dateTime](http://www.w3.org/2001/XMLSchema#dateTime) 

- [sst chodl GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072031) [id](http://www.geneontology.org/formats/oboInOwl#id) "CL:4072031"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [sst chodl GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072031) [in_subset](http://www.geneontology.org/formats/oboInOwl#inSubset) [cellxgene_subset](http://purl.obolibrary.org/obo/cl/cl-base#cellxgene_subset) 

- [sst chodl GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072031) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A transcriptomically distinct sst GABAergic interneuron that also expresses Chodl. These neurons are rare and correspond to the only known interneurons with long-range projection. The standard transcriptomic reference data for this cell type can be found on the CellxGene census under the collection: 'Transcriptomic cytoarchitecture reveals principles of human neocortex organization', dataset: 'Supercluster: MGE-derived interneurons', Author Categories: 'CrossArea_subclass', clusters Sst Chodl."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:15845086"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:31209381"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:33372656"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:37824655"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "https://cellxgene.cziscience.com/e/9c63201d-bfd9-41a8-bbbc-18d947556f3d.cxg/"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [sst chodl GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072031) [has_broad_synonym](http://www.geneontology.org/formats/oboInOwl#hasBroadSynonym) "long-range GABAergic interneuron (Homo sapiens)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:33372656"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [sst chodl GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072031) [comment](http://www.w3.org/2000/01/rdf-schema#comment) "The marker set NPY can identify the Human cell type sst chodl GABAergic interneuron in the neocortex with a confidence of 0.894026187 (NS-Forest FBeta value)."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "https://doi.org/10.5281/zenodo.11165918"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [sst chodl GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072031) [label](http://www.w3.org/2000/01/rdf-schema#label) "sst chodl GABAergic interneuron (Homo sapiens)"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [sst chodl GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072031) SubClassOf [sst chodl GABAergic interneuron](http://purl.obolibrary.org/obo/CL_4023121) 

- [sst chodl GABAergic interneuron (Homo sapiens)](http://purl.obolibrary.org/obo/CL_4072031) SubClassOf [RO_0002162](http://purl.obolibrary.org/obo/RO_0002162) some [NCBITaxon_9606](http://purl.obolibrary.org/obo/NCBITaxon_9606) 


### stellate neuron `http://purl.obolibrary.org/obo/CL_0000122`

#### Added
- [stellate neuron](http://purl.obolibrary.org/obo/CL_0000122) [description](http://purl.org/dc/terms/description) "Stellate neurons are a specialized type of interneurons found predominantly in the cerebral cortex and the cerebellum, two major components of the brain's central nervous system. These neurons are named for their distinct star-like shape, characterized by short dendrites that radiate from the cell body in multiple directions. They are a key component of the mammalian neural circuit due to their unique morphology and crucial functions in neural communication.
Stellate neurons play a pivotal role in the processing and transmission of information within the brain. In the cerebellum, these neurons are predominantly inhibitory and release the neurotransmitter gamma-aminobutyric acid (GABA), effectively regulating the excitability of other neurons within the network and thus fine-tuning the neuron-to-neuron communication. Their extensive dendritic network allows for connections with multiple neurons allowing for the integration of synaptic inputs from various sources in the brain. This widespread connectivity places stellate neurons at the heart of information processing in the central nervous system.
In addition to their important inhibitory role in regulating and processing information, the role of stellate neurons in the cerebellum differs slightly from their cortical counterparts. The cortical spiny stellate neurons are excitatory and release the neurotransmitter glutamate. They are involved in motor control and responsible for coordinating precise, voluntary movements and maintaining posture and balance. Any dysfunction in these cerebellar neurons can lead to movement disorders, highlighting their significance in our daily activities.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1002/cne.901460304"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/nn1698"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1007/s11055-006-0093-x"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1126/science.aai8178"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1523/JNEUROSCI.2929-06.2006"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [stellate neuron](http://purl.obolibrary.org/obo/CL_0000122) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0000122"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "stellate neuron on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### stem cell `http://purl.obolibrary.org/obo/CL_0000034`

#### Added
- [stem cell](http://purl.obolibrary.org/obo/CL_0000034) [description](http://purl.org/dc/terms/description) "Stem cells are a unique type of cell that has the remarkable ability to divide and develop into many different types of cells in the body. These cells are undifferentiated, meaning they haven't yet developed into a specific cell type, and they can renew themselves through cell division. As a result, stem cells have the potential to replace damaged or diseased cells in the body, making them a valuable tool in medicine and research.
There are two main types of stem cells: embryonic stem cells and adult stem cells. Embryonic stem cells are derived from embryos and can develop into any type of cell in the body. Adult stem cells, on the other hand, are found in various tissues and organs throughout the body and can only develop into certain types of cells.
Stem cells have numerous potential applications in medicine, including the treatment of various diseases and the repair of damaged tissues. For example, they could be used to replace damaged heart tissue after a heart attack or to repair spinal cord injuries. Stem cells also have potential uses in drug development and disease research, as they can be used to model diseases and test potential therapies in vitro.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.stem.2023.05.015"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.cell.2018.09.010"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "https://www.mayoclinic.org/tests-procedures/bone-marrow-transplant/in-depth/stem-cells/art-20048117#:~:text=These%20stem%20cells%20are%20manipulated,injected%20into%20the%20heart%20muscle."^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.cell.2016.05.043"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/nm.4502"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [stem cell](http://purl.obolibrary.org/obo/CL_0000034) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0000034"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "stem cell on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### stem cell of epidermis `http://purl.obolibrary.org/obo/CL_1000428`

#### Added
- [stem cell of epidermis](http://purl.obolibrary.org/obo/CL_1000428) [description](http://purl.org/dc/terms/description) "The epidermal stem cell, a critical component of the skin's structure, is essential for maintaining, repairing, and renewing the epidermal layer. Undifferentiated and capable of becoming various cell types within the epidermis, these cells typically remain quiescent, preserving the skin's longevity and regenerative ability.
The primary role of these stem cells is to replenish the population of keratinocytes, the key protective cells in the epidermis. When stimulated by signals like injury or normal wear, these stem cells activate, divide, and give rise to both new stem cells and differentiated cells. The latter form layers of stratified keratinocytes that make up the skin's protective barrier.
Beyond skin-related applications, the regenerative potential of epidermal stem cells has garnered interest in medical and cosmetic fields, particularly for treating burns, chronic wounds, and aging skin. Notably, these cells, specialized for skin tissue, might, under controlled conditions, have the potential to form diverse tissues, expanding possibilities in regenerative medicine.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.3390/ijms161025476"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1146/annurev.cellbio.22.010305.104357"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1186/s13287-019-1312-z"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "https://www.ncbi.nlm.nih.gov/books/NBK26865/"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "https://www.sciencedirect.com/topics/medicine-and-dentistry/epidermal-stem-cell"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [stem cell of epidermis](http://purl.obolibrary.org/obo/CL_1000428) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_1000428"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "stem cell of epidermis on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### stratified epithelial cell `http://purl.obolibrary.org/obo/CL_0000079`

#### Added
- [stratified epithelial cell](http://purl.obolibrary.org/obo/CL_0000079) [description](http://purl.org/dc/terms/description) "Stratified epithelial cells form layers of cells stacked on top of one another, and they primarily function as a protective barrier. Their appearance can be cuboidal, columnar or squamous, and they offer a layer of defense against foreign bodies, harmful substances, pathogens, and prevent water loss and electrolyte imbalance within the body. Depending on the organ and its specific function, the topmost layer of these cells may be specialized as keratinized or non-keratinized. The skin, being constantly exposed to external physical stress, is composed of keratinized stratified epithelial cells that prevent desiccation and provide additional protection. In contrast, non-keratinized stratified epithelial cells line internally exposed surfaces like the oral cavity, esophagus, and vagina, where lubrication is required. 
In addition to protection, stratified epithelial cells also contribute to tissue repair processes, referred to as epithelialization. Basal epithelial cells are capable of rapid division and migration to heal wounds and maintain the protective barrier function of the tissues they line. Despite having a limited lifespan, these cells ensure a constant replacement process, maintaining their population and the integrity of epithelial layers.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "https://www.ncbi.nlm.nih.gov/books/NBK534261/"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1002/wdev.146"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [stratified epithelial cell](http://purl.obolibrary.org/obo/CL_0000079) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0000079"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "stratified epithelial cell on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### stratified squamous epithelial cell `http://purl.obolibrary.org/obo/CL_0000240`

#### Added
- [stratified squamous epithelial cell](http://purl.obolibrary.org/obo/CL_0000240) [description](http://purl.org/dc/terms/description) "Stratified squamous epithelial cells are a specific type of epithelial cell that form layers and have a flat shape. These cells are part of the outermost layer of the skin, called the epidermis, and are also found lining surfaces of the body that are subjected to physical or chemical wear and tear, such as the mouth, esophagus, and vagina. The primary role of these cells is to create a barrier that protects underlying tissues from dehydration, abrasion, and infection.
The stratified squamous epithelium is able to provide a strong, force-resistant barrier due to the stratified arrangement of the cells; it consists primarily of keratinocytes in various stages of differentiation: The basal layer contains cells that continuously divide and push older cells towards the surface. As the cells move up through the layers, they gradually flatten and become filled with a protein called keratin. In high-abrasion areas like the skin, the outer layers of these cells can become so filled with keratin that they die (they become 'keratinized'), providing an even more robust barrier; eventually, they are shed off in a process known as desquamation.
In addition to forming a protective barrier, stratified squamous epithelial cells also play a part in the immune response. They can recognize pathogens, and in response, release chemical signals that attract immune cells to the site of potential infection. Moreover, they also carry out a variety of interactions with the immune system to ensure that invading pathogens cannot penetrate into deeper layers of the body.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.jdermsci.2018.01.012"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1007/s00281-018-0701-1"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/onc.2010.456"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.jdermsci.2004.05.004"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [stratified squamous epithelial cell](http://purl.obolibrary.org/obo/CL_0000240) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0000240"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "stratified squamous epithelial cell on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### striatal pthlh-expressing interneuron `http://purl.obolibrary.org/obo/CL_4042023`

#### Added
- [striatal pthlh-expressing interneuron](http://purl.obolibrary.org/obo/CL_4042023) [RO_0002175](http://purl.obolibrary.org/obo/RO_0002175) [NCBITaxon_9606](http://purl.obolibrary.org/obo/NCBITaxon_9606) 

- [striatal pthlh-expressing interneuron](http://purl.obolibrary.org/obo/CL_4042023) [in_subset](http://www.geneontology.org/formats/oboInOwl#inSubset) [human_reference_atlas](http://purl.obolibrary.org/obo/cl/cl-base#human_reference_atlas) 


### striosomal D1 medium spiny neuron `http://purl.obolibrary.org/obo/CL_4030048`

#### Added
- [striosomal D1 medium spiny neuron](http://purl.obolibrary.org/obo/CL_4030048) [RO_0002175](http://purl.obolibrary.org/obo/RO_0002175) [NCBITaxon_9606](http://purl.obolibrary.org/obo/NCBITaxon_9606) 

- [striosomal D1 medium spiny neuron](http://purl.obolibrary.org/obo/CL_4030048) [in_subset](http://www.geneontology.org/formats/oboInOwl#inSubset) [human_reference_atlas](http://purl.obolibrary.org/obo/cl/cl-base#human_reference_atlas) 


### striosomal D2 medium spiny neuron `http://purl.obolibrary.org/obo/CL_4030049`

#### Added
- [striosomal D2 medium spiny neuron](http://purl.obolibrary.org/obo/CL_4030049) [RO_0002175](http://purl.obolibrary.org/obo/RO_0002175) [NCBITaxon_9606](http://purl.obolibrary.org/obo/NCBITaxon_9606) 

- [striosomal D2 medium spiny neuron](http://purl.obolibrary.org/obo/CL_4030049) [in_subset](http://www.geneontology.org/formats/oboInOwl#inSubset) [human_reference_atlas](http://purl.obolibrary.org/obo/cl/cl-base#human_reference_atlas) 


### stromal cell `http://purl.obolibrary.org/obo/CL_0000499`

#### Added
- [stromal cell](http://purl.obolibrary.org/obo/CL_0000499) [description](http://purl.org/dc/terms/description) "Stromal cells constitute a significant portion of the structural cellular milieu (stroma) that supports the function of parenchymal (functional) cells within an organ. The stroma is the connective, supportive framework of biological tissues with various functions depending on the specific organ and cell type. Stromal cells, derived from the mesenchymal lineage, play crucial roles in creating, maintaining, and repairing the tissue microenvironment in virtually all mammalian organs, making these cells indispensable to biological function.
Additionally, stromal cells secrete growth factors, cytokines, and extracellular matrix (ECM) molecules to constitute an interactive cellular network and ensure an optimal environment for the functional cells. Beyond mere structural support, these cells are instrumental in wound healing, inflammation, and maintaining tissue integrity. They may also influence the behavior of other cell types through direct cell-cell interactions or paracrine signaling.
Clinically and in research, stromal cells often draw attention due to their interaction with tumors in the cancer microenvironment. Over the past years, the role of stromal cells as facilitators of tumor growth and metastasis has been recognized increasingly. They contribute to cancer progression not only by modulating the ECM but also through the secretion of various bioactive molecules and interacting with immune cells, thereby influencing the immunologic responses to tumors.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1634/stemcells.2007-0637"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.3389/fcell.2020.555378"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1186/s13058-016-0740-2"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/s41536-019-0083-6"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/nri2588"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [stromal cell](http://purl.obolibrary.org/obo/CL_0000499) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0000499"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "stromal cell on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### stromal cell of ovary `http://purl.obolibrary.org/obo/CL_0002132`

#### Added
- [stromal cell of ovary](http://purl.obolibrary.org/obo/CL_0002132) [description](http://purl.org/dc/terms/description) "Stromal cells of the ovary, also known as ovarian stromal cells, form a significant portion of the ovarian tissue. They are non-epithelial cells that primarily make up the supportive or connective tissue&mdash;similar to the stroma of other organs&mdash;in the ovaries. These cells play vital roles in folliculogenesis, including in the activation of primordial follicles and in the differentiation of theca cells. They are crucial for maintaining the structure, function, and overall health of the ovaries. In addition to roles in hormone production and follicle maturation, stromal cells of the ovary play a role in ovarian aging and age-related diseases such as ovarian cancer. They participate in cell-to-cell communication with oocyte and other ovarian cell types through gap junctions and paracrine signaling and help maintain the ovarian microenvironment.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1017/CBO9780511921896.037"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1371/journal.pone.0205494"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1530/REP-19-0501"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/s42003-022-04384-8"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1186/s13578-017-0163-5"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [stromal cell of ovary](http://purl.obolibrary.org/obo/CL_0002132) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0002132"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "stromal cell of ovary on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### subcutaneous adipocyte `http://purl.obolibrary.org/obo/CL_0002521`

#### Added
- [subcutaneous adipocyte](http://purl.obolibrary.org/obo/CL_0002521) [description](http://purl.org/dc/terms/description) "Subcutaneous fat cells, also known as adipocytes, reside under the skin, in the subcutaneous layer of our body. These cells primarily function as reserves for energy due to their ability to store lipids, particularly triglycerides, efficiently. Subcutaneous fat cells make up the most significant part of the adipose tissue, which provides insulation, cushioning, and serves as an energy reserve for individuals. The location of these cells varies with gender; typically, in males these cells are found in the area of the abdomen, whereas, in females, they are found in the thighs and the buttock region.
These cells play an integral role in maintaining metabolic homeostasis. Beyond energy storage, subcutaneous fat cells are also engaged in numerous physiological processes such as maintaining body temperature, protecting the body from mechanical trauma, and functioning as an endocrine organ. They produce and secrete various bioactive molecules known as adipokines which have wide-ranging impacts on various bodily processes, including regulation of appetite, insulin sensitivity, inflammation, and immune function.
Subcutaneous fat cells have a characteristic shape, being large, spherical, and filled with a large lipid droplet, unlike other cell types that contain smaller lipid droplets scattered throughout the cytoplasm. The diameter of these cells can range from 30 to over 300 micrometers, significantly larger than most other cells in the body. Of note, these cells also demonstrate a unique property called plasticity. They are able to change their volume dramatically, expanding to store more fat as it is taken up from the bloodstream, and shrinking when fats are released to provide energy.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.cell.2021.12.016"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1152/ajpregu.00257.2017"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.5114/aoms.2013.33181"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/nature05483"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.3389/fcvm.2020.00022"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [subcutaneous adipocyte](http://purl.obolibrary.org/obo/CL_0002521) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0002521"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "subcutaneous adipocyte on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### substantia nigra dopaminergic neuron `http://purl.obolibrary.org/obo/CL_4042025`

#### Added
- [substantia nigra dopaminergic neuron](http://purl.obolibrary.org/obo/CL_4042025) [RO_0002175](http://purl.obolibrary.org/obo/RO_0002175) [NCBITaxon_9606](http://purl.obolibrary.org/obo/NCBITaxon_9606) 

- [substantia nigra dopaminergic neuron](http://purl.obolibrary.org/obo/CL_4042025) [in_subset](http://www.geneontology.org/formats/oboInOwl#inSubset) [human_reference_atlas](http://purl.obolibrary.org/obo/cl/cl-base#human_reference_atlas) 


### suprabasal cell `http://purl.obolibrary.org/obo/CL_7770004`

#### Added
- Class: [suprabasal cell](http://purl.obolibrary.org/obo/CL_7770004) 

- [suprabasal cell](http://purl.obolibrary.org/obo/CL_7770004) [contributor](http://purl.org/dc/terms/contributor) [0009-0000-8480-9277](https://orcid.org/0009-0000-8480-9277) 

- [suprabasal cell](http://purl.obolibrary.org/obo/CL_7770004) [date](http://purl.org/dc/terms/date) "2025-09-05T07:59:28Z"^^[dateTime](http://www.w3.org/2001/XMLSchema#dateTime) 

- [suprabasal cell](http://purl.obolibrary.org/obo/CL_7770004) [id](http://www.geneontology.org/formats/oboInOwl#id) "CL:7770004"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [suprabasal cell](http://purl.obolibrary.org/obo/CL_7770004) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "An epithelial cell that resides in the layer(s) immediately above the basal layer in stratified or pseudostratified epithelia. This cell mainly originates from basal cells and is found across various tissues, including skin, esophagus, oral mucosa, airway, and cornea."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:32726565"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:37672481"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:30837987"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:17020902"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:32187560"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [suprabasal cell](http://purl.obolibrary.org/obo/CL_7770004) [label](http://www.w3.org/2000/01/rdf-schema#label) "suprabasal cell"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [suprabasal cell](http://purl.obolibrary.org/obo/CL_7770004) SubClassOf [epithelial cell](http://purl.obolibrary.org/obo/CL_0000066) 


### suprabasal keratinocyte `http://purl.obolibrary.org/obo/CL_4033013`
#### Removed
- [suprabasal keratinocyte](http://purl.obolibrary.org/obo/CL_4033013) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A keratinocyte of the epidermis suprabasal layer. This cell may express the differentiation markers keratin 10 and keratin 1."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "https://www.proteinatlas.org/humanproteome/single+cell+type/squamous+epithelial+cells#suprabasalkeratinocytes"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:35883599"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:9520465"^^[string](http://www.w3.org/2001/XMLSchema#string) 

#### Added
- [suprabasal keratinocyte](http://purl.obolibrary.org/obo/CL_4033013) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A keratinocyte that resides in the epidermal suprabasal layer and expresses differentiation markers, including keratin 1 and keratin 10, in both humans and mice. In human interfollicular epidermis, this cell retains proliferative capacity and possesses retrodifferentiation potential, acquiring basal-like properties during wound healing or basement membrane contact."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "https://www.proteinatlas.org/humanproteome/single+cell+type/squamous+epithelial+cells#suprabasalkeratinocytes"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:35883599"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:9520465"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:22375063"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [suprabasal keratinocyte](http://purl.obolibrary.org/obo/CL_4033013) SubClassOf [suprabasal cell](http://purl.obolibrary.org/obo/CL_7770004) 


### sympathetic neuron `http://purl.obolibrary.org/obo/CL_0011103`

#### Added
- [sympathetic neuron](http://purl.obolibrary.org/obo/CL_0011103) [description](http://purl.org/dc/terms/description) "Sympathetic neurons are a distinct class of neurons that are an integral part of the autonomic nervous system, specifically within the sympathetic division. They play a critical role in the body's 'fight or flight' responses and are primarily involved in regulating involuntary physiological processes, such as heart rate, blood pressure, body temperature, and metabolic processes. These neurons are essential for homeostatic mechanisms, working to ensure a balanced state is maintained within the body amidst changing environments.
Structurally, sympathetic neurons are characterized by their relatively long axons, which allow them to relay signals from the central nervous system to various peripheral organs and tissues. They typically originate within the spinal cord's thoracic and lumbar regions, extending their axons to connect with various targets, including cardiac muscle, smooth muscle, and glands. These neurons often utilize noradrenaline (norepinephrine) as a neurotransmitter, which binds to adrenergic receptors on the target cells to exert their effects.
Functionally, the sympathetic neurons help the body respond to stressful or threatening situations. When activated, they provoke a cascade of processes that ready the body for quick action. This includes accelerating the heart rate, dilating the pupils, slowing digestion, and increasing blood flow to muscles. Conversely, when the threat has passed, they encourage a return to normalcy by inhibiting these same processes. Beyond this 'fight or flight' response, sympathetic neurons also function to help regulate body temperature by inducing sweating or shivering, influence the dilation of blood vessels to control blood pressure, and can stimulate the release of glucose from the liver to provide energy. Overall, sympathetic neurons are key mediators of the body's response to various internal and external stimuli, maintaining homeostasis and preparing the body for diverse challenges.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/s41583-021-00523-y"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1023/A:1009889922985"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1161/HYPERTENSIONAHA.111.00633"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1152/physrev.00005.2023"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1146/annurev.neuro.22.1.541"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [sympathetic neuron](http://purl.obolibrary.org/obo/CL_0011103) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0011103"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "sympathetic neuron on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### taste receptor cell `http://purl.obolibrary.org/obo/CL_0000209`

#### Added
- [taste receptor cell](http://purl.obolibrary.org/obo/CL_0000209) [description](http://purl.org/dc/terms/description) "Taste receptor cells, also known as gustatory receptor cells, are specialized epithelial cells that are responsible for detecting taste stimuli. They also play a role in regulating feeding behavior and nutrient intake by signaling the presence of nutrients or toxins in food. 
These cells are located in taste buds, which are small structures found on the tongue, palate, and throat. Each taste bud is composed of 50-100 taste receptor cells, as well as supporting cells and basal cells.
Taste receptor cells are involved in the transduction of taste stimuli into neural signals that can be interpreted by the brain. They detect five primary tastes: sweet, sour, salty, umami (savory), and bitter. Each taste receptor cell is specialized to detect one or more of these tastes, depending on the expression of specific taste receptors on their membrane. For example, sweet taste receptors bind to sugars and artificial sweeteners, while bitter taste receptors recognize many toxins and alkaloids. Mechanistically, when a taste receptor cell is stimulated by a tastant, it depolarizes and releases neurotransmitters that activate sensory nerve fibers, which then transmit the information to the brain.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/nrendo.2015.7"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1111/j.1748-1716.2011.02308.x"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/nrn.2017.68"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [taste receptor cell](http://purl.obolibrary.org/obo/CL_0000209) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0000209"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "taste receptor cell on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### tendon cell `http://purl.obolibrary.org/obo/CL_0000388`
#### Removed
- [tendon cell](http://purl.obolibrary.org/obo/CL_0000388) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "An elongated fibrocyte that is part of a tendon. The cytoplasm is stretched between the collagen fibres of the tendon. They have a central cell nucleus with a prominent nucleolus. Tendon cells have a well-developed rough endoplasmic reticulum and they are responsible for synthesis and turnover of tendon fibres and ground substance."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "GOC:NV"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "GOC:tfm"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:957445"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [tendon cell](http://purl.obolibrary.org/obo/CL_0000388) EquivalentTo [circulating fibrocyte](http://purl.obolibrary.org/obo/CL_0000135) and ([BFO_0000050](http://purl.obolibrary.org/obo/BFO_0000050) some [UBERON_0000043](http://purl.obolibrary.org/obo/UBERON_0000043)) 

- [tendon cell](http://purl.obolibrary.org/obo/CL_0000388) SubClassOf [circulating fibrocyte](http://purl.obolibrary.org/obo/CL_0000135) 
  - [is_inferred](http://www.geneontology.org/formats/oboInOwl#is_inferred) "true"^^[string](http://www.w3.org/2001/XMLSchema#string) 

#### Added
- [tendon cell](http://purl.obolibrary.org/obo/CL_0000388) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "An elongated fibroblast that is part of a tendon. Its cytoplasm is stretched between the collagen fibres of the tendon, and it possesses a central nucleus with a prominent nucleolus. Tendon cell has a well-developed rough endoplasmic reticulum, and it is responsible for the synthesis and turnover of tendon fibres and ground substance."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:37894875"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "GOC:NV"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "GOC:tfm"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:957445"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [tendon cell](http://purl.obolibrary.org/obo/CL_0000388) EquivalentTo [fibroblast](http://purl.obolibrary.org/obo/CL_0000057) and ([BFO_0000050](http://purl.obolibrary.org/obo/BFO_0000050) some [UBERON_0000043](http://purl.obolibrary.org/obo/UBERON_0000043)) 

- [tendon cell](http://purl.obolibrary.org/obo/CL_0000388) SubClassOf [fibroblast of dense regular elastic tissue](http://purl.obolibrary.org/obo/CL_1000307) 


### theca cell `http://purl.obolibrary.org/obo/CL_0000503`

#### Added
- [theca cell](http://purl.obolibrary.org/obo/CL_0000503) [description](http://purl.org/dc/terms/description) "Theca cells are specialized stromal cells found within the ovaries, more specifically in the follicles that house and nurture the developing ovum. They exist in two forms: theca interna cells and theca externa cells, both playing significant roles in ovarian function and the reproductive system as a whole. Found in all mammalian species, these cells allow intricate processes like ovulation and hormonal control to function seamlessly.
Theca interna cells are highly vascularized and specialized to produce androgens, specifically androstenedione, a precursor to estradiol. When the follicle-stimulating hormone (FSH) stimulates granulosa cells to produce aromatase, this enzyme then converts the androgens into estrogens. Consequently, the hormonal interaction between theca interna cells and granulosa cells supports the production of estrogen necessary for the reproductive cycle. 
Theca externa cells, on the other hand, are mechanically supportive cells. They form an outer protective layer around the follicle. While they do not directly influence hormone production, they play an essential role in structural integrity and act in concert with the granulosa and theca interna cells during the rupture of the mature ovarian follicle, releasing the egg for potential fertilization. Simultaneously, these cells participate in the formation of the corpus luteum, which secretes progesterone to prepare the uterus for potential implantation.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1097/CM9.0000000000000850"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1155/2015/925691"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1210/er.2017-00164"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/ncomms7934"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.biocel.2005.01.016"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [theca cell](http://purl.obolibrary.org/obo/CL_0000503) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0000503"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "theca cell on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### thymic plasmacytoid dendritic cell `http://purl.obolibrary.org/obo/CL_0000942`
#### Removed
- [thymic plasmacytoid dendritic cell](http://purl.obolibrary.org/obo/CL_0000942) EquivalentTo [plasmacytoid dendritic cell](http://purl.obolibrary.org/obo/CL_0000784) and ([lacks_plasma_membrane_part](http://purl.obolibrary.org/obo/CL_4030046) some [PR_000001012](http://purl.obolibrary.org/obo/PR_000001012)) and ([RO_0002104](http://purl.obolibrary.org/obo/RO_0002104) some [PR_000001015](http://purl.obolibrary.org/obo/PR_000001015)) and ([RO_0002104](http://purl.obolibrary.org/obo/RO_0002104) some [PR_000001865](http://purl.obolibrary.org/obo/PR_000001865)) 

#### Added
- [thymic plasmacytoid dendritic cell](http://purl.obolibrary.org/obo/CL_0000942) EquivalentTo [plasmacytoid dendritic cell](http://purl.obolibrary.org/obo/CL_0000784) and ([BFO_0000050](http://purl.obolibrary.org/obo/BFO_0000050) some [UBERON_0002370](http://purl.obolibrary.org/obo/UBERON_0002370)) and ([lacks_plasma_membrane_part](http://purl.obolibrary.org/obo/CL_4030046) some [PR_000001012](http://purl.obolibrary.org/obo/PR_000001012)) and ([RO_0002104](http://purl.obolibrary.org/obo/RO_0002104) some [PR_000001015](http://purl.obolibrary.org/obo/PR_000001015)) and ([RO_0002104](http://purl.obolibrary.org/obo/RO_0002104) some [PR_000001865](http://purl.obolibrary.org/obo/PR_000001865)) 

- [thymic plasmacytoid dendritic cell](http://purl.obolibrary.org/obo/CL_0000942) SubClassOf [BFO_0000050](http://purl.obolibrary.org/obo/BFO_0000050) some [UBERON_0002370](http://purl.obolibrary.org/obo/UBERON_0002370) 


### tissue-resident macrophage `http://purl.obolibrary.org/obo/CL_0000864`

#### Added
- [tissue-resident macrophage](http://purl.obolibrary.org/obo/CL_0000864) [description](http://purl.org/dc/terms/description) "Tissue-resident macrophages refer to a heterogeneous group of specialized macrophages that permanently reside within specific tissues, performing integral functions essential to tissue homeostasis. Most tissue-resident macrophages arise from embryonic populations and self-renew. Some macrophages may also originate from circulating monocytes released from the bone marrow and differentiate into macrophages upon entering tissue. Tissue-resident macrophages are given different names depending upon the tissue in which they reside, such as Kupffer cells in the liver, alveolar macrophages in the lungs, and microglia in the brain.
Tissue-resident macrophages are responsible for the phagocytosis of pathogenic substances and dead cells, serving as part of the foremost line of defense against invading microorganisms. They utilize pattern recognition receptors to identify foreign substances and function as antigen-presenting cells to induce adaptive immunity. Additionally, they produce a variety of cytokines and chemokines, signaling molecules that recruit additional immune cells to sites of infection or inflammation and regulate immune responses.
In addition to their immunological roles, tissue-resident macrophages also have tissue-unique functions. In the liver, Kupffer cells play a role in iron homeostasis, while in the brain, microglia are responsible for synaptic pruning during development. In the lungs, alveolar macrophages are involved in the removal of airborne particles and debris. This underscores their significance in not only providing pathological immunity but also in maintaining the optimal function of the tissues in which they reside. &quot;

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1093/intimm/dxy054"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.immuni.2016.02.015"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1146/annurev-immunol-032414-112220"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/s41392-023-01452-1"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [tissue-resident macrophage](http://purl.obolibrary.org/obo/CL_0000864) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0000864"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "tissue-resident macrophage on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### trabecular meshwork cell `http://purl.obolibrary.org/obo/CL_0002367`
#### Removed
- [trabecular meshwork cell](http://purl.obolibrary.org/obo/CL_0002367) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "A cell that lines the trabecular meshwork, which is an area of tissue in the eye located around the base of the cornea, near the ciliary body, and is responsible for draining the aqueous humor from the eye via the anterior chamber (the chamber on the front of the eye covered by the cornea). This cell may play a role in regulating intraocular pressure."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:14500801"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "GOC:tfm"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [trabecular meshwork cell](http://purl.obolibrary.org/obo/CL_0002367) EquivalentTo [endothelial cell](http://purl.obolibrary.org/obo/CL_0000115) and ([BFO_0000050](http://purl.obolibrary.org/obo/BFO_0000050) some [UBERON_0005969](http://purl.obolibrary.org/obo/UBERON_0005969)) 

- [trabecular meshwork cell](http://purl.obolibrary.org/obo/CL_0002367) SubClassOf [endothelial cell](http://purl.obolibrary.org/obo/CL_0000115) 
  - [is_inferred](http://www.geneontology.org/formats/oboInOwl#is_inferred) "true"^^[string](http://www.w3.org/2001/XMLSchema#string) 

#### Added
- [trabecular meshwork cell](http://purl.obolibrary.org/obo/CL_0002367) [definition](http://purl.obolibrary.org/obo/IAO_0000115) "An ocular connective tissue cell that is part of the eye's trabecular meshwork and is directly responsible for regulating the outflow of aqueous humor from the anterior chamber of the eye, thereby maintaining proper intraocular pressure."^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:14500801"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "PMID:40136401"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "GOC:tfm"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [trabecular meshwork cell](http://purl.obolibrary.org/obo/CL_0002367) EquivalentTo [connective tissue cell](http://purl.obolibrary.org/obo/CL_0002320) and ([BFO_0000050](http://purl.obolibrary.org/obo/BFO_0000050) some [UBERON_0005969](http://purl.obolibrary.org/obo/UBERON_0005969)) 

- [trabecular meshwork cell](http://purl.obolibrary.org/obo/CL_0002367) SubClassOf [connective tissue cell](http://purl.obolibrary.org/obo/CL_0002320) 
  - [is_inferred](http://www.geneontology.org/formats/oboInOwl#is_inferred) "true"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### trophoblast giant cell `http://purl.obolibrary.org/obo/CL_0002488`

#### Added
- [trophoblast giant cell](http://purl.obolibrary.org/obo/CL_0002488) [description](http://purl.org/dc/terms/description) "Trophoblast giant cells are specialized cells that form during the early stages of mammalian embryonic development. Within the trophoblast cell category, these cells, marked by their large, multinucleated structure, play a pivotal role in the early stages of pregnancy, particularly during embryo implantation and the maintenance of pregnancy.
Trophoblast giant cells are predominantly located at the fetal-maternal interface, where they aid in establishing the connection between the maternal blood supply and the embryonic circulatory system. This is achieved by the invasion and remodeling of maternal spiral arteries, a process carried out by a subset of these cells known as invasive trophoblast giant cells - a necessary step to ensure sufficient nutrient supply to the growing fetus. Additionally, these cells participate in the immune defense mechanism, secreting immunosuppressive molecules to protect against maternal immune rejection.
These cells are also the primary sources of progesterone, crucial for maintaining the uterine lining during pregnancy, and placental lactogen, which influences the mother's metabolic state to promote fetal growth. In essence, trophoblast giant cells serve as integral mediators, connecting the developmental needs of the fetus with the adaptive physiological changes in the mother.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.3389/fimmu.2021.717884"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1007/s00018-019-03104-6"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1172/JCI22991"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1002/dvdy.10291"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [trophoblast giant cell](http://purl.obolibrary.org/obo/CL_0002488) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0002488"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "trophoblast giant cell on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### type B pancreatic cell `http://purl.obolibrary.org/obo/CL_0000169`

#### Added
- [type B pancreatic cell](http://purl.obolibrary.org/obo/CL_0000169) [description](http://purl.org/dc/terms/description) "Type B pancreatic cells, also known as beta cells, are one of the endocrine cell types located in the pancreas. They represent 70-80% of the cells in the islets of Langerhans, the endocrine part of this organ. Unique to mammals, Type B pancreatic cells are the predominant cell type in the pancreatic islets (50&ndash;80% of all islet endocrine cells) and play a crucial role in the body's metabolic regulation and glucose homeostasis. 
The primary function of type B pancreatic cells is the production, storage, and release of insulin, accounting for almost all insulin production in the human body: each cell contains about 20 pg insulin. Insulin has a critical function in regulating blood sugar levels: whenever glucose levels in the blood are high, such as after a meal, type B pancreatic cells secrete insulin to promote glucose uptake into cells, transforming it into usable energy or storing it for later use. Additionally, insulin also inhibits glucose production by the liver, therefore lowering blood glucose levels.
Impaired function of the type B cells is associated with type 1 and type 2 diabetes, metabolic diseases characterized by chronic hyperglycemia. In type 1 diabetes, an autoimmune response leads to the attack of the body&rsquo;s own type B cells, resulting in the insufficient production and secretion of insulin. In type 2 diabetes, insulin resistance occurs, and type B cells are unable to compensate by producing more insulin.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.3390/ijms22126403"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.3389/fcell.2017.00055"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1155/2017/1478294"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1007/s40828-021-00140-3"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/s41574-020-00443-4"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [type B pancreatic cell](http://purl.obolibrary.org/obo/CL_0000169) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0000169"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "type B pancreatic cell on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### type L enteroendocrine cell `http://purl.obolibrary.org/obo/CL_0002279`

#### Added
- [type L enteroendocrine cell](http://purl.obolibrary.org/obo/CL_0002279) [description](http://purl.org/dc/terms/description) "The type L enteroendocrine cell, also known as L-cell, is an integral part of the digestive system found in the intestine, primarily identified in the ileum and colon. These cells are distinguished among other cell types primarily because of their role in the secretion of glucagon-like peptide-1 (GLP-1) and peptide YY (PYY). Both peptides are essential for maintaining the body's energy homeostasis and the normal functioning of the gastrointestinal tract.
In response to the ingestion of food, type L enteroendrocrine cells secrete GLP-1, a potent hormone that stimulates insulin secretion, lowers glucagon secretion, slows gastric emptying, enhances satiety, and has the potential to promote proliferation of type B pancreatic cells and improve their function.
L-cells also secrete peptide YY (PYY), which plays a pivotal role in controlling appetite. Postprandially secreted into the bloodstream, PYY functions by decreasing appetite and food intake, thereby contributing to long-term energy balance. PYY can centralize the regulation of appetite via the hypothalamus.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/nrendo.2011.77"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1042/CS20150154"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1111/j.1749-6632.2003.tb03176.x"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.cmet.2008.11.002"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/ncpendmet0318"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [type L enteroendocrine cell](http://purl.obolibrary.org/obo/CL_0002279) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0002279"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "type L enteroendocrine cell on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### visceromotor neuron `http://purl.obolibrary.org/obo/CL_0005025`

#### Added
- [visceromotor neuron](http://purl.obolibrary.org/obo/CL_0005025) [description](http://purl.org/dc/terms/description) "Visceromotor neurons, also known as autonomic motor neurons, are an integral component of the body's autonomic nervous system, which controls and regulates internal body processes such as digestion, circulation, and respiration, autonomously from conscious control. These neurons regulate the motor functions of internal organs and glands, and serve as the effector neurons of the autonomic nervous system, conveying the signals from the central nervous system to execute actions.Visceromotor neurons are predominantly subdivided into two main functionally distinct categories: sympathetic and parasympathetic neurons.
Sympathetic visceromotor neurons tend to stimulate action, conserving energy for times of impending threat or when increased alertness is required. These neurons often act to increase heart rate, expand air passages allowing for greater oxygen flow to the muscles and brain, slow digestion, and produce a surge of glucose (sugar) in the bloodstream to cater to the energy demands during periods of heightened activity or stress.
Parasympathetic visceromotor neurons typically take effect during periods of calm, and are responsible for what's often termed as &quot;rest and digest&quot; activities. They govern a variety of activities collectively aimed at restoring and conserving bodily energy. Parasympathetic visceromotor neurons act to conserve energy by slowing the heart rate, increasing intestinal and gland activity, and relaxing sphincter muscles in the gastrointestinal tract. The complementary functions of these two types of visceromotor neurons enable the body to maintain an overall balance, or homeostasis, responding appropriately under different conditions and demands. Without the dynamic and essential functioning of visceromotor neurons, the body would lose its capability to autonomously regulate essential processes, thereby exposing it to potentially harmful internal conditions.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.3389/fncel.2014.00293"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1177/0148607108321705"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.3389/fphys.2021.700129"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/0165-1838(81)90068-0"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "https://www.ncbi.nlm.nih.gov/books/NBK554616/"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [visceromotor neuron](http://purl.obolibrary.org/obo/CL_0005025) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0005025"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "visceromotor neuron on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 


### white adipocyte `http://purl.obolibrary.org/obo/CL_0000448`

#### Added
- [white adipocyte](http://purl.obolibrary.org/obo/CL_0000448) [description](http://purl.org/dc/terms/description) "White fat cells, also known as white adipocytes, are cells found primarily in the white adipose tissue, a specialized type of connective tissue that functions as the major storage site for energy in the form of lipids. This type of adipose cell is characterized by its large droplets of liquid fat, called triglycerides, which almost completely fill the cell pushing the nucleus and other organelles to the periphery. They appear white or yellowish in color due to their high cytoplasm lipid content.
White fat cells are involved in lipogenesis, the metabolic process where excess glucose and other energy-rich compounds are converted into neutral fats for storage. Conversely, these cells also play a pivotal role in lipolysis, the breakdown of triglycerides to release glycerol and fatty acids, when required by the body. Moreover, these cells act as thermal insulators, helping to maintain the body's temperature.
Importantly, white fat cells are crucial for the proper functioning of numerous physiological processes including appetite regulation, inflammatory response, and immunity. They are also known to secrete various hormones and cytokines, such as leptin and adiponectin, which are instrumental in regulating metabolic activities and maintaining overall metabolic homeostasis. Dysregulation of these cells, along with an accumulation of excess white fat, has been linked to several health issues including obesity, metabolic syndrome, and type 2 diabetes.

(This extended description was generated by ChatGPT and reviewed by the CellGuide team, who added references, and by the CL editors, who approved it for inclusion in CL. It may contain information that applies only to some subtypes and species, and so should not be considered definitional.)"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/s42255-021-00493-6"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1016/j.arcmed.2008.09.005"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/s41574-019-0230-6"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1038/s41574-021-00471-8"^^[string](http://www.w3.org/2001/XMLSchema#string) 

  - [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "DOI:10.1146/annurev.nutr.27.061406.093734"^^[string](http://www.w3.org/2001/XMLSchema#string) 

- [white adipocyte](http://purl.obolibrary.org/obo/CL_0000448) [seeAlso](http://purl.obolibrary.org/obo/cl/cl-base#seeAlso) "https://cellxgene.cziscience.com/cellguide/CL_0000448"^^[string](http://www.w3.org/2001/XMLSchema#string) 
  - [label](http://www.w3.org/2000/01/rdf-schema#label) "white adipocyte on CELLxGENE CellGuide"^^[string](http://www.w3.org/2001/XMLSchema#string) 

